# Navigating the Data Portal

<div align="center">
    <video src="https://nmdc-edge.org/docs/videos/NMDC_portal_tutorial.mp4" width="80%" height="80%" controls />
</div>
<br><br>

>NMDC Data Portal Tutorial Practice 
>
>There are many ways to search the microbiome studies available in the Data Portal. This Tutorial will guide you through some of the most common methods.
>
>Task 1: Go to the [NMDC Data Portal](https://data.microbiomedata.org/) and log in with your ORCID account. (You can browse the data without logging in, but you will not be able to download data/processed data. You can create an account from the login page from the ORCID login link in the top corner of the Data Portal.)
>
>Task 2: Using the map features (zoom capability and "Search this region" to filter by latitude/longitude), find all the metagenomes collected closest to Corvallis, Oregon. 
>
>&nbsp;&nbsp;&nbsp;&nbsp;Question 1: How many metagenomes have been collected near Corvallis?
>
>Task 3: Using the Study box (which shows the number of microbiome studies related to the metagenomes identified in Task 2), click the arrow on the right side of this box to go to the Study Page for this study.
>
>&nbsp;&nbsp;&nbsp;&nbsp;Question 1:  What is the DOI for this study? (Note: This is also the DOI for the Dataset Citation.)
>
>Task 4: Go back to the main Data Portal page and clear the active query terms in the upper left corner. Use the Collection date option in the left menu bar or the timeline slide feature (below the Omics type and map) to filter to samples collected in 2015.  Use the "search" feature (upper left corner) to find metagenomes collected from freshwater river biomes.
>
>&nbsp;&nbsp;&nbsp;&nbsp;Question 1: How many samples collected from freshwater biomes in 2015 have metagenomic data?
>
>&nbsp;&nbsp;&nbsp;&nbsp;Question 2: What other types of omics data are available for these samples?
>
>Task 5: In the Omics box, click the additional omics types available for these samples. This will allow you to be able to download from any of the processed data. (You must be logged in to be able to download data.) Download some **small** files from the first sample in the list:
>
>&nbsp;&nbsp;&nbsp;&nbsp;1. Click the Metagenome button under the first sample. You can see all of the processed data available from this metagenome; download the QC Statistics. 
>
>&nbsp;&nbsp;&nbsp;&nbsp;2. Click on the Proteomics button. You can see all of the processed data available from this metaproteomic sample; download the Protein Report.
>
>&nbsp;&nbsp;&nbsp;&nbsp;3. Click on the Metabolomics button; download the GC-MS Metabolomics Results.
>
>&nbsp;&nbsp;&nbsp;&nbsp;4. In the purple box above the samples, test the **Bulk Download** feature. In the *Select file type* box dropdown menu, select nmdc:ReadQC Analysis Activity. Click to **remove** the Filtered Sequencing Reads. This will provide a zipped (compressed) files with just the QC Statistics for all of samples in the study.




## Answers to Tutorial Questions
>Task 2, Question 1: 108 metagenomes have been collected near Corvallis, Oregon.
>
>Task 3, Question 1: The DOI for this study and dataset is https://doi.org/10.46936/10.25585/60000017
>
>Task 4, Question 1: There are 32 samples collected from freshwater river biomes in 2015 which have metagenomic data.
>
>Task 4, Question 2: There are also proteomics and metabolomics data for these 32 samples.
