Class: activity
===============

a provence-generating activity

URI: https://microbiomedata/schema/workflow_execution_activity/Activity

|img|

Children
--------

-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

Referenced by Class
-------------------

-  **None** `was generated by <was_generated_by.md>`__ 0..1
   `Activity <Activity.md>`__
-  **None** `was informed by <was_informed_by.md>`__ 0..1
   `Activity <Activity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__
   `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1
   `Activity <Activity.md>`__

Attributes
----------

Own
~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `started at time <started_at_time.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `ended at time <ended_at_time.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `was informed by <was_informed_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `was associated with <was_associated_with.md>`__ 0..1

   -  Range: `Agent <Agent.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

Other properties
----------------

============= =============
**Mappings:**  prov:Activity
============= =============

Class: agent
============

a provence-generating agent

URI: https://microbiomedata/schema/workflow_execution_activity/Agent

|image1|

.. _referenced-by-class-1:

Referenced by Class
-------------------

-  **None** `acted on behalf of <acted_on_behalf_of.md>`__ 0..1
   `Agent <Agent.md>`__
-  **None** `was associated with <was_associated_with.md>`__ 0..1
   `Agent <Agent.md>`__

.. _attributes-1:

Attributes
----------

.. _own-1:

Own
~~~

-  `acted on behalf of <acted_on_behalf_of.md>`__ 0..1

   -  Range: `Agent <Agent.md>`__

-  `was informed by <was_informed_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-1:

Other properties
----------------

============= ==========
**Mappings:**  prov:Agent
============= ==========

Class: attribute value
======================

The value for any value of a attribute for a sample. This object can
hold both the un-normalized atomic value and the structured value

URI:
https://microbiomedata/schema/workflow_execution_activity/AttributeValue

|image2|

.. _children-1:

Children
--------

-  `BooleanValue <BooleanValue.md>`__ - A value that is a boolean
-  `ControlledTermValue <ControlledTermValue.md>`__ - A controlled term
   or class from an ontology
-  `GeolocationValue <GeolocationValue.md>`__ - A normalized value for a
   location on the earth’s surface
-  `ImageValue <ImageValue.md>`__ - An attribute value representing an
   image.
-  `IntegerValue <IntegerValue.md>`__ - A value that is an integer
-  `PersonValue <PersonValue.md>`__ - An attribute value representing a
   person
-  `QuantityValue <QuantityValue.md>`__ - A simple quantity, e.g. 2cm
-  `TextValue <TextValue.md>`__ - A basic string value
-  `TimestampValue <TimestampValue.md>`__ - A value that is a timestamp.
   The range should be ISO-8601
-  `UrlValue <UrlValue.md>`__ - A value that is a string that conforms
   to URL syntax

.. _referenced-by-class-2:

Referenced by Class
-------------------

.. _attributes-2:

Attributes
----------

.. _own-2:

Own
~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: biosample
================

A material sample. It may be environmental (encompassing many organisms)
or isolate or tissue. An environmental sample containing genetic
material from multiple individuals is commonly referred to as a
biosample.

URI: `nmdc:Biosample <https://microbiomedata/meta/Biosample>`__

|image3|

Identifier prefixes
-------------------

-  GOLD

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _referenced-by-class-3:

Referenced by Class
-------------------

-  `BiosampleProcessing <BiosampleProcessing.md>`__ `biosample
   processing➞has input <biosample_processing_has_input.md>`__ 0..\*
   `Biosample <Biosample.md>`__
-  `Database <Database.md>`__ `biosample set <biosample_set.md>`__ 0..\*
   `Biosample <Biosample.md>`__
-  `OmicsProcessing <OmicsProcessing.md>`__ `omics processing➞has
   input <omics_processing_has_input.md>`__ 1..\*
   `Biosample <Biosample.md>`__

.. _attributes-3:

Attributes
----------

.. _own-3:

Own
~~~

-  `type <type.md>`__ 0..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `agrochem_addition <agrochem_addition.md>`__ 0..1

   -  Description: “Addition of fertilizers, pesticides, etc. - amount
      and time of applications”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `alkalinity <alkalinity.md>`__ 0..1

   -  Description: “Alkalinity, the ability of a solution to neutralize
      acids to the equivalence point of carbonate or bicarbonate”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `alkalinity_method <alkalinity_method.md>`__ 0..1

   -  Description: Method used for alkalinity measurement
   -  Range: `TextValue <TextValue.md>`__

-  `alkyl_diethers <alkyl_diethers.md>`__ 0..1

   -  Description: Concentration of alkyl diethers
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `alt <alt.md>`__ 0..1

   -  Description: “Altitude is a term used to identify heights of
      objects such as airplanes, space shuttles, rockets, atmospheric
      balloons and heights of places such as atmospheric layers and
      clouds. It is used to measure the height of an object which is
      above the earthbs surface. In this context, the altitude
      measurement is the vertical distance between the earth’s surface
      above sea level and the sampled position in the air”
   -  Range: `QuantityValue <QuantityValue.md>`__
   -  in subsets: (environment)

-  `al_sat <al_sat.md>`__ 0..1

   -  Description: Aluminum saturation (esp. For tropical soils)
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `al_sat_meth <al_sat_meth.md>`__ 0..1

   -  Description: Reference or method used in determining Al saturation
   -  Range: `TextValue <TextValue.md>`__

-  `aminopept_act <aminopept_act.md>`__ 0..1

   -  Description: Measurement of aminopeptidase activity
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `ammonium <ammonium.md>`__ 0..1

   -  Description: Concentration of ammonium in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `annual_precpt <annual_precpt.md>`__ 0..1

   -  Description: “The average of all annual precipitation values
      known, or an estimated equivalent value derived by such methods as
      regional indexes or Isohyetal maps.”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `annual_temp <annual_temp.md>`__ 0..1

   -  Description: Mean annual temperature
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `bacteria_carb_prod <bacteria_carb_prod.md>`__ 0..1

   -  Description: Measurement of bacterial carbon production
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `bishomohopanol <bishomohopanol.md>`__ 0..1

   -  Description: Concentration of bishomohopanol
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `bromide <bromide.md>`__ 0..1

   -  Description: Concentration of bromide
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `calcium <calcium.md>`__ 0..1

   -  Description: Concentration of calcium in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `carb_nitro_ratio <carb_nitro_ratio.md>`__ 0..1

   -  Description: Ratio of amount or concentrations of carbon to
      nitrogen
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `chem_administration <chem_administration.md>`__ 0..1

   -  Description: “List of chemical compounds administered to the host
      or site where sampling occurred, and when (e.g. Antibiotics, n
      fertilizer, air filter); can include multiple compounds. For
      chemical entities of biological interest ontology (chebi) (v 163),
      http://purl.bioontology.org/ontology/chebi”
   -  Range: `ControlledTermValue <ControlledTermValue.md>`__

-  `chloride <chloride.md>`__ 0..1

   -  Description: Concentration of chloride in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `chlorophyll <chlorophyll.md>`__ 0..1

   -  Description: Concentration of chlorophyll
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `collection_date <collection_date.md>`__ 0..1

   -  Description: “The time of sampling, either as an instance (single
      point in time) or interval. In case no exact time is available,
      the date/time can be right truncated i.e. all of these are valid
      times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23;
      2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant”
   -  Range: `TimestampValue <TimestampValue.md>`__
   -  in subsets: (environment)

-  `cur_land_use <cur_land_use.md>`__ 0..1

   -  Description: Present state of sample site
   -  Range: `TextValue <TextValue.md>`__

-  `cur_vegetation <cur_vegetation.md>`__ 0..1

   -  Description: “Vegetation classification from one or more standard
      classification systems, or agricultural crop”
   -  Range: `TextValue <TextValue.md>`__

-  `cur_vegetation_meth <cur_vegetation_meth.md>`__ 0..1

   -  Description: Reference or method used in vegetation classification
   -  Range: `TextValue <TextValue.md>`__

-  `crop_rotation <crop_rotation.md>`__ 0..1

   -  Description: “Whether or not crop is rotated, and if yes, rotation
      schedule”
   -  Range: `TextValue <TextValue.md>`__

-  `density <density.md>`__ 0..1

   -  Description: “Density of the sample, which is its mass per unit
      volume (aka volumetric mass density)”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `depth <depth.md>`__ 0..1

   -  Description: Please refer to the definitions of depth in the
      environmental packages
   -  Range: `QuantityValue <QuantityValue.md>`__
   -  in subsets: (environment)

-  `diss_carb_dioxide <diss_carb_dioxide.md>`__ 0..1

   -  Description: Concentration of dissolved carbon dioxide in the
      sample or liquid portion of the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `diss_hydrogen <diss_hydrogen.md>`__ 0..1

   -  Description: Concentration of dissolved hydrogen
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `diss_inorg_carb <diss_inorg_carb.md>`__ 0..1

   -  Description: “Dissolved inorganic carbon concentration in the
      sample, typically measured after filtering the sample using a 0.45
      micrometer filter”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `diss_inorg_phosp <diss_inorg_phosp.md>`__ 0..1

   -  Description: Concentration of dissolved inorganic phosphorus in
      the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `diss_org_carb <diss_org_carb.md>`__ 0..1

   -  Description: “Concentration of dissolved organic carbon in the
      sample, liquid portion of the sample, or aqueous phase of the
      fluid”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `diss_org_nitro <diss_org_nitro.md>`__ 0..1

   -  Description: Dissolved organic nitrogen concentration measured as;
      total dissolved nitrogen - NH4 - NO3 - NO2
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `diss_oxygen <diss_oxygen.md>`__ 0..1

   -  Description: Concentration of dissolved oxygen
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `drainage_class <drainage_class.md>`__ 0..1

   -  Description: Drainage classification from a standard system such
      as the USDA system
   -  Range: `TextValue <TextValue.md>`__

-  `elev <elev.md>`__ 0..1

   -  Description: “Elevation of the sampling site is its height above a
      fixed reference point, most commonly the mean sea level. Elevation
      is mainly used when referring to points on the earth’s surface,
      while altitude is used for points above the surface, such as an
      aircraft in flight or a spacecraft in orbit”
   -  Range: `QuantityValue <QuantityValue.md>`__
   -  in subsets: (environment)

-  `env_package <env_package.md>`__ 0..1

   -  Description: “MIxS extension for reporting of measurements and
      observations obtained from one or more of the environments where
      the sample was obtained. All environmental packages listed here
      are further defined in separate subtables. By giving the name of
      the environmental package, a selection of fields can be made from
      the subtables and can be reported”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (mixs extension)

-  `biosample➞env_broad_scale <biosample_env_broad_scale.md>`__ 1..1

   -  Description: “In this field, report which major environmental
      system your sample or specimen came from. The systems identified
      should have a coarse spatial grain, to provide the general
      environmental context of where the sampling was done (e.g. were
      you in the desert or a rainforest?). We recommend using subclasses
      of ENVOUs biome class:
      http://purl.obolibrary.org/obo/ENVO_00000428. Format (one term):
      termLabel [termID], Format (multiple terms): termLabel
      [termID]|termLabel [termID]|termLabel [termID]. Example:
      Annotating a water sample from the photic zone in middle of the
      Atlantic Ocean, consider: oceanic epipelagic zone biome
      [ENVO:01000033]. Example: Annotating a sample from the Amazon
      rainforest consider: tropical moist broadleaf forest biome
      [ENVO:01000228]. If needed, request new terms on the ENVO tracker,
      identified here: http://www.obofoundry.org/ontology/envo.html”
   -  Range: `ControlledTermValue <ControlledTermValue.md>`__
   -  in subsets: (environment)

-  `biosample➞env_local_scale <biosample_env_local_scale.md>`__ 1..1

   -  Description: “In this field, report the entity or entities which
      are in your sample or specimenUs local vicinity and which you
      believe have significant causal influences on your sample or
      specimen. Please use terms that are present in ENVO and which are
      of smaller spatial grain than your entry for env_broad_scale.
      Format (one term): termLabel [termID]; Format (multiple terms):
      termLabel [termID]|termLabel [termID]|termLabel [termID]. Example:
      Annotating a pooled sample taken from various vegetation layers in
      a forest consider: canopy [ENVO:00000047]|herb and fern layer
      [ENVO:01000337]|litter layer [ENVO:01000338]|understory
      [01000335]|shrub layer [ENVO:01000336]. If needed, request new
      terms on the ENVO tracker, identified here:
      http://www.obofoundry.org/ontology/envo.html”
   -  Range: `ControlledTermValue <ControlledTermValue.md>`__
   -  in subsets: (environment)

-  `biosample➞env_medium <biosample_env_medium.md>`__ 1..1

   -  Description: “In this field, report which environmental material
      or materials (pipe separated) immediately surrounded your sample
      or specimen prior to sampling, using one or more subclasses of
      ENVOUs environmental material class:
      http://purl.obolibrary.org/obo/ENVO_00010483. Format (one term):
      termLabel [termID]; Format (multiple terms): termLabel
      [termID]|termLabel [termID]|termLabel [termID]. Example:
      Annotating a fish swimming in the upper 100 m of the Atlantic
      Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating
      a duck on a pond consider: pond water [ENVO:00002228]|air
      ENVO_00002005. If needed, request new terms on the ENVO tracker,
      identified here: http://www.obofoundry.org/ontology/envo.html”
   -  Range: `ControlledTermValue <ControlledTermValue.md>`__
   -  in subsets: (environment)

-  `extreme_event <extreme_event.md>`__ 0..1

   -  Description: Unusual physical events that may have affected
      microbial populations
   -  Range: `TimestampValue <TimestampValue.md>`__

-  `fao_class <fao_class.md>`__ 0..1

   -  Description: Soil classification from the FAO World Reference
      Database for Soil Resources. The list can be found at
      http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups
   -  Range: `TextValue <TextValue.md>`__

-  `fire <fire.md>`__ 0..1

   -  Description: Historical and/or physical evidence of fire
   -  Range: `TimestampValue <TimestampValue.md>`__

-  `flooding <flooding.md>`__ 0..1

   -  Description: Historical and/or physical evidence of flooding
   -  Range: `TimestampValue <TimestampValue.md>`__

-  `geo_loc_name <geo_loc_name.md>`__ 0..1

   -  Description: “The geographical origin of the sample as defined by
      the country or sea name followed by specific region name. Country
      or sea names should be chosen from the INSDC country list
      (http://insdc.org/country.html), or the GAZ ontology (v 1.512)
      (http://purl.bioontology.org/ontology/GAZ)”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (environment)

-  `glucosidase_act <glucosidase_act.md>`__ 0..1

   -  Description: Measurement of glucosidase activity
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `heavy_metals <heavy_metals.md>`__ 0..1

   -  Description: Heavy metals present and concentrationsany drug used
      by subject and the frequency of usage; can include multiple heavy
      metals and concentrations
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `heavy_metals_meth <heavy_metals_meth.md>`__ 0..1

   -  Description: Reference or method used in determining heavy metals
   -  Range: `TextValue <TextValue.md>`__

-  `horizon <horizon.md>`__ 0..1

   -  Description: Specific layer in the land area which measures
      parallel to the soil surface and possesses physical
      characteristics which differ from the layers above and beneath
   -  Range: `TextValue <TextValue.md>`__

-  `horizon_meth <horizon_meth.md>`__ 0..1

   -  Description: Reference or method used in determining the horizon
   -  Range: `TextValue <TextValue.md>`__

-  `biosample➞lat_lon <biosample_lat_lon.md>`__ 0..1

   -  Description: This is currently a required field but it’s not clear
      if this should be required for human hosts
   -  Range: `GeolocationValue <GeolocationValue.md>`__
   -  in subsets: (environment)

-  `link_addit_analys <link_addit_analys.md>`__ 0..1

   -  Description: Link to additional analysis results performed on the
      sample
   -  Range: `TextValue <TextValue.md>`__

-  `link_class_info <link_class_info.md>`__ 0..1

   -  Description: Link to digitized soil maps or other soil
      classification information
   -  Range: `TextValue <TextValue.md>`__

-  `link_climate_info <link_climate_info.md>`__ 0..1

   -  Description: Link to climate resource
   -  Range: `TextValue <TextValue.md>`__

-  `local_class <local_class.md>`__ 0..1

   -  Description: Soil classification based on local soil
      classification system
   -  Range: `TextValue <TextValue.md>`__

-  `local_class_meth <local_class_meth.md>`__ 0..1

   -  Description: Reference or method used in determining the local
      soil classification
   -  Range: `TextValue <TextValue.md>`__

-  `magnesium <magnesium.md>`__ 0..1

   -  Description: Concentration of magnesium in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `mean_frict_vel <mean_frict_vel.md>`__ 0..1

   -  Description: Measurement of mean friction velocity
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `mean_peak_frict_vel <mean_peak_frict_vel.md>`__ 0..1

   -  Description: Measurement of mean peak friction velocity
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `microbial_biomass <microbial_biomass.md>`__ 0..1

   -  Description: “The part of the organic matter in the soil that
      constitutes living microorganisms smaller than 5-10 micrometer. If
      you keep this, you would need to have correction factors used for
      conversion to the final units”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `microbial_biomass_meth <microbial_biomass_meth.md>`__ 0..1

   -  Description: Reference or method used in determining microbial
      biomass
   -  Range: `TextValue <TextValue.md>`__

-  `misc_param <misc_param.md>`__ 0..1

   -  Description: “Any other measurement performed or parameter
      collected, that is not listed here”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `n_alkanes <n_alkanes.md>`__ 0..1

   -  Description: Concentration of n-alkanes; can include multiple
      n-alkanes
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `nitrate <nitrate.md>`__ 0..1

   -  Description: Concentration of nitrate in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `nitrite <nitrite.md>`__ 0..1

   -  Description: Concentration of nitrite in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `org_matter <org_matter.md>`__ 0..1

   -  Description: Concentration of organic matter
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `org_nitro <org_nitro.md>`__ 0..1

   -  Description: Concentration of organic nitrogen
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `organism_count <organism_count.md>`__ 0..1

   -  Description: “Total cell count of any organism (or group of
      organisms) per gram, volume or area of sample, should include name
      of organism followed by count. The method that was used for the
      enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided.
      (example: total prokaryotes; 3.5e7 cells per ml; qpcr)”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `oxy_stat_samp <oxy_stat_samp.md>`__ 0..1

   -  Description: Oxygenation status of sample
   -  Range: `TextValue <TextValue.md>`__

-  `part_org_carb <part_org_carb.md>`__ 0..1

   -  Description: Concentration of particulate organic carbon
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `perturbation <perturbation.md>`__ 0..1

   -  Description: “Type of perturbation, e.g. chemical administration,
      physical disturbance, etc., coupled with perturbation regimen
      including how many times the perturbation was repeated, how long
      each perturbation lasted, and the start and end time of the entire
      perturbation period; can include multiple perturbation types”
   -  Range: `TextValue <TextValue.md>`__

-  `petroleum_hydrocarb <petroleum_hydrocarb.md>`__ 0..1

   -  Description: Concentration of petroleum hydrocarbon
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `ph <ph.md>`__ 0..1

   -  Description: “Ph measurement of the sample, or liquid portion of
      sample, or aqueous phase of the fluid”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `ph_meth <ph_meth.md>`__ 0..1

   -  Description: Reference or method used in determining ph
   -  Range: `TextValue <TextValue.md>`__

-  `phaeopigments <phaeopigments.md>`__ 0..1

   -  Description: Concentration of phaeopigments; can include multiple
      phaeopigments
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `phosplipid_fatt_acid <phosplipid_fatt_acid.md>`__ 0..1

   -  Description: Concentration of phospholipid fatty acids; can
      include multiple values
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `pool_dna_extracts <pool_dna_extracts.md>`__ 0..1

   -  Description: “Indicate whether multiple DNA extractions were
      mixed. If the answer yes, the number of extracts that were pooled
      should be given”
   -  Range: `TextValue <TextValue.md>`__

-  `potassium <potassium.md>`__ 0..1

   -  Description: Concentration of potassium in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `pressure <pressure.md>`__ 0..1

   -  Description: “Pressure to which the sample is subject to, in
      atmospheres”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `previous_land_use <previous_land_use.md>`__ 0..1

   -  Description: Previous land use and dates
   -  Range: `TextValue <TextValue.md>`__

-  `previous_land_use_meth <previous_land_use_meth.md>`__ 0..1

   -  Description: Reference or method used in determining previous land
      use and dates
   -  Range: `TextValue <TextValue.md>`__

-  `profile_position <profile_position.md>`__ 0..1

   -  Description: Cross-sectional position in the hillslope where
      sample was collected.sample area position in relation to
      surrounding areas
   -  Range: `TextValue <TextValue.md>`__

-  `redox_potential <redox_potential.md>`__ 0..1

   -  Description: “Redox potential, measured relative to a hydrogen
      cell, indicating oxidation or reduction potential”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `salinity <salinity.md>`__ 0..1

   -  Description: “Salinity is the total concentration of all dissolved
      salts in a water sample. While salinity can be measured by a
      complete chemical analysis, this method is difficult and time
      consuming. More often, it is instead derived from the conductivity
      measurement. This is known as practical salinity. These
      derivations compare the specific conductance of the sample to a
      salinity standard such as seawater”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `salinity_meth <salinity_meth.md>`__ 0..1

   -  Description: Reference or method used in determining salinity
   -  Range: `TextValue <TextValue.md>`__

-  `samp_collect_device <samp_collect_device.md>`__ 0..1

   -  Description: The method or device employed for collecting the
      sample
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (nucleic acid sequence source)

-  `samp_mat_process <samp_mat_process.md>`__ 0..1

   -  Description: “Any processing applied to the sample during or after
      retrieving the sample from environment. This field accepts OBI,
      for a browser of OBI (v 2018-02-12) terms please see
      http://purl.bioontology.org/ontology/OBI”
   -  Range: `ControlledTermValue <ControlledTermValue.md>`__
   -  in subsets: (nucleic acid sequence source)

-  `samp_store_dur <samp_store_dur.md>`__ 0..1

   -  Description: Duration for which the sample was stored
   -  Range: `TextValue <TextValue.md>`__

-  `samp_store_loc <samp_store_loc.md>`__ 0..1

   -  Description: “Location at which sample was stored, usually name of
      a specific freezer/room”
   -  Range: `TextValue <TextValue.md>`__

-  `samp_store_temp <samp_store_temp.md>`__ 0..1

   -  Description: “Temperature at which sample was stored, e.g. -80
      degree Celsius”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `samp_vol_we_dna_ext <samp_vol_we_dna_ext.md>`__ 0..1

   -  Description: “Volume (ml), weight (g) of processed sample, or
      surface area swabbed from sample for DNA extraction”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `season_temp <season_temp.md>`__ 0..1

   -  Description: Mean seasonal temperature
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `season_precpt <season_precpt.md>`__ 0..1

   -  Description: “The average of all seasonal precipitation values
      known, or an estimated equivalent value derived by such methods as
      regional indexes or Isohyetal maps.”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `sieving <sieving.md>`__ 0..1

   -  Description: Collection design of pooled samples and/or sieve size
      and amount of sample sieved
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `size_frac_low <size_frac_low.md>`__ 0..1

   -  Description: Refers to the mesh/pore size used to
      pre-filter/pre-sort the sample. Materials larger than the size
      threshold are excluded from the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `size_frac_up <size_frac_up.md>`__ 0..1

   -  Description: Refers to the mesh/pore size used to retain the
      sample. Materials smaller than the size threshold are excluded
      from the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `slope_gradient <slope_gradient.md>`__ 0..1

   -  Description: Commonly called ‘slope’. The angle between ground
      surface and a horizontal line (in percent). This is the direction
      that overland water would flow. This measure is usually taken with
      a hand level meter or clinometer
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `slope_aspect <slope_aspect.md>`__ 0..1

   -  Description: “The direction a slope faces. While looking down a
      slope use a compass to record the direction you are facing
      (direction or degrees); e.g., nw or 315 degrees. This measure
      provides an indication of sun and wind exposure that will
      influence soil temperature and evapotranspiration.”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `sodium <sodium.md>`__ 0..1

   -  Description: Sodium concentration in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `soil_type <soil_type.md>`__ 0..1

   -  Description: Soil series name or other lower-level classification
   -  Range: `TextValue <TextValue.md>`__

-  `soil_type_meth <soil_type_meth.md>`__ 0..1

   -  Description: Reference or method used in determining soil series
      name or other lower-level classification
   -  Range: `TextValue <TextValue.md>`__

-  `store_cond <store_cond.md>`__ 0..1

   -  Description: Explain how and for how long the soil sample was
      stored before DNA extraction
   -  Range: `TextValue <TextValue.md>`__

-  `sulfate <sulfate.md>`__ 0..1

   -  Description: Concentration of sulfate in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `sulfide <sulfide.md>`__ 0..1

   -  Description: Concentration of sulfide in the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `temp <temp.md>`__ 0..1

   -  Description: Temperature of the sample at the time of sampling
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `texture <texture.md>`__ 0..1

   -  Description: “The relative proportion of different grain sizes of
      mineral particles in a soil, as described using a standard system;
      express as % sand (50 um to 2 mm), silt (2 um to 50 um), and clay
      (<2 um) with textural name (e.g., silty clay loam) optional.”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `texture_meth <texture_meth.md>`__ 0..1

   -  Description: Reference or method used in determining soil texture
   -  Range: `TextValue <TextValue.md>`__

-  `tillage <tillage.md>`__ 0..1

   -  Description: Note method(s) used for tilling
   -  Range: `TextValue <TextValue.md>`__

-  `tidal_stage <tidal_stage.md>`__ 0..1

   -  Description: Stage of tide
   -  Range: `TextValue <TextValue.md>`__

-  `tot_carb <tot_carb.md>`__ 0..1

   -  Description: Total carbon content
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `tot_depth_water_col <tot_depth_water_col.md>`__ 0..1

   -  Description: Measurement of total depth of water column
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `tot_diss_nitro <tot_diss_nitro.md>`__ 0..1

   -  Description: “Total dissolved nitrogen concentration, reported as
      nitrogen, measured by: total dissolved nitrogen = NH4 + NO3NO2 +
      dissolved organic nitrogen”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `tot_org_carb <tot_org_carb.md>`__ 0..1

   -  Description: “Definition for soil: total organic carbon content of
      the soil, definition otherwise: total organic carbon content”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `tot_org_c_meth <tot_org_c_meth.md>`__ 0..1

   -  Description: Reference or method used in determining total organic
      carbon
   -  Range: `TextValue <TextValue.md>`__

-  `tot_nitro_content <tot_nitro_content.md>`__ 0..1

   -  Description: Total nitrogen content of the sample
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `tot_nitro_content_meth <tot_nitro_content_meth.md>`__ 0..1

   -  Description: Reference or method used in determining the total
      nitrogen
   -  Range: `TextValue <TextValue.md>`__

-  `tot_phosp <tot_phosp.md>`__ 0..1

   -  Description: “Total phosphorus concentration in the sample,
      calculated by: total phosphorus = total dissolved phosphorus +
      particulate phosphorus”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `water_content <water_content.md>`__ 0..1

   -  Description: Water content measurement
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `water_content_soil_meth <water_content_soil_meth.md>`__ 0..1

   -  Description: Reference or method used in determining the water
      content of soil
   -  Range: `TextValue <TextValue.md>`__

-  `ecosystem <ecosystem.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `ecosystem_category <ecosystem_category.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `ecosystem_type <ecosystem_type.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `ecosystem_subtype <ecosystem_subtype.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `specific_ecosystem <specific_ecosystem.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `add_date <add_date.md>`__ 0..1

   -  Description: The date on which the information was added to the
      database.
   -  Range: `String <types/String.md>`__

-  `community <community.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `depth2 <depth2.md>`__ 0..1

   -  Range: `QuantityValue <QuantityValue.md>`__

-  `habitat <habitat.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `host_name <host_name.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `identifier <identifier.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `location <location.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `mod_date <mod_date.md>`__ 0..1

   -  Description: The last date on which the database information was
      modified.
   -  Range: `String <types/String.md>`__

-  `ncbi_taxonomy_name <ncbi_taxonomy_name.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `proport_woa_temperature <proport_woa_temperature.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `salinity_category <salinity_category.md>`__ 0..1

   -  Description: Categorcial description of the sample’s salinity.
      Examples: halophile, halotolerant, hypersaline, huryhaline
   -  Range: `String <types/String.md>`__

-  `sample_collection_site <sample_collection_site.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `soluble_iron_micromol <soluble_iron_micromol.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `subsurface_depth <subsurface_depth.md>`__ 0..1

   -  Range: `QuantityValue <QuantityValue.md>`__

-  `subsurface_depth2 <subsurface_depth2.md>`__ 0..1

   -  Range: `QuantityValue <QuantityValue.md>`__

-  `GOLD sample identifiers <GOLD_sample_identifiers.md>`__ 0..\*

   -  Description: [‘identifiers for corresponding sample in GOLD’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/gold:GbTODO None

-  `INSDC biosample identifiers <INSDC_biosample_identifiers.md>`__
   0..\*

   -  Description: [‘identifiers for corresponding sample in INSDC’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/biosample:SAMEA5989477 None
   -  Example: https://identifiers.org/biosample:SAMD00212331 I13_N_5-10
      sample from Soil fungal diversity along elevational gradients

-  `INSDC secondary sample
   identifiers <INSDC_secondary_sample_identifiers.md>`__ 0..\*

   -  Description: [‘secondary identifiers for corresponding sample in
      INSDC’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/insdc.sra:DRS166340 I13_N_5-10
      sample from Soil fungal diversity along elevational gradients

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

.. _other-properties-2:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | sample                |
+-----------------------+-----------------------+-----------------------+
|                       |                       | material sample       |
+-----------------------+-----------------------+-----------------------+
|                       |                       | specimen              |
+-----------------------+-----------------------+-----------------------+
|                       |                       | biospecimen           |
+-----------------------+-----------------------+-----------------------+
| **Alt Descriptions:** |                       | A sample contains     |
|                       |                       | information about the |
|                       |                       | sequenced source      |
|                       |                       | material. Samples are |
|                       |                       | associated with       |
|                       |                       | checklists, which     |
|                       |                       | define the fields     |
|                       |                       | used to annotate the  |
|                       |                       | samples. Samples are  |
|                       |                       | always associated     |
|                       |                       | with a taxon.         |
|                       |                       | (embl.ena)            |
+-----------------------+-----------------------+-----------------------+
| **In Subsets:**       |                       | sample subset         |
+-----------------------+-----------------------+-----------------------+
| **Exact Mappings:**   |                       | OBI:0000747           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | NCIT:C43412           |
+-----------------------+-----------------------+-----------------------+

Class: biosample processing
===========================

A process that takes one or more biosamples as inputs and generates one
or as outputs. Examples of outputs include samples cultivated from
another sample or data objects created by instruments runs.

URI:
`nmdc:BiosampleProcessing <https://microbiomedata/meta/BiosampleProcessing>`__

|image4|

.. _parents-1:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _children-2:

Children
--------

-  `OmicsProcessing <OmicsProcessing.md>`__ - The methods and processes
   used to generate omics data from a biosample or organism.

.. _referenced-by-class-4:

Referenced by Class
-------------------

.. _attributes-4:

Attributes
----------

.. _own-4:

Own
~~~

-  `biosample processing➞has
   input <biosample_processing_has_input.md>`__ 0..\*

   -  Description: An input to a process.
   -  Range: `Biosample <Biosample.md>`__

.. _inherited-from-named-thing-1:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-3:

Other properties
----------------

=================== ===================
**Aliases:**         material processing
**Broad Mappings:**  OBI:0000094
=================== ===================

Class: boolean value
====================

A value that is a boolean

URI:
https://microbiomedata/schema/workflow_execution_activity/BooleanValue

|image5|

.. _parents-2:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _attributes-5:

Attributes
----------

.. _own-5:

Own
~~~

-  `has boolean value <has_boolean_value.md>`__ 0..1

   -  Description: Links a quantity value to a boolean
   -  Range: `Boolean <types/Boolean.md>`__

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: chemical entity
======================

An atom or molecule that can be represented with a chemical formula.
Include lipids, glycans, natural products, drugs. There may be different
terms for distinct acid-base forms, protonation states

URI:
https://microbiomedata/schema/workflow_execution_activity/ChemicalEntity

|image6|

.. _identifier-prefixes-1:

Identifier prefixes
-------------------

-  KEGG.COMPOUND
-  CHEBI
-  CHEMBL.COMPOUND
-  DRUGBANK
-  PUBCHEM.COMPOUND
-  CAS
-  HMDB
-  MESH

.. _parents-3:

Parents
-------

-  is_a: `OntologyClass <OntologyClass.md>`__

.. _referenced-by-class-5:

Referenced by Class
-------------------

-  `MetaboliteQuantification <MetaboliteQuantification.md>`__
   `metabolite quantification➞metabolite
   quantified <metabolite_quantification_metabolite_quantified.md>`__
   0..1 `ChemicalEntity <ChemicalEntity.md>`__
-  **None** `metabolite quantified <metabolite_quantified.md>`__ 0..1
   `ChemicalEntity <ChemicalEntity.md>`__

.. _attributes-6:

Attributes
----------

.. _own-6:

Own
~~~

-  `chemical entity➞inchi <chemical_entity_inchi.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `chemical entity➞inchi key <chemical_entity_inchi_key.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `chemical entity➞smiles <chemical_entity_smiles.md>`__ 0..\*

   -  Description: A string encoding of a molecular graph, no chiral or
      isotopic information. There are usually a large number of valid
      SMILES which represent a given structure. For example, CCO, OCC
      and C(O)C all specify the structure of ethanol.
   -  Range: `String <types/String.md>`__

-  `chemical entity➞chemical
   formula <chemical_entity_chemical_formula.md>`__ 0..1

   -  Description: A generic grouping for miolecular formulae and
      empirican formulae
   -  Range: `String <types/String.md>`__

Inherited from ontology class:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-4:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | metabolite            |
+-----------------------+-----------------------+-----------------------+
|                       |                       | chemical substance    |
+-----------------------+-----------------------+-----------------------+
|                       |                       | chemical compound     |
+-----------------------+-----------------------+-----------------------+
|                       |                       | chemical              |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://biocon        |
|                       |                       | ductor.org/packages/d |
|                       |                       | evel/data/annotation/ |
|                       |                       | vignettes/metaboliteI |
|                       |                       | Dmapping/inst/doc/met |
|                       |                       | aboliteIDmapping.html |
+-----------------------+-----------------------+-----------------------+
| **Exact Mappings:**   |                       | biol                  |
|                       |                       | ink:ChemicalSubstance |
+-----------------------+-----------------------+-----------------------+

Class: controlled term value
============================

A controlled term or class from an ontology

URI:
https://microbiomedata/schema/workflow_execution_activity/ControlledTermValue

|image7|

.. _parents-4:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _referenced-by-class-6:

Referenced by Class
-------------------

.. _attributes-7:

Attributes
----------

.. _own-7:

Own
~~~

-  `term <term.md>`__ 0..1

   -  Description: pointer to an ontology class
   -  Range: `OntologyClass <OntologyClass.md>`__

.. _inherited-from-attribute-value-1:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: credit association
=========================

This class supports binding associated researchers to studies. There
will be at least a slot for a CRediT Contributor Role
(https://casrai.org/credit/) and for a person value Specifically see the
associated researchers tab on the
NMDC_SampleMetadata-V4_CommentsForUpdates at
https://docs.google.com/spreadsheets/d/1INlBo5eoqn2efn4H2P2i8rwRBtnbDVTqXrochJEAPko/edit#gid=0

URI:
`nmdc:CreditAssociation <https://microbiomedata/meta/CreditAssociation>`__

|image8|

.. _referenced-by-class-7:

Referenced by Class
-------------------

-  `Study <Study.md>`__ `has credit
   associations <has_credit_associations.md>`__ 0..\*
   `CreditAssociation <CreditAssociation.md>`__

.. _attributes-8:

Attributes
----------

.. _own-8:

Own
~~~

-  `applies to person <applies_to_person.md>`__ 1..1

   -  Range: `PersonValue <PersonValue.md>`__

-  `applied role <applied_role.md>`__ 1..1

   -  Range: `credit enum <credit%20enum.md>`__

-  `type <type.md>`__ 0..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

.. _other-properties-5:

Other properties
----------------

============= ======================
**Aliases:**   study role
\              credit table
\              associated researchers
**Mappings:**  prov:Association
============= ======================

Class: data object
==================

An object that primarily consists of symbols that represent information.
Files, records, and omics data are examples of data objects.

URI: `nmdc:DataObject <https://microbiomedata/meta/DataObject>`__

|image9|

.. _identifier-prefixes-2:

Identifier prefixes
-------------------

-  GOLD

.. _parents-5:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _referenced-by-class-8:

Referenced by Class
-------------------

-  `Database <Database.md>`__ `data object set <data_object_set.md>`__
   0..\* `DataObject <DataObject.md>`__

.. _attributes-9:

Attributes
----------

.. _own-9:

Own
~~~

-  `file size bytes <file_size_bytes.md>`__ 0..1

   -  Description: Size of the file in bytes
   -  Range: `Bytes <types/Bytes.md>`__

-  `md5 checksum <md5_checksum.md>`__ 0..1

   -  Description: MD5 checksum of file (pre-compressed)
   -  Range: `String <types/String.md>`__

-  `data object type <data_object_type.md>`__ 0..1

   -  Description: The type of file represented by the data object.
   -  Range: `file type enum <file%20type%20enum.md>`__
   -  Example: FT ICR-MS Analysis Results None
   -  Example: GC-MS Metabolomics Results None

-  `compression type <compression_type.md>`__ 0..1

   -  Description: If provided, specifies the compression type
   -  Range: `String <types/String.md>`__
   -  Example: gzip None

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `url <url.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `type <type.md>`__ 0..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `data object➞name <data_object_name.md>`__ 1..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `data object➞description <data_object_description.md>`__ 1..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

.. _inherited-from-named-thing-2:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-6:

Other properties
----------------

=============== ==================
**In Subsets:**  data object subset
=============== ==================

Subset: data object subset
==========================

Subset consisting of the data objects that either inputs or outputs of
processes or workflows.

URI:
`nmdc:data_object_subset <https://microbiomedata/meta/data_object_subset>`__

Classes
-------

-  `DataObject <DataObject.md>`__ - An object that primarily consists of
   symbols that represent information. Files, records, and omics data
   are examples of data objects.

Mixins
------

Slots
-----

Types
-----

Enums
-----

Class: database
===============

An abstract holder for any set of metadata and data. It does not need to
correspond to an actual managed databse top level holder class. When
translated to JSON-Schema this is the ‘root’ object. It should contain
pointers to other objects of interest

URI: `nmdc:Database <https://microbiomedata/meta/Database>`__

|image10|

.. _referenced-by-class-9:

Referenced by Class
-------------------

.. _attributes-10:

Attributes
----------

.. _own-10:

Own
~~~

-  `biosample set <biosample_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      samples within it.
   -  Range: `Biosample <Biosample.md>`__

-  `study set <study_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      studies within it.
   -  Range: `Study <Study.md>`__

-  `data object set <data_object_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      data objects within it.
   -  Range: `DataObject <DataObject.md>`__

-  `activity set <activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      workflow activities.
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `mags activity set <mags_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      MAGs analysis activities.
   -  Range: `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

-  `metabolomics analysis activity
   set <metabolomics_analysis_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      metabolomics analysis activities.
   -  Range:
      `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__

-  `metaproteomics analysis activity
   set <metaproteomics_analysis_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      metaproteomics analysis activities.
   -  Range:
      `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__

-  `metagenome annotation activity
   set <metagenome_annotation_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      metagenome annotation activities.
   -  Range:
      `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__

-  `metagenome assembly set <metagenome_assembly_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      metagenome assembly activities.
   -  Range: `MetagenomeAssembly <MetagenomeAssembly.md>`__

-  `metatranscriptome activity
   set <metatranscriptome_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      metatranscriptome analysis activities.
   -  Range:
      `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__

-  `read QC analysis activity set <read_QC_analysis_activity_set.md>`__
   0..\*

   -  Description: This property links a database object to the set of
      read QC analysis activities.
   -  Range: `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

-  `read based analysis activity
   set <read_based_analysis_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      read based analysis activities.

   -  Range:
      `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__

-  `nom analysis activity set <nom_analysis_activity_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      natural organic matter (NOM) analysis activities.
   -  Range: `NomAnalysisActivity <NomAnalysisActivity.md>`__

-  `omics processing set <omics_processing_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      omics processings within it.
   -  Range: `OmicsProcessing <OmicsProcessing.md>`__

-  `functional annotation set <functional_annotation_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      all functional annotations
   -  Range: `FunctionalAnnotation <FunctionalAnnotation.md>`__

-  `genome feature set <genome_feature_set.md>`__ 0..\*

   -  Description: This property links a database object to the set of
      all features
   -  Range: `GenomeFeature <GenomeFeature.md>`__

-  `database➞nmdc schema version <database_nmdc_schema_version.md>`__
   0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `database➞date created <database_date_created.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `database➞etl software version <database_etl_software_version.md>`__
   0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

.. _other-properties-7:

Other properties
----------------

============ ====================
**Aliases:**  NMDC metadata object
============ ====================

Subset: environment
===================

URI:
https://microbiomedata/schema/workflow_execution_activity/environment

.. _classes-1:

Classes
-------

.. _mixins-1:

Mixins
------

.. _slots-1:

Slots
-----

.. _types-1:

Types
-----

.. _enums-1:

Enums
-----

Class: environmental material term
==================================

URI:
https://microbiomedata/schema/workflow_execution_activity/EnvironmentalMaterialTerm

|image11|

.. _parents-6:

Parents
-------

-  is_a: `OntologyClass <OntologyClass.md>`__

.. _attributes-11:

Attributes
----------

.. _inherited-from-ontology-class-1:

Inherited from ontology class:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

Class: functional annotation
============================

An assignment of a function term (e.g. reaction or pathway) that is
executed by a gene product, or which the gene product plays an active
role in. Functional annotations can be assigned manually by curators, or
automatically in workflows. In the context of NMDC, all function
annotation is performed automatically, typically using HMM or Blast type
methods

URI:
`nmdc:FunctionalAnnotation <https://microbiomedata/meta/FunctionalAnnotation>`__

|image12|

.. _referenced-by-class-10:

Referenced by Class
-------------------

-  `Database <Database.md>`__ `functional annotation
   set <functional_annotation_set.md>`__ 0..\*
   `FunctionalAnnotation <FunctionalAnnotation.md>`__

.. _attributes-12:

Attributes
----------

.. _own-11:

Own
~~~

-  `functional annotation➞was generated
   by <functional_annotation_was_generated_by.md>`__ 0..1

   -  Description: provenance for the annotation.
   -  Range:
      `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__

-  `subject <subject.md>`__ 0..1

   -  Range: `GeneProduct <GeneProduct.md>`__

-  `functional annotation➞has
   function <functional_annotation_has_function.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `functional annotation➞type <functional_annotation_type.md>`__ 0..1

   -  Description: TODO
   -  Range: `OntologyClass <OntologyClass.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

.. _other-properties-8:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | move id slot usage    |
|                       |                       | patterns to           |
|                       |                       | has_function slot     |
|                       |                       | usage?                |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://img.          |
|                       |                       | jgi.doe.gov/docs/func |
|                       |                       | tional-annotation.pdf |
+-----------------------+-----------------------+-----------------------+
|                       |                       | h                     |
|                       |                       | ttps://github.com/mic |
|                       |                       | robiomedata/mg_annota |
|                       |                       | tion/blob/master/func |
|                       |                       | tional-annotation.wdl |
+-----------------------+-----------------------+-----------------------+
| **Narrow Mappings:**  |                       | biolink:Ge            |
|                       |                       | neToGoTermAssociation |
+-----------------------+-----------------------+-----------------------+

Class: functional annotation term
=================================

Abstract grouping class for any term/descriptor that can be applied to a
functional unit of a genome (protein, ncRNA, complex).

URI:
`nmdc:FunctionalAnnotationTerm <https://microbiomedata/meta/FunctionalAnnotationTerm>`__

|image13|

.. _parents-7:

Parents
-------

-  is_a: `OntologyClass <OntologyClass.md>`__

.. _children-3:

Children
--------

-  `OrthologyGroup <OrthologyGroup.md>`__ - A set of genes or gene
   products in which all members are orthologous
-  `Pathway <Pathway.md>`__ - A pathway is a sequence of steps/reactions
   carried out by an organism or community of organisms
-  `Reaction <Reaction.md>`__ - An individual biochemical transformation
   carried out by a functional unit of an organism, in which a
   collection of substrates are transformed into a collection of
   products. Can also represent transporters

.. _referenced-by-class-11:

Referenced by Class
-------------------

.. _attributes-13:

Attributes
----------

.. _inherited-from-ontology-class-2:

Inherited from ontology class:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-9:

Other properties
----------------

============ =====================
**Aliases:**  function
\             functional annotation
============ =====================

Slot: GOLD analysis project identifiers
=======================================

[‘identifiers for corresponding analysis project in GOLD’]

URI:
https://microbiomedata/schema/workflow_execution_activity/GOLD_analysis_project_identifiers

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-8:

Parents
-------

-  is_a: `analysis identifiers <analysis_identifiers.md>`__

.. _children-4:

Children
--------

Used by
-------

.. _other-properties-10:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(va            |
|                       |                       | lue=‘https://identifi |
|                       |                       | ers.org/gold:GaTODO’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+

Slot: GOLD identifiers
======================

URI:
https://microbiomedata/schema/workflow_execution_activity/GOLD_identifiers

.. _domain-and-range-1:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-9:

Parents
-------

.. _children-5:

Children
--------

.. _used-by-1:

Used by
-------

.. _other-properties-11:

Other properties
----------------

============= =========================
**See also:**  https://gold.jgi.doe.gov/
============= =========================

Slot: GOLD sample identifiers
=============================

[‘identifiers for corresponding sample in GOLD’]

URI:
https://microbiomedata/schema/workflow_execution_activity/GOLD_sample_identifiers

.. _domain-and-range-2:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-10:

Parents
-------

-  is_a: `sample identifiers <sample_identifiers.md>`__

.. _children-6:

Children
--------

.. _used-by-2:

Used by
-------

.. _other-properties-12:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(va            |
|                       |                       | lue=‘https://identifi |
|                       |                       | ers.org/gold:GbTODO’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+

Slot: GOLD sequencing project identifiers
=========================================

[‘identifiers for corresponding sequencing project in GOLD’]

URI:
https://microbiomedata/schema/workflow_execution_activity/GOLD_sequencing_project_identifiers

.. _domain-and-range-3:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-11:

Parents
-------

-  is_a: `external database
   identifiers <external_database_identifiers.md>`__

.. _children-7:

Children
--------

.. _used-by-3:

Used by
-------

.. _other-properties-13:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(value         |
|                       |                       | =‘https://identifiers |
|                       |                       | .org/gold:Gp0108335’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+

Slot: GOLD study identifiers
============================

[‘identifiers for corresponding project in GOLD’]

URI:
https://microbiomedata/schema/workflow_execution_activity/GOLD_study_identifiers

.. _domain-and-range-4:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-12:

Parents
-------

-  is_a: `study identifiers <study_identifiers.md>`__

.. _children-8:

Children
--------

.. _used-by-4:

Used by
-------

.. _other-properties-14:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(value         |
|                       |                       | =‘https://identifiers |
|                       |                       | .org/gold:Gs0110115’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://gol           |
|                       |                       | d.jgi.doe.gov/studies |
+-----------------------+-----------------------+-----------------------+

Class: gene product
===================

A molecule encoded by a gene that has an evolved function

URI:
https://microbiomedata/schema/workflow_execution_activity/GeneProduct

|image14|

.. _identifier-prefixes-3:

Identifier prefixes
-------------------

-  UniProtKB
-  gtpo
-  PR

.. _parents-13:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _referenced-by-class-12:

Referenced by Class
-------------------

-  **None** `all proteins <all_proteins.md>`__ 0..\*
   `GeneProduct <GeneProduct.md>`__
-  **None** `best protein <best_protein.md>`__ 0..1
   `GeneProduct <GeneProduct.md>`__
-  `PeptideQuantification <PeptideQuantification.md>`__ `peptide
   quantification➞all
   proteins <peptide_quantification_all_proteins.md>`__ 0..\*
   `GeneProduct <GeneProduct.md>`__
-  `PeptideQuantification <PeptideQuantification.md>`__ `peptide
   quantification➞best
   protein <peptide_quantification_best_protein.md>`__ 0..1
   `GeneProduct <GeneProduct.md>`__
-  `ProteinQuantification <ProteinQuantification.md>`__ `protein
   quantification➞all
   proteins <protein_quantification_all_proteins.md>`__ 0..\*
   `GeneProduct <GeneProduct.md>`__
-  `ProteinQuantification <ProteinQuantification.md>`__ `protein
   quantification➞best
   protein <protein_quantification_best_protein.md>`__ 0..1
   `GeneProduct <GeneProduct.md>`__

.. _attributes-14:

Attributes
----------

.. _inherited-from-named-thing-3:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-15:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | we may include a more |
|                       |                       | general gene product  |
|                       |                       | class in future to    |
|                       |                       | allow for ncRNA       |
|                       |                       | annotation            |
+-----------------------+-----------------------+-----------------------+
| **Exact Mappings:**   |                       | biolink:GeneProduct   |
+-----------------------+-----------------------+-----------------------+

Class: genome feature
=====================

A feature localized to an interval along a genome

URI: `nmdc:GenomeFeature <https://microbiomedata/meta/GenomeFeature>`__

|image15|

.. _referenced-by-class-13:

Referenced by Class
-------------------

-  `Database <Database.md>`__ `genome feature
   set <genome_feature_set.md>`__ 0..\*
   `GenomeFeature <GenomeFeature.md>`__

.. _attributes-15:

Attributes
----------

.. _own-12:

Own
~~~

-  `genome feature➞seqid <genome_feature_seqid.md>`__ 1..1

   -  Description: The ID of the landmark used to establish the
      coordinate system for the current feature.
   -  Range: `String <types/String.md>`__

-  `genome feature➞type <genome_feature_type.md>`__ 0..1

   -  Description: A type from the sequence ontology
   -  Range: `OntologyClass <OntologyClass.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `genome feature➞start <genome_feature_start.md>`__ 1..1

   -  Description: The start of the feature in positive 1-based integer
      coordinates
   -  Range: `Integer <types/Integer.md>`__

-  `genome feature➞end <genome_feature_end.md>`__ 1..1

   -  Description: The end of the feature in positive 1-based integer
      coordinates
   -  Range: `Integer <types/Integer.md>`__

-  `genome feature➞strand <genome_feature_strand.md>`__ 0..1

   -  Description: The strand on which a feature is located. Has a value
      of ‘+’ (sense strand or forward strand) or ‘-’ (anti-sense strand
      or reverse strand).
   -  Range: `String <types/String.md>`__

-  `genome feature➞phase <genome_feature_phase.md>`__ 0..1

   -  Description: The phase for a coding sequence entity. For example,
      phase of a CDS as represented in a GFF3 with a value of 0, 1 or 2.
   -  Range: `Integer <types/Integer.md>`__

-  `genome feature➞encodes <genome_feature_encodes.md>`__ 0..1

   -  Description: The gene product encoded by this feature. Typically
      this is used for a CDS feature or gene feature which will encode a
      protein. It can also be used by a nc transcript ot gene feature
      that encoded a ncRNA
   -  Range: `GeneProduct <GeneProduct.md>`__

-  `genome feature➞feature type <genome_feature_feature_type.md>`__ 0..1

   -  Description: TODO: Yuri to write
   -  Range: `String <types/String.md>`__

.. _other-properties-16:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | corresponds to an     |
|                       |                       | entry in GFF3         |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://gith          |
|                       |                       | ub.com/The-Sequence-O |
|                       |                       | ntology/Specification |
|                       |                       | s/blob/master/gff3.md |
+-----------------------+-----------------------+-----------------------+

Class: geolocation value
========================

A normalized value for a location on the earth’s surface

URI:
https://microbiomedata/schema/workflow_execution_activity/GeolocationValue

|image16|

.. _parents-14:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _referenced-by-class-14:

Referenced by Class
-------------------

.. _attributes-16:

Attributes
----------

.. _own-13:

Own
~~~

-  `latitude <latitude.md>`__ 0..1

   -  Description: latitude
   -  Range: `DecimalDegree <types/DecimalDegree.md>`__

-  `longitude <longitude.md>`__ 0..1

   -  Description: longitude
   -  Range: `DecimalDegree <types/DecimalDegree.md>`__

-  `geolocation value➞has raw
   value <geolocation_value_has_raw_value.md>`__ 0..1

   -  Description: The raw value for a geolocation should follow {lat}
      {long}
   -  Range: `String <types/String.md>`__

.. _inherited-from-attribute-value-2:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

.. _other-properties-17:

Other properties
----------------

============= =====================
**Mappings:**  schema:GeoCoordinates
============= =====================

Slot: INSDC SRA ENA study identifiers
=====================================

[‘identifiers for corresponding project in INSDC SRA / ENA’]

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_SRA_ENA_study_identifiers

.. _domain-and-range-5:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-15:

Parents
-------

-  is_a: `study identifiers <study_identifiers.md>`__

.. _children-9:

Children
--------

.. _used-by-5:

Used by
-------

.. _other-properties-18:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | EBI ENA study         |
|                       |                       | identifiers           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | NCBI SRA identifiers  |
+-----------------------+-----------------------+-----------------------+
|                       |                       | DDBJ SRA identifiers  |
+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(value=‘htt    |
|                       |                       | ps://identifiers.org/ |
|                       |                       | insdc.sra:SRP121659’, |
|                       |                       | description=‘Avena    |
|                       |                       | fatua rhizosphere     |
|                       |                       | microbial communities |
|                       |                       | - H1_Rhizo_Litter_2   |
|                       |                       | metatranscriptome’)   |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://git           |
|                       |                       | hub.com/bioregistry/b |
|                       |                       | ioregistry/issues/109 |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https://trace.ncbi.n  |
|                       |                       | lm.nih.gov/Traces/sra |
|                       |                       | /sra.cgi?view=studies |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https://trace.ncbi.n  |
|                       |                       | lm.nih.gov/Traces/sra |
|                       |                       | /sra.cgi?view=studies |
+-----------------------+-----------------------+-----------------------+

Slot: INSDC analysis identifiers
================================

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_analysis_identifiers

.. _domain-and-range-6:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-16:

Parents
-------

-  is_a: `analysis identifiers <analysis_identifiers.md>`__

.. _children-10:

Children
--------

.. _used-by-6:

Used by
-------

.. _other-properties-19:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | in INSDC this is a    |
|                       |                       | run but it            |
|                       |                       | corresponds to a GOLD |
|                       |                       | analysis              |
+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Exa                   |
|                       |                       | mple(value=‘https://w |
|                       |                       | ww.ebi.ac.uk/metageno |
|                       |                       | mics/runs/DRR218479’, |
|                       |                       | description=‘Illumina |
|                       |                       | MiSeq paired end      |
|                       |                       | sequencing of         |
|                       |                       | SAMD00212331’)        |
+-----------------------+-----------------------+-----------------------+
|                       |                       | Ex                    |
|                       |                       | ample(value=‘https:// |
|                       |                       | www.ebi.ac.uk/ena/bro |
|                       |                       | wser/view/ERR436051’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+

Slot: INSDC assembly identifiers
================================

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_assembly_identifiers

.. _domain-and-range-7:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-17:

Parents
-------

-  is_a: `assembly identifiers <assembly_identifiers.md>`__

.. _children-11:

Children
--------

.. _used-by-7:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: INSDC bioproject identifiers
==================================

[‘identifiers for corresponding project in INSDC Bioproject’]

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_bioproject_identifiers

.. _domain-and-range-8:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-18:

Parents
-------

-  is_a: `study identifiers <study_identifiers.md>`__

.. _children-12:

Children
--------

.. _used-by-8:

Used by
-------

.. _other-properties-20:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | NCBI bioproject       |
|                       |                       | identifiers           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | DDBJ bioproject       |
|                       |                       | identifiers           |
+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | these are distinct    |
|                       |                       | IDs from INSDC        |
|                       |                       | SRA/ENA project       |
|                       |                       | identifiers, but are  |
|                       |                       | usually(?) one to one |
+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(value=‘https: |
|                       |                       | //identifiers.org/bio |
|                       |                       | project:PRJNA366857’, |
|                       |                       | description=‘Avena    |
|                       |                       | fatua rhizosphere     |
|                       |                       | microbial communities |
|                       |                       | - H1_Rhizo_Litter_2   |
|                       |                       | metatranscriptome’)   |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://www.ncbi.nl   |
|                       |                       | m.nih.gov/bioproject/ |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https://              |
|                       |                       | www.ddbj.nig.ac.jp/bi |
|                       |                       | oproject/index-e.html |
+-----------------------+-----------------------+-----------------------+

Slot: INSDC biosample identifiers
=================================

[‘identifiers for corresponding sample in INSDC’]

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_biosample_identifiers

.. _domain-and-range-9:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-19:

Parents
-------

-  is_a: `sample identifiers <sample_identifiers.md>`__

.. _children-13:

Children
--------

.. _used-by-9:

Used by
-------

.. _other-properties-21:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | EBI biosample         |
|                       |                       | identifiers           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | NCBI biosample        |
|                       |                       | identifiers           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | DDBJ biosample        |
|                       |                       | identifiers           |
+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(value=‘https: |
|                       |                       | //identifiers.org/bio |
|                       |                       | sample:SAMEA5989477’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+
|                       |                       | Example(value=‘https: |
|                       |                       | //identifiers.org/bio |
|                       |                       | sample:SAMD00212331’, |
|                       |                       | de                    |
|                       |                       | scription=‘I13_N_5-10 |
|                       |                       | sample from Soil      |
|                       |                       | fungal diversity      |
|                       |                       | along elevational     |
|                       |                       | gradients’)           |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | https://git           |
|                       |                       | hub.com/bioregistry/b |
|                       |                       | ioregistry/issues/108 |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https://www.          |
|                       |                       | ebi.ac.uk/biosamples/ |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https://www.ncbi.     |
|                       |                       | nlm.nih.gov/biosample |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https:/               |
|                       |                       | /www.ddbj.nig.ac.jp/b |
|                       |                       | iosample/index-e.html |
+-----------------------+-----------------------+-----------------------+

Slot: INSDC experiment identifiers
==================================

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_experiment_identifiers

.. _domain-and-range-10:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-20:

Parents
-------

-  is_a: `external database
   identifiers <external_database_identifiers.md>`__

.. _children-14:

Children
--------

.. _used-by-10:

Used by
-------

Slot: INSDC identifiers
=======================

Any identifier covered by the International Nucleotide Sequence Database
Collaboration

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_identifiers

.. _domain-and-range-11:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-21:

Parents
-------

.. _children-15:

Children
--------

.. _used-by-11:

Used by
-------

.. _other-properties-22:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | EBI identifiers       |
+-----------------------+-----------------------+-----------------------+
|                       |                       | NCBI identifiers      |
+-----------------------+-----------------------+-----------------------+
|                       |                       | DDBJ identifiers      |
+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | note that we          |
|                       |                       | deliberately abstract |
|                       |                       | over which of the     |
|                       |                       | partner databases     |
|                       |                       | accepted the initial  |
|                       |                       | submission            |
+-----------------------+-----------------------+-----------------------+
|                       |                       | {‘the first letter of |
|                       |                       | the accession         |
|                       |                       | indicates which       |
|                       |                       | partner accepted the  |
|                       |                       | initial submission’:  |
|                       |                       | ‘E for ENA, D for     |
|                       |                       | DDBJ, or S or N for   |
|                       |                       | NCBI.’}               |
+-----------------------+-----------------------+-----------------------+
| **See also:**         |                       | h                     |
|                       |                       | ttps://www.insdc.org/ |
+-----------------------+-----------------------+-----------------------+
|                       |                       | https://ena-doc       |
|                       |                       | s.readthedocs.io/en/l |
|                       |                       | atest/submit/general- |
|                       |                       | guide/accessions.html |
+-----------------------+-----------------------+-----------------------+

Slot: INSDC secondary sample identifiers
========================================

[‘secondary identifiers for corresponding sample in INSDC’]

URI:
https://microbiomedata/schema/workflow_execution_activity/INSDC_secondary_sample_identifiers

.. _domain-and-range-12:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-22:

Parents
-------

-  is_a: `sample identifiers <sample_identifiers.md>`__

.. _children-16:

Children
--------

.. _used-by-12:

Used by
-------

.. _other-properties-23:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | ENA redirects these   |
|                       |                       | to primary IDs,       |
|                       |                       | e.g. https:           |
|                       |                       | //www.ebi.ac.uk/ena/b |
|                       |                       | rowser/view/DRS166340 |
|                       |                       | -> SAMD00212331       |
+-----------------------+-----------------------+-----------------------+
|                       |                       | MGnify uses these as  |
|                       |                       | their primary sample  |
|                       |                       | IDs                   |
+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(value=‘htt    |
|                       |                       | ps://identifiers.org/ |
|                       |                       | insdc.sra:DRS166340’, |
|                       |                       | de                    |
|                       |                       | scription=‘I13_N_5-10 |
|                       |                       | sample from Soil      |
|                       |                       | fungal diversity      |
|                       |                       | along elevational     |
|                       |                       | gradients’)           |
+-----------------------+-----------------------+-----------------------+

Class: image value
==================

An attribute value representing an image.

URI:
https://microbiomedata/schema/workflow_execution_activity/ImageValue

|image17|

.. _parents-23:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _attributes-17:

Attributes
----------

.. _own-14:

Own
~~~

-  `url <url.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `display order <display_order.md>`__ 0..1

   -  Description: When rendering information, this attribute to specify
      the order in which the information should be rendered.
   -  Range: `String <types/String.md>`__

.. _inherited-from-attribute-value-3:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: instrument
=================

A material entity that is designed to perform a function in a scientific
investigation, but is not a reagent[OBI].

URI:
https://microbiomedata/schema/workflow_execution_activity/Instrument

|image18|

.. _parents-24:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _referenced-by-class-15:

Referenced by Class
-------------------

-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
   `metabolomics analysis
   activity➞used <metabolomics_analysis_activity_used.md>`__ 0..1
   `Instrument <Instrument.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
   `metaproteomics analysis
   activity➞used <metaproteomics_analysis_activity_used.md>`__ 0..1
   `Instrument <Instrument.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__ `nom analysis
   activity➞used <nom_analysis_activity_used.md>`__ 0..1
   `Instrument <Instrument.md>`__

.. _attributes-18:

Attributes
----------

.. _inherited-from-named-thing-4:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-24:

Other properties
----------------

=================== ===========
**Aliases:**         device
**Exact Mappings:**  OBI:0000485
=================== ===========

Class: integer value
====================

A value that is an integer

URI:
https://microbiomedata/schema/workflow_execution_activity/IntegerValue

|image19|

.. _parents-25:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _attributes-19:

Attributes
----------

.. _own-15:

Own
~~~

-  `has numeric value <has_numeric_value.md>`__ 0..1

   -  Description: Links a quantity value to a number
   -  Range: `Float <types/Float.md>`__

.. _inherited-from-attribute-value-4:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Subset: investigation
=====================

URI:
https://microbiomedata/schema/workflow_execution_activity/investigation

.. _classes-2:

Classes
-------

.. _mixins-2:

Mixins
------

.. _slots-2:

Slots
-----

.. _types-2:

Types
-----

.. _enums-2:

Enums
-----

Class: MAG bin
==============

URI: https://microbiomedata/schema/workflow_execution_activity/MAGBin

|image20|

.. _referenced-by-class-16:

Referenced by Class
-------------------

-  **None** `mags list <mags_list.md>`__ 0..\* `MAGBin <MAGBin.md>`__

.. _attributes-20:

Attributes
----------

.. _own-16:

Own
~~~

-  `type <type.md>`__ 0..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `➞bin name <mAGBin__bin_name.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞number of contig <mAGBin__number_of_contig.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `➞completeness <mAGBin__completeness.md>`__ 0..1

   -  Range: `Float <types/Float.md>`__

-  `➞contamination <mAGBin__contamination.md>`__ 0..1

   -  Range: `Float <types/Float.md>`__

-  `➞gene count <mAGBin__gene_count.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `➞bin quality <mAGBin__bin_quality.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞num 16s <mAGBin__num_16s.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `➞num 5s <mAGBin__num_5s.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `➞num 23s <mAGBin__num_23s.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `➞num tRNA <mAGBin__num_tRNA.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `➞gtdbtk domain <mAGBin__gtdbtk_domain.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞gtdbtk phylum <mAGBin__gtdbtk_phylum.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞gtdbtk class <mAGBin__gtdbtk_class.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞gtdbtk order <mAGBin__gtdbtk_order.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞gtdbtk family <mAGBin__gtdbtk_family.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞gtdbtk genus <mAGBin__gtdbtk_genus.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `➞gtdbtk species <mAGBin__gtdbtk_species.md>`__ 0..1

   -  Range: `String <types/String.md>`__

Class: MAGs analysis activity
=============================

URI:
https://microbiomedata/schema/workflow_execution_activity/MAGsAnalysisActivity

|image21|

.. _parents-26:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-21:

Attributes
----------

.. _own-17:

Own
~~~

-  `input contig num <input_contig_num.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `binned contig num <binned_contig_num.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `too short contig num <too_short_contig_num.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `lowDepth contig num <lowDepth_contig_num.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `unbinned contig num <unbinned_contig_num.md>`__ 0..1

   -  Range: `Integer <types/Integer.md>`__

-  `mags list <mags_list.md>`__ 0..\*

   -  Range: `MAGBin <MAGBin.md>`__

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-25:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Slot: MGnify analysis identifiers
=================================

URI:
https://microbiomedata/schema/workflow_execution_activity/MGnify_analysis_identifiers

.. _domain-and-range-13:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-27:

Parents
-------

-  is_a: `analysis identifiers <analysis_identifiers.md>`__

.. _children-17:

Children
--------

.. _used-by-13:

Used by
-------

.. _other-properties-26:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Example(va            |
|                       |                       | lue=‘https://www.ebi. |
|                       |                       | ac.uk/metagenomics/an |
|                       |                       | alyses/MGYA00002270’, |
|                       |                       | description=‘combined |
|                       |                       | analyses (taxonomic,  |
|                       |                       | functional) of sample |
|                       |                       | ERS438107’)           |
+-----------------------+-----------------------+-----------------------+

Slot: MGnify identifiers
========================

URI:
https://microbiomedata/schema/workflow_execution_activity/MGnify_identifiers

.. _domain-and-range-14:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-28:

Parents
-------

.. _children-18:

Children
--------

.. _used-by-14:

Used by
-------

.. _other-properties-27:

Other properties
----------------

============= ===================================
**See also:**  https://www.ebi.ac.uk/metagenomics/
============= ===================================

Slot: MGnify project identifiers
================================

[‘identifiers for corresponding project in MGnify’]

URI:
https://microbiomedata/schema/workflow_execution_activity/MGnify_project_identifiers

.. _domain-and-range-15:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-29:

Parents
-------

-  is_a: `study identifiers <study_identifiers.md>`__

.. _children-19:

Children
--------

.. _used-by-15:

Used by
-------

.. _other-properties-28:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Examples:**         |                       | Ex                    |
|                       |                       | ample(value=‘https:// |
|                       |                       | identifiers.org/mgnif |
|                       |                       | y.proj:MGYS00005757’, |
|                       |                       | description=None)     |
+-----------------------+-----------------------+-----------------------+

Class: metabolite quantification
================================

This is used to link a metabolomics analysis workflow to a specific
metabolite

URI:
https://microbiomedata/schema/workflow_execution_activity/MetaboliteQuantification

|image22|

.. _referenced-by-class-17:

Referenced by Class
-------------------

-  **None** `has metabolite
   quantifications <has_metabolite_quantifications.md>`__ 0..\*
   `MetaboliteQuantification <MetaboliteQuantification.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
   `metabolomics analysis activity➞has metabolite
   quantifications <metabolomics_analysis_activity_has_metabolite_quantifications.md>`__
   0..\* `MetaboliteQuantification <MetaboliteQuantification.md>`__

.. _attributes-22:

Attributes
----------

.. _own-18:

Own
~~~

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

-  `metabolite quantification➞metabolite
   quantified <metabolite_quantification_metabolite_quantified.md>`__
   0..1

   -  Description: the specific metabolite identifier
   -  Range: `ChemicalEntity <ChemicalEntity.md>`__

-  `metabolite quantification➞highest similarity
   score <metabolite_quantification_highest_similarity_score.md>`__ 0..1

   -  Description: TODO: Yuri to fill in
   -  Range: `Float <types/Float.md>`__

Class: metabolomics analysis activity
=====================================

URI:
https://microbiomedata/schema/workflow_execution_activity/MetabolomicsAnalysisActivity

|image23|

.. _parents-30:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _referenced-by-class-18:

Referenced by Class
-------------------

.. _attributes-23:

Attributes
----------

.. _own-19:

Own
~~~

-  `metabolomics analysis
   activity➞used <metabolomics_analysis_activity_used.md>`__ 0..1

   -  Description: The instrument used to collect the data used in the
      analysis
   -  Range: `Instrument <Instrument.md>`__

-  `metabolomics analysis activity➞has metabolite
   quantifications <metabolomics_analysis_activity_has_metabolite_quantifications.md>`__
   0..\*

   -  Range: `MetaboliteQuantification <MetaboliteQuantification.md>`__

-  `metabolomics analysis activity➞has
   calibration <metabolomics_analysis_activity_has_calibration.md>`__
   0..1

   -  Description: TODO: Yuri to fill in
   -  Range: `String <types/String.md>`__

.. _inherited-from-workflow-execution-activity-1:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-29:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: metagenome annotation activity
=====================================

URI:
https://microbiomedata/schema/workflow_execution_activity/MetagenomeAnnotationActivity

|image24|

.. _parents-31:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-24:

Attributes
----------

.. _inherited-from-workflow-execution-activity-2:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-30:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: metagenome assembly
==========================

URI:
https://microbiomedata/schema/workflow_execution_activity/MetagenomeAssembly

|image25|

.. _parents-32:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-25:

Attributes
----------

.. _own-20:

Own
~~~

-  `asm_score <asm_score.md>`__ 0..1

   -  Description: A score for comparing metagenomic assembly quality
      from same sample.
   -  Range: `Float <types/Float.md>`__

-  `scaffolds <scaffolds.md>`__ 0..1

   -  Description: Total sequence count of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `scaf_logsum <scaf_logsum.md>`__ 0..1

   -  Description: The sum of the (length*log(length)) of all scaffolds,
      times some constant. Increase the contiguity, the score will
      increase
   -  Range: `Float <types/Float.md>`__

-  `scaf_powsum <scaf_powsum.md>`__ 0..1

   -  Description: Powersum of all scaffolds is the same as logsum
      except that it uses the sum of (length*(length^P)) for some power
      P (default P=0.25).
   -  Range: `Float <types/Float.md>`__

-  `scaf_max <scaf_max.md>`__ 0..1

   -  Description: Maximum scaffold length.
   -  Range: `Float <types/Float.md>`__

-  `scaf_bp <scaf_bp.md>`__ 0..1

   -  Description: Total size in bp of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `scaf_N50 <scaf_N50.md>`__ 0..1

   -  Description: Given a set of scaffolds, each with its own length,
      the N50 count is defined as the smallest number of scaffolds whose
      length sum makes up half of genome size.
   -  Range: `Float <types/Float.md>`__

-  `scaf_N90 <scaf_N90.md>`__ 0..1

   -  Description: Given a set of scaffolds, each with its own length,
      the N90 count is defined as the smallest number of scaffolds whose
      length sum makes up 90% of genome size.
   -  Range: `Float <types/Float.md>`__

-  `scaf_L50 <scaf_L50.md>`__ 0..1

   -  Description: Given a set of scaffolds, the L50 is defined as the
      sequence length of the shortest scaffold at 50% of the total
      genome length.
   -  Range: `Float <types/Float.md>`__

-  `scaf_L90 <scaf_L90.md>`__ 0..1

   -  Description: The L90 statistic is less than or equal to the L50
      statistic; it is the length for which the collection of all
      scaffolds of that length or longer contains at least 90% of the
      sum of the lengths of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `scaf_n_gt50K <scaf_n_gt50K.md>`__ 0..1

   -  Description: Total sequence count of scaffolds greater than 50 KB.
   -  Range: `Float <types/Float.md>`__

-  `scaf_l_gt50K <scaf_l_gt50K.md>`__ 0..1

   -  Description: Total size in bp of all scaffolds greater than 50 KB.
   -  Range: `Float <types/Float.md>`__

-  `scaf_pct_gt50K <scaf_pct_gt50K.md>`__ 0..1

   -  Description: Total sequence size percentage of scaffolds greater
      than 50 KB.
   -  Range: `Float <types/Float.md>`__

-  `contigs <contigs.md>`__ 0..1

   -  Description: The sum of the (length*log(length)) of all contigs,
      times some constant. Increase the contiguity, the score will
      increase
   -  Range: `Float <types/Float.md>`__

-  `contig_bp <contig_bp.md>`__ 0..1

   -  Description: Total size in bp of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `ctg_N50 <ctg_N50.md>`__ 0..1

   -  Description: Given a set of contigs, each with its own length, the
      N50 count is defined as the smallest number of contigs whose
      length sum makes up half of genome size.
   -  Range: `Float <types/Float.md>`__

-  `ctg_L50 <ctg_L50.md>`__ 0..1

   -  Description: Given a set of contigs, the L50 is defined as the
      sequence length of the shortest contig at 50% of the total genome
      length.
   -  Range: `Float <types/Float.md>`__

-  `ctg_N90 <ctg_N90.md>`__ 0..1

   -  Description: Given a set of contigs, each with its own length, the
      N90 count is defined as the smallest number of contigs whose
      length sum makes up 90% of genome size.
   -  Range: `Float <types/Float.md>`__

-  `ctg_L90 <ctg_L90.md>`__ 0..1

   -  Description: The L90 statistic is less than or equal to the L50
      statistic; it is the length for which the collection of all
      contigs of that length or longer contains at least 90% of the sum
      of the lengths of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `ctg_logsum <ctg_logsum.md>`__ 0..1

   -  Description: Maximum contig length.
   -  Range: `Float <types/Float.md>`__

-  `ctg_powsum <ctg_powsum.md>`__ 0..1

   -  Description: Powersum of all contigs is the same as logsum except
      that it uses the sum of (length*(length^P)) for some power P
      (default P=0.25).
   -  Range: `Float <types/Float.md>`__

-  `ctg_max <ctg_max.md>`__ 0..1

   -  Description: Maximum contig length.
   -  Range: `Float <types/Float.md>`__

-  `gap_pct <gap_pct.md>`__ 0..1

   -  Description: The gap size percentage of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `gc_std <gc_std.md>`__ 0..1

   -  Description: Standard deviation of GC content of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `gc_avg <gc_avg.md>`__ 0..1

   -  Description: Average of GC content of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `num_input_reads <num_input_reads.md>`__ 0..1

   -  Description: The sequence count number of input reads for
      assembly.
   -  Range: `Float <types/Float.md>`__

-  `num_aligned_reads <num_aligned_reads.md>`__ 0..1

   -  Description: The sequence count number of input reads aligned to
      assembled contigs.
   -  Range: `Float <types/Float.md>`__

-  `INSDC assembly identifiers <INSDC_assembly_identifiers.md>`__ 0..1

   -  Range: `String <types/String.md>`__

.. _inherited-from-workflow-execution-activity-3:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-31:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: metaproteomics analysis activity
=======================================

URI:
https://microbiomedata/schema/workflow_execution_activity/MetaproteomicsAnalysisActivity

|image26|

.. _parents-33:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _referenced-by-class-19:

Referenced by Class
-------------------

.. _attributes-26:

Attributes
----------

.. _own-21:

Own
~~~

-  `metaproteomics analysis
   activity➞used <metaproteomics_analysis_activity_used.md>`__ 0..1

   -  Description: The instrument used to collect the data used in the
      analysis
   -  Range: `Instrument <Instrument.md>`__

-  `metaproteomics analysis activity➞has peptide
   quantifications <metaproteomics_analysis_activity_has_peptide_quantifications.md>`__
   0..\*

   -  Range: `PeptideQuantification <PeptideQuantification.md>`__

.. _inherited-from-workflow-execution-activity-4:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-32:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: metatranscriptome activity
=================================

A metatranscriptome activity that e.g. pools assembly and annotation
activity.

URI:
https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeActivity

|image27|

.. _parents-34:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-27:

Attributes
----------

.. _inherited-from-workflow-execution-activity-5:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-33:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: metatranscriptome annotation activity
============================================

URI:
https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeAnnotationActivity

|image28|

.. _parents-35:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-28:

Attributes
----------

.. _inherited-from-workflow-execution-activity-6:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-34:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: metatranscriptome assembly
=================================

URI:
https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeAssembly

|image29|

.. _parents-36:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-29:

Attributes
----------

.. _own-22:

Own
~~~

-  `asm_score <asm_score.md>`__ 0..1

   -  Description: A score for comparing metagenomic assembly quality
      from same sample.
   -  Range: `Float <types/Float.md>`__

-  `scaffolds <scaffolds.md>`__ 0..1

   -  Description: Total sequence count of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `scaf_logsum <scaf_logsum.md>`__ 0..1

   -  Description: The sum of the (length*log(length)) of all scaffolds,
      times some constant. Increase the contiguity, the score will
      increase
   -  Range: `Float <types/Float.md>`__

-  `scaf_powsum <scaf_powsum.md>`__ 0..1

   -  Description: Powersum of all scaffolds is the same as logsum
      except that it uses the sum of (length*(length^P)) for some power
      P (default P=0.25).
   -  Range: `Float <types/Float.md>`__

-  `scaf_max <scaf_max.md>`__ 0..1

   -  Description: Maximum scaffold length.
   -  Range: `Float <types/Float.md>`__

-  `scaf_bp <scaf_bp.md>`__ 0..1

   -  Description: Total size in bp of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `scaf_N50 <scaf_N50.md>`__ 0..1

   -  Description: Given a set of scaffolds, each with its own length,
      the N50 count is defined as the smallest number of scaffolds whose
      length sum makes up half of genome size.
   -  Range: `Float <types/Float.md>`__

-  `scaf_N90 <scaf_N90.md>`__ 0..1

   -  Description: Given a set of scaffolds, each with its own length,
      the N90 count is defined as the smallest number of scaffolds whose
      length sum makes up 90% of genome size.
   -  Range: `Float <types/Float.md>`__

-  `scaf_L50 <scaf_L50.md>`__ 0..1

   -  Description: Given a set of scaffolds, the L50 is defined as the
      sequence length of the shortest scaffold at 50% of the total
      genome length.
   -  Range: `Float <types/Float.md>`__

-  `scaf_L90 <scaf_L90.md>`__ 0..1

   -  Description: The L90 statistic is less than or equal to the L50
      statistic; it is the length for which the collection of all
      scaffolds of that length or longer contains at least 90% of the
      sum of the lengths of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `scaf_n_gt50K <scaf_n_gt50K.md>`__ 0..1

   -  Description: Total sequence count of scaffolds greater than 50 KB.
   -  Range: `Float <types/Float.md>`__

-  `scaf_l_gt50K <scaf_l_gt50K.md>`__ 0..1

   -  Description: Total size in bp of all scaffolds greater than 50 KB.
   -  Range: `Float <types/Float.md>`__

-  `scaf_pct_gt50K <scaf_pct_gt50K.md>`__ 0..1

   -  Description: Total sequence size percentage of scaffolds greater
      than 50 KB.
   -  Range: `Float <types/Float.md>`__

-  `contigs <contigs.md>`__ 0..1

   -  Description: The sum of the (length*log(length)) of all contigs,
      times some constant. Increase the contiguity, the score will
      increase
   -  Range: `Float <types/Float.md>`__

-  `contig_bp <contig_bp.md>`__ 0..1

   -  Description: Total size in bp of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `ctg_N50 <ctg_N50.md>`__ 0..1

   -  Description: Given a set of contigs, each with its own length, the
      N50 count is defined as the smallest number of contigs whose
      length sum makes up half of genome size.
   -  Range: `Float <types/Float.md>`__

-  `ctg_L50 <ctg_L50.md>`__ 0..1

   -  Description: Given a set of contigs, the L50 is defined as the
      sequence length of the shortest contig at 50% of the total genome
      length.
   -  Range: `Float <types/Float.md>`__

-  `ctg_N90 <ctg_N90.md>`__ 0..1

   -  Description: Given a set of contigs, each with its own length, the
      N90 count is defined as the smallest number of contigs whose
      length sum makes up 90% of genome size.
   -  Range: `Float <types/Float.md>`__

-  `ctg_L90 <ctg_L90.md>`__ 0..1

   -  Description: The L90 statistic is less than or equal to the L50
      statistic; it is the length for which the collection of all
      contigs of that length or longer contains at least 90% of the sum
      of the lengths of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `ctg_logsum <ctg_logsum.md>`__ 0..1

   -  Description: Maximum contig length.
   -  Range: `Float <types/Float.md>`__

-  `ctg_powsum <ctg_powsum.md>`__ 0..1

   -  Description: Powersum of all contigs is the same as logsum except
      that it uses the sum of (length*(length^P)) for some power P
      (default P=0.25).
   -  Range: `Float <types/Float.md>`__

-  `ctg_max <ctg_max.md>`__ 0..1

   -  Description: Maximum contig length.
   -  Range: `Float <types/Float.md>`__

-  `gap_pct <gap_pct.md>`__ 0..1

   -  Description: The gap size percentage of all scaffolds.
   -  Range: `Float <types/Float.md>`__

-  `gc_std <gc_std.md>`__ 0..1

   -  Description: Standard deviation of GC content of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `gc_avg <gc_avg.md>`__ 0..1

   -  Description: Average of GC content of all contigs.
   -  Range: `Float <types/Float.md>`__

-  `num_input_reads <num_input_reads.md>`__ 0..1

   -  Description: The sequence count number of input reads for
      assembly.
   -  Range: `Float <types/Float.md>`__

-  `num_aligned_reads <num_aligned_reads.md>`__ 0..1

   -  Description: The sequence count number of input reads aligned to
      assembled contigs.
   -  Range: `Float <types/Float.md>`__

-  `INSDC assembly identifiers <INSDC_assembly_identifiers.md>`__ 0..1

   -  Range: `String <types/String.md>`__

.. _inherited-from-workflow-execution-activity-7:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-35:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Subset: mixs extension
======================

URI:
https://microbiomedata/schema/workflow_execution_activity/mixs_extension

.. _classes-3:

Classes
-------

.. _mixins-3:

Mixins
------

.. _slots-3:

Slots
-----

.. _types-3:

Types
-----

.. _enums-3:

Enums
-----

Class: named thing
==================

a databased entity or concept/class

URI:
https://microbiomedata/schema/workflow_execution_activity/NamedThing

|image30|

.. _children-20:

Children
--------

-  `GeneProduct <GeneProduct.md>`__ - A molecule encoded by a gene that
   has an evolved function
-  `Instrument <Instrument.md>`__ - A material entity that is designed
   to perform a function in a scientific investigation, but is not a
   reagent[OBI].
-  `OntologyClass <OntologyClass.md>`__
-  `Person <Person.md>`__ - represents a person, such as a researcher

.. _referenced-by-class-20:

Referenced by Class
-------------------

-  `NamedThing <NamedThing.md>`__ `has input <has_input.md>`__ 0..\*
   `NamedThing <NamedThing.md>`__
-  `NamedThing <NamedThing.md>`__ `has output <has_output.md>`__ 0..\*
   `NamedThing <NamedThing.md>`__
-  `NamedThing <NamedThing.md>`__ `part of <part_of.md>`__ 0..\*
   `NamedThing <NamedThing.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__ `read QC
   analysis activity➞has
   input <read_QC_analysis_activity_has_input.md>`__ 1..\*
   `NamedThing <NamedThing.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__ `read QC
   analysis activity➞has
   output <read_QC_analysis_activity_has_output.md>`__ 1..\*
   `NamedThing <NamedThing.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__
   `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*
   `NamedThing <NamedThing.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__
   `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*
   `NamedThing <NamedThing.md>`__

.. _attributes-30:

Attributes
----------

.. _own-23:

Own
~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

Class: nom analysis activity
============================

URI:
https://microbiomedata/schema/workflow_execution_activity/NomAnalysisActivity

|image31|

.. _parents-37:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _referenced-by-class-21:

Referenced by Class
-------------------

.. _attributes-31:

Attributes
----------

.. _own-24:

Own
~~~

-  `nom analysis activity➞used <nom_analysis_activity_used.md>`__ 0..1

   -  Description: The instrument used to collect the data used in the
      analysis
   -  Range: `Instrument <Instrument.md>`__

-  `nom analysis activity➞has
   calibration <nom_analysis_activity_has_calibration.md>`__ 0..1

   -  Description: A reference to a file that holds calibration
      information.
   -  Range: `String <types/String.md>`__

.. _inherited-from-workflow-execution-activity-8:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-36:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Subset: nucleic acid sequence source
====================================

URI:
https://microbiomedata/schema/workflow_execution_activity/nucleic_acid_sequence_source

.. _classes-4:

Classes
-------

.. _mixins-4:

Mixins
------

.. _slots-4:

Slots
-----

.. _types-4:

Types
-----

.. _enums-4:

Enums
-----

Class: omics processing
=======================

The methods and processes used to generate omics data from a biosample
or organism.

URI:
`nmdc:OmicsProcessing <https://microbiomedata/meta/OmicsProcessing>`__

|image32|

.. _parents-38:

Parents
-------

-  is_a: `BiosampleProcessing <BiosampleProcessing.md>`__ - A process
   that takes one or more biosamples as inputs and generates one or as
   outputs. Examples of outputs include samples cultivated from another
   sample or data objects created by instruments runs.

.. _referenced-by-class-22:

Referenced by Class
-------------------

-  `Database <Database.md>`__ `omics processing
   set <omics_processing_set.md>`__ 0..\*
   `OmicsProcessing <OmicsProcessing.md>`__

.. _attributes-32:

Attributes
----------

.. _own-25:

Own
~~~

-  `add_date <add_date.md>`__ 0..1

   -  Description: The date on which the information was added to the
      database.
   -  Range: `String <types/String.md>`__

-  `mod_date <mod_date.md>`__ 0..1

   -  Description: The last date on which the database information was
      modified.
   -  Range: `String <types/String.md>`__

-  `has input <has_input.md>`__ 0..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `has output <has_output.md>`__ 0..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `instrument_name <instrument_name.md>`__ 0..1

   -  Description: The name of the instrument that was used for
      processing the sample.

   -  Range: `String <types/String.md>`__

-  `ncbi_project_name <ncbi_project_name.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `omics type <omics_type.md>`__ 0..1

   -  Description: The type of omics data
   -  Range: `ControlledTermValue <ControlledTermValue.md>`__
   -  Example: metatranscriptome None
   -  Example: metagenome None

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `principal investigator <principal_investigator.md>`__ 0..1

   -  Description: represents the PI
   -  Range: `PersonValue <PersonValue.md>`__

-  `processing_institution <processing_institution.md>`__ 0..1

   -  Description: The organization that processed the sample.
   -  Range: `String <types/String.md>`__

-  `type <type.md>`__ 0..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `GOLD sequencing project
   identifiers <GOLD_sequencing_project_identifiers.md>`__ 0..\*

   -  Description: [‘identifiers for corresponding sequencing project in
      GOLD’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/gold:Gp0108335 None

-  `INSDC experiment identifiers <INSDC_experiment_identifiers.md>`__
   0..\*

   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__

-  `samp_vol_we_dna_ext <samp_vol_we_dna_ext.md>`__ 0..1

   -  Description: “Volume (ml), weight (g) of processed sample, or
      surface area swabbed from sample for DNA extraction”
   -  Range: `QuantityValue <QuantityValue.md>`__

-  `nucl_acid_ext <nucl_acid_ext.md>`__ 0..1

   -  Description: “A link to a literature reference, electronic
      resource or a standard operating procedure (SOP), that describes
      the material separation to recover the nucleic acid fraction from
      a sample”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `nucl_acid_amp <nucl_acid_amp.md>`__ 0..1

   -  Description: “A link to a literature reference, electronic
      resource or a standard operating procedure (SOP), that describes
      the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic
      acids”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `target_gene <target_gene.md>`__ 0..1

   -  Description: Targeted gene or locus name for marker gene studies
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `target_subfragment <target_subfragment.md>`__ 0..1

   -  Description: Name of subfragment of a gene or locus. Important to
      e.g. identify special regions on marker genes like V6 on 16S rRNA
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `pcr_primers <pcr_primers.md>`__ 0..1

   -  Description: “PCR primers that were used to amplify the sequence
      of the targeted gene, locus or subfragment. This field should
      contain all the primers used for a single PCR reaction if multiple
      forward or reverse primers are present in a single PCR reaction.
      The primer sequence should be reported in uppercase letters”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `pcr_cond <pcr_cond.md>`__ 0..1

   -  Description: Description of reaction conditions and components of
      PCR in the form of ‘initial denaturation:94degC_1.5min;
      annealing=…’
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `seq_meth <seq_meth.md>`__ 0..1

   -  Description: “Sequencing method used; e.g. Sanger, pyrosequencing,
      ABI-solid”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `seq_quality_check <seq_quality_check.md>`__ 0..1

   -  Description: “Indicate if the sequence has been called by
      automatic systems (none) or undergone a manual editing procedure
      (e.g. by inspecting the raw data or chromatograms). Applied only
      for sequences that are not submitted to SRA,ENA or DRA”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `chimera_check <chimera_check.md>`__ 0..1

   -  Description: “A chimeric sequence, or chimera for short, is a
      sequence comprised of two or more phylogenetically distinct parent
      sequences. Chimeras are usually PCR artifacts thought to occur
      when a prematurely terminated amplicon reanneals to a foreign DNA
      strand and is copied to completion in the following PCR cycles.
      The point at which the chimeric sequence changes from one parent
      to the next is called the breakpoint or conversion point”
   -  Range: `TextValue <TextValue.md>`__
   -  in subsets: (sequencing)

-  `omics processing➞has input <omics_processing_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `Biosample <Biosample.md>`__

Inherited from biosample processing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-37:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | omics assay           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | sequencing project    |
+-----------------------+-----------------------+-----------------------+
|                       |                       | experiment            |
+-----------------------+-----------------------+-----------------------+
| **Alt Descriptions:** |                       | An experiment         |
|                       |                       | contains information  |
|                       |                       | about a sequencing    |
|                       |                       | experiment including  |
|                       |                       | library and           |
|                       |                       | instrument details.   |
|                       |                       | (embl.ena)            |
+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | The IDs for objects   |
|                       |                       | coming from GOLD will |
|                       |                       | have prefixes         |
|                       |                       | gold:GpNNNN           |
+-----------------------+-----------------------+-----------------------+
| **In Subsets:**       |                       | sample subset         |
+-----------------------+-----------------------+-----------------------+
| **Broad Mappings:**   |                       | OBI:0000070           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | ISA:Assay             |
+-----------------------+-----------------------+-----------------------+

Class: ontology class
=====================

URI:
https://microbiomedata/schema/workflow_execution_activity/OntologyClass

|image33|

.. _parents-39:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _children-21:

Children
--------

-  `ChemicalEntity <ChemicalEntity.md>`__ - An atom or molecule that can
   be represented with a chemical formula. Include lipids, glycans,
   natural products, drugs. There may be different terms for distinct
   acid-base forms, protonation states
-  `EnvironmentalMaterialTerm <EnvironmentalMaterialTerm.md>`__

.. _referenced-by-class-23:

Referenced by Class
-------------------

-  `ControlledTermValue <ControlledTermValue.md>`__ `term <term.md>`__
   0..1 `OntologyClass <OntologyClass.md>`__

.. _attributes-33:

Attributes
----------

.. _inherited-from-named-thing-5:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

Class: orthology group
======================

A set of genes or gene products in which all members are orthologous

URI:
`nmdc:OrthologyGroup <https://microbiomedata/meta/OrthologyGroup>`__

|image34|

.. _identifier-prefixes-4:

Identifier prefixes
-------------------

-  KEGG.ORTHOLOGY
-  EGGNOG
-  PFAM
-  TIGRFAM
-  SUPFAM
-  CATH
-  PANTHER.FAMILY

.. _parents-40:

Parents
-------

-  is_a: `FunctionalAnnotationTerm <FunctionalAnnotationTerm.md>`__ -
   Abstract grouping class for any term/descriptor that can be applied
   to a functional unit of a genome (protein, ncRNA, complex).

.. _attributes-34:

Attributes
----------

Inherited from functional annotation term:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-38:

Other properties
----------------

=================== ==================
**Exact Mappings:**  biolink:GeneFamily
=================== ==================

Class: pathway
==============

A pathway is a sequence of steps/reactions carried out by an organism or
community of organisms

URI: `nmdc:Pathway <https://microbiomedata/meta/Pathway>`__

|image35|

.. _identifier-prefixes-5:

Identifier prefixes
-------------------

-  KEGG.PATHWAY
-  COG

.. _parents-41:

Parents
-------

-  is_a: `FunctionalAnnotationTerm <FunctionalAnnotationTerm.md>`__ -
   Abstract grouping class for any term/descriptor that can be applied
   to a functional unit of a genome (protein, ncRNA, complex).

.. _referenced-by-class-24:

Referenced by Class
-------------------

.. _attributes-35:

Attributes
----------

.. _own-26:

Own
~~~

-  `pathway➞has_part <pathway_has_part.md>`__ 0..\*

   -  Description: A pathway can be broken down to a series of reaction
      step
   -  Range: `Reaction <Reaction.md>`__

.. _inherited-from-functional-annotation-term-1:

Inherited from functional annotation term:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-39:

Other properties
----------------

=================== ==================
**Aliases:**         biological process
\                    metabolic pathway
\                    signaling pathway
**Exact Mappings:**  biolink:Pathway
=================== ==================

Class: peptide quantification
=============================

This is used to link a metaproteomics analysis workflow to a specific
peptide sequence and related information

URI:
https://microbiomedata/schema/workflow_execution_activity/PeptideQuantification

|image36|

.. _referenced-by-class-25:

Referenced by Class
-------------------

-  **None** `has peptide
   quantifications <has_peptide_quantifications.md>`__ 0..\*
   `PeptideQuantification <PeptideQuantification.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
   `metaproteomics analysis activity➞has peptide
   quantifications <metaproteomics_analysis_activity_has_peptide_quantifications.md>`__
   0..\* `PeptideQuantification <PeptideQuantification.md>`__

.. _attributes-36:

Attributes
----------

.. _own-27:

Own
~~~

-  `peptide quantification➞peptide
   sequence <peptide_quantification_peptide_sequence.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `peptide quantification➞best
   protein <peptide_quantification_best_protein.md>`__ 0..1

   -  Description: the specific protein identifier most correctly
      associated with the peptide sequence
   -  Range: `GeneProduct <GeneProduct.md>`__

-  `peptide quantification➞all
   proteins <peptide_quantification_all_proteins.md>`__ 0..\*

   -  Description: the list of protein identifiers that are associated
      with the peptide sequence
   -  Range: `GeneProduct <GeneProduct.md>`__

-  `peptide
   quantification➞min_q_value <peptide_quantification_min_q_value.md>`__
   0..1

   -  Description: smallest Q-Value associated with the peptide sequence
      as provided by MSGFPlus tool
   -  Range: `Float <types/Float.md>`__

-  `peptide
   quantification➞peptide_spectral_count <peptide_quantification_peptide_spectral_count.md>`__
   0..1

   -  Description: sum of filter passing MS2 spectra associated with the
      peptide sequence within a given LC-MS/MS data file
   -  Range: `Integer <types/Integer.md>`__

-  `peptide
   quantification➞peptide_sum_masic_abundance <peptide_quantification_peptide_sum_masic_abundance.md>`__
   0..1

   -  Description: combined MS1 extracted ion chromatograms derived from
      MS2 spectra associated with the peptide sequence from a given
      LC-MS/MS data file using the MASIC tool
   -  Range: `Integer <types/Integer.md>`__

Class: person
=============

represents a person, such as a researcher

URI: https://microbiomedata/schema/workflow_execution_activity/Person

|image37|

.. _parents-42:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _referenced-by-class-26:

Referenced by Class
-------------------

.. _attributes-37:

Attributes
----------

.. _own-28:

Own
~~~

-  `person➞id <person_id.md>`__ 1..1

   -  Description: Should be an ORCID. Specify in CURIE format. E.g
      ORCID:0000-1111-…
   -  Range: `String <types/String.md>`__

.. _inherited-from-named-thing-6:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-40:

Other properties
----------------

============= ==============
**Comments:**  not yet in use
============= ==============

Class: person value
===================

An attribute value representing a person

URI:
https://microbiomedata/schema/workflow_execution_activity/PersonValue

|image38|

.. _parents-43:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _referenced-by-class-27:

Referenced by Class
-------------------

.. _attributes-38:

Attributes
----------

.. _own-29:

Own
~~~

-  `orcid <orcid.md>`__ 0..1

   -  Description: The ORICD of a person.
   -  Range: `String <types/String.md>`__

-  `profile image url <profile_image_url.md>`__ 0..1

   -  Description: A url that points to an image of a person.
   -  Range: `String <types/String.md>`__

-  `email <email.md>`__ 0..1

   -  Description: An email address for an entity such as a person. This
      should be the primarly email address used.
   -  Range: `String <types/String.md>`__

-  `person value➞name <person_value_name.md>`__ 0..1

   -  Description: The full name of the Investgator. It should follow
      the format FIRST [MIDDLE NAME\| MIDDLE INITIAL] LAST, where MIDDLE
      NAME\| MIDDLE INITIAL is optional.
   -  Range: `String <types/String.md>`__

-  `websites <websites.md>`__ 0..\*

   -  Description: A list of websites that are assocatiated with the
      entity.
   -  Range: `String <types/String.md>`__

-  `person value➞has raw value <person_value_has_raw_value.md>`__ 0..1

   -  Description: The full name of the Investgator in format FIRST
      LAST.
   -  Range: `String <types/String.md>`__

.. _inherited-from-attribute-value-5:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: protein quantification
=============================

This is used to link a metaproteomics analysis workflow to a specific
protein

URI:
https://microbiomedata/schema/workflow_execution_activity/ProteinQuantification

|image39|

.. _referenced-by-class-28:

Referenced by Class
-------------------

.. _attributes-39:

Attributes
----------

.. _own-30:

Own
~~~

-  `protein quantification➞best
   protein <protein_quantification_best_protein.md>`__ 0..1

   -  Description: the specific protein identifier most correctly
      grouped to its associated peptide sequences
   -  Range: `GeneProduct <GeneProduct.md>`__

-  `protein quantification➞all
   proteins <protein_quantification_all_proteins.md>`__ 0..\*

   -  Description: the grouped list of protein identifiers associated
      with the peptide sequences that were grouped to a best protein
   -  Range: `GeneProduct <GeneProduct.md>`__

-  `protein
   quantification➞peptide_sequence_count <protein_quantification_peptide_sequence_count.md>`__
   0..1

   -  Description: count of peptide sequences grouped to the
      best_protein
   -  Range: `Integer <types/Integer.md>`__

-  `protein
   quantification➞protein_spectral_count <protein_quantification_protein_spectral_count.md>`__
   0..1

   -  Description: sum of filter passing MS2 spectra associated with the
      best protein within a given LC-MS/MS data file
   -  Range: `Integer <types/Integer.md>`__

-  `protein
   quantification➞protein_sum_masic_abundance <protein_quantification_protein_sum_masic_abundance.md>`__
   0..1

   -  Description: combined MS1 extracted ion chromatograms derived from
      MS2 spectra associated with the best protein from a given LC-MS/MS
      data file using the MASIC tool
   -  Range: `Integer <types/Integer.md>`__

Class: quantity value
=====================

A simple quantity, e.g. 2cm

URI:
https://microbiomedata/schema/workflow_execution_activity/QuantityValue

|image40|

.. _parents-44:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _referenced-by-class-29:

Referenced by Class
-------------------

.. _attributes-40:

Attributes
----------

.. _own-31:

Own
~~~

-  `quantity value➞has unit <quantity_value_has_unit.md>`__ 0..1

   -  Description: The unit of the quantity
   -  Range: `Unit <types/Unit.md>`__

-  `quantity value➞has numeric
   value <quantity_value_has_numeric_value.md>`__ 0..1

   -  Description: The number part of the quantity
   -  Range: `Double <types/Double.md>`__

-  `has minimum numeric value <has_minimum_numeric_value.md>`__ 0..1

   -  Description: The minimum value part, expressed as number, of the
      quantity value when the value covers a range.
   -  Range: `Float <types/Float.md>`__

-  `has maximum numeric value <has_maximum_numeric_value.md>`__ 0..1

   -  Description: The maximum value part, expressed as number, of the
      quantity value when the value covers a range.
   -  Range: `Float <types/Float.md>`__

-  `quantity value➞has raw value <quantity_value_has_raw_value.md>`__
   0..1

   -  Description: Unnormalized atomic string representation, should in
      syntax {number} {unit}
   -  Range: `String <types/String.md>`__

.. _inherited-from-attribute-value-6:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

.. _other-properties-41:

Other properties
----------------

============= ====================
**Mappings:**  schema:QuantityValue
============= ====================

Class: reaction
===============

An individual biochemical transformation carried out by a functional
unit of an organism, in which a collection of substrates are transformed
into a collection of products. Can also represent transporters

URI: `nmdc:Reaction <https://microbiomedata/meta/Reaction>`__

|image41|

.. _identifier-prefixes-6:

Identifier prefixes
-------------------

-  KEGG.REACTION
-  RHEA
-  MetaCyc
-  EC
-  GO
-  MetaNetX
-  SEED
-  RetroRules

.. _parents-45:

Parents
-------

-  is_a: `FunctionalAnnotationTerm <FunctionalAnnotationTerm.md>`__ -
   Abstract grouping class for any term/descriptor that can be applied
   to a functional unit of a genome (protein, ncRNA, complex).

.. _referenced-by-class-30:

Referenced by Class
-------------------

-  **None** `has_part <has_part.md>`__ 0..\* `Reaction <Reaction.md>`__
-  `Pathway <Pathway.md>`__ `pathway➞has_part <pathway_has_part.md>`__
   0..\* `Reaction <Reaction.md>`__

.. _attributes-41:

Attributes
----------

.. _own-32:

Own
~~~

-  `reaction➞left participants <reaction_left_participants.md>`__ 0..\*

   -  Range: `ReactionParticipant <ReactionParticipant.md>`__

-  `reaction➞right participants <reaction_right_participants.md>`__
   0..\*

   -  Range: `ReactionParticipant <ReactionParticipant.md>`__

-  `reaction➞direction <reaction_direction.md>`__ 0..1

   -  Description: One of l->r, r->l, bidirectional, neutral
   -  Range: `String <types/String.md>`__

-  `reaction➞smarts string <reaction_smarts_string.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `reaction➞is
   diastereoselective <reaction_is_diastereoselective.md>`__ 0..1

   -  Range: `Boolean <types/Boolean.md>`__

-  `reaction➞is stereo <reaction_is_stereo.md>`__ 0..1

   -  Range: `Boolean <types/Boolean.md>`__

-  `reaction➞is balanced <reaction_is_balanced.md>`__ 0..1

   -  Range: `Boolean <types/Boolean.md>`__

-  `reaction➞is transport <reaction_is_transport.md>`__ 0..1

   -  Range: `Boolean <types/Boolean.md>`__

-  `reaction➞is fully
   characterized <reaction_is_fully_characterized.md>`__ 0..1

   -  Description: False if includes R-groups
   -  Range: `Boolean <types/Boolean.md>`__

.. _inherited-from-functional-annotation-term-2:

Inherited from functional annotation term:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-42:

Other properties
----------------

=================== =========================
**Exact Mappings:**  biolink:MolecularActivity
=================== =========================

Class: reaction participant
===========================

Instances of this link a reaction to a chemical entity participant

URI:
`nmdc:ReactionParticipant <https://microbiomedata/meta/ReactionParticipant>`__

|image42|

.. _referenced-by-class-31:

Referenced by Class
-------------------

-  **None** `left participants <left_participants.md>`__ 0..\*
   `ReactionParticipant <ReactionParticipant.md>`__
-  `Reaction <Reaction.md>`__ `reaction➞left
   participants <reaction_left_participants.md>`__ 0..\*
   `ReactionParticipant <ReactionParticipant.md>`__
-  `Reaction <Reaction.md>`__ `reaction➞right
   participants <reaction_right_participants.md>`__ 0..\*
   `ReactionParticipant <ReactionParticipant.md>`__
-  **None** `right participants <right_participants.md>`__ 0..\*
   `ReactionParticipant <ReactionParticipant.md>`__

.. _attributes-42:

Attributes
----------

.. _own-33:

Own
~~~

-  `reaction participant➞chemical <reaction_participant_chemical.md>`__
   0..1

   -  Range: `ChemicalEntity <ChemicalEntity.md>`__

-  `reaction
   participant➞stoichiometry <reaction_participant_stoichiometry.md>`__
   0..1

   -  Range: `Integer <types/Integer.md>`__

Class: read based analysis activity
===================================

URI:
https://microbiomedata/schema/workflow_execution_activity/ReadBasedAnalysisActivity

|image43|

.. _parents-46:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _attributes-43:

Attributes
----------

.. _inherited-from-workflow-execution-activity-9:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-43:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Class: read QC analysis activity
================================

URI:
https://microbiomedata/schema/workflow_execution_activity/ReadQCAnalysisActivity

|image44|

.. _parents-47:

Parents
-------

-  is_a: `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _referenced-by-class-32:

Referenced by Class
-------------------

.. _attributes-44:

Attributes
----------

.. _own-34:

Own
~~~

-  `input read count <input_read_count.md>`__ 0..1

   -  Description: The sequence count number of input reads for QC
      analysis.
   -  Range: `Float <types/Float.md>`__

-  `input base count <input_base_count.md>`__ 0..1

   -  Description: The nucleotide base count number of input reads for
      QC analysis.
   -  Range: `Float <types/Float.md>`__

-  `output read count <output_read_count.md>`__ 0..1

   -  Description: After QC analysis sequence count number.
   -  Range: `Float <types/Float.md>`__

-  `output base count <output_base_count.md>`__ 0..1

   -  Description: After QC analysis nucleotide base count number.
   -  Range: `Float <types/Float.md>`__

-  `read QC analysis
   activity➞input_read_bases <read_QC_analysis_activity_input_read_bases.md>`__
   0..1

   -  Description: TODO
   -  Range: `Float <types/Float.md>`__

-  `read QC analysis
   activity➞output_read_bases <read_QC_analysis_activity_output_read_bases.md>`__
   0..1

   -  Description: TODO
   -  Range: `Float <types/Float.md>`__

-  `read QC analysis activity➞has
   input <read_QC_analysis_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `read QC analysis activity➞has
   output <read_QC_analysis_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

.. _inherited-from-workflow-execution-activity-10:

Inherited from workflow execution activity:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

.. _other-properties-44:

Other properties
----------------

=============== ===============
**In Subsets:**  workflow subset
=============== ===============

Subset: sample subset
=====================

Subset consisting of entities linked to the processing of samples.
Currently, this subset consists of study, omics process, and biosample.

URI: `nmdc:sample_subset <https://microbiomedata/meta/sample_subset>`__

.. _classes-5:

Classes
-------

-  `Biosample <Biosample.md>`__ - A material sample. It may be
   environmental (encompassing many organisms) or isolate or tissue. An
   environmental sample containing genetic material from multiple
   individuals is commonly referred to as a biosample.
-  `OmicsProcessing <OmicsProcessing.md>`__ - The methods and processes
   used to generate omics data from a biosample or organism.
-  `Study <Study.md>`__ - A study summarizes the overall goal of a
   research initiative and outlines the key objective of its underlying
   projects.

.. _mixins-5:

Mixins
------

.. _slots-5:

Slots
-----

.. _types-5:

Types
-----

.. _enums-5:

Enums
-----

Subset: sequencing
==================

URI:
https://microbiomedata/schema/workflow_execution_activity/sequencing

.. _classes-6:

Classes
-------

.. _mixins-6:

Mixins
------

.. _slots-6:

Slots
-----

.. _types-6:

Types
-----

.. _enums-6:

Enums
-----

Subset: soil
============

URI: https://microbiomedata/schema/workflow_execution_activity/soil

.. _classes-7:

Classes
-------

.. _mixins-7:

Mixins
------

.. _slots-7:

Slots
-----

.. _types-7:

Types
-----

.. _enums-7:

Enums
-----

Class: study
============

A study summarizes the overall goal of a research initiative and
outlines the key objective of its underlying projects.

URI: `nmdc:Study <https://microbiomedata/meta/Study>`__

|image45|

.. _identifier-prefixes-7:

Identifier prefixes
-------------------

-  GOLD
-  insdc.srs
-  mgnify

.. _parents-48:

Parents
-------

-  is_a: `NamedThing <NamedThing.md>`__ - a databased entity or
   concept/class

.. _referenced-by-class-33:

Referenced by Class
-------------------

-  `Database <Database.md>`__ `study set <study_set.md>`__ 0..\*
   `Study <Study.md>`__

.. _attributes-45:

Attributes
----------

.. _own-35:

Own
~~~

-  `ecosystem <ecosystem.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `ecosystem_category <ecosystem_category.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `ecosystem_type <ecosystem_type.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `ecosystem_subtype <ecosystem_subtype.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `specific_ecosystem <specific_ecosystem.md>`__ 0..1

   -  Description: TODO
   -  Range: `String <types/String.md>`__

-  `principal investigator <principal_investigator.md>`__ 0..1

   -  Description: represents the PI
   -  Range: `PersonValue <PersonValue.md>`__

-  `study➞doi <study_doi.md>`__ 0..1

   -  Description: The dataset citation for this study
   -  Range: `AttributeValue <AttributeValue.md>`__

-  `title <title.md>`__ 0..1

   -  Description: A name given to the entity that differs from the
      name/label programatically assigned to it. For example, when
      extracting study information for GOLD, the GOLD system has
      assigned a name/label. However, for display purposes, we may also
      wish the capture the title of the proposal that was used to fund
      the study.
   -  Range: `String <types/String.md>`__

-  `alternative titles <alternative_titles.md>`__ 0..\*

   -  Description: A list of alternative titles for the entity. The
      distinction between title and alternative titles is
      application-specific.
   -  Range: `String <types/String.md>`__

-  `alternative descriptions <alternative_descriptions.md>`__ 0..\*

   -  Description: A list of alternative descriptions for the entity.
      The distinction between desciption and alternative descriptions is
      application-specific.
   -  Range: `String <types/String.md>`__

-  `alternative names <alternative_names.md>`__ 0..\*

   -  Description: A list of alternative names used to refer to the
      entity. The distinction between name and alternative names is
      application-specific.
   -  Range: `String <types/String.md>`__

-  `abstract <abstract.md>`__ 0..1

   -  Description: The abstract of manuscript/grant associated with the
      entity; i.e., a summary of the resource.
   -  Range: `String <types/String.md>`__

-  `objective <objective.md>`__ 0..1

   -  Description: The scientific objectives associated with the entity.
      It SHOULD correspond to scientific norms for objectives field in a
      structured abstract.
   -  Range: `String <types/String.md>`__

-  `websites <websites.md>`__ 0..\*

   -  Description: A list of websites that are assocatiated with the
      entity.
   -  Range: `String <types/String.md>`__

-  `publications <publications.md>`__ 0..\*

   -  Description: A list of publications that are assocatiated with the
      entity. The publicatons SHOULD be given using an identifier, such
      as a DOI or Pubmed ID, if possible.
   -  Range: `String <types/String.md>`__

-  `ess dive datasets <ess_dive_datasets.md>`__ 0..\*

   -  Description: List of ESS-DIVE dataset DOIs
   -  Range: `String <types/String.md>`__

-  `type <type.md>`__ 0..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `relevant protocols <relevant_protocols.md>`__ 0..\*

   -  Range: `String <types/String.md>`__

-  `funding sources <funding_sources.md>`__ 0..\*

   -  Range: `String <types/String.md>`__

-  `INSDC bioproject identifiers <INSDC_bioproject_identifiers.md>`__
   0..\*

   -  Description: [‘identifiers for corresponding project in INSDC
      Bioproject’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/bioproject:PRJNA366857 Avena
      fatua rhizosphere microbial communities - H1_Rhizo_Litter_2
      metatranscriptome

-  `INSDC SRA ENA study
   identifiers <INSDC_SRA_ENA_study_identifiers.md>`__ 0..\*

   -  Description: [‘identifiers for corresponding project in INSDC SRA
      / ENA’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/insdc.sra:SRP121659 Avena fatua
      rhizosphere microbial communities - H1_Rhizo_Litter_2
      metatranscriptome

-  `GOLD study identifiers <GOLD_study_identifiers.md>`__ 0..\*

   -  Description: [‘identifiers for corresponding project in GOLD’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/gold:Gs0110115 None

-  `MGnify project identifiers <MGnify_project_identifiers.md>`__ 0..\*

   -  Description: [‘identifiers for corresponding project in MGnify’]
   -  Range: `ExternalIdentifier <types/ExternalIdentifier.md>`__
   -  Example: https://identifiers.org/mgnify.proj:MGYS00005757 None

-  `has credit associations <has_credit_associations.md>`__ 0..\*

   -  Description: This slot links a study to a credit association. The
      credit association will be linked to a person value and to a
      CRediT Contributor Roles term. Overall semantics: person should
      get credit X for their participation in the study
   -  Range: `CreditAssociation <CreditAssociation.md>`__

-  `study image <study_image.md>`__ 0..\*

   -  Description: Links a study to one or more images.
   -  Range: `ImageValue <ImageValue.md>`__

.. _inherited-from-named-thing-7:

Inherited from named thing:
~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `description <description.md>`__ 0..1

   -  Description: a human-readable description of a thing
   -  Range: `String <types/String.md>`__

-  `alternative identifiers <alternative_identifiers.md>`__ 0..\*

   -  Description: A list of alternative identifiers for the entity.
   -  Range: `String <types/String.md>`__

.. _other-properties-45:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | proposal              |
+-----------------------+-----------------------+-----------------------+
|                       |                       | research proposal     |
+-----------------------+-----------------------+-----------------------+
|                       |                       | research study        |
+-----------------------+-----------------------+-----------------------+
|                       |                       | investigation         |
+-----------------------+-----------------------+-----------------------+
|                       |                       | project               |
+-----------------------+-----------------------+-----------------------+
|                       |                       | umbrella project      |
+-----------------------+-----------------------+-----------------------+
| **Alt Descriptions:** |                       | A study (project)     |
|                       |                       | groups together data  |
|                       |                       | submitted to the      |
|                       |                       | archive and controls  |
|                       |                       | its release date. A   |
|                       |                       | study accession is    |
|                       |                       | typically used when   |
|                       |                       | citing data submitted |
|                       |                       | to ENA (embl.ena)     |
+-----------------------+-----------------------+-----------------------+
| **In Subsets:**       |                       | sample subset         |
+-----------------------+-----------------------+-----------------------+
| **Exact Mappings:**   |                       | SIO:001066            |
+-----------------------+-----------------------+-----------------------+
|                       |                       | NCIT:C63536           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | OBI:0000066           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | ISA:Study             |
+-----------------------+-----------------------+-----------------------+
| **Close Mappings:**   |                       | NCIT:C41198           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | ISA:Investigation     |
+-----------------------+-----------------------+-----------------------+
| **Broad Mappings:**   |                       | prov:Activity         |
+-----------------------+-----------------------+-----------------------+

Class: text value
=================

A basic string value

URI: https://microbiomedata/schema/workflow_execution_activity/TextValue

|image46|

.. _parents-49:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _attributes-46:

Attributes
----------

.. _own-36:

Own
~~~

-  `language <language.md>`__ 0..1

   -  Description: Should use ISO 639-1 code e.g. “en”, “fr”
   -  Range: `LanguageCode <types/LanguageCode.md>`__

.. _inherited-from-attribute-value-7:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: timestamp value
======================

A value that is a timestamp. The range should be ISO-8601

URI:
https://microbiomedata/schema/workflow_execution_activity/TimestampValue

|image47|

.. _parents-50:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _attributes-47:

Attributes
----------

.. _inherited-from-attribute-value-8:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: url value
================

A value that is a string that conforms to URL syntax

URI: https://microbiomedata/schema/workflow_execution_activity/UrlValue

|image48|

.. _parents-51:

Parents
-------

-  is_a: `AttributeValue <AttributeValue.md>`__ - The value for any
   value of a attribute for a sample. This object can hold both the
   un-normalized atomic value and the structured value

.. _attributes-48:

Attributes
----------

.. _inherited-from-attribute-value-9:

Inherited from attribute value:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-  `has raw value <has_raw_value.md>`__ 0..1

   -  Description: The value that was specified for an annotation in raw
      form, i.e. a string. E.g. “2 cm” or “2-4 cm”
   -  Range: `String <types/String.md>`__

-  `was generated by <was_generated_by.md>`__ 0..1

   -  Range: `Activity <Activity.md>`__

-  `attribute value➞type <attribute_value_type.md>`__ 0..1

   -  Description: An optional string that specified the type of object.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

Class: workflow execution activity
==================================

Represents an instance of an execution of a particular workflow

URI:
https://microbiomedata/schema/workflow_execution_activity/WorkflowExecutionActivity

|image49|

.. _parents-52:

Parents
-------

-  is_a: `Activity <Activity.md>`__ - a provence-generating activity

.. _children-22:

Children
--------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__ - A
   metatranscriptome activity that e.g. pools assembly and annotation
   activity.
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__

.. _referenced-by-class-34:

Referenced by Class
-------------------

-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__
   `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1
   `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _attributes-49:

Attributes
----------

.. _own-37:

Own
~~~

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__ 1..1

   -  Description: Example: NERSC-Cori
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__ 1..1

   -  Description: Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1
   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__ 1..\*

   -  Description: An input to a process.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__ 1..\*

   -  Description: An output biosample to a processing step
   -  Range: `NamedThing <NamedThing.md>`__

-  `part of <part_of.md>`__ 0..\*

   -  Description: Links a resource to another resource that either
      logically or physically includes it.
   -  Range: `NamedThing <NamedThing.md>`__

-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__ 1..1

   -  Description: An optional string that specifies the type object.
      This is used to allow for searches for different kinds of objects.
   -  Range: `String <types/String.md>`__
   -  Example: nmdc:Biosample None
   -  Example: nmdc:Study None

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__ 0..1

   -  Description: the agent/entity associated with the generation of
      the file
   -  Range:
      `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__ 1..1

   -  Range: `String <types/String.md>`__

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__ 1..1

   -  Range: `Activity <Activity.md>`__

Inherited from activity:
~~~~~~~~~~~~~~~~~~~~~~~~

-  `id <id.md>`__ 1..1

   -  Description: A unique identifier for a thing. Must be either a
      CURIE shorthand for a URI or a complete URI
   -  Range: `String <types/String.md>`__

-  `name <name.md>`__ 0..1

   -  Description: A human readable label for an entity
   -  Range: `String <types/String.md>`__

-  `used <used.md>`__ 0..1

   -  Range: `String <types/String.md>`__

.. _other-properties-46:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Aliases:**          |                       | analysis              |
+-----------------------+-----------------------+-----------------------+
| **Alt Descriptions:** |                       | An analysis contains  |
|                       |                       | secondary analysis    |
|                       |                       | results derived from  |
|                       |                       | sequence reads        |
|                       |                       | (e.g. a genome        |
|                       |                       | assembly) (embl.ena)  |
+-----------------------+-----------------------+-----------------------+
| **In Subsets:**       |                       | workflow subset       |
+-----------------------+-----------------------+-----------------------+

Subset: workflow subset
=======================

Subset consisting of just the workflow exectution activities

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_subset

.. _classes-8:

Classes
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__ - A
   metatranscriptome activity that e.g. pools assembly and annotation
   activity.
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
   Represents an instance of an execution of a particular workflow

.. _mixins-8:

Mixins
------

.. _slots-8:

Slots
-----

.. _types-8:

Types
-----

.. _enums-8:

Enums
-----

Slot: \_16s_recover
===================

Can a 16S gene be recovered from the submitted SAG or MAG?

URI: [nmdc:_16s_recover](https://microbiomedata/meta/_16s_recover)

.. _domain-and-range-16:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-53:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-23:

Children
--------

.. _used-by-16:

Used by
-------

.. _other-properties-47:

Other properties
----------------

=============== =================
**Aliases:**     16S recovered
**Mappings:**    mixs:_16s_recover
**In Subsets:**  sequencing
=============== =================

Slot: \_16s_recover_software
============================

Tools used for 16S rRNA gene extraction

URI:
[nmdc:_16s_recover_software](https://microbiomedata/meta/_16s_recover_software)

.. _domain-and-range-17:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-54:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-24:

Children
--------

.. _used-by-17:

Used by
-------

.. _other-properties-48:

Other properties
----------------

=============== ==========================
**Aliases:**     16S recovery software
**Mappings:**    mixs:_16s_recover_software
**In Subsets:**  sequencing
=============== ==========================

Slot: abs_air_humidity
======================

Actual mass of water vapor - mh20 - present in the air water vapor
mixture

URI:
`nmdc:abs_air_humidity <https://microbiomedata/meta/abs_air_humidity>`__

.. _domain-and-range-18:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-55:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-25:

Children
--------

.. _used-by-18:

Used by
-------

.. _other-properties-49:

Other properties
----------------

============= =====================
**Aliases:**   absolute air humidity
**Mappings:**  mixs:abs_air_humidity
============= =====================

Slot: abstract
==============

The abstract of manuscript/grant associated with the entity; i.e., a
summary of the resource.

URI: https://microbiomedata/schema/workflow_execution_activity/abstract

.. _domain-and-range-19:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-56:

Parents
-------

.. _children-26:

Children
--------

.. _used-by-19:

Used by
-------

.. _other-properties-50:

Other properties
----------------

=================== ================
**Exact Mappings:**  dcterms:abstract
=================== ================

Slot: acted on behalf of
========================

URI:
https://microbiomedata/schema/workflow_execution_activity/acted_on_behalf_of

.. _domain-and-range-20:

Domain and Range
----------------

None → 0..1 `Agent <Agent.md>`__

.. _parents-57:

Parents
-------

.. _children-27:

Children
--------

.. _used-by-20:

Used by
-------

-  `Agent <Agent.md>`__

.. _other-properties-51:

Other properties
----------------

============= ====================
**Mappings:**  prov:actedOnBehalfOf
============= ====================

Slot: activity set
==================

This property links a database object to the set of workflow activities.

URI: `nmdc:activity_set <https://microbiomedata/meta/activity_set>`__

.. _domain-and-range-21:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _parents-58:

Parents
-------

.. _children-28:

Children
--------

.. _used-by-21:

Used by
-------

-  `Database <Database.md>`__

Slot: adapters
==============

Adapters provide priming sequences for both amplification and sequencing
of the sample-library fragments. Both adapters should be reported; in
uppercase letters

URI: `nmdc:adapters <https://microbiomedata/meta/adapters>`__

.. _domain-and-range-22:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-59:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-29:

Children
--------

.. _used-by-22:

Used by
-------

.. _other-properties-52:

Other properties
----------------

=============== =============
**Aliases:**     adapters
**Mappings:**    mixs:adapters
**In Subsets:**  sequencing
=============== =============

Slot: add_date
==============

The date on which the information was added to the database.

URI: `nmdc:add_date <https://microbiomedata/meta/add_date>`__

.. _domain-and-range-23:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-60:

Parents
-------

.. _children-30:

Children
--------

.. _used-by-23:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `OmicsProcessing <OmicsProcessing.md>`__

Slot: add_recov_method
======================

“Additional (i.e. Secondary, tertiary, etc.) recovery methods deployed
for increase of hydrocarbon recovery from resource and start date for
each one of them. If”“other”” is specified, please propose entry in
““additional info”” field”

URI:
`nmdc:add_recov_method <https://microbiomedata/meta/add_recov_method>`__

.. _domain-and-range-24:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-61:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-31:

Children
--------

.. _used-by-24:

Used by
-------

.. _other-properties-53:

Other properties
----------------

============= ======================================================
**Aliases:**   secondary and tertiary recovery methods and start date
**Mappings:**  mixs:add_recov_method
============= ======================================================

Slot: additional_info
=====================

Information that doesn’t fit anywhere else. Can also be used to propose
new entries for fields with controlled vocabulary

URI:
`nmdc:additional_info <https://microbiomedata/meta/additional_info>`__

.. _domain-and-range-25:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-62:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-32:

Children
--------

.. _used-by-25:

Used by
-------

.. _other-properties-54:

Other properties
----------------

============= ====================
**Aliases:**   additional info
**Mappings:**  mixs:additional_info
============= ====================

Slot: address
=============

The street name and building number where the sampling occurred.

URI: `nmdc:address <https://microbiomedata/meta/address>`__

.. _domain-and-range-26:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-63:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-33:

Children
--------

.. _used-by-26:

Used by
-------

.. _other-properties-55:

Other properties
----------------

============= ============
**Aliases:**   address
**Mappings:**  mixs:address
============= ============

Slot: adj_room
==============

“List of rooms (room number, room name) immediately adjacent to the
sampling room”

URI: `nmdc:adj_room <https://microbiomedata/meta/adj_room>`__

.. _domain-and-range-27:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-64:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-34:

Children
--------

.. _used-by-27:

Used by
-------

.. _other-properties-56:

Other properties
----------------

============= ==============
**Aliases:**   adjacent rooms
**Mappings:**  mixs:adj_room
============= ==============

Slot: aero_struc
================

“Aerospace structures typically consist of thin plates with stiffeners
for the external surfaces, bulkheads and frames to support the shape and
fasteners such as welds, rivets, screws and bolts to hold the components
together”

URI: `nmdc:aero_struc <https://microbiomedata/meta/aero_struc>`__

.. _domain-and-range-28:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-65:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-35:

Children
--------

.. _used-by-28:

Used by
-------

.. _other-properties-57:

Other properties
----------------

============= ===================
**Aliases:**   aerospace structure
**Mappings:**  mixs:aero_struc
============= ===================

Slot: agrochem_addition
=======================

“Addition of fertilizers, pesticides, etc. - amount and time of
applications”

URI:
`nmdc:agrochem_addition <https://microbiomedata/meta/agrochem_addition>`__

.. _domain-and-range-29:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-66:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-36:

Children
--------

.. _used-by-29:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-58:

Other properties
----------------

============= ==============================
**Aliases:**   history/agrochemical additions
**Mappings:**  mixs:agrochem_addition
============= ==============================

Slot: air_temp
==============

Temperature of the air at the time of sampling

URI: `nmdc:air_temp <https://microbiomedata/meta/air_temp>`__

.. _domain-and-range-30:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-67:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-37:

Children
--------

.. _used-by-30:

Used by
-------

.. _other-properties-59:

Other properties
----------------

============= ===============
**Aliases:**   air temperature
**Mappings:**  mixs:air_temp
============= ===============

Slot: air_temp_regm
===================

“Information about treatment involving an exposure to varying
temperatures; should include the temperature, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include different temperature regimens”

URI: `nmdc:air_temp_regm <https://microbiomedata/meta/air_temp_regm>`__

.. _domain-and-range-31:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-68:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-38:

Children
--------

.. _used-by-31:

Used by
-------

.. _other-properties-60:

Other properties
----------------

============= =======================
**Aliases:**   air temperature regimen
**Mappings:**  mixs:air_temp_regm
============= =======================

Slot: al_sat
============

Aluminum saturation (esp. For tropical soils)

URI: `nmdc:al_sat <https://microbiomedata/meta/al_sat>`__

.. _domain-and-range-32:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-69:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-39:

Children
--------

.. _used-by-32:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-61:

Other properties
----------------

============= ========================================
**Aliases:**   extreme_unusual_properties/Al saturation
**Mappings:**  mixs:al_sat
============= ========================================

Slot: al_sat_meth
=================

Reference or method used in determining Al saturation

URI: `nmdc:al_sat_meth <https://microbiomedata/meta/al_sat_meth>`__

.. _domain-and-range-33:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-70:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-40:

Children
--------

.. _used-by-33:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-62:

Other properties
----------------

============= ===============================================
**Aliases:**   extreme_unusual_properties/Al saturation method
**Mappings:**  mixs:al_sat_meth
============= ===============================================

Slot: alkalinity
================

“Alkalinity, the ability of a solution to neutralize acids to the
equivalence point of carbonate or bicarbonate”

URI: `nmdc:alkalinity <https://microbiomedata/meta/alkalinity>`__

.. _domain-and-range-34:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-71:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-41:

Children
--------

.. _used-by-34:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-63:

Other properties
----------------

============= ===============
**Aliases:**   alkalinity
**Mappings:**  mixs:alkalinity
============= ===============

Slot: alkalinity_method
=======================

Method used for alkalinity measurement

URI:
`nmdc:alkalinity_method <https://microbiomedata/meta/alkalinity_method>`__

.. _domain-and-range-35:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-72:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-42:

Children
--------

.. _used-by-35:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-64:

Other properties
----------------

============= ======================
**Aliases:**   alkalinity method
**Mappings:**  mixs:alkalinity_method
============= ======================

Slot: alkyl_diethers
====================

Concentration of alkyl diethers

URI:
`nmdc:alkyl_diethers <https://microbiomedata/meta/alkyl_diethers>`__

.. _domain-and-range-36:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-73:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-43:

Children
--------

.. _used-by-36:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-65:

Other properties
----------------

============= ===================
**Aliases:**   alkyl diethers
**Mappings:**  mixs:alkyl_diethers
============= ===================

Slot: all proteins
==================

the list of protein identifiers that are associated with the peptide
sequence

URI:
https://microbiomedata/schema/workflow_execution_activity/all_proteins

.. _domain-and-range-37:

Domain and Range
----------------

None → 0..\* `GeneProduct <GeneProduct.md>`__

.. _parents-74:

Parents
-------

.. _children-44:

Children
--------

-  `peptide quantification➞all
   proteins <peptide_quantification_all_proteins.md>`__
-  `protein quantification➞all
   proteins <protein_quantification_all_proteins.md>`__

.. _used-by-37:

Used by
-------

Slot: alt
=========

“Altitude is a term used to identify heights of objects such as
airplanes, space shuttles, rockets, atmospheric balloons and heights of
places such as atmospheric layers and clouds. It is used to measure the
height of an object which is above the earthbs surface. In this context,
the altitude measurement is the vertical distance between the earth’s
surface above sea level and the sampled position in the air”

URI: `nmdc:alt <https://microbiomedata/meta/alt>`__

.. _domain-and-range-38:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-75:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-45:

Children
--------

.. _used-by-38:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-66:

Other properties
----------------

=============== ===========
**Aliases:**     altitude
**Mappings:**    mixs:alt
**In Subsets:**  environment
=============== ===========

Slot: alternate emails
======================

One or more other email addresses for an entity.

URI:
https://microbiomedata/schema/workflow_execution_activity/alternate_emails

.. _domain-and-range-39:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-76:

Parents
-------

.. _children-46:

Children
--------

.. _used-by-39:

Used by
-------

Slot: alternative descriptions
==============================

A list of alternative descriptions for the entity. The distinction
between desciption and alternative descriptions is application-specific.

URI:
https://microbiomedata/schema/workflow_execution_activity/alternative_descriptions

.. _domain-and-range-40:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-77:

Parents
-------

.. _children-47:

Children
--------

.. _used-by-40:

Used by
-------

Slot: alternative identifiers
=============================

A list of alternative identifiers for the entity.

URI:
https://microbiomedata/schema/workflow_execution_activity/alternative_identifiers

.. _domain-and-range-41:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-78:

Parents
-------

.. _children-48:

Children
--------

-  `external database identifiers <external_database_identifiers.md>`__

.. _used-by-41:

Used by
-------

-  `ChemicalEntity <ChemicalEntity.md>`__
-  `EnvironmentalMaterialTerm <EnvironmentalMaterialTerm.md>`__
-  `GeneProduct <GeneProduct.md>`__
-  `Instrument <Instrument.md>`__
-  `MetaboliteQuantification <MetaboliteQuantification.md>`__
-  `NamedThing <NamedThing.md>`__
-  `OntologyClass <OntologyClass.md>`__
-  `Person <Person.md>`__

Slot: alternative names
=======================

A list of alternative names used to refer to the entity. The distinction
between name and alternative names is application-specific.

URI:
https://microbiomedata/schema/workflow_execution_activity/alternative_names

.. _domain-and-range-42:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-79:

Parents
-------

.. _children-49:

Children
--------

.. _used-by-42:

Used by
-------

.. _other-properties-67:

Other properties
----------------

=================== ===================
**Exact Mappings:**  dcterms:alternative
\                    skos:altLabel
=================== ===================

Slot: alternative titles
========================

A list of alternative titles for the entity. The distinction between
title and alternative titles is application-specific.

URI:
https://microbiomedata/schema/workflow_execution_activity/alternative_titles

.. _domain-and-range-43:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-80:

Parents
-------

.. _children-50:

Children
--------

.. _used-by-43:

Used by
-------

.. _other-properties-68:

Other properties
----------------

=================== ===================
**Exact Mappings:**  dcterms:alternative
=================== ===================

Slot: aminopept_act
===================

Measurement of aminopeptidase activity

URI: `nmdc:aminopept_act <https://microbiomedata/meta/aminopept_act>`__

.. _domain-and-range-44:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-81:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-51:

Children
--------

.. _used-by-44:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-69:

Other properties
----------------

============= =======================
**Aliases:**   aminopeptidase activity
**Mappings:**  mixs:aminopept_act
============= =======================

Slot: ammonium
==============

Concentration of ammonium in the sample

URI: `nmdc:ammonium <https://microbiomedata/meta/ammonium>`__

.. _domain-and-range-45:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-82:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-52:

Children
--------

.. _used-by-45:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-70:

Other properties
----------------

============= =============
**Aliases:**   ammonium
**Mappings:**  mixs:ammonium
============= =============

Slot: amniotic_fluid_color
==========================

Specification of the color of the amniotic fluid sample

URI:
`nmdc:amniotic_fluid_color <https://microbiomedata/meta/amniotic_fluid_color>`__

.. _domain-and-range-46:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-83:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-53:

Children
--------

.. _used-by-46:

Used by
-------

.. _other-properties-71:

Other properties
----------------

============= =========================
**Aliases:**   amniotic fluid/color
**Mappings:**  mixs:amniotic_fluid_color
============= =========================

Slot: amount_light
==================

“The unit of illuminance and luminous emittance, measuring luminous flux
per unit area”

URI: `nmdc:amount_light <https://microbiomedata/meta/amount_light>`__

.. _domain-and-range-47:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-84:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-54:

Children
--------

.. _used-by-47:

Used by
-------

.. _other-properties-72:

Other properties
----------------

============= =================
**Aliases:**   amount of light
**Mappings:**  mixs:amount_light
============= =================

Slot: analysis identifiers
==========================

URI:
https://microbiomedata/schema/workflow_execution_activity/analysis_identifiers

.. _domain-and-range-48:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-85:

Parents
-------

-  is_a: `external database
   identifiers <external_database_identifiers.md>`__

.. _children-55:

Children
--------

-  `GOLD analysis project
   identifiers <GOLD_analysis_project_identifiers.md>`__
-  `INSDC analysis identifiers <INSDC_analysis_identifiers.md>`__
-  `MGnify analysis identifiers <MGnify_analysis_identifiers.md>`__

.. _used-by-48:

Used by
-------

Slot: ances_data
================

“Information about either pedigree or other ancestral information
description (e.g. parental variety in case of mutant or selection),
e.g. A/3*B (meaning [(A x B) x B] x B)”

URI: `nmdc:ances_data <https://microbiomedata/meta/ances_data>`__

.. _domain-and-range-49:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-86:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-56:

Children
--------

.. _used-by-49:

Used by
-------

.. _other-properties-73:

Other properties
----------------

============= ===============
**Aliases:**   ancestral data
**Mappings:**  mixs:ances_data
============= ===============

Slot: annot
===========

“Tool used for annotation, or for cases where annotation was provided by
a community jamboree or model organism database rather than by a
specific submitter”

URI: `nmdc:annot <https://microbiomedata/meta/annot>`__

.. _domain-and-range-50:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-87:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-57:

Children
--------

.. _used-by-50:

Used by
-------

.. _other-properties-74:

Other properties
----------------

=============== ==========
**Aliases:**     annotation
**Mappings:**    mixs:annot
**In Subsets:**  sequencing
=============== ==========

Slot: annual_precpt
===================

“The average of all annual precipitation values known, or an estimated
equivalent value derived by such methods as regional indexes or
Isohyetal maps.”

URI: `nmdc:annual_precpt <https://microbiomedata/meta/annual_precpt>`__

.. _domain-and-range-51:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-88:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-58:

Children
--------

.. _used-by-51:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-75:

Other properties
----------------

============= =========================
**Aliases:**   mean annual precipitation
**Mappings:**  mixs:annual_precpt
============= =========================

Slot: annual_temp
=================

Mean annual temperature

URI: `nmdc:annual_temp <https://microbiomedata/meta/annual_temp>`__

.. _domain-and-range-52:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-89:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-59:

Children
--------

.. _used-by-52:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-76:

Other properties
----------------

============= =======================
**Aliases:**   mean annual temperature
**Mappings:**  mixs:annual_temp
============= =======================

Slot: antibiotic_regm
=====================

“Information about treatment involving antibiotic administration; should
include the name of antibiotic, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple antibiotic regimens”

URI:
`nmdc:antibiotic_regm <https://microbiomedata/meta/antibiotic_regm>`__

.. _domain-and-range-53:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-90:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-60:

Children
--------

.. _used-by-53:

Used by
-------

.. _other-properties-77:

Other properties
----------------

============= ====================
**Aliases:**   antibiotic regimen
**Mappings:**  mixs:antibiotic_regm
============= ====================

Slot: api
=========

API gravity is a measure of how heavy or light a petroleum liquid is
compared to water (source: https://en.wikipedia.org/wiki/API_gravity)
(e.g. 31.1B0 API)

URI: `nmdc:api <https://microbiomedata/meta/api>`__

.. _domain-and-range-54:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-91:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-61:

Children
--------

.. _used-by-54:

Used by
-------

.. _other-properties-78:

Other properties
----------------

============= ===========
**Aliases:**   API gravity
**Mappings:**  mixs:api
============= ===========

Slot: applied role
==================

URI: `nmdc:applied_role <https://microbiomedata/meta/applied_role>`__

.. _domain-and-range-55:

Domain and Range
----------------

`CreditAssociation <CreditAssociation.md>`__ → 1..1 `credit
enum <credit%20enum.md>`__

.. _parents-92:

Parents
-------

.. _children-62:

Children
--------

.. _used-by-55:

Used by
-------

-  `CreditAssociation <CreditAssociation.md>`__

.. _other-properties-79:

Other properties
----------------

============= ============
**Mappings:**  prov:hadRole
============= ============

Slot: applies to person
=======================

URI:
`nmdc:applies_to_person <https://microbiomedata/meta/applies_to_person>`__

.. _domain-and-range-56:

Domain and Range
----------------

`CreditAssociation <CreditAssociation.md>`__ → 1..1
`PersonValue <PersonValue.md>`__

.. _parents-93:

Parents
-------

.. _children-63:

Children
--------

.. _used-by-56:

Used by
-------

-  `CreditAssociation <CreditAssociation.md>`__

.. _other-properties-80:

Other properties
----------------

============= ==========
**Mappings:**  prov:agent
============= ==========

Slot: arch_struc
================

“An architectural structure is a human-made, free-standing, immobile
outdoor construction”

URI: `nmdc:arch_struc <https://microbiomedata/meta/arch_struc>`__

.. _domain-and-range-57:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-94:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-64:

Children
--------

.. _used-by-57:

Used by
-------

.. _other-properties-81:

Other properties
----------------

============= =======================
**Aliases:**   architectural structure
**Mappings:**  mixs:arch_struc
============= =======================

Slot: aromatics_pc
==================

“Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)”

URI: `nmdc:aromatics_pc <https://microbiomedata/meta/aromatics_pc>`__

.. _domain-and-range-58:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-95:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-65:

Children
--------

.. _used-by-58:

Used by
-------

.. _other-properties-82:

Other properties
----------------

============= =================
**Aliases:**   aromatics wt%
**Mappings:**  mixs:aromatics_pc
============= =================

Slot: asm_score
===============

A score for comparing metagenomic assembly quality from same sample.

URI: https://microbiomedata/schema/workflow_execution_activity/asm_score

.. _domain-and-range-59:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-96:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-66:

Children
--------

.. _used-by-59:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: asphaltenes_pc
====================

“Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)”

URI:
`nmdc:asphaltenes_pc <https://microbiomedata/meta/asphaltenes_pc>`__

.. _domain-and-range-60:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-97:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-67:

Children
--------

.. _used-by-60:

Used by
-------

.. _other-properties-83:

Other properties
----------------

============= ===================
**Aliases:**   asphaltenes wt%
**Mappings:**  mixs:asphaltenes_pc
============= ===================

Slot: assembly identifiers
==========================

URI:
https://microbiomedata/schema/workflow_execution_activity/assembly_identifiers

.. _domain-and-range-61:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-98:

Parents
-------

.. _children-68:

Children
--------

-  `INSDC assembly identifiers <INSDC_assembly_identifiers.md>`__

.. _used-by-61:

Used by
-------

Slot: assembly_name
===================

Name/version of the assembly provided by the submitter that is used in
the genome browsers and in the community

URI: `nmdc:assembly_name <https://microbiomedata/meta/assembly_name>`__

.. _domain-and-range-62:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-99:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-69:

Children
--------

.. _used-by-62:

Used by
-------

.. _other-properties-84:

Other properties
----------------

=============== ==================
**Aliases:**     assembly name
**Mappings:**    mixs:assembly_name
**In Subsets:**  sequencing
=============== ==================

Slot: assembly_qual
===================

“The assembly quality category is based on sets of criteria outlined for
each assembly quality category. For MISAG/MIMAG; Finished: Single,
validated, contiguous sequence per replicon without gaps or ambiguities
with a consensus error rate equivalent to Q50 or better. High Quality
Draft:Multiple fragments where gaps span repetitive regions. Presence of
the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality
Draft:Many fragments with little to no review of assembly other than
reporting of standard assembly statistics. Low Quality Draft:Many
fragments with little to no review of assembly other than reporting of
standard assembly statistics. Assembly statistics include, but are not
limited to total assembly size, number of contigs, contig N50/L50, and
maximum contig length. For MIUVIG; Finished: Single, validated,
contiguous sequence per replicon without gaps or ambiguities, with
extensive manual review and editing to annotate putative gene functions
and transcriptional units. High-quality draft genome: One or multiple
fragments, totaling 3 90% of the expected genome or replicon sequence or
predicted complete. Genome fragment(s): One or multiple fragments,
totalling < 90% of the expected genome or replicon sequence, or for
which no genome size could be estimated”

URI: `nmdc:assembly_qual <https://microbiomedata/meta/assembly_qual>`__

.. _domain-and-range-63:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-100:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-70:

Children
--------

.. _used-by-63:

Used by
-------

.. _other-properties-85:

Other properties
----------------

=============== ==================
**Aliases:**     assembly quality
**Mappings:**    mixs:assembly_qual
**In Subsets:**  sequencing
=============== ==================

Slot: assembly_software
=======================

“Tool(s) used for assembly, including version number and parameters”

URI:
`nmdc:assembly_software <https://microbiomedata/meta/assembly_software>`__

.. _domain-and-range-64:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-101:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-71:

Children
--------

.. _used-by-64:

Used by
-------

.. _other-properties-86:

Other properties
----------------

=============== ======================
**Aliases:**     assembly software
**Mappings:**    mixs:assembly_software
**In Subsets:**  sequencing
=============== ======================

Slot: atmospheric_data
======================

Measurement of atmospheric data; can include multiple data

URI:
`nmdc:atmospheric_data <https://microbiomedata/meta/atmospheric_data>`__

.. _domain-and-range-65:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-102:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-72:

Children
--------

.. _used-by-65:

Used by
-------

.. _other-properties-87:

Other properties
----------------

============= =====================
**Aliases:**   atmospheric data
**Mappings:**  mixs:atmospheric_data
============= =====================

Slot: attribute
===============

A attribute of a biosample. Examples: depth, habitat, material. For
NMDC, attributes SHOULD be mapped to terms within a MIxS template

URI: https://microbiomedata/schema/workflow_execution_activity/attribute

.. _domain-and-range-66:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-103:

Parents
-------

.. _children-73:

Children
--------

-  `display order <display_order.md>`__
-  `execution resource <execution_resource.md>`__
-  `file size bytes <file_size_bytes.md>`__
-  `git url <git_url.md>`__
-  `md5 checksum <md5_checksum.md>`__
-  `url <url.md>`__

.. _used-by-66:

Used by
-------

.. _other-properties-88:

Other properties
----------------

============ ==============
**Aliases:**  field
\             property
\             template field
\             key
\             characteristic
============ ==============

Slot: type
==========

An optional string that specified the type of object.

URI:
https://microbiomedata/schema/workflow_execution_activity/attribute_value_type

.. _domain-and-range-67:

Domain and Range
----------------

`AttributeValue <AttributeValue.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-104:

Parents
-------

-  is_a: `type <type.md>`__

.. _children-74:

Children
--------

.. _used-by-67:

Used by
-------

-  `AttributeValue <AttributeValue.md>`__
-  `BooleanValue <BooleanValue.md>`__
-  `ControlledTermValue <ControlledTermValue.md>`__
-  `GeolocationValue <GeolocationValue.md>`__
-  `ImageValue <ImageValue.md>`__
-  `IntegerValue <IntegerValue.md>`__
-  `PersonValue <PersonValue.md>`__
-  `QuantityValue <QuantityValue.md>`__
-  `TextValue <TextValue.md>`__
-  `TimestampValue <TimestampValue.md>`__
-  `UrlValue <UrlValue.md>`__

.. _other-properties-89:

Other properties
----------------

============= =================================================
**Examples:**  Example(value=‘nmdc:Biosample’, description=None)
\              Example(value=‘nmdc:Study’, description=None)
============= =================================================

Slot: avg_dew_point
===================

The average of dew point measures taken at the beginning of every hour
over a 24 hour period on the sampling day

URI: `nmdc:avg_dew_point <https://microbiomedata/meta/avg_dew_point>`__

.. _domain-and-range-68:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-105:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-75:

Children
--------

.. _used-by-68:

Used by
-------

.. _other-properties-90:

Other properties
----------------

============= ==================
**Aliases:**   average dew point
**Mappings:**  mixs:avg_dew_point
============= ==================

Slot: avg_occup
===============

Daily average occupancy of room

URI: `nmdc:avg_occup <https://microbiomedata/meta/avg_occup>`__

.. _domain-and-range-69:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-106:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-76:

Children
--------

.. _used-by-69:

Used by
-------

.. _other-properties-91:

Other properties
----------------

============= =======================
**Aliases:**   average daily occupancy
**Mappings:**  mixs:avg_occup
============= =======================

Slot: avg_temp
==============

The average of temperatures taken at the beginning of every hour over a
24 hour period on the sampling day

URI: `nmdc:avg_temp <https://microbiomedata/meta/avg_temp>`__

.. _domain-and-range-70:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-107:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-77:

Children
--------

.. _used-by-70:

Used by
-------

.. _other-properties-92:

Other properties
----------------

============= ===================
**Aliases:**   average temperature
**Mappings:**  mixs:avg_temp
============= ===================

Slot: bac_prod
==============

Bacterial production in the water column measured by isotope uptake

URI: `nmdc:bac_prod <https://microbiomedata/meta/bac_prod>`__

.. _domain-and-range-71:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-108:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-78:

Children
--------

.. _used-by-71:

Used by
-------

.. _other-properties-93:

Other properties
----------------

============= ====================
**Aliases:**   bacterial production
**Mappings:**  mixs:bac_prod
============= ====================

Slot: bac_resp
==============

Measurement of bacterial respiration in the water column

URI: `nmdc:bac_resp <https://microbiomedata/meta/bac_resp>`__

.. _domain-and-range-72:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-109:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-79:

Children
--------

.. _used-by-72:

Used by
-------

.. _other-properties-94:

Other properties
----------------

============= =====================
**Aliases:**   bacterial respiration
**Mappings:**  mixs:bac_resp
============= =====================

Slot: bacteria_carb_prod
========================

Measurement of bacterial carbon production

URI:
`nmdc:bacteria_carb_prod <https://microbiomedata/meta/bacteria_carb_prod>`__

.. _domain-and-range-73:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-110:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-80:

Children
--------

.. _used-by-73:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-95:

Other properties
----------------

============= ===========================
**Aliases:**   bacterial carbon production
**Mappings:**  mixs:bacteria_carb_prod
============= ===========================

Slot: barometric_press
======================

Force per unit area exerted against a surface by the weight of air above
that surface

URI:
`nmdc:barometric_press <https://microbiomedata/meta/barometric_press>`__

.. _domain-and-range-74:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-111:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-81:

Children
--------

.. _used-by-74:

Used by
-------

.. _other-properties-96:

Other properties
----------------

============= =====================
**Aliases:**   barometric pressure
**Mappings:**  mixs:barometric_press
============= =====================

Slot: basin
===========

Name of the basin (e.g. Campos)

URI: `nmdc:basin <https://microbiomedata/meta/basin>`__

.. _domain-and-range-75:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-112:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-82:

Children
--------

.. _used-by-75:

Used by
-------

.. _other-properties-97:

Other properties
----------------

============= ==========
**Aliases:**   basin name
**Mappings:**  mixs:basin
============= ==========

Slot: bathroom_count
====================

The number of bathrooms in the building

URI:
`nmdc:bathroom_count <https://microbiomedata/meta/bathroom_count>`__

.. _domain-and-range-76:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-113:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-83:

Children
--------

.. _used-by-76:

Used by
-------

.. _other-properties-98:

Other properties
----------------

============= ===================
**Aliases:**   bathroom count
**Mappings:**  mixs:bathroom_count
============= ===================

Slot: bedroom_count
===================

The number of bedrooms in the building

URI: `nmdc:bedroom_count <https://microbiomedata/meta/bedroom_count>`__

.. _domain-and-range-77:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-114:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-84:

Children
--------

.. _used-by-77:

Used by
-------

.. _other-properties-99:

Other properties
----------------

============= ==================
**Aliases:**   bedroom count
**Mappings:**  mixs:bedroom_count
============= ==================

Slot: benzene
=============

Concentration of benzene in the sample

URI: `nmdc:benzene <https://microbiomedata/meta/benzene>`__

.. _domain-and-range-78:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-115:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-85:

Children
--------

.. _used-by-78:

Used by
-------

.. _other-properties-100:

Other properties
----------------

============= ============
**Aliases:**   benzene
**Mappings:**  mixs:benzene
============= ============

Slot: best protein
==================

the specific protein identifier most correctly associated with the
peptide sequence

URI:
https://microbiomedata/schema/workflow_execution_activity/best_protein

.. _domain-and-range-79:

Domain and Range
----------------

None → 0..1 `GeneProduct <GeneProduct.md>`__

.. _parents-116:

Parents
-------

.. _children-86:

Children
--------

-  `peptide quantification➞best
   protein <peptide_quantification_best_protein.md>`__
-  `protein quantification➞best
   protein <protein_quantification_best_protein.md>`__

.. _used-by-79:

Used by
-------

Slot: bin_param
===============

The parameters that have been applied during the extraction of genomes
from metagenomic datasets

URI: `nmdc:bin_param <https://microbiomedata/meta/bin_param>`__

.. _domain-and-range-80:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-117:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-87:

Children
--------

.. _used-by-80:

Used by
-------

.. _other-properties-101:

Other properties
----------------

=============== ==================
**Aliases:**     binning parameters
**Mappings:**    mixs:bin_param
**In Subsets:**  sequencing
=============== ==================

Slot: bin_software
==================

Tool(s) used for the extraction of genomes from metagenomic datasets

URI: `nmdc:bin_software <https://microbiomedata/meta/bin_software>`__

.. _domain-and-range-81:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-118:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-88:

Children
--------

.. _used-by-81:

Used by
-------

.. _other-properties-102:

Other properties
----------------

=============== =================
**Aliases:**     binning software
**Mappings:**    mixs:bin_software
**In Subsets:**  sequencing
=============== =================

Slot: binned contig num
=======================

URI:
https://microbiomedata/schema/workflow_execution_activity/binned_contig_num

.. _domain-and-range-82:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-119:

Parents
-------

.. _children-89:

Children
--------

.. _used-by-82:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

Slot: biochem_oxygen_dem
========================

Amount of dissolved oxygen needed by aerobic biological organisms in a
body of water to break down organic material present in a given water
sample at certain temperature over a specific time period

URI:
`nmdc:biochem_oxygen_dem <https://microbiomedata/meta/biochem_oxygen_dem>`__

.. _domain-and-range-83:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-120:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-90:

Children
--------

.. _used-by-83:

Used by
-------

.. _other-properties-103:

Other properties
----------------

============= =========================
**Aliases:**   biochemical oxygen demand
**Mappings:**  mixs:biochem_oxygen_dem
============= =========================

Slot: biocide
=============

List of biocides (commercial name of product and supplier) and date of
administration

URI: `nmdc:biocide <https://microbiomedata/meta/biocide>`__

.. _domain-and-range-84:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-121:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-91:

Children
--------

.. _used-by-84:

Used by
-------

.. _other-properties-104:

Other properties
----------------

============= ======================
**Aliases:**   biocide administration
**Mappings:**  mixs:biocide
============= ======================

Slot: biocide_admin_method
==========================

“Method of biocide administration (dose, frequency, duration, time
elapsed between last biociding and sampling) (e.g. 150 mg/l; weekly; 4
hr; 3 days)”

URI:
`nmdc:biocide_admin_method <https://microbiomedata/meta/biocide_admin_method>`__

.. _domain-and-range-85:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-122:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-92:

Children
--------

.. _used-by-85:

Used by
-------

.. _other-properties-105:

Other properties
----------------

============= =============================
**Aliases:**   biocide administration method
**Mappings:**  mixs:biocide_admin_method
============= =============================

Slot: biogas_retention_time
===========================

URI:
`nmdc:biogas_retention_time <https://microbiomedata/meta/biogas_retention_time>`__

.. _domain-and-range-86:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-123:

Parents
-------

.. _children-93:

Children
--------

.. _used-by-86:

Used by
-------

Slot: biogas_temperature
========================

URI:
`nmdc:biogas_temperature <https://microbiomedata/meta/biogas_temperature>`__

.. _domain-and-range-87:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-124:

Parents
-------

.. _children-94:

Children
--------

.. _used-by-87:

Used by
-------

Slot: biol_stat
===============

The level of genome modification

URI: `nmdc:biol_stat <https://microbiomedata/meta/biol_stat>`__

.. _domain-and-range-88:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-125:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-95:

Children
--------

.. _used-by-88:

Used by
-------

.. _other-properties-106:

Other properties
----------------

============= =================
**Aliases:**   biological status
**Mappings:**  mixs:biol_stat
============= =================

Slot: biomass
=============

“Amount of biomass; should include the name for the part of biomass
measured, e.g. Microbial, total. Can include multiple measurements”

URI: `nmdc:biomass <https://microbiomedata/meta/biomass>`__

.. _domain-and-range-89:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-126:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-96:

Children
--------

.. _used-by-89:

Used by
-------

.. _other-properties-107:

Other properties
----------------

============= ============
**Aliases:**   biomass
**Mappings:**  mixs:biomass
============= ============

Slot: env_broad_scale
=====================

“In this field, report which major environmental system your sample or
specimen came from. The systems identified should have a coarse spatial
grain, to provide the general environmental context of where the
sampling was done (e.g. were you in the desert or a rainforest?). We
recommend using subclasses of ENVOUs biome class:
http://purl.obolibrary.org/obo/ENVO_00000428. Format (one term):
termLabel [termID], Format (multiple terms): termLabel
[termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a
water sample from the photic zone in middle of the Atlantic Ocean,
consider: oceanic epipelagic zone biome [ENVO:01000033]. Example:
Annotating a sample from the Amazon rainforest consider: tropical moist
broadleaf forest biome [ENVO:01000228]. If needed, request new terms on
the ENVO tracker, identified here:
http://www.obofoundry.org/ontology/envo.html”

URI:
`nmdc:biosample_env_broad_scale <https://microbiomedata/meta/biosample_env_broad_scale>`__

.. _domain-and-range-90:

Domain and Range
----------------

`Biosample <Biosample.md>`__ → 1..1
`ControlledTermValue <ControlledTermValue.md>`__

.. _parents-127:

Parents
-------

-  is_a: `env_broad_scale <env_broad_scale.md>`__

.. _children-97:

Children
--------

.. _used-by-90:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-108:

Other properties
----------------

=============== =================================
**Aliases:**     broad-scale environmental context
**Mappings:**    mixs:env_broad_scale
**In Subsets:**  environment
=============== =================================

Slot: env_local_scale
=====================

“In this field, report the entity or entities which are in your sample
or specimenUs local vicinity and which you believe have significant
causal influences on your sample or specimen. Please use terms that are
present in ENVO and which are of smaller spatial grain than your entry
for env_broad_scale. Format (one term): termLabel [termID]; Format
(multiple terms): termLabel [termID]|termLabel [termID]|termLabel
[termID]. Example: Annotating a pooled sample taken from various
vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and
fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory
[01000335]|shrub layer [ENVO:01000336]. If needed, request new terms on
the ENVO tracker, identified here:
http://www.obofoundry.org/ontology/envo.html”

URI:
`nmdc:biosample_env_local_scale <https://microbiomedata/meta/biosample_env_local_scale>`__

.. _domain-and-range-91:

Domain and Range
----------------

`Biosample <Biosample.md>`__ → 1..1
`ControlledTermValue <ControlledTermValue.md>`__

.. _parents-128:

Parents
-------

-  is_a: `env_local_scale <env_local_scale.md>`__

.. _children-98:

Children
--------

.. _used-by-91:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-109:

Other properties
----------------

=============== ===========================
**Aliases:**     local environmental context
**Mappings:**    mixs:env_local_scale
**In Subsets:**  environment
=============== ===========================

Slot: env_medium
================

“In this field, report which environmental material or materials (pipe
separated) immediately surrounded your sample or specimen prior to
sampling, using one or more subclasses of ENVOUs environmental material
class: http://purl.obolibrary.org/obo/ENVO_00010483. Format (one term):
termLabel [termID]; Format (multiple terms): termLabel
[termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a
fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean
water [ENVO:00002151]. Example: Annotating a duck on a pond consider:
pond water [ENVO:00002228]|air ENVO_00002005. If needed, request new
terms on the ENVO tracker, identified here:
http://www.obofoundry.org/ontology/envo.html”

URI:
`nmdc:biosample_env_medium <https://microbiomedata/meta/biosample_env_medium>`__

.. _domain-and-range-92:

Domain and Range
----------------

`Biosample <Biosample.md>`__ → 1..1
`ControlledTermValue <ControlledTermValue.md>`__

.. _parents-129:

Parents
-------

-  is_a: `env_medium <env_medium.md>`__

.. _children-99:

Children
--------

.. _used-by-92:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-110:

Other properties
----------------

=============== ====================
**Aliases:**     environmental medium
**Mappings:**    mixs:env_medium
**In Subsets:**  environment
=============== ====================

Slot: lat_lon
=============

This is currently a required field but it’s not clear if this should be
required for human hosts

URI:
`nmdc:biosample_lat_lon <https://microbiomedata/meta/biosample_lat_lon>`__

.. _domain-and-range-93:

Domain and Range
----------------

`Biosample <Biosample.md>`__ → 0..1
`GeolocationValue <GeolocationValue.md>`__

.. _parents-130:

Parents
-------

-  is_a: `lat_lon <lat_lon.md>`__

.. _children-100:

Children
--------

.. _used-by-93:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-111:

Other properties
----------------

=============== ============================================
**Aliases:**     geographic location (latitude and longitude)
**Mappings:**    mixs:lat_lon
**In Subsets:**  environment
=============== ============================================

Slot: has input
===============

An input to a process.

URI:
`nmdc:biosample_processing_has_input <https://microbiomedata/meta/biosample_processing_has_input>`__

.. _domain-and-range-94:

Domain and Range
----------------

`BiosampleProcessing <BiosampleProcessing.md>`__ → 0..\*
`Biosample <Biosample.md>`__

.. _parents-131:

Parents
-------

-  is_a: `has input <has_input.md>`__

.. _children-101:

Children
--------

-  `omics processing➞has input <omics_processing_has_input.md>`__

.. _used-by-94:

Used by
-------

-  `BiosampleProcessing <BiosampleProcessing.md>`__

.. _other-properties-112:

Other properties
----------------

============ =====
**Aliases:**  input
============ =====

Slot: biosample set
===================

This property links a database object to the set of samples within it.

URI: `nmdc:biosample_set <https://microbiomedata/meta/biosample_set>`__

.. _domain-and-range-95:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\* `Biosample <Biosample.md>`__

.. _parents-132:

Parents
-------

.. _children-102:

Children
--------

.. _used-by-95:

Used by
-------

-  `Database <Database.md>`__

Slot: biotic_regm
=================

“Information about treatment(s) involving use of biotic factors, such as
bacteria, viruses or fungi.”

URI: `nmdc:biotic_regm <https://microbiomedata/meta/biotic_regm>`__

.. _domain-and-range-96:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-133:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-103:

Children
--------

.. _used-by-96:

Used by
-------

.. _other-properties-113:

Other properties
----------------

============= ================
**Aliases:**   biotic regimen
**Mappings:**  mixs:biotic_regm
============= ================

Slot: biotic_relationship
=========================

“Description of relationship(s) between the subject organism and other
organism(s) it is associated with. E.g., parasite on species X;
mutualist with species Y. The target organism is the subject of the
relationship, and the other organism(s) is the object”

URI:
`nmdc:biotic_relationship <https://microbiomedata/meta/biotic_relationship>`__

.. _domain-and-range-97:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-134:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-104:

Children
--------

.. _used-by-97:

Used by
-------

.. _other-properties-114:

Other properties
----------------

=============== ============================
**Aliases:**     observed biotic relationship
**Mappings:**    mixs:biotic_relationship
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: birth_control
===================

Specification of birth control medication used

URI: `nmdc:birth_control <https://microbiomedata/meta/birth_control>`__

.. _domain-and-range-98:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-135:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-105:

Children
--------

.. _used-by-98:

Used by
-------

.. _other-properties-115:

Other properties
----------------

============= ==================
**Aliases:**   birth control
**Mappings:**  mixs:birth_control
============= ==================

Slot: bishomohopanol
====================

Concentration of bishomohopanol

URI:
`nmdc:bishomohopanol <https://microbiomedata/meta/bishomohopanol>`__

.. _domain-and-range-99:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-136:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-106:

Children
--------

.. _used-by-99:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-116:

Other properties
----------------

============= ===================
**Aliases:**   bishomohopanol
**Mappings:**  mixs:bishomohopanol
============= ===================

Slot: blood_blood_disord
========================

History of blood disorders; can include multiple disorders

URI:
`nmdc:blood_blood_disord <https://microbiomedata/meta/blood_blood_disord>`__

.. _domain-and-range-100:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-137:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-107:

Children
--------

.. _used-by-100:

Used by
-------

.. _other-properties-117:

Other properties
----------------

============= =======================
**Aliases:**   blood/blood disorder
**Mappings:**  mixs:blood_blood_disord
============= =======================

Slot: bromide
=============

Concentration of bromide

URI: `nmdc:bromide <https://microbiomedata/meta/bromide>`__

.. _domain-and-range-101:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-138:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-108:

Children
--------

.. _used-by-101:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-118:

Other properties
----------------

============= ============
**Aliases:**   bromide
**Mappings:**  mixs:bromide
============= ============

Slot: build_docs
================

“The building design, construction and operation documents”

URI: `nmdc:build_docs <https://microbiomedata/meta/build_docs>`__

.. _domain-and-range-102:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-139:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-109:

Children
--------

.. _used-by-102:

Used by
-------

.. _other-properties-119:

Other properties
----------------

============= =============================================
**Aliases:**   design, construction, and operation documents
**Mappings:**  mixs:build_docs
============= =============================================

Slot: build_occup_type
======================

The primary function for which a building or discrete part of a building
is intended to be used

URI:
`nmdc:build_occup_type <https://microbiomedata/meta/build_occup_type>`__

.. _domain-and-range-103:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-140:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-110:

Children
--------

.. _used-by-103:

Used by
-------

.. _other-properties-120:

Other properties
----------------

============= =======================
**Aliases:**   building occupancy type
**Mappings:**  mixs:build_occup_type
============= =======================

Slot: building_setting
======================

A location (geography) where a building is set

URI:
`nmdc:building_setting <https://microbiomedata/meta/building_setting>`__

.. _domain-and-range-104:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-141:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-111:

Children
--------

.. _used-by-104:

Used by
-------

.. _other-properties-121:

Other properties
----------------

============= =====================
**Aliases:**   building setting
**Mappings:**  mixs:building_setting
============= =====================

Slot: built_struc_age
=====================

The age of the built structure since construction

URI:
`nmdc:built_struc_age <https://microbiomedata/meta/built_struc_age>`__

.. _domain-and-range-105:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-142:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-112:

Children
--------

.. _used-by-105:

Used by
-------

.. _other-properties-122:

Other properties
----------------

============= ====================
**Aliases:**   built structure age
**Mappings:**  mixs:built_struc_age
============= ====================

Slot: built_struc_set
=====================

The characterization of the location of the built structure as high or
low human density

URI:
`nmdc:built_struc_set <https://microbiomedata/meta/built_struc_set>`__

.. _domain-and-range-106:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-143:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-113:

Children
--------

.. _used-by-106:

Used by
-------

.. _other-properties-123:

Other properties
----------------

============= =======================
**Aliases:**   built structure setting
**Mappings:**  mixs:built_struc_set
============= =======================

Slot: built_struc_type
======================

A physical structure that is a body or assemblage of bodies in space to
form a system capable of supporting loads

URI:
`nmdc:built_struc_type <https://microbiomedata/meta/built_struc_type>`__

.. _domain-and-range-107:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-144:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-114:

Children
--------

.. _used-by-107:

Used by
-------

.. _other-properties-124:

Other properties
----------------

============= =====================
**Aliases:**   built structure type
**Mappings:**  mixs:built_struc_type
============= =====================

Slot: calcium
=============

Concentration of calcium in the sample

URI: `nmdc:calcium <https://microbiomedata/meta/calcium>`__

.. _domain-and-range-108:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-145:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-115:

Children
--------

.. _used-by-108:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-125:

Other properties
----------------

============= ============
**Aliases:**   calcium
**Mappings:**  mixs:calcium
============= ============

Slot: carb_dioxide
==================

Carbon dioxide (gas) amount or concentration at the time of sampling

URI: `nmdc:carb_dioxide <https://microbiomedata/meta/carb_dioxide>`__

.. _domain-and-range-109:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-146:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-116:

Children
--------

.. _used-by-109:

Used by
-------

.. _other-properties-126:

Other properties
----------------

============= =================
**Aliases:**   carbon dioxide
**Mappings:**  mixs:carb_dioxide
============= =================

Slot: carb_monoxide
===================

Carbon monoxide (gas) amount or concentration at the time of sampling

URI: `nmdc:carb_monoxide <https://microbiomedata/meta/carb_monoxide>`__

.. _domain-and-range-110:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-147:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-117:

Children
--------

.. _used-by-110:

Used by
-------

.. _other-properties-127:

Other properties
----------------

============= ==================
**Aliases:**   carbon monoxide
**Mappings:**  mixs:carb_monoxide
============= ==================

Slot: carb_nitro_ratio
======================

Ratio of amount or concentrations of carbon to nitrogen

URI:
`nmdc:carb_nitro_ratio <https://microbiomedata/meta/carb_nitro_ratio>`__

.. _domain-and-range-111:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-148:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-118:

Children
--------

.. _used-by-111:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-128:

Other properties
----------------

============= =====================
**Aliases:**   carbon/nitrogen ratio
**Mappings:**  mixs:carb_nitro_ratio
============= =====================

Slot: ceil_area
===============

The area of the ceiling space within the room

URI: `nmdc:ceil_area <https://microbiomedata/meta/ceil_area>`__

.. _domain-and-range-112:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-149:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-119:

Children
--------

.. _used-by-112:

Used by
-------

.. _other-properties-129:

Other properties
----------------

============= ==============
**Aliases:**   ceiling area
**Mappings:**  mixs:ceil_area
============= ==============

Slot: ceil_cond
===============

The physical condition of the ceiling at the time of sampling; photos or
video preferred; use drawings to indicate location of damaged areas

URI: `nmdc:ceil_cond <https://microbiomedata/meta/ceil_cond>`__

.. _domain-and-range-113:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-150:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-120:

Children
--------

.. _used-by-113:

Used by
-------

.. _other-properties-130:

Other properties
----------------

============= =================
**Aliases:**   ceiling condition
**Mappings:**  mixs:ceil_cond
============= =================

Slot: ceil_finish_mat
=====================

The type of material used to finish a ceiling

URI:
`nmdc:ceil_finish_mat <https://microbiomedata/meta/ceil_finish_mat>`__

.. _domain-and-range-114:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-151:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-121:

Children
--------

.. _used-by-114:

Used by
-------

.. _other-properties-131:

Other properties
----------------

============= =======================
**Aliases:**   ceiling finish material
**Mappings:**  mixs:ceil_finish_mat
============= =======================

Slot: ceil_struc
================

The construction format of the ceiling

URI: `nmdc:ceil_struc <https://microbiomedata/meta/ceil_struc>`__

.. _domain-and-range-115:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-152:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-122:

Children
--------

.. _used-by-115:

Used by
-------

.. _other-properties-132:

Other properties
----------------

============= =================
**Aliases:**   ceiling structure
**Mappings:**  mixs:ceil_struc
============= =================

Slot: ceil_texture
==================

“The feel, appearance, or consistency of a ceiling surface”

URI: `nmdc:ceil_texture <https://microbiomedata/meta/ceil_texture>`__

.. _domain-and-range-116:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-153:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-123:

Children
--------

.. _used-by-116:

Used by
-------

.. _other-properties-133:

Other properties
----------------

============= =================
**Aliases:**   ceiling texture
**Mappings:**  mixs:ceil_texture
============= =================

Slot: ceil_thermal_mass
=======================

The ability of the ceiling to provide inertia against temperature
fluctuations. Generally this means concrete that is exposed. A metal
deck that supports a concrete slab will act thermally as long as it is
exposed to room air flow

URI:
`nmdc:ceil_thermal_mass <https://microbiomedata/meta/ceil_thermal_mass>`__

.. _domain-and-range-117:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-154:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-124:

Children
--------

.. _used-by-117:

Used by
-------

.. _other-properties-134:

Other properties
----------------

============= ======================
**Aliases:**   ceiling thermal mass
**Mappings:**  mixs:ceil_thermal_mass
============= ======================

Slot: ceil_type
===============

The type of ceiling according to the ceiling’s appearance or
construction

URI: `nmdc:ceil_type <https://microbiomedata/meta/ceil_type>`__

.. _domain-and-range-118:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-155:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-125:

Children
--------

.. _used-by-118:

Used by
-------

.. _other-properties-135:

Other properties
----------------

============= ==============
**Aliases:**   ceiling type
**Mappings:**  mixs:ceil_type
============= ==============

Slot: ceil_water_mold
=====================

Signs of the presence of mold or mildew on the ceiling

URI:
`nmdc:ceil_water_mold <https://microbiomedata/meta/ceil_water_mold>`__

.. _domain-and-range-119:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-156:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-126:

Children
--------

.. _used-by-119:

Used by
-------

.. _other-properties-136:

Other properties
----------------

============= ===========================
**Aliases:**   ceiling signs of water/mold
**Mappings:**  mixs:ceil_water_mold
============= ===========================

Slot: chem_administration
=========================

“List of chemical compounds administered to the host or site where
sampling occurred, and when (e.g. Antibiotics, n fertilizer, air
filter); can include multiple compounds. For chemical entities of
biological interest ontology (chebi) (v 163),
http://purl.bioontology.org/ontology/chebi”

URI:
`nmdc:chem_administration <https://microbiomedata/meta/chem_administration>`__

.. _domain-and-range-120:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-157:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-127:

Children
--------

.. _used-by-120:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-137:

Other properties
----------------

============= ========================
**Aliases:**   chemical administration
**Mappings:**  mixs:chem_administration
============= ========================

Slot: chem_mutagen
==================

“Treatment involving use of mutagens; should include the name of
mutagen, amount administered, treatment regimen including how many times
the treatment was repeated, how long each treatment lasted, and the
start and end time of the entire treatment; can include multiple mutagen
regimens”

URI: `nmdc:chem_mutagen <https://microbiomedata/meta/chem_mutagen>`__

.. _domain-and-range-121:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-158:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-128:

Children
--------

.. _used-by-121:

Used by
-------

.. _other-properties-138:

Other properties
----------------

============= =================
**Aliases:**   chemical mutagen
**Mappings:**  mixs:chem_mutagen
============= =================

Slot: chem_oxygen_dem
=====================

A measure of the capacity of water to consume oxygen during the
decomposition of organic matter and the oxidation of inorganic chemicals
such as ammonia and nitrite

URI:
`nmdc:chem_oxygen_dem <https://microbiomedata/meta/chem_oxygen_dem>`__

.. _domain-and-range-122:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-159:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-129:

Children
--------

.. _used-by-122:

Used by
-------

.. _other-properties-139:

Other properties
----------------

============= ======================
**Aliases:**   chemical oxygen demand
**Mappings:**  mixs:chem_oxygen_dem
============= ======================

Slot: chem_treatment
====================

“List of chemical compounds administered upstream the sampling location
where sampling occurred (e.g. Glycols, H2S scavenger, corrosion and
scale inhibitors, demulsifiers, and other production chemicals etc.).
The commercial name of the product and name of the supplier should be
provided. The date of administration should also be included”

URI:
`nmdc:chem_treatment <https://microbiomedata/meta/chem_treatment>`__

.. _domain-and-range-123:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-160:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-130:

Children
--------

.. _used-by-123:

Used by
-------

.. _other-properties-140:

Other properties
----------------

============= ===================
**Aliases:**   chemical treatment
**Mappings:**  mixs:chem_treatment
============= ===================

Slot: chem_treatment_method
===========================

“Method of chemical administration(dose, frequency, duration, time
elapsed between administration and sampling) (e.g. 50 mg/l; twice a
week; 1 hr; 0 days)”

URI:
`nmdc:chem_treatment_method <https://microbiomedata/meta/chem_treatment_method>`__

.. _domain-and-range-124:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-161:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-131:

Children
--------

.. _used-by-124:

Used by
-------

.. _other-properties-141:

Other properties
----------------

============= ==========================
**Aliases:**   chemical treatment method
**Mappings:**  mixs:chem_treatment_method
============= ==========================

Slot: chemical
==============

URI: `nmdc:chemical <https://microbiomedata/meta/chemical>`__

.. _domain-and-range-125:

Domain and Range
----------------

None → 0..1 `ChemicalEntity <ChemicalEntity.md>`__

.. _parents-162:

Parents
-------

.. _children-132:

Children
--------

-  `reaction participant➞chemical <reaction_participant_chemical.md>`__

.. _used-by-125:

Used by
-------

Slot: chemical formula
======================

A generic grouping for miolecular formulae and empirican formulae

URI:
https://microbiomedata/schema/workflow_execution_activity/chemical_entity_chemical_formula

.. _domain-and-range-126:

Domain and Range
----------------

`ChemicalEntity <ChemicalEntity.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-163:

Parents
-------

-  is_a: `chemical formula <chemical_formula.md>`__

.. _children-133:

Children
--------

.. _used-by-126:

Used by
-------

-  `ChemicalEntity <ChemicalEntity.md>`__

Slot: inchi
===========

URI:
https://microbiomedata/schema/workflow_execution_activity/chemical_entity_inchi

.. _domain-and-range-127:

Domain and Range
----------------

`ChemicalEntity <ChemicalEntity.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-164:

Parents
-------

-  is_a: `inchi <inchi.md>`__

.. _children-134:

Children
--------

.. _used-by-127:

Used by
-------

-  `ChemicalEntity <ChemicalEntity.md>`__

Slot: inchi key
===============

URI:
https://microbiomedata/schema/workflow_execution_activity/chemical_entity_inchi_key

.. _domain-and-range-128:

Domain and Range
----------------

`ChemicalEntity <ChemicalEntity.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-165:

Parents
-------

-  is_a: `inchi key <inchi_key.md>`__

.. _children-135:

Children
--------

.. _used-by-128:

Used by
-------

-  `ChemicalEntity <ChemicalEntity.md>`__

Slot: smiles
============

A string encoding of a molecular graph, no chiral or isotopic
information. There are usually a large number of valid SMILES which
represent a given structure. For example, CCO, OCC and C(O)C all specify
the structure of ethanol.

URI:
https://microbiomedata/schema/workflow_execution_activity/chemical_entity_smiles

.. _domain-and-range-129:

Domain and Range
----------------

`ChemicalEntity <ChemicalEntity.md>`__ → 0..\*
`String <types/String.md>`__

.. _parents-166:

Parents
-------

-  is_a: `smiles <smiles.md>`__

.. _children-136:

Children
--------

.. _used-by-129:

Used by
-------

-  `ChemicalEntity <ChemicalEntity.md>`__

.. _slot-chemical-formula-1:

Slot: chemical formula
======================

A generic grouping for miolecular formulae and empirican formulae

URI:
https://microbiomedata/schema/workflow_execution_activity/chemical_formula

.. _domain-and-range-130:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-167:

Parents
-------

.. _children-137:

Children
--------

-  `chemical entity➞chemical
   formula <chemical_entity_chemical_formula.md>`__

.. _used-by-130:

Used by
-------

Slot: chimera_check
===================

“A chimeric sequence, or chimera for short, is a sequence comprised of
two or more phylogenetically distinct parent sequences. Chimeras are
usually PCR artifacts thought to occur when a prematurely terminated
amplicon reanneals to a foreign DNA strand and is copied to completion
in the following PCR cycles. The point at which the chimeric sequence
changes from one parent to the next is called the breakpoint or
conversion point”

URI: `nmdc:chimera_check <https://microbiomedata/meta/chimera_check>`__

.. _domain-and-range-131:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-168:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-138:

Children
--------

.. _used-by-131:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-142:

Other properties
----------------

=============== ==================
**Aliases:**     chimera check
**Mappings:**    mixs:chimera_check
**In Subsets:**  sequencing
=============== ==================

Slot: chloride
==============

Concentration of chloride in the sample

URI: `nmdc:chloride <https://microbiomedata/meta/chloride>`__

.. _domain-and-range-132:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-169:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-139:

Children
--------

.. _used-by-132:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-143:

Other properties
----------------

============= =============
**Aliases:**   chloride
**Mappings:**  mixs:chloride
============= =============

Slot: chlorophyll
=================

Concentration of chlorophyll

URI: `nmdc:chlorophyll <https://microbiomedata/meta/chlorophyll>`__

.. _domain-and-range-133:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-170:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-140:

Children
--------

.. _used-by-133:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-144:

Other properties
----------------

============= ================
**Aliases:**   chlorophyll
**Mappings:**  mixs:chlorophyll
============= ================

Slot: climate_environment
=========================

“Treatment involving an exposure to a particular climate; treatment
regimen including how many times the treatment was repeated, how long
each treatment lasted, and the start and end time of the entire
treatment; can include multiple climates”

URI:
`nmdc:climate_environment <https://microbiomedata/meta/climate_environment>`__

.. _domain-and-range-134:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-171:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-141:

Children
--------

.. _used-by-134:

Used by
-------

.. _other-properties-145:

Other properties
----------------

============= ========================
**Aliases:**   climate environment
**Mappings:**  mixs:climate_environment
============= ========================

Slot: collection_date
=====================

“The time of sampling, either as an instance (single point in time) or
interval. In case no exact time is available, the date/time can be right
truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00;
2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008
all are ISO8601 compliant”

URI:
`nmdc:collection_date <https://microbiomedata/meta/collection_date>`__

.. _domain-and-range-135:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-172:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-142:

Children
--------

.. _used-by-135:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-146:

Other properties
----------------

=============== ====================
**Aliases:**     collection date
**Mappings:**    mixs:collection_date
**In Subsets:**  environment
=============== ====================

Slot: community
===============

URI: `nmdc:community <https://microbiomedata/meta/community>`__

.. _domain-and-range-136:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-173:

Parents
-------

.. _children-143:

Children
--------

.. _used-by-136:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: compl_appr
================

“The approach used to determine the completeness of a given SAG or MAG,
which would typically make use of a set of conserved marker genes or a
closely related reference genome. For UViG completeness, include
reference genome or group used, and contig feature suggesting a complete
genome”

URI: `nmdc:compl_appr <https://microbiomedata/meta/compl_appr>`__

.. _domain-and-range-137:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-174:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-144:

Children
--------

.. _used-by-137:

Used by
-------

.. _other-properties-147:

Other properties
----------------

=============== =====================
**Aliases:**     completeness approach
**Mappings:**    mixs:compl_appr
**In Subsets:**  sequencing
=============== =====================

Slot: compl_score
=================

“Completeness score is typically based on either the fraction of markers
found as compared to a database or the percent of a genome found as
compared to a closely related reference genome. High Quality Draft:
>90%, Medium Quality Draft: >50%, and Low Quality Draft: < 50% should
have the indicated completeness scores”

URI: `nmdc:compl_score <https://microbiomedata/meta/compl_score>`__

.. _domain-and-range-138:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-175:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-145:

Children
--------

.. _used-by-138:

Used by
-------

.. _other-properties-148:

Other properties
----------------

=============== ==================
**Aliases:**     completeness score
**Mappings:**    mixs:compl_score
**In Subsets:**  sequencing
=============== ==================

Slot: compl_software
====================

“Tools used for completion estimate, i.e. checkm, anvi’o, busco”

URI:
`nmdc:compl_software <https://microbiomedata/meta/compl_software>`__

.. _domain-and-range-139:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-176:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-146:

Children
--------

.. _used-by-139:

Used by
-------

.. _other-properties-149:

Other properties
----------------

=============== =====================
**Aliases:**     completeness software
**Mappings:**    mixs:compl_software
**In Subsets:**  sequencing
=============== =====================

Slot: completion_date
=====================

URI:
`nmdc:completion_date <https://microbiomedata/meta/completion_date>`__

.. _domain-and-range-140:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-177:

Parents
-------

.. _children-147:

Children
--------

.. _used-by-140:

Used by
-------

Slot: compression type
======================

If provided, specifies the compression type

URI:
`nmdc:compression_type <https://microbiomedata/meta/compression_type>`__

.. _domain-and-range-141:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-178:

Parents
-------

.. _children-148:

Children
--------

.. _used-by-141:

Used by
-------

-  `DataObject <DataObject.md>`__

.. _other-properties-150:

Other properties
----------------

============= =======================================
**Examples:**  Example(value=‘gzip’, description=None)
============= =======================================

Slot: conduc
============

Electrical conductivity of water

URI: `nmdc:conduc <https://microbiomedata/meta/conduc>`__

.. _domain-and-range-142:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-179:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-149:

Children
--------

.. _used-by-142:

Used by
-------

.. _other-properties-151:

Other properties
----------------

============= ============
**Aliases:**   conductivity
**Mappings:**  mixs:conduc
============= ============

Slot: contam_score
==================

“The contamination score is based on the fraction of single-copy genes
that are observed more than once in a query genome. The following scores
are acceptable for; High Quality Draft: < 5%, Medium Quality Draft: <
10%, Low Quality Draft: < 10%. Contamination must be below 5% for a SAG
or MAG to be deposited into any of the public databases”

URI: `nmdc:contam_score <https://microbiomedata/meta/contam_score>`__

.. _domain-and-range-143:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-180:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-150:

Children
--------

.. _used-by-143:

Used by
-------

.. _other-properties-152:

Other properties
----------------

=============== ===================
**Aliases:**     contamination score
**Mappings:**    mixs:contam_score
**In Subsets:**  sequencing
=============== ===================

Slot: contam_screen_input
=========================

The type of sequence data used as input

URI:
`nmdc:contam_screen_input <https://microbiomedata/meta/contam_screen_input>`__

.. _domain-and-range-144:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-181:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-151:

Children
--------

.. _used-by-144:

Used by
-------

.. _other-properties-153:

Other properties
----------------

=============== =============================
**Aliases:**     contamination screening input
**Mappings:**    mixs:contam_screen_input
**In Subsets:**  sequencing
=============== =============================

Slot: contam_screen_param
=========================

“Specific parameters used in the decontamination sofware, such as
reference database, coverage, and kmers. Combinations of these
parameters may also be used, i.e. kmer and coverage, or reference
database and kmer”

URI:
`nmdc:contam_screen_param <https://microbiomedata/meta/contam_screen_param>`__

.. _domain-and-range-145:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-182:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-152:

Children
--------

.. _used-by-145:

Used by
-------

.. _other-properties-154:

Other properties
----------------

=============== ==================================
**Aliases:**     contamination screening parameters
**Mappings:**    mixs:contam_screen_param
**In Subsets:**  sequencing
=============== ==================================

Slot: contig_bp
===============

Total size in bp of all contigs.

URI: https://microbiomedata/schema/workflow_execution_activity/contig_bp

.. _domain-and-range-146:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-183:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-153:

Children
--------

.. _used-by-146:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: contigs
=============

The sum of the (length*log(length)) of all contigs, times some constant.
Increase the contiguity, the score will increase

URI: https://microbiomedata/schema/workflow_execution_activity/contigs

.. _domain-and-range-147:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-184:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-154:

Children
--------

.. _used-by-147:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: cool_syst_id
==================

The cooling system identifier

URI: `nmdc:cool_syst_id <https://microbiomedata/meta/cool_syst_id>`__

.. _domain-and-range-148:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-185:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-155:

Children
--------

.. _used-by-148:

Used by
-------

.. _other-properties-155:

Other properties
----------------

============= =========================
**Aliases:**   cooling system identifier
**Mappings:**  mixs:cool_syst_id
============= =========================

Enum: credit enum
=================

URI: `nmdc:credit_enum <https://microbiomedata/meta/credit_enum>`__

.. _other-properties-156:

Other properties
----------------

Permissible Values
------------------

+-----------------+-----------------+-------------+-----------------+
| Text            | Description     | Meaning     | Other           |
|                 |                 |             | Information     |
+=================+=================+=============+=================+
| Co              | Co              |             |                 |
| nceptualization | nceptualization |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Data curation   | Data curation   |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Formal Analysis | Formal Analysis |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Funding         | Funding         |             |                 |
| acquisition     | acquisition     |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Investigation   | Investigation   |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Methodology     | Methodology     |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Project         | Project         |             |                 |
| administration  | administration  |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Resources       | Resources       |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Software        | Software        |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Supervision     | Supervision     |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Validation      | Validation      |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Visualization   | Visualization   |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Writing         | Writing –       |             |                 |
| original draft  | original draft  |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Writing review  | Writing –       |             |                 |
| and editing     | review &        |             |                 |
|                 | editing         |             |                 |
+-----------------+-----------------+-------------+-----------------+
| Principal       | principal       | OBI:0000103 |                 |
| Investigator    | investigator    |             |                 |
|                 | role            |             |                 |
+-----------------+-----------------+-------------+-----------------+

Slot: crop_rotation
===================

“Whether or not crop is rotated, and if yes, rotation schedule”

URI: `nmdc:crop_rotation <https://microbiomedata/meta/crop_rotation>`__

.. _domain-and-range-149:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-186:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-156:

Children
--------

.. _used-by-149:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-157:

Other properties
----------------

============= =====================
**Aliases:**   history/crop rotation
**Mappings:**  mixs:crop_rotation
============= =====================

Slot: ctg_L50
=============

Given a set of contigs, the L50 is defined as the sequence length of the
shortest contig at 50% of the total genome length.

URI: https://microbiomedata/schema/workflow_execution_activity/ctg_L50

.. _domain-and-range-150:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-187:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-157:

Children
--------

.. _used-by-150:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: ctg_L90
=============

The L90 statistic is less than or equal to the L50 statistic; it is the
length for which the collection of all contigs of that length or longer
contains at least 90% of the sum of the lengths of all contigs.

URI: https://microbiomedata/schema/workflow_execution_activity/ctg_L90

.. _domain-and-range-151:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-188:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-158:

Children
--------

.. _used-by-151:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: ctg_N50
=============

Given a set of contigs, each with its own length, the N50 count is
defined as the smallest number of contigs whose length sum makes up half
of genome size.

URI: https://microbiomedata/schema/workflow_execution_activity/ctg_N50

.. _domain-and-range-152:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-189:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-159:

Children
--------

.. _used-by-152:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: ctg_N90
=============

Given a set of contigs, each with its own length, the N90 count is
defined as the smallest number of contigs whose length sum makes up 90%
of genome size.

URI: https://microbiomedata/schema/workflow_execution_activity/ctg_N90

.. _domain-and-range-153:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-190:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-160:

Children
--------

.. _used-by-153:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: ctg_logsum
================

Maximum contig length.

URI:
https://microbiomedata/schema/workflow_execution_activity/ctg_logsum

.. _domain-and-range-154:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-191:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-161:

Children
--------

.. _used-by-154:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: ctg_max
=============

Maximum contig length.

URI: https://microbiomedata/schema/workflow_execution_activity/ctg_max

.. _domain-and-range-155:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-192:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-162:

Children
--------

.. _used-by-155:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: ctg_powsum
================

Powersum of all contigs is the same as logsum except that it uses the
sum of (length*(length^P)) for some power P (default P=0.25).

URI:
https://microbiomedata/schema/workflow_execution_activity/ctg_powsum

.. _domain-and-range-156:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-193:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-163:

Children
--------

.. _used-by-156:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: cult_root_med
===================

“Name or reference for the hydroponic or in vitro culture rooting
medium; can be the name of a commonly used medium or reference to a
specific medium, e.g. Murashige and Skoog medium. If the medium has not
been formally published, use the rooting medium descriptors.”

URI: `nmdc:cult_root_med <https://microbiomedata/meta/cult_root_med>`__

.. _domain-and-range-157:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-194:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-164:

Children
--------

.. _used-by-157:

Used by
-------

.. _other-properties-158:

Other properties
----------------

============= ======================
**Aliases:**   culture rooting medium
**Mappings:**  mixs:cult_root_med
============= ======================

Slot: cur_land_use
==================

Present state of sample site

URI: `nmdc:cur_land_use <https://microbiomedata/meta/cur_land_use>`__

.. _domain-and-range-158:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-195:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-165:

Children
--------

.. _used-by-158:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-159:

Other properties
----------------

============= =================
**Aliases:**   current land use
**Mappings:**  mixs:cur_land_use
============= =================

Slot: cur_vegetation
====================

“Vegetation classification from one or more standard classification
systems, or agricultural crop”

URI:
`nmdc:cur_vegetation <https://microbiomedata/meta/cur_vegetation>`__

.. _domain-and-range-159:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-196:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-166:

Children
--------

.. _used-by-159:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-160:

Other properties
----------------

============= ===================
**Aliases:**   current vegetation
**Mappings:**  mixs:cur_vegetation
============= ===================

Slot: cur_vegetation_meth
=========================

Reference or method used in vegetation classification

URI:
`nmdc:cur_vegetation_meth <https://microbiomedata/meta/cur_vegetation_meth>`__

.. _domain-and-range-160:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-197:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-167:

Children
--------

.. _used-by-160:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-161:

Other properties
----------------

============= =========================
**Aliases:**   current vegetation method
**Mappings:**  mixs:cur_vegetation_meth
============= =========================

Slot: description
=================

a human-readable description of a thing

URI:
`nmdc:data_object_description <https://microbiomedata/meta/data_object_description>`__

.. _domain-and-range-161:

Domain and Range
----------------

`DataObject <DataObject.md>`__ → 1..1 `String <types/String.md>`__

.. _parents-198:

Parents
-------

-  is_a: `description <description.md>`__

.. _children-168:

Children
--------

.. _used-by-161:

Used by
-------

-  `DataObject <DataObject.md>`__

.. _other-properties-162:

Other properties
----------------

============= ===================
**Mappings:**  dcterms:description
============= ===================

Slot: name
==========

A human readable label for an entity

URI:
`nmdc:data_object_name <https://microbiomedata/meta/data_object_name>`__

.. _domain-and-range-162:

Domain and Range
----------------

`DataObject <DataObject.md>`__ → 1..1 `String <types/String.md>`__

.. _parents-199:

Parents
-------

-  is_a: `name <name.md>`__

.. _children-169:

Children
--------

.. _used-by-162:

Used by
-------

-  `DataObject <DataObject.md>`__

Slot: data object set
=====================

This property links a database object to the set of data objects within
it.

URI:
`nmdc:data_object_set <https://microbiomedata/meta/data_object_set>`__

.. _domain-and-range-163:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\* `DataObject <DataObject.md>`__

.. _parents-200:

Parents
-------

.. _children-170:

Children
--------

.. _used-by-163:

Used by
-------

-  `Database <Database.md>`__

Slot: data object type
======================

The type of file represented by the data object.

URI:
`nmdc:data_object_type <https://microbiomedata/meta/data_object_type>`__

.. _domain-and-range-164:

Domain and Range
----------------

None → 0..1 `file type enum <file%20type%20enum.md>`__

.. _parents-201:

Parents
-------

.. _children-171:

Children
--------

.. _used-by-164:

Used by
-------

-  `DataObject <DataObject.md>`__

.. _other-properties-163:

Other properties
----------------

+---------------+---+------------------------------------------------+
| **Examples:** |   | Example(value=‘FT ICR-MS Analysis Results’,    |
|               |   | description=None)                              |
+---------------+---+------------------------------------------------+
|               |   | Example(value=‘GC-MS Metabolomics Results’,    |
|               |   | description=None)                              |
+---------------+---+------------------------------------------------+

Slot: date created
==================

TODO

URI:
`nmdc:database_date_created <https://microbiomedata/meta/database_date_created>`__

.. _domain-and-range-165:

Domain and Range
----------------

`Database <Database.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-202:

Parents
-------

-  is_a: `date created <date_created.md>`__

.. _children-172:

Children
--------

.. _used-by-165:

Used by
-------

-  `Database <Database.md>`__

Slot: etl software version
==========================

TODO

URI:
`nmdc:database_etl_software_version <https://microbiomedata/meta/database_etl_software_version>`__

.. _domain-and-range-166:

Domain and Range
----------------

`Database <Database.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-203:

Parents
-------

-  is_a: `etl software version <etl_software_version.md>`__

.. _children-173:

Children
--------

.. _used-by-166:

Used by
-------

-  `Database <Database.md>`__

Slot: nmdc schema version
=========================

TODO

URI:
`nmdc:database_nmdc_schema_version <https://microbiomedata/meta/database_nmdc_schema_version>`__

.. _domain-and-range-167:

Domain and Range
----------------

`Database <Database.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-204:

Parents
-------

-  is_a: `nmdc schema version <nmdc_schema_version.md>`__

.. _children-174:

Children
--------

.. _used-by-167:

Used by
-------

-  `Database <Database.md>`__

.. _slot-date-created-1:

Slot: date created
==================

TODO

URI: `nmdc:date_created <https://microbiomedata/meta/date_created>`__

.. _domain-and-range-168:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-205:

Parents
-------

.. _children-175:

Children
--------

-  `database➞date created <database_date_created.md>`__

.. _used-by-168:

Used by
-------

Slot: date_last_rain
====================

The date of the last time it rained

URI:
`nmdc:date_last_rain <https://microbiomedata/meta/date_last_rain>`__

.. _domain-and-range-169:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-206:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-176:

Children
--------

.. _used-by-169:

Used by
-------

.. _other-properties-164:

Other properties
----------------

============= ===================
**Aliases:**   date last rain
**Mappings:**  mixs:date_last_rain
============= ===================

Slot: decontam_software
=======================

Tool(s) used in contamination screening

URI:
`nmdc:decontam_software <https://microbiomedata/meta/decontam_software>`__

.. _domain-and-range-170:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-207:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-177:

Children
--------

.. _used-by-170:

Used by
-------

.. _other-properties-165:

Other properties
----------------

=============== ========================
**Aliases:**     decontamination software
**Mappings:**    mixs:decontam_software
**In Subsets:**  sequencing
=============== ========================

Slot: density
=============

“Density of the sample, which is its mass per unit volume (aka
volumetric mass density)”

URI: `nmdc:density <https://microbiomedata/meta/density>`__

.. _domain-and-range-171:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-208:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-178:

Children
--------

.. _used-by-171:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-166:

Other properties
----------------

============= ============
**Aliases:**   density
**Mappings:**  mixs:density
============= ============

Slot: depos_env
===============

“Main depositional environment
(https://en.wikipedia.org/wiki/Depositional_environment). If”“other”” is
specified, please propose entry in ““additional info”” field”

URI: `nmdc:depos_env <https://microbiomedata/meta/depos_env>`__

.. _domain-and-range-172:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-209:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-179:

Children
--------

.. _used-by-172:

Used by
-------

.. _other-properties-167:

Other properties
----------------

============= ========================
**Aliases:**   depositional environment
**Mappings:**  mixs:depos_env
============= ========================

Slot: depth
===========

Please refer to the definitions of depth in the environmental packages

URI: `nmdc:depth <https://microbiomedata/meta/depth>`__

.. _domain-and-range-173:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-210:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-180:

Children
--------

.. _used-by-173:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-168:

Other properties
----------------

=============== ===========================
**Aliases:**     geographic location (depth)
**Mappings:**    mixs:depth
**In Subsets:**  environment
=============== ===========================

Slot: depth2
============

URI: `nmdc:depth2 <https://microbiomedata/meta/depth2>`__

.. _domain-and-range-174:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-211:

Parents
-------

.. _children-181:

Children
--------

.. _used-by-174:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: dermatology_disord
========================

History of dermatology disorders; can include multiple disorders

URI:
`nmdc:dermatology_disord <https://microbiomedata/meta/dermatology_disord>`__

.. _domain-and-range-175:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-212:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-182:

Children
--------

.. _used-by-175:

Used by
-------

.. _other-properties-169:

Other properties
----------------

============= =======================
**Aliases:**   dermatology disorder
**Mappings:**  mixs:dermatology_disord
============= =======================

.. _slot-description-1:

Slot: description
=================

a human-readable description of a thing

URI:
https://microbiomedata/schema/workflow_execution_activity/description

.. _domain-and-range-176:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-213:

Parents
-------

.. _children-183:

Children
--------

.. _used-by-176:

Used by
-------

-  `ChemicalEntity <ChemicalEntity.md>`__
-  `EnvironmentalMaterialTerm <EnvironmentalMaterialTerm.md>`__
-  `GeneProduct <GeneProduct.md>`__
-  `ImageValue <ImageValue.md>`__
-  `Instrument <Instrument.md>`__
-  `NamedThing <NamedThing.md>`__
-  `OntologyClass <OntologyClass.md>`__
-  `Person <Person.md>`__

.. _other-properties-170:

Other properties
----------------

============= ===================
**Mappings:**  dcterms:description
============= ===================

Slot: detec_type
================

Type of UViG detection

URI: `nmdc:detec_type <https://microbiomedata/meta/detec_type>`__

.. _domain-and-range-177:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-214:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-184:

Children
--------

.. _used-by-177:

Used by
-------

.. _other-properties-171:

Other properties
----------------

=============== ===============
**Aliases:**     detection type
**Mappings:**    mixs:detec_type
**In Subsets:**  sequencing
=============== ===============

Slot: dew_point
===============

“The temperature to which a given parcel of humid air must be cooled, at
constant barometric pressure, for water vapor to condense into water.”

URI: `nmdc:dew_point <https://microbiomedata/meta/dew_point>`__

.. _domain-and-range-178:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-215:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-185:

Children
--------

.. _used-by-178:

Used by
-------

.. _other-properties-172:

Other properties
----------------

============= ==============
**Aliases:**   dew point
**Mappings:**  mixs:dew_point
============= ==============

Slot: diet_last_six_month
=========================

“Specification of major diet changes in the last six months, if yes the
change should be specified”

URI:
`nmdc:diet_last_six_month <https://microbiomedata/meta/diet_last_six_month>`__

.. _domain-and-range-179:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-216:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-186:

Children
--------

.. _used-by-179:

Used by
-------

.. _other-properties-173:

Other properties
----------------

============= ====================================
**Aliases:**   major diet change in last six months
**Mappings:**  mixs:diet_last_six_month
============= ====================================

Slot: diether_lipids
====================

Concentration of diether lipids; can include multiple types of diether
lipids

URI:
`nmdc:diether_lipids <https://microbiomedata/meta/diether_lipids>`__

.. _domain-and-range-180:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-217:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-187:

Children
--------

.. _used-by-180:

Used by
-------

.. _other-properties-174:

Other properties
----------------

============= ===================
**Aliases:**   diether lipids
**Mappings:**  mixs:diether_lipids
============= ===================

Slot: direction
===============

One of l->r, r->l, bidirectional, neutral

URI: `nmdc:direction <https://microbiomedata/meta/direction>`__

.. _domain-and-range-181:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-218:

Parents
-------

.. _children-188:

Children
--------

-  `reaction➞direction <reaction_direction.md>`__

.. _used-by-181:

Used by
-------

Slot: display order
===================

When rendering information, this attribute to specify the order in which
the information should be rendered.

URI:
https://microbiomedata/schema/workflow_execution_activity/display_order

.. _domain-and-range-182:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-219:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-189:

Children
--------

.. _used-by-182:

Used by
-------

-  `ImageValue <ImageValue.md>`__

Slot: diss_carb_dioxide
=======================

Concentration of dissolved carbon dioxide in the sample or liquid
portion of the sample

URI:
`nmdc:diss_carb_dioxide <https://microbiomedata/meta/diss_carb_dioxide>`__

.. _domain-and-range-183:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-220:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-190:

Children
--------

.. _used-by-183:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-175:

Other properties
----------------

============= ========================
**Aliases:**   dissolved carbon dioxide
**Mappings:**  mixs:diss_carb_dioxide
============= ========================

Slot: diss_hydrogen
===================

Concentration of dissolved hydrogen

URI: `nmdc:diss_hydrogen <https://microbiomedata/meta/diss_hydrogen>`__

.. _domain-and-range-184:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-221:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-191:

Children
--------

.. _used-by-184:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-176:

Other properties
----------------

============= ==================
**Aliases:**   dissolved hydrogen
**Mappings:**  mixs:diss_hydrogen
============= ==================

Slot: diss_inorg_carb
=====================

“Dissolved inorganic carbon concentration in the sample, typically
measured after filtering the sample using a 0.45 micrometer filter”

URI:
`nmdc:diss_inorg_carb <https://microbiomedata/meta/diss_inorg_carb>`__

.. _domain-and-range-185:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-222:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-192:

Children
--------

.. _used-by-185:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-177:

Other properties
----------------

============= ==========================
**Aliases:**   dissolved inorganic carbon
**Mappings:**  mixs:diss_inorg_carb
============= ==========================

Slot: diss_inorg_nitro
======================

Concentration of dissolved inorganic nitrogen

URI:
`nmdc:diss_inorg_nitro <https://microbiomedata/meta/diss_inorg_nitro>`__

.. _domain-and-range-186:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-223:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-193:

Children
--------

.. _used-by-186:

Used by
-------

.. _other-properties-178:

Other properties
----------------

============= ============================
**Aliases:**   dissolved inorganic nitrogen
**Mappings:**  mixs:diss_inorg_nitro
============= ============================

Slot: diss_inorg_phosp
======================

Concentration of dissolved inorganic phosphorus in the sample

URI:
`nmdc:diss_inorg_phosp <https://microbiomedata/meta/diss_inorg_phosp>`__

.. _domain-and-range-187:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-224:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-194:

Children
--------

.. _used-by-187:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-179:

Other properties
----------------

============= ==============================
**Aliases:**   dissolved inorganic phosphorus
**Mappings:**  mixs:diss_inorg_phosp
============= ==============================

Slot: diss_iron
===============

Concentration of dissolved iron in the sample

URI: `nmdc:diss_iron <https://microbiomedata/meta/diss_iron>`__

.. _domain-and-range-188:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-225:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-195:

Children
--------

.. _used-by-188:

Used by
-------

.. _other-properties-180:

Other properties
----------------

============= ==============
**Aliases:**   dissolved iron
**Mappings:**  mixs:diss_iron
============= ==============

Slot: diss_org_carb
===================

“Concentration of dissolved organic carbon in the sample, liquid portion
of the sample, or aqueous phase of the fluid”

URI: `nmdc:diss_org_carb <https://microbiomedata/meta/diss_org_carb>`__

.. _domain-and-range-189:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-226:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-196:

Children
--------

.. _used-by-189:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-181:

Other properties
----------------

============= ========================
**Aliases:**   dissolved organic carbon
**Mappings:**  mixs:diss_org_carb
============= ========================

Slot: diss_org_nitro
====================

Dissolved organic nitrogen concentration measured as; total dissolved
nitrogen - NH4 - NO3 - NO2

URI:
`nmdc:diss_org_nitro <https://microbiomedata/meta/diss_org_nitro>`__

.. _domain-and-range-190:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-227:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-197:

Children
--------

.. _used-by-190:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-182:

Other properties
----------------

============= ==========================
**Aliases:**   dissolved organic nitrogen
**Mappings:**  mixs:diss_org_nitro
============= ==========================

Slot: diss_oxygen
=================

Concentration of dissolved oxygen

URI: `nmdc:diss_oxygen <https://microbiomedata/meta/diss_oxygen>`__

.. _domain-and-range-191:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-228:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-198:

Children
--------

.. _used-by-191:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-183:

Other properties
----------------

============= ================
**Aliases:**   dissolved oxygen
**Mappings:**  mixs:diss_oxygen
============= ================

Slot: diss_oxygen_fluid
=======================

Concentration of dissolved oxygen in the oil field produced fluids as it
contributes to oxgen-corrosion and microbial activity (e.g. Mic).

URI:
`nmdc:diss_oxygen_fluid <https://microbiomedata/meta/diss_oxygen_fluid>`__

.. _domain-and-range-192:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-229:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-199:

Children
--------

.. _used-by-192:

Used by
-------

.. _other-properties-184:

Other properties
----------------

============= ==========================
**Aliases:**   dissolved oxygen in fluids
**Mappings:**  mixs:diss_oxygen_fluid
============= ==========================

Slot: doi
=========

URI: `nmdc:doi <https://microbiomedata/meta/doi>`__

.. _domain-and-range-193:

Domain and Range
----------------

None → 0..1 `AttributeValue <AttributeValue.md>`__

.. _parents-230:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-200:

Children
--------

-  `study➞doi <study_doi.md>`__

.. _used-by-193:

Used by
-------

Slot: dominant_hand
===================

Dominant hand of the subject

URI: `nmdc:dominant_hand <https://microbiomedata/meta/dominant_hand>`__

.. _domain-and-range-194:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-231:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-201:

Children
--------

.. _used-by-194:

Used by
-------

.. _other-properties-185:

Other properties
----------------

============= ==================
**Aliases:**   dominant hand
**Mappings:**  mixs:dominant_hand
============= ==================

Slot: door_comp_type
====================

The composite type of the door

URI:
`nmdc:door_comp_type <https://microbiomedata/meta/door_comp_type>`__

.. _domain-and-range-195:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-232:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-202:

Children
--------

.. _used-by-195:

Used by
-------

.. _other-properties-186:

Other properties
----------------

============= ====================
**Aliases:**   door type, composite
**Mappings:**  mixs:door_comp_type
============= ====================

Slot: door_cond
===============

The phsical condition of the door

URI: `nmdc:door_cond <https://microbiomedata/meta/door_cond>`__

.. _domain-and-range-196:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-233:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-203:

Children
--------

.. _used-by-196:

Used by
-------

.. _other-properties-187:

Other properties
----------------

============= ==============
**Aliases:**   door condition
**Mappings:**  mixs:door_cond
============= ==============

Slot: door_direct
=================

The direction the door opens

URI: `nmdc:door_direct <https://microbiomedata/meta/door_direct>`__

.. _domain-and-range-197:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-234:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-204:

Children
--------

.. _used-by-197:

Used by
-------

.. _other-properties-188:

Other properties
----------------

============= =========================
**Aliases:**   door direction of opening
**Mappings:**  mixs:door_direct
============= =========================

Slot: door_loc
==============

The relative location of the door in the room

URI: `nmdc:door_loc <https://microbiomedata/meta/door_loc>`__

.. _domain-and-range-198:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-235:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-205:

Children
--------

.. _used-by-198:

Used by
-------

.. _other-properties-189:

Other properties
----------------

============= =============
**Aliases:**   door location
**Mappings:**  mixs:door_loc
============= =============

Slot: door_mat
==============

The material the door is composed of

URI: `nmdc:door_mat <https://microbiomedata/meta/door_mat>`__

.. _domain-and-range-199:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-236:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-206:

Children
--------

.. _used-by-199:

Used by
-------

.. _other-properties-190:

Other properties
----------------

============= =============
**Aliases:**   door material
**Mappings:**  mixs:door_mat
============= =============

Slot: door_move
===============

The type of movement of the door

URI: `nmdc:door_move <https://microbiomedata/meta/door_move>`__

.. _domain-and-range-200:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-237:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-207:

Children
--------

.. _used-by-200:

Used by
-------

.. _other-properties-191:

Other properties
----------------

============= ==============
**Aliases:**   door movement
**Mappings:**  mixs:door_move
============= ==============

Slot: door_size
===============

The size of the door

URI: `nmdc:door_size <https://microbiomedata/meta/door_size>`__

.. _domain-and-range-201:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-238:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-208:

Children
--------

.. _used-by-201:

Used by
-------

.. _other-properties-192:

Other properties
----------------

============= =================
**Aliases:**   door area or size
**Mappings:**  mixs:door_size
============= =================

Slot: door_type
===============

The type of door material

URI: `nmdc:door_type <https://microbiomedata/meta/door_type>`__

.. _domain-and-range-202:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-239:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-209:

Children
--------

.. _used-by-202:

Used by
-------

.. _other-properties-193:

Other properties
----------------

============= ==============
**Aliases:**   door type
**Mappings:**  mixs:door_type
============= ==============

Slot: door_type_metal
=====================

The type of metal door

URI:
`nmdc:door_type_metal <https://microbiomedata/meta/door_type_metal>`__

.. _domain-and-range-203:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-240:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-210:

Children
--------

.. _used-by-203:

Used by
-------

.. _other-properties-194:

Other properties
----------------

============= ====================
**Aliases:**   door type, metal
**Mappings:**  mixs:door_type_metal
============= ====================

Slot: door_type_wood
====================

The type of wood door

URI:
`nmdc:door_type_wood <https://microbiomedata/meta/door_type_wood>`__

.. _domain-and-range-204:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-241:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-211:

Children
--------

.. _used-by-204:

Used by
-------

.. _other-properties-195:

Other properties
----------------

============= ===================
**Aliases:**   door type, wood
**Mappings:**  mixs:door_type_wood
============= ===================

Slot: door_water_mold
=====================

Signs of the presence of mold or mildew on a door

URI:
`nmdc:door_water_mold <https://microbiomedata/meta/door_water_mold>`__

.. _domain-and-range-205:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-242:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-212:

Children
--------

.. _used-by-205:

Used by
-------

.. _other-properties-196:

Other properties
----------------

============= ========================
**Aliases:**   door signs of water/mold
**Mappings:**  mixs:door_water_mold
============= ========================

Slot: douche
============

Date of most recent douche

URI: `nmdc:douche <https://microbiomedata/meta/douche>`__

.. _domain-and-range-206:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-243:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-213:

Children
--------

.. _used-by-206:

Used by
-------

.. _other-properties-197:

Other properties
----------------

============= ===========
**Aliases:**   douche
**Mappings:**  mixs:douche
============= ===========

Slot: down_par
==============

Visible waveband radiance and irradiance measurements in the water
column

URI: `nmdc:down_par <https://microbiomedata/meta/down_par>`__

.. _domain-and-range-207:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-244:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-214:

Children
--------

.. _used-by-207:

Used by
-------

.. _other-properties-198:

Other properties
----------------

============= =============
**Aliases:**   downward PAR
**Mappings:**  mixs:down_par
============= =============

Slot: drainage_class
====================

Drainage classification from a standard system such as the USDA system

URI:
`nmdc:drainage_class <https://microbiomedata/meta/drainage_class>`__

.. _domain-and-range-208:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-245:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-215:

Children
--------

.. _used-by-208:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-199:

Other properties
----------------

============= =======================
**Aliases:**   drainage classification
**Mappings:**  mixs:drainage_class
============= =======================

Slot: drawings
==============

“The buildings architectural drawings; if design is chosen, indicate
phase-conceptual, schematic, design development, and construction
documents”

URI: `nmdc:drawings <https://microbiomedata/meta/drawings>`__

.. _domain-and-range-209:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-246:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-216:

Children
--------

.. _used-by-209:

Used by
-------

.. _other-properties-200:

Other properties
----------------

============= =============
**Aliases:**   drawings
**Mappings:**  mixs:drawings
============= =============

Slot: drug_usage
================

Any drug used by subject and the frequency of usage; can include
multiple drugs used

URI: `nmdc:drug_usage <https://microbiomedata/meta/drug_usage>`__

.. _domain-and-range-210:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-247:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-217:

Children
--------

.. _used-by-210:

Used by
-------

.. _other-properties-201:

Other properties
----------------

============= ===============
**Aliases:**   drug usage
**Mappings:**  mixs:drug_usage
============= ===============

Slot: ecosystem
===============

TODO

URI: `nmdc:ecosystem <https://microbiomedata/meta/ecosystem>`__

.. _domain-and-range-211:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-248:

Parents
-------

-  is_a: `gold_path_field <gold_path_field.md>`__

.. _children-218:

Children
--------

.. _used-by-211:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `Study <Study.md>`__

Slot: ecosystem_category
========================

TODO

URI:
`nmdc:ecosystem_category <https://microbiomedata/meta/ecosystem_category>`__

.. _domain-and-range-212:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-249:

Parents
-------

-  is_a: `gold_path_field <gold_path_field.md>`__

.. _children-219:

Children
--------

.. _used-by-212:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `Study <Study.md>`__

Slot: ecosystem_path_id
=======================

A unique id representing the GOLD classifers associated with a sample.

URI:
`nmdc:ecosystem_path_id <https://microbiomedata/meta/ecosystem_path_id>`__

.. _domain-and-range-213:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-250:

Parents
-------

.. _children-220:

Children
--------

.. _used-by-213:

Used by
-------

Slot: ecosystem_subtype
=======================

TODO

URI:
`nmdc:ecosystem_subtype <https://microbiomedata/meta/ecosystem_subtype>`__

.. _domain-and-range-214:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-251:

Parents
-------

-  is_a: `gold_path_field <gold_path_field.md>`__

.. _children-221:

Children
--------

.. _used-by-214:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `Study <Study.md>`__

Slot: ecosystem_type
====================

TODO

URI:
`nmdc:ecosystem_type <https://microbiomedata/meta/ecosystem_type>`__

.. _domain-and-range-215:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-252:

Parents
-------

-  is_a: `gold_path_field <gold_path_field.md>`__

.. _children-222:

Children
--------

.. _used-by-215:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `Study <Study.md>`__

Slot: efficiency_percent
========================

Percentage of volatile solids removed from the anaerobic digestor

URI:
`nmdc:efficiency_percent <https://microbiomedata/meta/efficiency_percent>`__

.. _domain-and-range-216:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-253:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-223:

Children
--------

.. _used-by-216:

Used by
-------

.. _other-properties-202:

Other properties
----------------

============= =======================
**Aliases:**   efficiency percent
**Mappings:**  mixs:efficiency_percent
============= =======================

Slot: elev
==========

“Elevation of the sampling site is its height above a fixed reference
point, most commonly the mean sea level. Elevation is mainly used when
referring to points on the earth’s surface, while altitude is used for
points above the surface, such as an aircraft in flight or a spacecraft
in orbit”

URI: `nmdc:elev <https://microbiomedata/meta/elev>`__

.. _domain-and-range-217:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-254:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-224:

Children
--------

.. _used-by-217:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-203:

Other properties
----------------

=============== ===========
**Aliases:**     elevation
**Mappings:**    mixs:elev
**In Subsets:**  environment
=============== ===========

Slot: elevator
==============

The number of elevators within the built structure

URI: `nmdc:elevator <https://microbiomedata/meta/elevator>`__

.. _domain-and-range-218:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-255:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-225:

Children
--------

.. _used-by-218:

Used by
-------

.. _other-properties-204:

Other properties
----------------

============= ==============
**Aliases:**   elevator count
**Mappings:**  mixs:elevator
============= ==============

Slot: email
===========

An email address for an entity such as a person. This should be the
primarly email address used.

URI: https://microbiomedata/schema/workflow_execution_activity/email

.. _domain-and-range-219:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-256:

Parents
-------

.. _children-226:

Children
--------

.. _used-by-219:

Used by
-------

-  `PersonValue <PersonValue.md>`__

.. _other-properties-205:

Other properties
----------------

============= ============
**Mappings:**  schema:email
============= ============

Slot: emulsions
===============

“Amount or concentration of substances such as paints, adhesives,
mayonnaise, hair colorants, emulsified oils, etc.; can include multiple
emulsion types”

URI: `nmdc:emulsions <https://microbiomedata/meta/emulsions>`__

.. _domain-and-range-220:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-257:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-227:

Children
--------

.. _used-by-220:

Used by
-------

.. _other-properties-206:

Other properties
----------------

============= ==============
**Aliases:**   emulsions
**Mappings:**  mixs:emulsions
============= ==============

Slot: encoded_traits
====================

“Should include key traits like antibiotic resistance or xenobiotic
degradation phenotypes for plasmids, converting genes for phage”

URI:
`nmdc:encoded_traits <https://microbiomedata/meta/encoded_traits>`__

.. _domain-and-range-221:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-258:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-228:

Children
--------

.. _used-by-221:

Used by
-------

.. _other-properties-207:

Other properties
----------------

=============== ============================
**Aliases:**     encoded traits
**Mappings:**    mixs:encoded_traits
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: encodes
=============

The gene product encoded by this feature. Typically this is used for a
CDS feature or gene feature which will encode a protein. It can also be
used by a nc transcript ot gene feature that encoded a ncRNA

URI: `nmdc:encodes <https://microbiomedata/meta/encodes>`__

.. _domain-and-range-222:

Domain and Range
----------------

None → 0..1 `GeneProduct <GeneProduct.md>`__

.. _parents-259:

Parents
-------

.. _children-229:

Children
--------

-  `genome feature➞encodes <genome_feature_encodes.md>`__

.. _used-by-222:

Used by
-------

Slot: end
=========

The end of the feature in positive 1-based integer coordinates

URI: `nmdc:end <https://microbiomedata/meta/end>`__

.. _domain-and-range-223:

Domain and Range
----------------

None → 1..1 `Integer <types/Integer.md>`__

.. _parents-260:

Parents
-------

-  is_a: `gff coordinate <gff_coordinate.md>`__

.. _children-230:

Children
--------

.. _used-by-223:

Used by
-------

.. _other-properties-208:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | - “constraint: end >  |
|                       |                       | start” - “For         |
|                       |                       | features that cross   |
|                       |                       | the origin of a       |
|                       |                       | circular feature, end |
|                       |                       | = the position of the |
|                       |                       | end + the length of   |
|                       |                       | the landmark          |
|                       |                       | feature.”             |
+-----------------------+-----------------------+-----------------------+
| **Close Mappings:**   |                       | biolink:end           |
|                       |                       | _interbase_coordinate |
+-----------------------+-----------------------+-----------------------+

Slot: ended at time
===================

URI:
https://microbiomedata/schema/workflow_execution_activity/ended_at_time

.. _domain-and-range-224:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-261:

Parents
-------

.. _children-231:

Children
--------

-  `workflow execution activity➞ended at
   time <workflow_execution_activity_ended_at_time.md>`__

.. _used-by-224:

Used by
-------

-  `Activity <Activity.md>`__

.. _other-properties-209:

Other properties
----------------

============= ================
**Mappings:**  prov:endedAtTime
============= ================

.. _slot-env_broad_scale-1:

Slot: env_broad_scale
=====================

“In this field, report which major environmental system your sample or
specimen came from. The systems identified should have a coarse spatial
grain, to provide the general environmental context of where the
sampling was done (e.g. were you in the desert or a rainforest?). We
recommend using subclasses of ENVOUs biome class:
http://purl.obolibrary.org/obo/ENVO_00000428. Format (one term):
termLabel [termID], Format (multiple terms): termLabel
[termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a
water sample from the photic zone in middle of the Atlantic Ocean,
consider: oceanic epipelagic zone biome [ENVO:01000033]. Example:
Annotating a sample from the Amazon rainforest consider: tropical moist
broadleaf forest biome [ENVO:01000228]. If needed, request new terms on
the ENVO tracker, identified here:
http://www.obofoundry.org/ontology/envo.html”

URI:
`nmdc:env_broad_scale <https://microbiomedata/meta/env_broad_scale>`__

.. _domain-and-range-225:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-262:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-232:

Children
--------

-  `biosample➞env_broad_scale <biosample_env_broad_scale.md>`__

.. _used-by-225:

Used by
-------

.. _other-properties-210:

Other properties
----------------

=============== =================================
**Aliases:**     broad-scale environmental context
**Mappings:**    mixs:env_broad_scale
**In Subsets:**  environment
=============== =================================

.. _slot-env_local_scale-1:

Slot: env_local_scale
=====================

“In this field, report the entity or entities which are in your sample
or specimenUs local vicinity and which you believe have significant
causal influences on your sample or specimen. Please use terms that are
present in ENVO and which are of smaller spatial grain than your entry
for env_broad_scale. Format (one term): termLabel [termID]; Format
(multiple terms): termLabel [termID]|termLabel [termID]|termLabel
[termID]. Example: Annotating a pooled sample taken from various
vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and
fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory
[01000335]|shrub layer [ENVO:01000336]. If needed, request new terms on
the ENVO tracker, identified here:
http://www.obofoundry.org/ontology/envo.html”

URI:
`nmdc:env_local_scale <https://microbiomedata/meta/env_local_scale>`__

.. _domain-and-range-226:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-263:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-233:

Children
--------

-  `biosample➞env_local_scale <biosample_env_local_scale.md>`__

.. _used-by-226:

Used by
-------

.. _other-properties-211:

Other properties
----------------

=============== ===========================
**Aliases:**     local environmental context
**Mappings:**    mixs:env_local_scale
**In Subsets:**  environment
=============== ===========================

.. _slot-env_medium-1:

Slot: env_medium
================

“In this field, report which environmental material or materials (pipe
separated) immediately surrounded your sample or specimen prior to
sampling, using one or more subclasses of ENVOUs environmental material
class: http://purl.obolibrary.org/obo/ENVO_00010483. Format (one term):
termLabel [termID]; Format (multiple terms): termLabel
[termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a
fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean
water [ENVO:00002151]. Example: Annotating a duck on a pond consider:
pond water [ENVO:00002228]|air ENVO_00002005. If needed, request new
terms on the ENVO tracker, identified here:
http://www.obofoundry.org/ontology/envo.html”

URI: `nmdc:env_medium <https://microbiomedata/meta/env_medium>`__

.. _domain-and-range-227:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-264:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-234:

Children
--------

-  `biosample➞env_medium <biosample_env_medium.md>`__

.. _used-by-227:

Used by
-------

.. _other-properties-212:

Other properties
----------------

=============== ====================
**Aliases:**     environmental medium
**Mappings:**    mixs:env_medium
**In Subsets:**  environment
=============== ====================

Slot: env_package
=================

“MIxS extension for reporting of measurements and observations obtained
from one or more of the environments where the sample was obtained. All
environmental packages listed here are further defined in separate
subtables. By giving the name of the environmental package, a selection
of fields can be made from the subtables and can be reported”

URI: `nmdc:env_package <https://microbiomedata/meta/env_package>`__

.. _domain-and-range-228:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-265:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-235:

Children
--------

.. _used-by-228:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-213:

Other properties
----------------

=============== =====================
**Aliases:**     environmental package
**Mappings:**    mixs:env_package
**In Subsets:**  mixs extension
=============== =====================

Slot: escalator
===============

The number of escalators within the built structure

URI: `nmdc:escalator <https://microbiomedata/meta/escalator>`__

.. _domain-and-range-229:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-266:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-236:

Children
--------

.. _used-by-229:

Used by
-------

.. _other-properties-214:

Other properties
----------------

============= ===============
**Aliases:**   escalator count
**Mappings:**  mixs:escalator
============= ===============

Slot: ess dive datasets
=======================

List of ESS-DIVE dataset DOIs

URI:
`nmdc:ess_dive_datasets <https://microbiomedata/meta/ess_dive_datasets>`__

.. _domain-and-range-230:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-267:

Parents
-------

.. _children-237:

Children
--------

.. _used-by-230:

Used by
-------

-  `Study <Study.md>`__

Slot: estimated_size
====================

The estimated size of the genome prior to sequencing. Of particular
importance in the sequencing of (eukaryotic) genome which could remain
in draft form for a long or unspecified period.

URI:
`nmdc:estimated_size <https://microbiomedata/meta/estimated_size>`__

.. _domain-and-range-231:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-268:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-238:

Children
--------

.. _used-by-231:

Used by
-------

.. _other-properties-215:

Other properties
----------------

=============== ============================
**Aliases:**     estimated size
**Mappings:**    mixs:estimated_size
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: ethylbenzene
==================

Concentration of ethylbenzene in the sample

URI: `nmdc:ethylbenzene <https://microbiomedata/meta/ethylbenzene>`__

.. _domain-and-range-232:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-269:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-239:

Children
--------

.. _used-by-232:

Used by
-------

.. _other-properties-216:

Other properties
----------------

============= =================
**Aliases:**   ethylbenzene
**Mappings:**  mixs:ethylbenzene
============= =================

.. _slot-etl-software-version-1:

Slot: etl software version
==========================

TODO

URI:
`nmdc:etl_software_version <https://microbiomedata/meta/etl_software_version>`__

.. _domain-and-range-233:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-270:

Parents
-------

.. _children-240:

Children
--------

-  `database➞etl software version <database_etl_software_version.md>`__

.. _used-by-233:

Used by
-------

Slot: execution resource
========================

Example: NERSC-Cori

URI:
https://microbiomedata/schema/workflow_execution_activity/execution_resource

.. _domain-and-range-234:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-271:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-241:

Children
--------

-  `workflow execution activity➞execution
   resource <workflow_execution_activity_execution_resource.md>`__

.. _used-by-234:

Used by
-------

Slot: exp_duct
==============

The amount of exposed ductwork in the room

URI: `nmdc:exp_duct <https://microbiomedata/meta/exp_duct>`__

.. _domain-and-range-235:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-272:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-242:

Children
--------

.. _used-by-235:

Used by
-------

.. _other-properties-217:

Other properties
----------------

============= ================
**Aliases:**   exposed ductwork
**Mappings:**  mixs:exp_duct
============= ================

Slot: exp_pipe
==============

The number of exposed pipes in the room

URI: `nmdc:exp_pipe <https://microbiomedata/meta/exp_pipe>`__

.. _domain-and-range-236:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-273:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-243:

Children
--------

.. _used-by-236:

Used by
-------

.. _other-properties-218:

Other properties
----------------

============= =============
**Aliases:**   exposed pipes
**Mappings:**  mixs:exp_pipe
============= =============

Slot: experimental_factor
=========================

“Experimental factors are essentially the variable aspects of an
experiment design which can be used to describe an experiment, or set of
experiments, in an increasingly detailed manner. This field accepts
ontology terms from Experimental Factor Ontology (EFO) and/or Ontology
for Biomedical Investigations (OBI). For a browser of EFO (v 2.95)
terms, please see http://purl.bioontology.org/ontology/EFO; for a
browser of OBI (v 2018-02-12) terms please see
http://purl.bioontology.org/ontology/OBI”

URI:
`nmdc:experimental_factor <https://microbiomedata/meta/experimental_factor>`__

.. _domain-and-range-237:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-274:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-244:

Children
--------

.. _used-by-237:

Used by
-------

.. _other-properties-219:

Other properties
----------------

=============== ========================
**Aliases:**     experimental factor
**Mappings:**    mixs:experimental_factor
**In Subsets:**  investigation
=============== ========================

Slot: ext_door
==============

The number of exterior doors in the built structure

URI: `nmdc:ext_door <https://microbiomedata/meta/ext_door>`__

.. _domain-and-range-238:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-275:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-245:

Children
--------

.. _used-by-238:

Used by
-------

.. _other-properties-220:

Other properties
----------------

============= ===================
**Aliases:**   exterior door count
**Mappings:**  mixs:ext_door
============= ===================

Slot: ext_wall_orient
=====================

The orientation of the exterior wall

URI:
`nmdc:ext_wall_orient <https://microbiomedata/meta/ext_wall_orient>`__

.. _domain-and-range-239:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-276:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-246:

Children
--------

.. _used-by-239:

Used by
-------

.. _other-properties-221:

Other properties
----------------

============= =============================
**Aliases:**   orientations of exterior wall
**Mappings:**  mixs:ext_wall_orient
============= =============================

Slot: ext_window_orient
=======================

The compass direction the exterior window of the room is facing

URI:
`nmdc:ext_window_orient <https://microbiomedata/meta/ext_window_orient>`__

.. _domain-and-range-240:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-277:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-247:

Children
--------

.. _used-by-240:

Used by
-------

.. _other-properties-222:

Other properties
----------------

============= ===============================
**Aliases:**   orientations of exterior window
**Mappings:**  mixs:ext_window_orient
============= ===============================

Slot: external database identifiers
===================================

Link to corresponding identifier in external database

URI:
https://microbiomedata/schema/workflow_execution_activity/external_database_identifiers

.. _domain-and-range-241:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-278:

Parents
-------

-  is_a: `alternative identifiers <alternative_identifiers.md>`__

.. _children-248:

Children
--------

-  `GOLD sequencing project
   identifiers <GOLD_sequencing_project_identifiers.md>`__
-  `INSDC experiment identifiers <INSDC_experiment_identifiers.md>`__
-  `analysis identifiers <analysis_identifiers.md>`__
-  `omics processing identifiers <omics_processing_identifiers.md>`__
-  `sample identifiers <sample_identifiers.md>`__
-  `study identifiers <study_identifiers.md>`__

.. _used-by-241:

Used by
-------

.. _other-properties-223:

Other properties
----------------

+---------------------+---+------------------------------------------------------+
| **Comments:**       |   | The value of this field is always a registered CURIE |
+---------------------+---+------------------------------------------------------+
| **Close Mappings:** |   | skos:closeMatch                                      |
+---------------------+---+------------------------------------------------------+

Slot: extrachrom_elements
=========================

Do plasmids exist of significant phenotypic consequence (e.g. ones that
determine virulence or antibiotic resistance). Megaplasmids? Other
plasmids (borrelia has 15+ plasmids)

URI:
`nmdc:extrachrom_elements <https://microbiomedata/meta/extrachrom_elements>`__

.. _domain-and-range-242:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-279:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-249:

Children
--------

.. _used-by-242:

Used by
-------

.. _other-properties-224:

Other properties
----------------

=============== ============================
**Aliases:**     extrachromosomal elements
**Mappings:**    mixs:extrachrom_elements
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: extreme_event
===================

Unusual physical events that may have affected microbial populations

URI: `nmdc:extreme_event <https://microbiomedata/meta/extreme_event>`__

.. _domain-and-range-243:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-280:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-250:

Children
--------

.. _used-by-243:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-225:

Other properties
----------------

============= ======================
**Aliases:**   history/extreme events
**Mappings:**  mixs:extreme_event
============= ======================

Slot: extreme_salinity
======================

Measured salinity

URI:
`nmdc:extreme_salinity <https://microbiomedata/meta/extreme_salinity>`__

.. _domain-and-range-244:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-281:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-251:

Children
--------

.. _used-by-244:

Used by
-------

.. _other-properties-226:

Other properties
----------------

============= ===================================
**Aliases:**   extreme_unusual_properties/salinity
**Mappings:**  mixs:extreme_salinity
============= ===================================

Slot: fao_class
===============

Soil classification from the FAO World Reference Database for Soil
Resources. The list can be found at
http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups

URI: `nmdc:fao_class <https://microbiomedata/meta/fao_class>`__

.. _domain-and-range-245:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-282:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-252:

Children
--------

.. _used-by-245:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-227:

Other properties
----------------

============= =================================
**Aliases:**   soil_taxonomic/FAO classification
**Mappings:**  mixs:fao_class
============= =================================

Slot: feat_pred
===============

“Method used to predict UViGs features such as ORFs, integration site,
etc.”

URI: `nmdc:feat_pred <https://microbiomedata/meta/feat_pred>`__

.. _domain-and-range-246:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-283:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-253:

Children
--------

.. _used-by-246:

Used by
-------

.. _other-properties-228:

Other properties
----------------

=============== ==================
**Aliases:**     feature prediction
**Mappings:**    mixs:feat_pred
**In Subsets:**  sequencing
=============== ==================

Slot: feature type
==================

TODO: Yuri to write

URI: `nmdc:feature_type <https://microbiomedata/meta/feature_type>`__

.. _domain-and-range-247:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-284:

Parents
-------

.. _children-254:

Children
--------

-  `genome feature➞feature type <genome_feature_feature_type.md>`__

.. _used-by-247:

Used by
-------

Slot: fertilizer_regm
=====================

“Information about treatment involving the use of fertilizers; should
include the name of fertilizer, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple fertilizer regimens”

URI:
`nmdc:fertilizer_regm <https://microbiomedata/meta/fertilizer_regm>`__

.. _domain-and-range-248:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-285:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-255:

Children
--------

.. _used-by-248:

Used by
-------

.. _other-properties-229:

Other properties
----------------

============= ====================
**Aliases:**   fertilizer regimen
**Mappings:**  mixs:fertilizer_regm
============= ====================

Slot: field
===========

Name of the hydrocarbon field (e.g. Albacora)

URI: `nmdc:field <https://microbiomedata/meta/field>`__

.. _domain-and-range-249:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-286:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-256:

Children
--------

.. _used-by-249:

Used by
-------

.. _other-properties-230:

Other properties
----------------

============= ==========
**Aliases:**   field name
**Mappings:**  mixs:field
============= ==========

Slot: file size bytes
=====================

Size of the file in bytes

URI:
https://microbiomedata/schema/workflow_execution_activity/file_size_bytes

.. _domain-and-range-250:

Domain and Range
----------------

None → 0..1 `Bytes <types/Bytes.md>`__

.. _parents-287:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-257:

Children
--------

.. _used-by-250:

Used by
-------

Enum: file type enum
====================

URI:
`nmdc:file_type_enum <https://microbiomedata/meta/file_type_enum>`__

.. _other-properties-231:

Other properties
----------------

.. _permissible-values-1:

Permissible Values
------------------

+---------------+-------------------+-------------------+---------------+
| Text          | Description       | Meaning           | Other         |
|               |                   |                   | Information   |
+===============+===================+===================+===============+
| FT ICR-MS     | FT ICR-MS-based   |                   |               |
| Analysis      | metabolite        |                   |               |
| Results       | assignment        |                   |               |
|               | results table     |                   |               |
+---------------+-------------------+-------------------+---------------+
| GC-MS         | GC-MS-based       |                   |               |
| Metabolomics  | metabolite        |                   |               |
| Results       | assignment        |                   |               |
|               | results table     |                   |               |
+---------------+-------------------+-------------------+---------------+
| M             | Aggregate         |                   |               |
| etaproteomics | workflow          |                   |               |
| Workflow      | statistics file   |                   |               |
| Statistics    |                   |                   |               |
+---------------+-------------------+-------------------+---------------+
| Protein       | Filtered protein  |                   |               |
| Report        | report file       |                   |               |
+---------------+-------------------+-------------------+---------------+
| Peptide       | Filtered peptide  |                   |               |
| Report        | report file       |                   |               |
+---------------+-------------------+-------------------+---------------+
| Unfiltered    | MSGFjobs and      |                   |               |
| M             | MASIC output file |                   |               |
| etaproteomics |                   |                   |               |
| Results       |                   |                   |               |
+---------------+-------------------+-------------------+---------------+
| Read Count    | Annotation read   |                   |               |
| and RPKM      | count and RPKM    |                   |               |
|               | per feature JSON  |                   |               |
+---------------+-------------------+-------------------+---------------+
| QC non-rRNA   | QC removed rRNA   |                   |               |
| R2            | reads (R2) fastq  |                   |               |
+---------------+-------------------+-------------------+---------------+
| QC non-rRNA   | QC removed rRNA   |                   |               |
| R1            | reads (R1) fastq  |                   |               |
+---------------+-------------------+-------------------+---------------+
| Metagenome    | Metagenome bin    |                   |               |
| Bins          | contigs fasta     |                   |               |
+---------------+-------------------+-------------------+---------------+
| CheckM        | CheckM statistics |                   |               |
| Statistics    | report            |                   |               |
+---------------+-------------------+-------------------+---------------+
| GOTTCHA2      | GOTTCHA2 krona    |                   |               |
| Krona Plot    | plot HTML file    |                   |               |
+---------------+-------------------+-------------------+---------------+
| Kraken2 Krona | Kraken2 krona     |                   |               |
| Plot          | plot HTML file    |                   |               |
+---------------+-------------------+-------------------+---------------+
| Centrifuge    | Centrifug krona   |                   |               |
| Krona Plot    | plot HTML file    |                   |               |
+---------------+-------------------+-------------------+---------------+
| Kraken2       | Kraken2 output    |                   |               |
| C             | report file       |                   |               |
| lassification |                   |                   |               |
| Report        |                   |                   |               |
+---------------+-------------------+-------------------+---------------+
| Kraken2       | Kraken2 output    |                   |               |
| Taxonomic     | read              |                   |               |
| C             | classification    |                   |               |
| lassification | file              |                   |               |
+---------------+-------------------+-------------------+---------------+
| Centrifuge    | Centrifuge output |                   |               |
| C             | report file       |                   |               |
| lassification |                   |                   |               |
| Report        |                   |                   |               |
+---------------+-------------------+-------------------+---------------+
| Centrifuge    | Centrifuge output |                   |               |
| Taxonomic     | read              |                   |               |
| C             | classification    |                   |               |
| lassification | file              |                   |               |
+---------------+-------------------+-------------------+---------------+
| Structural    | GFF3 format file  |                   |               |
| Annotation    | with structural   |                   |               |
| GFF           | annotations       |                   |               |
+---------------+-------------------+-------------------+---------------+
| Functional    | GFF3 format file  |                   |               |
| Annotation    | with functional   |                   |               |
| GFF           | annotations       |                   |               |
+---------------+-------------------+-------------------+---------------+
| Annotation    | FASTA amino acid  |                   |               |
| Amino Acid    | file for          |                   |               |
| FASTA         | annotated         |                   |               |
|               | proteins          |                   |               |
+---------------+-------------------+-------------------+---------------+
| Annotation    | Tab delimited     |                   |               |
| Enzyme        | file for EC       |                   |               |
| Commission    | annotation        |                   |               |
+---------------+-------------------+-------------------+---------------+
| Annotation    | Tab delimited     |                   |               |
| KEGG          | file for KO       |                   |               |
| Orthology     | annotation        |                   |               |
+---------------+-------------------+-------------------+---------------+
| Assembly      | Sorted bam file   |                   |               |
| Coverage BAM  | of reads mapping  |                   |               |
|               | back to the final |                   |               |
|               | assembly          |                   |               |
+---------------+-------------------+-------------------+---------------+
| Assembly AGP  | An AGP format     |                   |               |
|               | file that         |                   |               |
|               | describes the     |                   |               |
|               | assembly          |                   |               |
+---------------+-------------------+-------------------+---------------+
| Assembly      | Final assembly    |                   |               |
| Scaffolds     | scaffolds fasta   |                   |               |
+---------------+-------------------+-------------------+---------------+
| Assembly      | Final assembly    |                   |               |
| Contigs       | contigs fasta     |                   |               |
+---------------+-------------------+-------------------+---------------+
| Assembly      | Assembled contigs |                   |               |
| Coverage      | coverage          |                   |               |
| Stats         | information       |                   |               |
+---------------+-------------------+-------------------+---------------+
| Filtered      | Reads QC result   |                   |               |
| Sequencing    | fastq (clean      |                   |               |
| Reads         | data)             |                   |               |
+---------------+-------------------+-------------------+---------------+
| QC Statistics | Reads QC summary  |                   |               |
|               | statistics        |                   |               |
+---------------+-------------------+-------------------+---------------+
| TIGRFam       | GFF3 format file  |                   | {‘            |
| Annotation    | with TIGRfam      |                   | annotations’: |
| GFF           |                   |                   | {‘file_n      |
|               |                   |                   | ame_pattern’: |
|               |                   |                   | Annotatio     |
|               |                   |                   | n(tag=‘file_n |
|               |                   |                   | ame_pattern’, |
|               |                   |                   | valu          |
|               |                   |                   | e=’[GOLD-AP]_ |
|               |                   |                   | tigrfam.gff’, |
|               |                   |                   | e             |
|               |                   |                   | xtensions={}, |
|               |                   |                   | anno          |
|               |                   |                   | tations={})}} |
+---------------+-------------------+-------------------+---------------+
| Clusters of   | GFF3 format file  |                   | {‘            |
| Orthologous   | with COGs         |                   | annotations’: |
| Groups (COG)  |                   |                   | {‘file_n      |
| Annotation    |                   |                   | ame_pattern’: |
| GFF           |                   |                   | Annotatio     |
|               |                   |                   | n(tag=‘file_n |
|               |                   |                   | ame_pattern’, |
|               |                   |                   | value=’[GOLD- |
|               |                   |                   | AP]_cog.gff’, |
|               |                   |                   | e             |
|               |                   |                   | xtensions={}, |
|               |                   |                   | anno          |
|               |                   |                   | tations={})}} |
+---------------+-------------------+-------------------+---------------+
| CATH FunFams  | GFF3 format file  |                   | {‘            |
| (Functional   | with CATH FunFams |                   | annotations’: |
| Families)     |                   |                   | {‘file_n      |
| Annotation    |                   |                   | ame_pattern’: |
| GFF           |                   |                   | Annotatio     |
|               |                   |                   | n(tag=‘file_n |
|               |                   |                   | ame_pattern’, |
|               |                   |                   | value=’[      |
|               |                   |                   | GOLD-AP]_cath |
|               |                   |                   | _funfam.gff’, |
|               |                   |                   | e             |
|               |                   |                   | xtensions={}, |
|               |                   |                   | anno          |
|               |                   |                   | tations={})}} |
+---------------+-------------------+-------------------+---------------+
| SUPERFam      | GFF3 format file  |                   | {‘            |
| Annotation    | with SUPERFam     |                   | annotations’: |
| GFF           |                   |                   | {‘file_n      |
|               |                   |                   | ame_pattern’: |
|               |                   |                   | Annotatio     |
|               |                   |                   | n(tag=‘file_n |
|               |                   |                   | ame_pattern’, |
|               |                   |                   | val           |
|               |                   |                   | ue=’[GOLD-AP] |
|               |                   |                   | _supfam.gff’, |
|               |                   |                   | e             |
|               |                   |                   | xtensions={}, |
|               |                   |                   | anno          |
|               |                   |                   | tations={})}} |
+---------------+-------------------+-------------------+---------------+
| SMART         | GFF3 format file  |                   | {‘            |
| Annotation    | with SMART        |                   | annotations’: |
| GFF           |                   |                   | {‘file_n      |
|               |                   |                   | ame_pattern’: |
|               |                   |                   | Annotatio     |
|               |                   |                   | n(tag=‘file_n |
|               |                   |                   | ame_pattern’, |
|               |                   |                   | va            |
|               |                   |                   | lue=’[GOLD-AP |
|               |                   |                   | ]_smart.gff’, |
|               |                   |                   | e             |
|               |                   |                   | xtensions={}, |
|               |                   |                   | anno          |
|               |                   |                   | tations={})}} |
+---------------+-------------------+-------------------+---------------+
| Pfam          | GFF3 format file  |                   | {‘            |
| Annotation    | with Pfam         |                   | annotations’: |
| GFF           |                   |                   | {‘file_n      |
|               |                   |                   | ame_pattern’: |
|               |                   |                   | Annotatio     |
|               |                   |                   | n(tag=‘file_n |
|               |                   |                   | ame_pattern’, |
|               |                   |                   | v             |
|               |                   |                   | alue=’[GOLD-A |
|               |                   |                   | P]_pfam.gff’, |
|               |                   |                   | e             |
|               |                   |                   | xtensions={}, |
|               |                   |                   | anno          |
|               |                   |                   | tations={})}} |
+---------------+-------------------+-------------------+---------------+

Slot: filter_type
=================

A device which removes solid particulates or airborne molecular
contaminants

URI: `nmdc:filter_type <https://microbiomedata/meta/filter_type>`__

.. _domain-and-range-251:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-288:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-258:

Children
--------

.. _used-by-251:

Used by
-------

.. _other-properties-232:

Other properties
----------------

============= ================
**Aliases:**   filter type
**Mappings:**  mixs:filter_type
============= ================

Slot: fire
==========

Historical and/or physical evidence of fire

URI: `nmdc:fire <https://microbiomedata/meta/fire>`__

.. _domain-and-range-252:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-289:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-259:

Children
--------

.. _used-by-252:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-233:

Other properties
----------------

============= ============
**Aliases:**   history/fire
**Mappings:**  mixs:fire
============= ============

Slot: fireplace_type
====================

A firebox with chimney

URI:
`nmdc:fireplace_type <https://microbiomedata/meta/fireplace_type>`__

.. _domain-and-range-253:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-290:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-260:

Children
--------

.. _used-by-253:

Used by
-------

.. _other-properties-234:

Other properties
----------------

============= ===================
**Aliases:**   fireplace type
**Mappings:**  mixs:fireplace_type
============= ===================

Slot: flooding
==============

Historical and/or physical evidence of flooding

URI: `nmdc:flooding <https://microbiomedata/meta/flooding>`__

.. _domain-and-range-254:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-291:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-261:

Children
--------

.. _used-by-254:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-235:

Other properties
----------------

============= ================
**Aliases:**   history/flooding
**Mappings:**  mixs:flooding
============= ================

Slot: floor_age
===============

The time period since installment of the carpet or flooring

URI: `nmdc:floor_age <https://microbiomedata/meta/floor_age>`__

.. _domain-and-range-255:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-292:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-262:

Children
--------

.. _used-by-255:

Used by
-------

.. _other-properties-236:

Other properties
----------------

============= ==============
**Aliases:**   floor age
**Mappings:**  mixs:floor_age
============= ==============

Slot: floor_area
================

The area of the floor space within the room

URI: `nmdc:floor_area <https://microbiomedata/meta/floor_area>`__

.. _domain-and-range-256:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-293:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-263:

Children
--------

.. _used-by-256:

Used by
-------

.. _other-properties-237:

Other properties
----------------

============= ===============
**Aliases:**   floor area
**Mappings:**  mixs:floor_area
============= ===============

Slot: floor_cond
================

The physical condition of the floor at the time of sampling; photos or
video preferred; use drawings to indicate location of damaged areas

URI: `nmdc:floor_cond <https://microbiomedata/meta/floor_cond>`__

.. _domain-and-range-257:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-294:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-264:

Children
--------

.. _used-by-257:

Used by
-------

.. _other-properties-238:

Other properties
----------------

============= ===============
**Aliases:**   floor condition
**Mappings:**  mixs:floor_cond
============= ===============

Slot: floor_count
=================

“The number of floors in the building, including basements and
mechanical penthouse”

URI: `nmdc:floor_count <https://microbiomedata/meta/floor_count>`__

.. _domain-and-range-258:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-295:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-265:

Children
--------

.. _used-by-258:

Used by
-------

.. _other-properties-239:

Other properties
----------------

============= ================
**Aliases:**   floor count
**Mappings:**  mixs:floor_count
============= ================

Slot: floor_finish_mat
======================

The floor covering type; the finished surface that is walked on

URI:
`nmdc:floor_finish_mat <https://microbiomedata/meta/floor_finish_mat>`__

.. _domain-and-range-259:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-296:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-266:

Children
--------

.. _used-by-259:

Used by
-------

.. _other-properties-240:

Other properties
----------------

============= =====================
**Aliases:**   floor finish material
**Mappings:**  mixs:floor_finish_mat
============= =====================

Slot: floor_struc
=================

Refers to the structural elements and subfloor upon which the finish
flooring is installed

URI: `nmdc:floor_struc <https://microbiomedata/meta/floor_struc>`__

.. _domain-and-range-260:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-297:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-267:

Children
--------

.. _used-by-260:

Used by
-------

.. _other-properties-241:

Other properties
----------------

============= ================
**Aliases:**   floor structure
**Mappings:**  mixs:floor_struc
============= ================

Slot: floor_thermal_mass
========================

The ability of the floor to provide inertia against temperature
fluctuations

URI:
`nmdc:floor_thermal_mass <https://microbiomedata/meta/floor_thermal_mass>`__

.. _domain-and-range-261:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-298:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-268:

Children
--------

.. _used-by-261:

Used by
-------

.. _other-properties-242:

Other properties
----------------

============= =======================
**Aliases:**   floor thermal mass
**Mappings:**  mixs:floor_thermal_mass
============= =======================

Slot: floor_water_mold
======================

Signs of the presence of mold or mildew in a room

URI:
`nmdc:floor_water_mold <https://microbiomedata/meta/floor_water_mold>`__

.. _domain-and-range-262:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-299:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-269:

Children
--------

.. _used-by-262:

Used by
-------

.. _other-properties-243:

Other properties
----------------

============= =========================
**Aliases:**   floor signs of water/mold
**Mappings:**  mixs:floor_water_mold
============= =========================

Slot: fluor
===========

Raw or converted fluorescence of water

URI: `nmdc:fluor <https://microbiomedata/meta/fluor>`__

.. _domain-and-range-263:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-300:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-270:

Children
--------

.. _used-by-263:

Used by
-------

.. _other-properties-244:

Other properties
----------------

============= ============
**Aliases:**   fluorescence
**Mappings:**  mixs:fluor
============= ============

Slot: foetal_health_stat
========================

“Specification of foetal health status, should also include abortion”

URI:
`nmdc:foetal_health_stat <https://microbiomedata/meta/foetal_health_stat>`__

.. _domain-and-range-264:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-301:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-271:

Children
--------

.. _used-by-264:

Used by
-------

.. _other-properties-245:

Other properties
----------------

============= ===================================
**Aliases:**   amniotic fluid/foetal health status
**Mappings:**  mixs:foetal_health_stat
============= ===================================

Slot: freq_clean
================

The number of times the building is cleaned per week

URI: `nmdc:freq_clean <https://microbiomedata/meta/freq_clean>`__

.. _domain-and-range-265:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-302:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-272:

Children
--------

.. _used-by-265:

Used by
-------

.. _other-properties-246:

Other properties
----------------

============= =====================
**Aliases:**   frequency of cleaning
**Mappings:**  mixs:freq_clean
============= =====================

Slot: freq_cook
===============

The number of times a meal is cooked per week

URI: `nmdc:freq_cook <https://microbiomedata/meta/freq_cook>`__

.. _domain-and-range-266:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-303:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-273:

Children
--------

.. _used-by-266:

Used by
-------

.. _other-properties-247:

Other properties
----------------

============= ====================
**Aliases:**   frequency of cooking
**Mappings:**  mixs:freq_cook
============= ====================

Slot: has function
==================

URI:
`nmdc:functional_annotation_has_function <https://microbiomedata/meta/functional_annotation_has_function>`__

.. _domain-and-range-267:

Domain and Range
----------------

`FunctionalAnnotation <FunctionalAnnotation.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-304:

Parents
-------

-  is_a: `has function <has_function.md>`__

.. _children-274:

Children
--------

.. _used-by-267:

Used by
-------

-  `FunctionalAnnotation <FunctionalAnnotation.md>`__

.. _other-properties-248:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | Still missing         |
|                       |                       | patterns for COG and  |
|                       |                       | RetroRules.           |
+-----------------------+-----------------------+-----------------------+
|                       |                       | These patterns aren’t |
|                       |                       | tied to the listed    |
|                       |                       | prefixes. A           |
|                       |                       | discussion about that |
|                       |                       | possibility had been  |
|                       |                       | started, including    |
|                       |                       | the question of       |
|                       |                       | whether these lists   |
|                       |                       | are intended to be    |
|                       |                       | open examples or      |
|                       |                       | closed                |
+-----------------------+-----------------------+-----------------------+

Slot: functional annotation set
===============================

This property links a database object to the set of all functional
annotations

URI:
`nmdc:functional_annotation_set <https://microbiomedata/meta/functional_annotation_set>`__

.. _domain-and-range-268:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`FunctionalAnnotation <FunctionalAnnotation.md>`__

.. _parents-305:

Parents
-------

.. _children-275:

Children
--------

.. _used-by-268:

Used by
-------

-  `Database <Database.md>`__

.. _slot-type-1:

Slot: type
==========

TODO

URI:
`nmdc:functional_annotation_type <https://microbiomedata/meta/functional_annotation_type>`__

.. _domain-and-range-269:

Domain and Range
----------------

`FunctionalAnnotation <FunctionalAnnotation.md>`__ → 0..1
`OntologyClass <OntologyClass.md>`__

.. _parents-306:

Parents
-------

-  is_a: `type <type.md>`__

.. _children-276:

Children
--------

.. _used-by-269:

Used by
-------

-  `FunctionalAnnotation <FunctionalAnnotation.md>`__

.. _other-properties-249:

Other properties
----------------

============= =================================================
**Examples:**  Example(value=‘nmdc:Biosample’, description=None)
\              Example(value=‘nmdc:Study’, description=None)
============= =================================================

Slot: was generated by
======================

provenance for the annotation.

URI:
`nmdc:functional_annotation_was_generated_by <https://microbiomedata/meta/functional_annotation_was_generated_by>`__

.. _domain-and-range-270:

Domain and Range
----------------

`FunctionalAnnotation <FunctionalAnnotation.md>`__ → 0..1
`MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__

.. _parents-307:

Parents
-------

-  is_a: `was generated by <was_generated_by.md>`__

.. _children-277:

Children
--------

.. _used-by-270:

Used by
-------

-  `FunctionalAnnotation <FunctionalAnnotation.md>`__

.. _other-properties-250:

Other properties
----------------

============= ===================
**Mappings:**  prov:wasGeneratedBy
============= ===================

Slot: funding sources
=====================

URI:
`nmdc:funding_sources <https://microbiomedata/meta/funding_sources>`__

.. _domain-and-range-271:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-308:

Parents
-------

.. _children-278:

Children
--------

.. _used-by-271:

Used by
-------

-  `Study <Study.md>`__

Slot: fungicide_regm
====================

“Information about treatment involving use of fungicides; should include
the name of fungicide, amount administered, treatment regimen including
how many times the treatment was repeated, how long each treatment
lasted, and the start and end time of the entire treatment; can include
multiple fungicide regimens”

URI:
`nmdc:fungicide_regm <https://microbiomedata/meta/fungicide_regm>`__

.. _domain-and-range-272:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-309:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-279:

Children
--------

.. _used-by-272:

Used by
-------

.. _other-properties-251:

Other properties
----------------

============= ===================
**Aliases:**   fungicide regimen
**Mappings:**  mixs:fungicide_regm
============= ===================

Slot: furniture
===============

The types of furniture present in the sampled room

URI: `nmdc:furniture <https://microbiomedata/meta/furniture>`__

.. _domain-and-range-273:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-310:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-280:

Children
--------

.. _used-by-273:

Used by
-------

.. _other-properties-252:

Other properties
----------------

============= ==============
**Aliases:**   furniture
**Mappings:**  mixs:furniture
============= ==============

Slot: gap_pct
=============

The gap size percentage of all scaffolds.

URI: https://microbiomedata/schema/workflow_execution_activity/gap_pct

.. _domain-and-range-274:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-311:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-281:

Children
--------

.. _used-by-274:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: gaseous_environment
=========================

“Use of conditions with differing gaseous environments; should include
the name of gaseous compound, amount administered, treatment duration,
interval and total experimental duration; can include multiple gaseous
environment regimens”

URI:
`nmdc:gaseous_environment <https://microbiomedata/meta/gaseous_environment>`__

.. _domain-and-range-275:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-312:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-282:

Children
--------

.. _used-by-275:

Used by
-------

.. _other-properties-253:

Other properties
----------------

============= ========================
**Aliases:**   gaseous environment
**Mappings:**  mixs:gaseous_environment
============= ========================

Slot: gaseous_substances
========================

“Amount or concentration of substances such as hydrogen sulfide, carbon
dioxide, methane, etc.; can include multiple substances”

URI:
`nmdc:gaseous_substances <https://microbiomedata/meta/gaseous_substances>`__

.. _domain-and-range-276:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-313:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-283:

Children
--------

.. _used-by-276:

Used by
-------

.. _other-properties-254:

Other properties
----------------

============= =======================
**Aliases:**   gaseous substances
**Mappings:**  mixs:gaseous_substances
============= =======================

Slot: gastrointest_disord
=========================

History of gastrointestinal tract disorders; can include multiple
disorders

URI:
`nmdc:gastrointest_disord <https://microbiomedata/meta/gastrointest_disord>`__

.. _domain-and-range-277:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-314:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-284:

Children
--------

.. _used-by-277:

Used by
-------

.. _other-properties-255:

Other properties
----------------

============= ===============================
**Aliases:**   gastrointestinal tract disorder
**Mappings:**  mixs:gastrointest_disord
============= ===============================

Slot: gc_avg
============

Average of GC content of all contigs.

URI: https://microbiomedata/schema/workflow_execution_activity/gc_avg

.. _domain-and-range-278:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-315:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-285:

Children
--------

.. _used-by-278:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: gc_std
============

Standard deviation of GC content of all contigs.

URI: https://microbiomedata/schema/workflow_execution_activity/gc_std

.. _domain-and-range-279:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-316:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-286:

Children
--------

.. _used-by-279:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: gender_restroom
=====================

The gender type of the restroom

URI:
`nmdc:gender_restroom <https://microbiomedata/meta/gender_restroom>`__

.. _domain-and-range-280:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-317:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-287:

Children
--------

.. _used-by-280:

Used by
-------

.. _other-properties-256:

Other properties
----------------

============= ====================
**Aliases:**   gender of restroom
**Mappings:**  mixs:gender_restroom
============= ====================

Slot: genetic_mod
=================

“Genetic modifications of the genome of an organism, which may occur
naturally by spontaneous mutation, or be introduced by some experimental
means, e.g. specification of a transgene or the gene knocked-out or
details of transient transfection”

URI: `nmdc:genetic_mod <https://microbiomedata/meta/genetic_mod>`__

.. _domain-and-range-281:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-318:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-288:

Children
--------

.. _used-by-281:

Used by
-------

.. _other-properties-257:

Other properties
----------------

============= ====================
**Aliases:**   genetic modification
**Mappings:**  mixs:genetic_mod
============= ====================

.. _slot-encodes-1:

Slot: encodes
=============

The gene product encoded by this feature. Typically this is used for a
CDS feature or gene feature which will encode a protein. It can also be
used by a nc transcript ot gene feature that encoded a ncRNA

URI:
`nmdc:genome_feature_encodes <https://microbiomedata/meta/genome_feature_encodes>`__

.. _domain-and-range-282:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 0..1
`GeneProduct <GeneProduct.md>`__

.. _parents-319:

Parents
-------

-  is_a: `encodes <encodes.md>`__

.. _children-289:

Children
--------

.. _used-by-282:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

.. _slot-end-1:

Slot: end
=========

The end of the feature in positive 1-based integer coordinates

URI:
`nmdc:genome_feature_end <https://microbiomedata/meta/genome_feature_end>`__

.. _domain-and-range-283:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 1..1
`Integer <types/Integer.md>`__

.. _parents-320:

Parents
-------

-  is_a: `gff coordinate <gff_coordinate.md>`__

.. _children-290:

Children
--------

.. _used-by-283:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

.. _other-properties-258:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | - “constraint: end >  |
|                       |                       | start” - “For         |
|                       |                       | features that cross   |
|                       |                       | the origin of a       |
|                       |                       | circular feature, end |
|                       |                       | = the position of the |
|                       |                       | end + the length of   |
|                       |                       | the landmark          |
|                       |                       | feature.”             |
+-----------------------+-----------------------+-----------------------+
| **Close Mappings:**   |                       | biolink:end           |
|                       |                       | _interbase_coordinate |
+-----------------------+-----------------------+-----------------------+

.. _slot-feature-type-1:

Slot: feature type
==================

TODO: Yuri to write

URI:
`nmdc:genome_feature_feature_type <https://microbiomedata/meta/genome_feature_feature_type>`__

.. _domain-and-range-284:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-321:

Parents
-------

-  is_a: `feature type <feature_type.md>`__

.. _children-291:

Children
--------

.. _used-by-284:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

Slot: phase
===========

The phase for a coding sequence entity. For example, phase of a CDS as
represented in a GFF3 with a value of 0, 1 or 2.

URI:
`nmdc:genome_feature_phase <https://microbiomedata/meta/genome_feature_phase>`__

.. _domain-and-range-285:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-322:

Parents
-------

-  is_a: `phase <phase.md>`__

.. _children-292:

Children
--------

.. _used-by-285:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

.. _other-properties-259:

Other properties
----------------

=================== =============
**Exact Mappings:**  biolink:phase
=================== =============

Slot: seqid
===========

The ID of the landmark used to establish the coordinate system for the
current feature.

URI:
`nmdc:genome_feature_seqid <https://microbiomedata/meta/genome_feature_seqid>`__

.. _domain-and-range-286:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 1..1 `String <types/String.md>`__

.. _parents-323:

Parents
-------

-  is_a: `seqid <seqid.md>`__

.. _children-293:

Children
--------

.. _used-by-286:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

Slot: genome feature set
========================

This property links a database object to the set of all features

URI:
`nmdc:genome_feature_set <https://microbiomedata/meta/genome_feature_set>`__

.. _domain-and-range-287:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\* `GenomeFeature <GenomeFeature.md>`__

.. _parents-324:

Parents
-------

.. _children-294:

Children
--------

.. _used-by-287:

Used by
-------

-  `Database <Database.md>`__

Slot: start
===========

The start of the feature in positive 1-based integer coordinates

URI:
`nmdc:genome_feature_start <https://microbiomedata/meta/genome_feature_start>`__

.. _domain-and-range-288:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 1..1
`Integer <types/Integer.md>`__

.. _parents-325:

Parents
-------

-  is_a: `gff coordinate <gff_coordinate.md>`__

.. _children-295:

Children
--------

.. _used-by-288:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

.. _other-properties-260:

Other properties
----------------

=================== ==================================
**Close Mappings:**  biolink:start_interbase_coordinate
=================== ==================================

Slot: strand
============

The strand on which a feature is located. Has a value of ‘+’ (sense
strand or forward strand) or ‘-’ (anti-sense strand or reverse strand).

URI:
`nmdc:genome_feature_strand <https://microbiomedata/meta/genome_feature_strand>`__

.. _domain-and-range-289:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-326:

Parents
-------

-  is_a: `strand <strand.md>`__

.. _children-296:

Children
--------

.. _used-by-289:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

.. _other-properties-261:

Other properties
----------------

=================== ==============
**Exact Mappings:**  biolink:strand
=================== ==============

.. _slot-type-2:

Slot: type
==========

A type from the sequence ontology

URI:
`nmdc:genome_feature_type <https://microbiomedata/meta/genome_feature_type>`__

.. _domain-and-range-290:

Domain and Range
----------------

`GenomeFeature <GenomeFeature.md>`__ → 0..1
`OntologyClass <OntologyClass.md>`__

.. _parents-327:

Parents
-------

-  is_a: `type <type.md>`__

.. _children-297:

Children
--------

.. _used-by-290:

Used by
-------

-  `GenomeFeature <GenomeFeature.md>`__

.. _other-properties-262:

Other properties
----------------

============= =================================================
**Mappings:**  rdf:type
**Examples:**  Example(value=‘nmdc:Biosample’, description=None)
\              Example(value=‘nmdc:Study’, description=None)
============= =================================================

Slot: geo_loc_name
==================

“The geographical origin of the sample as defined by the country or sea
name followed by specific region name. Country or sea names should be
chosen from the INSDC country list (http://insdc.org/country.html), or
the GAZ ontology (v 1.512) (http://purl.bioontology.org/ontology/GAZ)”

URI: `nmdc:geo_loc_name <https://microbiomedata/meta/geo_loc_name>`__

.. _domain-and-range-291:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-328:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-298:

Children
--------

.. _used-by-291:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-263:

Other properties
----------------

=============== ===============================================
**Aliases:**     geographic location (country and/or sea,region)
**Mappings:**    mixs:geo_loc_name
**In Subsets:**  environment
=============== ===============================================

Slot: has raw value
===================

The raw value for a geolocation should follow {lat} {long}

URI:
https://microbiomedata/schema/workflow_execution_activity/geolocation_value_has_raw_value

.. _domain-and-range-292:

Domain and Range
----------------

`GeolocationValue <GeolocationValue.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-329:

Parents
-------

-  is_a: `has raw value <has_raw_value.md>`__

.. _children-299:

Children
--------

.. _used-by-292:

Used by
-------

-  `GeolocationValue <GeolocationValue.md>`__

Slot: gestation_state
=====================

Specification of the gestation state

URI:
`nmdc:gestation_state <https://microbiomedata/meta/gestation_state>`__

.. _domain-and-range-293:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-330:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-300:

Children
--------

.. _used-by-293:

Used by
-------

.. _other-properties-264:

Other properties
----------------

============= ==============================
**Aliases:**   amniotic fluid/gestation state
**Mappings:**  mixs:gestation_state
============= ==============================

Slot: gff coordinate
====================

A positive 1-based integer coordinate indicating start or end

URI:
`nmdc:gff_coordinate <https://microbiomedata/meta/gff_coordinate>`__

.. _domain-and-range-294:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-331:

Parents
-------

.. _children-301:

Children
--------

-  `end <end.md>`__
-  `genome feature➞end <genome_feature_end.md>`__
-  `genome feature➞start <genome_feature_start.md>`__
-  `start <start.md>`__

.. _used-by-294:

Used by
-------

.. _other-properties-265:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | For features that     |
|                       |                       | cross the origin of a |
|                       |                       | circular feature      |
|                       |                       | (e.g. most bacterial  |
|                       |                       | genomes, plasmids,    |
|                       |                       | and some viral        |
|                       |                       | genomes), the         |
|                       |                       | requirement for start |
|                       |                       | to be less than or    |
|                       |                       | equal to end is       |
|                       |                       | satisfied by making   |
|                       |                       | end = the position of |
|                       |                       | the end + the length  |
|                       |                       | of the landmark       |
|                       |                       | feature.              |
+-----------------------+-----------------------+-----------------------+

Slot: git url
=============

Example:
https://github.com/microbiomedata/mg_annotation/releases/tag/0.1

URI: https://microbiomedata/schema/workflow_execution_activity/git_url

.. _domain-and-range-295:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-332:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-302:

Children
--------

-  `workflow execution activity➞git
   url <workflow_execution_activity_git_url.md>`__

.. _used-by-295:

Used by
-------

Slot: glucosidase_act
=====================

Measurement of glucosidase activity

URI:
`nmdc:glucosidase_act <https://microbiomedata/meta/glucosidase_act>`__

.. _domain-and-range-296:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-333:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-303:

Children
--------

.. _used-by-296:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-266:

Other properties
----------------

============= ====================
**Aliases:**   glucosidase activity
**Mappings:**  mixs:glucosidase_act
============= ====================

Slot: gold_path_field
=====================

This is a grouping for any of the gold path fields

URI:
`nmdc:gold_path_field <https://microbiomedata/meta/gold_path_field>`__

.. _domain-and-range-297:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-334:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-304:

Children
--------

-  `ecosystem <ecosystem.md>`__
-  `ecosystem_category <ecosystem_category.md>`__
-  `ecosystem_subtype <ecosystem_subtype.md>`__
-  `ecosystem_type <ecosystem_type.md>`__
-  `specific_ecosystem <specific_ecosystem.md>`__

.. _used-by-297:

Used by
-------

Slot: gravidity
===============

“Whether or not subject is gravid, and if yes date due or date
post-conception, specifying which is used”

URI: `nmdc:gravidity <https://microbiomedata/meta/gravidity>`__

.. _domain-and-range-298:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-335:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-305:

Children
--------

.. _used-by-298:

Used by
-------

.. _other-properties-267:

Other properties
----------------

============= ==============
**Aliases:**   gravidity
**Mappings:**  mixs:gravidity
============= ==============

Slot: gravity
=============

“Information about treatment involving use of gravity factor to study
various types of responses in presence, absence or modified levels of
gravity; treatment regimen including how many times the treatment was
repeated, how long each treatment lasted, and the start and end time of
the entire treatment; can include multiple treatments”

URI: `nmdc:gravity <https://microbiomedata/meta/gravity>`__

.. _domain-and-range-299:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-336:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-306:

Children
--------

.. _used-by-299:

Used by
-------

.. _other-properties-268:

Other properties
----------------

============= ============
**Aliases:**   gravity
**Mappings:**  mixs:gravity
============= ============

Slot: growth_facil
==================

“Type of facility where the sampled plant was grown; controlled
vocabulary: growth chamber, open top chamber, glasshouse, experimental
garden, field. Alternatively use Crop Ontology (CO) terms, see
http://www.cropontology.org/ontology/CO_715/Crop%20Research”

URI: `nmdc:growth_facil <https://microbiomedata/meta/growth_facil>`__

.. _domain-and-range-300:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-337:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-307:

Children
--------

.. _used-by-300:

Used by
-------

.. _other-properties-269:

Other properties
----------------

============= =================
**Aliases:**   growth facility
**Mappings:**  mixs:growth_facil
============= =================

Slot: growth_habit
==================

“Characteristic shape, appearance or growth form of a plant species”

URI: `nmdc:growth_habit <https://microbiomedata/meta/growth_habit>`__

.. _domain-and-range-301:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-338:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-308:

Children
--------

.. _used-by-301:

Used by
-------

.. _other-properties-270:

Other properties
----------------

============= =================
**Aliases:**   growth habit
**Mappings:**  mixs:growth_habit
============= =================

Slot: growth_hormone_regm
=========================

“Information about treatment involving use of growth hormones; should
include the name of growth hormone, amount administered, treatment
regimen including how many times the treatment was repeated, how long
each treatment lasted, and the start and end time of the entire
treatment; can include multiple growth hormone regimens”

URI:
`nmdc:growth_hormone_regm <https://microbiomedata/meta/growth_hormone_regm>`__

.. _domain-and-range-302:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-339:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-309:

Children
--------

.. _used-by-302:

Used by
-------

.. _other-properties-271:

Other properties
----------------

============= ========================
**Aliases:**   growth hormone regimen
**Mappings:**  mixs:growth_hormone_regm
============= ========================

Slot: gynecologic_disord
========================

History of gynecological disorders; can include multiple disorders

URI:
`nmdc:gynecologic_disord <https://microbiomedata/meta/gynecologic_disord>`__

.. _domain-and-range-303:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-340:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-310:

Children
--------

.. _used-by-303:

Used by
-------

.. _other-properties-272:

Other properties
----------------

============= =======================
**Aliases:**   gynecological disorder
**Mappings:**  mixs:gynecologic_disord
============= =======================

Slot: habitat
=============

URI: `nmdc:habitat <https://microbiomedata/meta/habitat>`__

.. _domain-and-range-304:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-341:

Parents
-------

.. _children-311:

Children
--------

.. _used-by-304:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: hall_count
================

The total count of hallways and cooridors in the built structure

URI: `nmdc:hall_count <https://microbiomedata/meta/hall_count>`__

.. _domain-and-range-305:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-342:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-312:

Children
--------

.. _used-by-305:

Used by
-------

.. _other-properties-273:

Other properties
----------------

============= ======================
**Aliases:**   hallway/corridor count
**Mappings:**  mixs:hall_count
============= ======================

Slot: handidness
================

The handidness of the individual sampled

URI: `nmdc:handidness <https://microbiomedata/meta/handidness>`__

.. _domain-and-range-306:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-343:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-313:

Children
--------

.. _used-by-306:

Used by
-------

.. _other-properties-274:

Other properties
----------------

============= ===============
**Aliases:**   handidness
**Mappings:**  mixs:handidness
============= ===============

Slot: has boolean value
=======================

Links a quantity value to a boolean

URI:
https://microbiomedata/schema/workflow_execution_activity/has_boolean_value

.. _domain-and-range-307:

Domain and Range
----------------

None → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-344:

Parents
-------

.. _children-314:

Children
--------

.. _used-by-307:

Used by
-------

-  `BooleanValue <BooleanValue.md>`__

Slot: has calibration
=====================

TODO: Yuri to fill in

URI:
https://microbiomedata/schema/workflow_execution_activity/has_calibration

.. _domain-and-range-308:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-345:

Parents
-------

.. _children-315:

Children
--------

-  `metabolomics analysis activity➞has
   calibration <metabolomics_analysis_activity_has_calibration.md>`__
-  `nom analysis activity➞has
   calibration <nom_analysis_activity_has_calibration.md>`__

.. _used-by-308:

Used by
-------

Slot: has credit associations
=============================

This slot links a study to a credit association. The credit association
will be linked to a person value and to a CRediT Contributor Roles term.
Overall semantics: person should get credit X for their participation in
the study

URI:
`nmdc:has_credit_associations <https://microbiomedata/meta/has_credit_associations>`__

.. _domain-and-range-309:

Domain and Range
----------------

`Study <Study.md>`__ → 0..\*
`CreditAssociation <CreditAssociation.md>`__

.. _parents-346:

Parents
-------

.. _children-316:

Children
--------

.. _used-by-309:

Used by
-------

-  `Study <Study.md>`__

.. _other-properties-275:

Other properties
----------------

============= =========================
**Mappings:**  prov:qualifiedAssociation
============= =========================

.. _slot-has-function-1:

Slot: has function
==================

URI: `nmdc:has_function <https://microbiomedata/meta/has_function>`__

.. _domain-and-range-310:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-347:

Parents
-------

.. _children-317:

Children
--------

-  `functional annotation➞has
   function <functional_annotation_has_function.md>`__

.. _used-by-310:

Used by
-------

.. _other-properties-276:

Other properties
----------------

+-----------------------+-----------------------+-----------------------+
| **Comments:**         |                       | the range for         |
|                       |                       | has_function was      |
|                       |                       | asserted as           |
|                       |                       | functi                |
|                       |                       | onal_annotation_term, |
+-----------------------+-----------------------+-----------------------+
|                       |                       | but is actually       |
|                       |                       | taking string         |
|                       |                       | arguments in the      |
|                       |                       | Polyneme MongoDB,     |
+-----------------------+-----------------------+-----------------------+
|                       |                       | and those are         |
|                       |                       | frequently fulltext,  |
|                       |                       | not CURIEs. MAM       |
|                       |                       | 2021-06-23            |
+-----------------------+-----------------------+-----------------------+

.. _slot-has-input-1:

Slot: has input
===============

An input to a process.

URI: https://microbiomedata/schema/workflow_execution_activity/has_input

.. _domain-and-range-311:

Domain and Range
----------------

`NamedThing <NamedThing.md>`__ → 0..\* `NamedThing <NamedThing.md>`__

.. _parents-348:

Parents
-------

.. _children-318:

Children
--------

-  `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__

.. _used-by-311:

Used by
-------

.. _other-properties-277:

Other properties
----------------

============ =====
**Aliases:**  input
============ =====

Slot: has maximum numeric value
===============================

The maximum value part, expressed as number, of the quantity value when
the value covers a range.

URI:
https://microbiomedata/schema/workflow_execution_activity/has_maximum_numeric_value

.. _domain-and-range-312:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-349:

Parents
-------

-  is_a: `has numeric value <has_numeric_value.md>`__

.. _children-319:

Children
--------

.. _used-by-312:

Used by
-------

-  `QuantityValue <QuantityValue.md>`__

Slot: has metabolite quantifications
====================================

URI:
https://microbiomedata/schema/workflow_execution_activity/has_metabolite_quantifications

.. _domain-and-range-313:

Domain and Range
----------------

None → 0..\* `MetaboliteQuantification <MetaboliteQuantification.md>`__

.. _parents-350:

Parents
-------

.. _children-320:

Children
--------

-  `metabolomics analysis activity➞has metabolite
   quantifications <metabolomics_analysis_activity_has_metabolite_quantifications.md>`__

.. _used-by-313:

Used by
-------

Slot: has minimum numeric value
===============================

The minimum value part, expressed as number, of the quantity value when
the value covers a range.

URI:
https://microbiomedata/schema/workflow_execution_activity/has_minimum_numeric_value

.. _domain-and-range-314:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-351:

Parents
-------

-  is_a: `has numeric value <has_numeric_value.md>`__

.. _children-321:

Children
--------

.. _used-by-314:

Used by
-------

-  `QuantityValue <QuantityValue.md>`__

Slot: has numeric value
=======================

Links a quantity value to a number

URI:
https://microbiomedata/schema/workflow_execution_activity/has_numeric_value

.. _domain-and-range-315:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-352:

Parents
-------

.. _children-322:

Children
--------

-  `has maximum numeric value <has_maximum_numeric_value.md>`__
-  `has minimum numeric value <has_minimum_numeric_value.md>`__
-  `quantity value➞has numeric
   value <quantity_value_has_numeric_value.md>`__

.. _used-by-315:

Used by
-------

-  `IntegerValue <IntegerValue.md>`__

.. _other-properties-278:

Other properties
----------------

============= =================
**Mappings:**  qud:quantityValue
\              schema:value
============= =================

Slot: has output
================

An output biosample to a processing step

URI:
https://microbiomedata/schema/workflow_execution_activity/has_output

.. _domain-and-range-316:

Domain and Range
----------------

`NamedThing <NamedThing.md>`__ → 0..\* `NamedThing <NamedThing.md>`__

.. _parents-353:

Parents
-------

.. _children-323:

Children
--------

-  `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__

.. _used-by-316:

Used by
-------

.. _other-properties-279:

Other properties
----------------

============ ======
**Aliases:**  output
============ ======

Slot: has_part
==============

A pathway can be broken down to a series of reaction step

URI: `nmdc:has_part <https://microbiomedata/meta/has_part>`__

.. _domain-and-range-317:

Domain and Range
----------------

None → 0..\* `Reaction <Reaction.md>`__

.. _parents-354:

Parents
-------

.. _children-324:

Children
--------

-  `pathway➞has_part <pathway_has_part.md>`__

.. _used-by-317:

Used by
-------

Slot: has participants
======================

URI:
`nmdc:has_participants <https://microbiomedata/meta/has_participants>`__

.. _domain-and-range-318:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-355:

Parents
-------

.. _children-325:

Children
--------

-  `left participants <left_participants.md>`__
-  `reaction➞left participants <reaction_left_participants.md>`__
-  `reaction➞right participants <reaction_right_participants.md>`__
-  `right participants <right_participants.md>`__

.. _used-by-318:

Used by
-------

.. _other-properties-280:

Other properties
----------------

=================== ==========
**Exact Mappings:**  RO:0000057
=================== ==========

Slot: has peptide quantifications
=================================

URI:
https://microbiomedata/schema/workflow_execution_activity/has_peptide_quantifications

.. _domain-and-range-319:

Domain and Range
----------------

None → 0..\* `PeptideQuantification <PeptideQuantification.md>`__

.. _parents-356:

Parents
-------

.. _children-326:

Children
--------

-  `metaproteomics analysis activity➞has peptide
   quantifications <metaproteomics_analysis_activity_has_peptide_quantifications.md>`__

.. _used-by-319:

Used by
-------

.. _slot-has-raw-value-1:

Slot: has raw value
===================

The value that was specified for an annotation in raw form, i.e. a
string. E.g. “2 cm” or “2-4 cm”

URI:
https://microbiomedata/schema/workflow_execution_activity/has_raw_value

.. _domain-and-range-320:

Domain and Range
----------------

`AttributeValue <AttributeValue.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-357:

Parents
-------

.. _children-327:

Children
--------

-  `geolocation value➞has raw
   value <geolocation_value_has_raw_value.md>`__
-  `person value➞has raw value <person_value_has_raw_value.md>`__
-  `quantity value➞has raw value <quantity_value_has_raw_value.md>`__

.. _used-by-320:

Used by
-------

-  `AttributeValue <AttributeValue.md>`__
-  `BooleanValue <BooleanValue.md>`__
-  `ControlledTermValue <ControlledTermValue.md>`__
-  `ImageValue <ImageValue.md>`__
-  `IntegerValue <IntegerValue.md>`__
-  `TextValue <TextValue.md>`__
-  `TimestampValue <TimestampValue.md>`__
-  `UrlValue <UrlValue.md>`__

Slot: has unit
==============

Links a quantity value to a unit

URI: https://microbiomedata/schema/workflow_execution_activity/has_unit

.. _domain-and-range-321:

Domain and Range
----------------

None → 0..1 `Unit <types/Unit.md>`__

.. _parents-358:

Parents
-------

.. _children-328:

Children
--------

-  `quantity value➞has unit <quantity_value_has_unit.md>`__

.. _used-by-321:

Used by
-------

.. _other-properties-281:

Other properties
----------------

============= ===============
**Aliases:**   scale
**Mappings:**  qud:unit
\              schema:unitCode
============= ===============

Slot: hc_produced
=================

“Main hydrocarbon type produced from resource (i.e. Oil, gas,
condensate, etc). If”“other”” is specified, please propose entry in
““additional info”” field”

URI: `nmdc:hc_produced <https://microbiomedata/meta/hc_produced>`__

.. _domain-and-range-322:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-359:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-329:

Children
--------

.. _used-by-322:

Used by
-------

.. _other-properties-282:

Other properties
----------------

============= =========================
**Aliases:**   hydrocarbon type produced
**Mappings:**  mixs:hc_produced
============= =========================

Slot: hcr
=========

“Main Hydrocarbon Resource type. The term”“Hydrocarbon Resource”” HCR
defined as a natural environmental feature containing large amounts of
hydrocarbons at high concentrations potentially suitable for commercial
exploitation. This term should not be confused with the Hydrocarbon
Occurrence term which also includes hydrocarbon-rich environments with
currently limited commercial interest such as seeps, outcrops, gas
hydrates etc. If ““other”” is specified, please propose entry in
““additional info”” field”

URI: `nmdc:hcr <https://microbiomedata/meta/hcr>`__

.. _domain-and-range-323:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-360:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-330:

Children
--------

.. _used-by-323:

Used by
-------

.. _other-properties-283:

Other properties
----------------

============= =========================
**Aliases:**   hydrocarbon resource type
**Mappings:**  mixs:hcr
============= =========================

Slot: hcr_fw_salinity
=====================

Original formation water salinity (prior to secondary recovery
e.g. Waterflooding) expressed as TDS

URI:
`nmdc:hcr_fw_salinity <https://microbiomedata/meta/hcr_fw_salinity>`__

.. _domain-and-range-324:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-361:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-331:

Children
--------

.. _used-by-324:

Used by
-------

.. _other-properties-284:

Other properties
----------------

============= ========================
**Aliases:**   formation water salinity
**Mappings:**  mixs:hcr_fw_salinity
============= ========================

Slot: hcr_geol_age
==================

“Geological age of hydrocarbon resource (Additional info:
https://en.wikipedia.org/wiki/Period_(geology)). If”“other”” is
specified, please propose entry in ““additional info”” field”

URI: `nmdc:hcr_geol_age <https://microbiomedata/meta/hcr_geol_age>`__

.. _domain-and-range-325:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-362:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-332:

Children
--------

.. _used-by-325:

Used by
-------

.. _other-properties-285:

Other properties
----------------

============= ===================================
**Aliases:**   hydrocarbon resource geological age
**Mappings:**  mixs:hcr_geol_age
============= ===================================

Slot: hcr_pressure
==================

Original pressure of the hydrocarbon resource

URI: `nmdc:hcr_pressure <https://microbiomedata/meta/hcr_pressure>`__

.. _domain-and-range-326:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-363:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-333:

Children
--------

.. _used-by-326:

Used by
-------

.. _other-properties-286:

Other properties
----------------

============= ======================================
**Aliases:**   hydrocarbon resource original pressure
**Mappings:**  mixs:hcr_pressure
============= ======================================

Slot: hcr_temp
==============

Original temperature of the hydrocarbon resource

URI: `nmdc:hcr_temp <https://microbiomedata/meta/hcr_temp>`__

.. _domain-and-range-327:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-364:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-334:

Children
--------

.. _used-by-327:

Used by
-------

.. _other-properties-287:

Other properties
----------------

============= =========================================
**Aliases:**   hydrocarbon resource original temperature
**Mappings:**  mixs:hcr_temp
============= =========================================

Slot: health_disease_stat
=========================

Health or disease status of specific host at time of collection

URI:
`nmdc:health_disease_stat <https://microbiomedata/meta/health_disease_stat>`__

.. _domain-and-range-328:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-365:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-335:

Children
--------

.. _used-by-328:

Used by
-------

.. _other-properties-288:

Other properties
----------------

+-----------------+---+----------------------------------------------+
| **Aliases:**    |   | health or disease status of specific host at |
|                 |   | time of collection                           |
+-----------------+---+----------------------------------------------+
| **Mappings:**   |   | mixs:health_disease_stat                     |
+-----------------+---+----------------------------------------------+
| **In Subsets:** |   | nucleic acid sequence source                 |
+-----------------+---+----------------------------------------------+

Slot: heat_cool_type
====================

Methods of conditioning or heating a room or building

URI:
`nmdc:heat_cool_type <https://microbiomedata/meta/heat_cool_type>`__

.. _domain-and-range-329:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-366:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-336:

Children
--------

.. _used-by-329:

Used by
-------

.. _other-properties-289:

Other properties
----------------

============= ===============================
**Aliases:**   heating and cooling system type
**Mappings:**  mixs:heat_cool_type
============= ===============================

Slot: heat_deliv_loc
====================

The location of heat delivery within the room

URI:
`nmdc:heat_deliv_loc <https://microbiomedata/meta/heat_deliv_loc>`__

.. _domain-and-range-330:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-367:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-337:

Children
--------

.. _used-by-330:

Used by
-------

.. _other-properties-290:

Other properties
----------------

============= ==========================
**Aliases:**   heating delivery locations
**Mappings:**  mixs:heat_deliv_loc
============= ==========================

Slot: heat_system_deliv_meth
============================

The method by which the heat is delivered through the system

URI:
`nmdc:heat_system_deliv_meth <https://microbiomedata/meta/heat_system_deliv_meth>`__

.. _domain-and-range-331:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-368:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-338:

Children
--------

.. _used-by-331:

Used by
-------

.. _other-properties-291:

Other properties
----------------

============= ==============================
**Aliases:**   heating system delivery method
**Mappings:**  mixs:heat_system_deliv_meth
============= ==============================

Slot: heat_system_id
====================

The heating system identifier

URI:
`nmdc:heat_system_id <https://microbiomedata/meta/heat_system_id>`__

.. _domain-and-range-332:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-369:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-339:

Children
--------

.. _used-by-332:

Used by
-------

.. _other-properties-292:

Other properties
----------------

============= =========================
**Aliases:**   heating system identifier
**Mappings:**  mixs:heat_system_id
============= =========================

Slot: heavy_metals
==================

Heavy metals present and concentrationsany drug used by subject and the
frequency of usage; can include multiple heavy metals and concentrations

URI: `nmdc:heavy_metals <https://microbiomedata/meta/heavy_metals>`__

.. _domain-and-range-333:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-370:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-340:

Children
--------

.. _used-by-333:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-293:

Other properties
----------------

============= =======================================
**Aliases:**   extreme_unusual_properties/heavy metals
**Mappings:**  mixs:heavy_metals
============= =======================================

Slot: heavy_metals_meth
=======================

Reference or method used in determining heavy metals

URI:
`nmdc:heavy_metals_meth <https://microbiomedata/meta/heavy_metals_meth>`__

.. _domain-and-range-334:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-371:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-341:

Children
--------

.. _used-by-334:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-294:

Other properties
----------------

============= ==============================================
**Aliases:**   extreme_unusual_properties/heavy metals method
**Mappings:**  mixs:heavy_metals_meth
============= ==============================================

Slot: height_carper_fiber
=========================

The average carpet fiber height in the indoor environment

URI:
`nmdc:height_carper_fiber <https://microbiomedata/meta/height_carper_fiber>`__

.. _domain-and-range-335:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-372:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-342:

Children
--------

.. _used-by-335:

Used by
-------

.. _other-properties-295:

Other properties
----------------

============= ========================
**Aliases:**   height carpet fiber mat
**Mappings:**  mixs:height_carper_fiber
============= ========================

Slot: herbicide_regm
====================

“Information about treatment involving use of herbicides; information
about treatment involving use of growth hormones; should include the
name of herbicide, amount administered, treatment regimen including how
many times the treatment was repeated, how long each treatment lasted,
and the start and end time of the entire treatment; can include multiple
regimens”

URI:
`nmdc:herbicide_regm <https://microbiomedata/meta/herbicide_regm>`__

.. _domain-and-range-336:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-373:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-343:

Children
--------

.. _used-by-336:

Used by
-------

.. _other-properties-296:

Other properties
----------------

============= ===================
**Aliases:**   herbicide regimen
**Mappings:**  mixs:herbicide_regm
============= ===================

Slot: highest similarity score
==============================

TODO: Yuri to fill in

URI:
https://microbiomedata/schema/workflow_execution_activity/highest_similarity_score

.. _domain-and-range-337:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-374:

Parents
-------

.. _children-344:

Children
--------

-  `metabolite quantification➞highest similarity
   score <metabolite_quantification_highest_similarity_score.md>`__

.. _used-by-337:

Used by
-------

Slot: horizon
=============

Specific layer in the land area which measures parallel to the soil
surface and possesses physical characteristics which differ from the
layers above and beneath

URI: `nmdc:horizon <https://microbiomedata/meta/horizon>`__

.. _domain-and-range-338:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-375:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-345:

Children
--------

.. _used-by-338:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-297:

Other properties
----------------

============= ============
**Aliases:**   horizon
**Mappings:**  mixs:horizon
============= ============

Slot: horizon_meth
==================

Reference or method used in determining the horizon

URI: `nmdc:horizon_meth <https://microbiomedata/meta/horizon_meth>`__

.. _domain-and-range-339:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-376:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-346:

Children
--------

.. _used-by-339:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-298:

Other properties
----------------

============= =================
**Aliases:**   horizon method
**Mappings:**  mixs:horizon_meth
============= =================

Slot: host_age
==============

“Age of host at the time of sampling; relevant scale depends on species
and study, e.g. Could be seconds for amoebae or centuries for trees”

URI: `nmdc:host_age <https://microbiomedata/meta/host_age>`__

.. _domain-and-range-340:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-377:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-347:

Children
--------

.. _used-by-340:

Used by
-------

.. _other-properties-299:

Other properties
----------------

============= =============
**Aliases:**   host age
**Mappings:**  mixs:host_age
============= =============

Slot: host_blood_press_diast
============================

“Resting diastolic blood pressure, measured as mm mercury”

URI:
`nmdc:host_blood_press_diast <https://microbiomedata/meta/host_blood_press_diast>`__

.. _domain-and-range-341:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-378:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-348:

Children
--------

.. _used-by-341:

Used by
-------

.. _other-properties-300:

Other properties
----------------

============= =============================
**Aliases:**   host blood pressure diastolic
**Mappings:**  mixs:host_blood_press_diast
============= =============================

Slot: host_blood_press_syst
===========================

“Resting systolic blood pressure, measured as mm mercury”

URI:
`nmdc:host_blood_press_syst <https://microbiomedata/meta/host_blood_press_syst>`__

.. _domain-and-range-342:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-379:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-349:

Children
--------

.. _used-by-342:

Used by
-------

.. _other-properties-301:

Other properties
----------------

============= ============================
**Aliases:**   host blood pressure systolic
**Mappings:**  mixs:host_blood_press_syst
============= ============================

Slot: host_body_habitat
=======================

Original body habitat where the sample was obtained from

URI:
`nmdc:host_body_habitat <https://microbiomedata/meta/host_body_habitat>`__

.. _domain-and-range-343:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-380:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-350:

Children
--------

.. _used-by-343:

Used by
-------

.. _other-properties-302:

Other properties
----------------

============= ======================
**Aliases:**   host body habitat
**Mappings:**  mixs:host_body_habitat
============= ======================

Slot: host_body_mass_index
==========================

“Body mass index, calculated as weight/(height)squared”

URI:
`nmdc:host_body_mass_index <https://microbiomedata/meta/host_body_mass_index>`__

.. _domain-and-range-344:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-381:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-351:

Children
--------

.. _used-by-344:

Used by
-------

.. _other-properties-303:

Other properties
----------------

============= =========================
**Aliases:**   host body-mass index
**Mappings:**  mixs:host_body_mass_index
============= =========================

Slot: host_body_product
=======================

“Substance produced by the body, e.g. Stool, mucus, where the sample was
obtained from. For foundational model of anatomy ontology (fma) or
Uber-anatomy ontology (UBERON) terms, please see
https://www.ebi.ac.uk/ols/ontologies/fma or
https://www.ebi.ac.uk/ols/ontologies/uberon”

URI:
`nmdc:host_body_product <https://microbiomedata/meta/host_body_product>`__

.. _domain-and-range-345:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-382:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-352:

Children
--------

.. _used-by-345:

Used by
-------

.. _other-properties-304:

Other properties
----------------

============= ======================
**Aliases:**   host body product
**Mappings:**  mixs:host_body_product
============= ======================

Slot: host_body_site
====================

“Name of body site where the sample was obtained from, such as a
specific organ or tissue (tongue, lung etc…). For foundational model of
anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v
releases/2014-06-15) terms, please see
http://purl.bioontology.org/ontology/FMA or
http://purl.bioontology.org/ontology/UBERON”

URI:
`nmdc:host_body_site <https://microbiomedata/meta/host_body_site>`__

.. _domain-and-range-346:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-383:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-353:

Children
--------

.. _used-by-346:

Used by
-------

.. _other-properties-305:

Other properties
----------------

============= ===================
**Aliases:**   host body site
**Mappings:**  mixs:host_body_site
============= ===================

Slot: host_body_temp
====================

Core body temperature of the host when sample was collected

URI:
`nmdc:host_body_temp <https://microbiomedata/meta/host_body_temp>`__

.. _domain-and-range-347:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-384:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-354:

Children
--------

.. _used-by-347:

Used by
-------

.. _other-properties-306:

Other properties
----------------

============= =====================
**Aliases:**   host body temperature
**Mappings:**  mixs:host_body_temp
============= =====================

Slot: host_color
================

The color of host

URI: `nmdc:host_color <https://microbiomedata/meta/host_color>`__

.. _domain-and-range-348:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-385:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-355:

Children
--------

.. _used-by-348:

Used by
-------

.. _other-properties-307:

Other properties
----------------

============= ===============
**Aliases:**   host color
**Mappings:**  mixs:host_color
============= ===============

Slot: host_common_name
======================

“Common name of the host, e.g. Human”

URI:
`nmdc:host_common_name <https://microbiomedata/meta/host_common_name>`__

.. _domain-and-range-349:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-386:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-356:

Children
--------

.. _used-by-349:

Used by
-------

.. _other-properties-308:

Other properties
----------------

============= =====================
**Aliases:**   host common name
**Mappings:**  mixs:host_common_name
============= =====================

Slot: host_diet
===============

“Type of diet depending on the host, for animals omnivore, herbivore
etc., for humans high-fat, meditteranean etc.; can include multiple diet
types”

URI: `nmdc:host_diet <https://microbiomedata/meta/host_diet>`__

.. _domain-and-range-350:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-387:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-357:

Children
--------

.. _used-by-350:

Used by
-------

.. _other-properties-309:

Other properties
----------------

============= ==============
**Aliases:**   host diet
**Mappings:**  mixs:host_diet
============= ==============

Slot: host_disease_stat
=======================

“List of diseases with which the host has been diagnosed; can include
multiple diagnoses. The value of the field depends on host; for humans
the terms should be chosen from do (disease ontology) at
http://www.disease-ontology.org, other hosts are free text”

URI:
`nmdc:host_disease_stat <https://microbiomedata/meta/host_disease_stat>`__

.. _domain-and-range-351:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-388:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-358:

Children
--------

.. _used-by-351:

Used by
-------

.. _other-properties-310:

Other properties
----------------

============= ======================
**Aliases:**   host disease status
**Mappings:**  mixs:host_disease_stat
============= ======================

Slot: host_dry_mass
===================

Measurement of dry mass

URI: `nmdc:host_dry_mass <https://microbiomedata/meta/host_dry_mass>`__

.. _domain-and-range-352:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-389:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-359:

Children
--------

.. _used-by-352:

Used by
-------

.. _other-properties-311:

Other properties
----------------

============= ==================
**Aliases:**   host dry mass
**Mappings:**  mixs:host_dry_mass
============= ==================

Slot: host_family_relationship
==============================

Relationships to other hosts in the same study; can include multiple
relationships

URI:
`nmdc:host_family_relationship <https://microbiomedata/meta/host_family_relationship>`__

.. _domain-and-range-353:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-390:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-360:

Children
--------

.. _used-by-353:

Used by
-------

.. _other-properties-312:

Other properties
----------------

============= =============================
**Aliases:**   host family relationship
**Mappings:**  mixs:host_family_relationship
============= =============================

Slot: host_genotype
===================

Observed genotype

URI: `nmdc:host_genotype <https://microbiomedata/meta/host_genotype>`__

.. _domain-and-range-354:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-391:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-361:

Children
--------

.. _used-by-354:

Used by
-------

.. _other-properties-313:

Other properties
----------------

============= ==================
**Aliases:**   host genotype
**Mappings:**  mixs:host_genotype
============= ==================

Slot: host_growth_cond
======================

Literature reference giving growth conditions of the host

URI:
`nmdc:host_growth_cond <https://microbiomedata/meta/host_growth_cond>`__

.. _domain-and-range-355:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-392:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-362:

Children
--------

.. _used-by-355:

Used by
-------

.. _other-properties-314:

Other properties
----------------

============= ======================
**Aliases:**   host growth conditions
**Mappings:**  mixs:host_growth_cond
============= ======================

Slot: host_height
=================

The height of subject

URI: `nmdc:host_height <https://microbiomedata/meta/host_height>`__

.. _domain-and-range-356:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-393:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-363:

Children
--------

.. _used-by-356:

Used by
-------

.. _other-properties-315:

Other properties
----------------

============= ================
**Aliases:**   host height
**Mappings:**  mixs:host_height
============= ================

Slot: host_hiv_stat
===================

“HIV status of subject, if yes HAART initiation status should also be
indicated as [YES or NO]”

URI: `nmdc:host_hiv_stat <https://microbiomedata/meta/host_hiv_stat>`__

.. _domain-and-range-357:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-394:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-364:

Children
--------

.. _used-by-357:

Used by
-------

.. _other-properties-316:

Other properties
----------------

============= ==================
**Aliases:**   host HIV status
**Mappings:**  mixs:host_hiv_stat
============= ==================

Slot: host_infra_specific_name
==============================

Taxonomic information about the host below subspecies level

URI:
`nmdc:host_infra_specific_name <https://microbiomedata/meta/host_infra_specific_name>`__

.. _domain-and-range-358:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-395:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-365:

Children
--------

.. _used-by-358:

Used by
-------

.. _other-properties-317:

Other properties
----------------

============= =============================
**Aliases:**   host infra-specific name
**Mappings:**  mixs:host_infra_specific_name
============= =============================

Slot: host_infra_specific_rank
==============================

“Taxonomic rank information about the host below subspecies level, such
as variety, form, rank etc.”

URI:
`nmdc:host_infra_specific_rank <https://microbiomedata/meta/host_infra_specific_rank>`__

.. _domain-and-range-359:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-396:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-366:

Children
--------

.. _used-by-359:

Used by
-------

.. _other-properties-318:

Other properties
----------------

============= =============================
**Aliases:**   host infra-specific rank
**Mappings:**  mixs:host_infra_specific_rank
============= =============================

Slot: host_last_meal
====================

Content of last meal and time since feeding; can include multiple values

URI:
`nmdc:host_last_meal <https://microbiomedata/meta/host_last_meal>`__

.. _domain-and-range-360:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-397:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-367:

Children
--------

.. _used-by-360:

Used by
-------

.. _other-properties-319:

Other properties
----------------

============= ===================
**Aliases:**   host last meal
**Mappings:**  mixs:host_last_meal
============= ===================

Slot: host_length
=================

The length of subject

URI: `nmdc:host_length <https://microbiomedata/meta/host_length>`__

.. _domain-and-range-361:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-398:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-368:

Children
--------

.. _used-by-361:

Used by
-------

.. _other-properties-320:

Other properties
----------------

============= ================
**Aliases:**   host length
**Mappings:**  mixs:host_length
============= ================

Slot: host_life_stage
=====================

Description of life stage of host

URI:
`nmdc:host_life_stage <https://microbiomedata/meta/host_life_stage>`__

.. _domain-and-range-362:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-399:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-369:

Children
--------

.. _used-by-362:

Used by
-------

.. _other-properties-321:

Other properties
----------------

============= ====================
**Aliases:**   host life stage
**Mappings:**  mixs:host_life_stage
============= ====================

Slot: host_name
===============

URI: `nmdc:host_name <https://microbiomedata/meta/host_name>`__

.. _domain-and-range-363:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-400:

Parents
-------

.. _children-370:

Children
--------

.. _used-by-363:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: host_occupation
=====================

Most frequent job performed by subject

URI:
`nmdc:host_occupation <https://microbiomedata/meta/host_occupation>`__

.. _domain-and-range-364:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-401:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-371:

Children
--------

.. _used-by-364:

Used by
-------

.. _other-properties-322:

Other properties
----------------

============= ====================
**Aliases:**   host occupation
**Mappings:**  mixs:host_occupation
============= ====================

Slot: host_phenotype
====================

“Phenotype of human or other host. For phenotypic quality ontology
(pato) (v 2018-03-27) terms, please see
http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology
(HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP”

URI:
`nmdc:host_phenotype <https://microbiomedata/meta/host_phenotype>`__

.. _domain-and-range-365:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-402:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-372:

Children
--------

.. _used-by-365:

Used by
-------

.. _other-properties-323:

Other properties
----------------

============= ===================
**Aliases:**   host phenotype
**Mappings:**  mixs:host_phenotype
============= ===================

Slot: host_pred_appr
====================

Tool or approach used for host prediction

URI:
`nmdc:host_pred_appr <https://microbiomedata/meta/host_pred_appr>`__

.. _domain-and-range-366:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-403:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-373:

Children
--------

.. _used-by-366:

Used by
-------

.. _other-properties-324:

Other properties
----------------

=============== ========================
**Aliases:**     host prediction approach
**Mappings:**    mixs:host_pred_appr
**In Subsets:**  sequencing
=============== ========================

Slot: host_pred_est_acc
=======================

“For each tool or approach used for host prediction, estimated false
discovery rates should be included, either computed de novo or from the
literature”

URI:
`nmdc:host_pred_est_acc <https://microbiomedata/meta/host_pred_est_acc>`__

.. _domain-and-range-367:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-404:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-374:

Children
--------

.. _used-by-367:

Used by
-------

.. _other-properties-325:

Other properties
----------------

=============== ==================================
**Aliases:**     host prediction estimated accuracy
**Mappings:**    mixs:host_pred_est_acc
**In Subsets:**  sequencing
=============== ==================================

Slot: host_pulse
================

“Resting pulse, measured as beats per minute”

URI: `nmdc:host_pulse <https://microbiomedata/meta/host_pulse>`__

.. _domain-and-range-368:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-405:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-375:

Children
--------

.. _used-by-368:

Used by
-------

.. _other-properties-326:

Other properties
----------------

============= ===============
**Aliases:**   host pulse
**Mappings:**  mixs:host_pulse
============= ===============

Slot: host_sex
==============

Physical sex of the host

URI: `nmdc:host_sex <https://microbiomedata/meta/host_sex>`__

.. _domain-and-range-369:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-406:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-376:

Children
--------

.. _used-by-369:

Used by
-------

.. _other-properties-327:

Other properties
----------------

============= =============
**Aliases:**   host sex
**Mappings:**  mixs:host_sex
============= =============

Slot: host_shape
================

Morphological shape of host

URI: `nmdc:host_shape <https://microbiomedata/meta/host_shape>`__

.. _domain-and-range-370:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-407:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-377:

Children
--------

.. _used-by-370:

Used by
-------

.. _other-properties-328:

Other properties
----------------

============= ===============
**Aliases:**   host shape
**Mappings:**  mixs:host_shape
============= ===============

Slot: host_spec_range
=====================

The NCBI taxonomy identifier of the specific host if it is known

URI:
`nmdc:host_spec_range <https://microbiomedata/meta/host_spec_range>`__

.. _domain-and-range-371:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-408:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-378:

Children
--------

.. _used-by-371:

Used by
-------

.. _other-properties-329:

Other properties
----------------

=============== ============================
**Aliases:**     host specificity or range
**Mappings:**    mixs:host_spec_range
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: host_subject_id
=====================

“A unique identifier by which each subject can be referred to,
de-identified, e.g. #131”

URI:
`nmdc:host_subject_id <https://microbiomedata/meta/host_subject_id>`__

.. _domain-and-range-372:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-409:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-379:

Children
--------

.. _used-by-372:

Used by
-------

.. _other-properties-330:

Other properties
----------------

============= ====================
**Aliases:**   host subject id
**Mappings:**  mixs:host_subject_id
============= ====================

Slot: host_substrate
====================

The growth substrate of the host

URI:
`nmdc:host_substrate <https://microbiomedata/meta/host_substrate>`__

.. _domain-and-range-373:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-410:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-380:

Children
--------

.. _used-by-373:

Used by
-------

.. _other-properties-331:

Other properties
----------------

============= ===================
**Aliases:**   host substrate
**Mappings:**  mixs:host_substrate
============= ===================

Slot: host_taxid
================

“NCBI taxon id of the host, e.g. 9606”

URI: `nmdc:host_taxid <https://microbiomedata/meta/host_taxid>`__

.. _domain-and-range-374:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-411:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-381:

Children
--------

.. _used-by-374:

Used by
-------

.. _other-properties-332:

Other properties
----------------

============= ===============
**Aliases:**   host taxid
**Mappings:**  mixs:host_taxid
============= ===============

Slot: host_tot_mass
===================

“Total mass of the host at collection, the unit depends on host”

URI: `nmdc:host_tot_mass <https://microbiomedata/meta/host_tot_mass>`__

.. _domain-and-range-375:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-412:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-382:

Children
--------

.. _used-by-375:

Used by
-------

.. _other-properties-333:

Other properties
----------------

============= ==================
**Aliases:**   host total mass
**Mappings:**  mixs:host_tot_mass
============= ==================

Slot: host_wet_mass
===================

Measurement of wet mass

URI: `nmdc:host_wet_mass <https://microbiomedata/meta/host_wet_mass>`__

.. _domain-and-range-376:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-413:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-383:

Children
--------

.. _used-by-376:

Used by
-------

.. _other-properties-334:

Other properties
----------------

============= ==================
**Aliases:**   host wet mass
**Mappings:**  mixs:host_wet_mass
============= ==================

Slot: hrt
=========

“Whether subject had hormone replacement theraphy, and if yes start
date”

URI: `nmdc:hrt <https://microbiomedata/meta/hrt>`__

.. _domain-and-range-377:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-414:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-384:

Children
--------

.. _used-by-377:

Used by
-------

.. _other-properties-335:

Other properties
----------------

============= ========
**Aliases:**   HRT
**Mappings:**  mixs:hrt
============= ========

Slot: humidity
==============

“Amount of water vapour in the air, at the time of sampling”

URI: `nmdc:humidity <https://microbiomedata/meta/humidity>`__

.. _domain-and-range-378:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-415:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-385:

Children
--------

.. _used-by-378:

Used by
-------

.. _other-properties-336:

Other properties
----------------

============= =============
**Aliases:**   humidity
**Mappings:**  mixs:humidity
============= =============

Slot: humidity_regm
===================

“Information about treatment involving an exposure to varying degree of
humidity; information about treatment involving use of growth hormones;
should include amount of humidity administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple regimens”

URI: `nmdc:humidity_regm <https://microbiomedata/meta/humidity_regm>`__

.. _domain-and-range-379:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-416:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-386:

Children
--------

.. _used-by-379:

Used by
-------

.. _other-properties-337:

Other properties
----------------

============= ==================
**Aliases:**   humidity regimen
**Mappings:**  mixs:humidity_regm
============= ==================

Slot: hysterectomy
==================

Specification of whether hysterectomy was performed

URI: `nmdc:hysterectomy <https://microbiomedata/meta/hysterectomy>`__

.. _domain-and-range-380:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-417:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-387:

Children
--------

.. _used-by-380:

Used by
-------

.. _other-properties-338:

Other properties
----------------

============= =================
**Aliases:**   hysterectomy
**Mappings:**  mixs:hysterectomy
============= =================

Slot: id
========

A unique identifier for a thing. Must be either a CURIE shorthand for a
URI or a complete URI

URI: https://microbiomedata/schema/workflow_execution_activity/id

.. _domain-and-range-381:

Domain and Range
----------------

None → 1..1 `String <types/String.md>`__

.. _parents-418:

Parents
-------

.. _children-388:

Children
--------

-  `person➞id <person_id.md>`__

.. _used-by-381:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `Activity <Activity.md>`__
-  `ChemicalEntity <ChemicalEntity.md>`__
-  `EnvironmentalMaterialTerm <EnvironmentalMaterialTerm.md>`__
-  `GeneProduct <GeneProduct.md>`__
-  `Instrument <Instrument.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NamedThing <NamedThing.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `OntologyClass <OntologyClass.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

Slot: identifier
================

URI: `nmdc:identifier <https://microbiomedata/meta/identifier>`__

.. _domain-and-range-382:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-419:

Parents
-------

.. _children-389:

Children
--------

.. _used-by-382:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: ihmc_ethnicity
====================

Ethnicity of the subject

URI:
`nmdc:ihmc_ethnicity <https://microbiomedata/meta/ihmc_ethnicity>`__

.. _domain-and-range-383:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-420:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-390:

Children
--------

.. _used-by-383:

Used by
-------

.. _other-properties-339:

Other properties
----------------

============= ===================
**Aliases:**   IHMC ethnicity
**Mappings:**  mixs:ihmc_ethnicity
============= ===================

Slot: ihmc_medication_code
==========================

Can include multiple medication codes

URI:
`nmdc:ihmc_medication_code <https://microbiomedata/meta/ihmc_medication_code>`__

.. _domain-and-range-384:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-421:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-391:

Children
--------

.. _used-by-384:

Used by
-------

.. _other-properties-340:

Other properties
----------------

============= =========================
**Aliases:**   IHMC medication code
**Mappings:**  mixs:ihmc_medication_code
============= =========================

.. _slot-inchi-1:

Slot: inchi
===========

URI: https://microbiomedata/schema/workflow_execution_activity/inchi

.. _domain-and-range-385:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-422:

Parents
-------

.. _children-392:

Children
--------

-  `chemical entity➞inchi <chemical_entity_inchi.md>`__

.. _used-by-385:

Used by
-------

.. _slot-inchi-key-1:

Slot: inchi key
===============

URI: https://microbiomedata/schema/workflow_execution_activity/inchi_key

.. _domain-and-range-386:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-423:

Parents
-------

.. _children-393:

Children
--------

-  `chemical entity➞inchi key <chemical_entity_inchi_key.md>`__

.. _used-by-386:

Used by
-------

NMDC-Workflow-Exectution schema
===============================

.. _classes-9:

Classes
-------

-  `MAGBin <MAGBin.md>`__
-  `Activity <Activity.md>`__ - a provence-generating activity

   -  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ -
      Represents an instance of an execution of a particular workflow

      -  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
      -  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
      -  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
      -  `MetagenomeAssembly <MetagenomeAssembly.md>`__
      -  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
      -  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__ -
         A metatranscriptome activity that e.g. pools assembly and
         annotation activity.
      -  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
      -  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
      -  `NomAnalysisActivity <NomAnalysisActivity.md>`__
      -  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
      -  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__

-  `Agent <Agent.md>`__ - a provence-generating agent
-  `AttributeValue <AttributeValue.md>`__ - The value for any value of a
   attribute for a sample. This object can hold both the un-normalized
   atomic value and the structured value

   -  `BooleanValue <BooleanValue.md>`__ - A value that is a boolean
   -  `ControlledTermValue <ControlledTermValue.md>`__ - A controlled
      term or class from an ontology
   -  `GeolocationValue <GeolocationValue.md>`__ - A normalized value
      for a location on the earth’s surface
   -  `ImageValue <ImageValue.md>`__ - An attribute value representing
      an image.
   -  `IntegerValue <IntegerValue.md>`__ - A value that is an integer
   -  `PersonValue <PersonValue.md>`__ - An attribute value representing
      a person
   -  `QuantityValue <QuantityValue.md>`__ - A simple quantity, e.g. 2cm
   -  `TextValue <TextValue.md>`__ - A basic string value
   -  `TimestampValue <TimestampValue.md>`__ - A value that is a
      timestamp. The range should be ISO-8601
   -  `UrlValue <UrlValue.md>`__ - A value that is a string that
      conforms to URL syntax

-  `MetaboliteQuantification <MetaboliteQuantification.md>`__ - This is
   used to link a metabolomics analysis workflow to a specific
   metabolite
-  `NamedThing <NamedThing.md>`__ - a databased entity or concept/class

   -  `GeneProduct <GeneProduct.md>`__ - A molecule encoded by a gene
      that has an evolved function
   -  `Instrument <Instrument.md>`__ - A material entity that is
      designed to perform a function in a scientific investigation, but
      is not a reagent[OBI].
   -  `OntologyClass <OntologyClass.md>`__

      -  `ChemicalEntity <ChemicalEntity.md>`__ - An atom or molecule
         that can be represented with a chemical formula. Include
         lipids, glycans, natural products, drugs. There may be
         different terms for distinct acid-base forms, protonation
         states
      -  `EnvironmentalMaterialTerm <EnvironmentalMaterialTerm.md>`__

   -  `Person <Person.md>`__ - represents a person, such as a researcher

-  `PeptideQuantification <PeptideQuantification.md>`__ - This is used
   to link a metaproteomics analysis workflow to a specific peptide
   sequence and related information
-  `ProteinQuantification <ProteinQuantification.md>`__ - This is used
   to link a metaproteomics analysis workflow to a specific protein

.. _mixins-9:

Mixins
------

.. _slots-9:

Slots
-----

-  `GOLD identifiers <GOLD_identifiers.md>`__
-  `INSDC identifiers <INSDC_identifiers.md>`__ - Any identifier covered
   by the International Nucleotide Sequence Database Collaboration
-  `MGnify identifiers <MGnify_identifiers.md>`__
-  `abstract <abstract.md>`__ - The abstract of manuscript/grant
   associated with the entity; i.e., a summary of the resource.
-  `acted on behalf of <acted_on_behalf_of.md>`__
-  `all proteins <all_proteins.md>`__ - the list of protein identifiers
   that are associated with the peptide sequence

   -  `peptide quantification➞all
      proteins <peptide_quantification_all_proteins.md>`__
   -  `protein quantification➞all
      proteins <protein_quantification_all_proteins.md>`__ - the grouped
      list of protein identifiers associated with the peptide sequences
      that were grouped to a best protein

-  `alternate emails <alternate_emails.md>`__ - One or more other email
   addresses for an entity.
-  `alternative descriptions <alternative_descriptions.md>`__ - A list
   of alternative descriptions for the entity. The distinction between
   desciption and alternative descriptions is application-specific.
-  `alternative identifiers <alternative_identifiers.md>`__ - A list of
   alternative identifiers for the entity.

   -  `external database
      identifiers <external_database_identifiers.md>`__ - Link to
      corresponding identifier in external database

      -  `GOLD sequencing project
         identifiers <GOLD_sequencing_project_identifiers.md>`__ -
         [‘identifiers for corresponding sequencing project in GOLD’]
      -  `INSDC experiment
         identifiers <INSDC_experiment_identifiers.md>`__
      -  `analysis identifiers <analysis_identifiers.md>`__

         -  `GOLD analysis project
            identifiers <GOLD_analysis_project_identifiers.md>`__ -
            [‘identifiers for corresponding analysis project in GOLD’]
         -  `INSDC analysis
            identifiers <INSDC_analysis_identifiers.md>`__
         -  `MGnify analysis
            identifiers <MGnify_analysis_identifiers.md>`__

      -  `omics processing
         identifiers <omics_processing_identifiers.md>`__
      -  `sample identifiers <sample_identifiers.md>`__

         -  `GOLD sample identifiers <GOLD_sample_identifiers.md>`__ -
            [‘identifiers for corresponding sample in GOLD’]
         -  `INSDC biosample
            identifiers <INSDC_biosample_identifiers.md>`__ -
            [‘identifiers for corresponding sample in INSDC’]
         -  `INSDC secondary sample
            identifiers <INSDC_secondary_sample_identifiers.md>`__ -
            [‘secondary identifiers for corresponding sample in INSDC’]

      -  `study identifiers <study_identifiers.md>`__

         -  `GOLD study identifiers <GOLD_study_identifiers.md>`__ -
            [‘identifiers for corresponding project in GOLD’]
         -  `INSDC SRA ENA study
            identifiers <INSDC_SRA_ENA_study_identifiers.md>`__ -
            [‘identifiers for corresponding project in INSDC SRA / ENA’]
         -  `INSDC bioproject
            identifiers <INSDC_bioproject_identifiers.md>`__ -
            [‘identifiers for corresponding project in INSDC
            Bioproject’]
         -  `MGnify project
            identifiers <MGnify_project_identifiers.md>`__ -
            [‘identifiers for corresponding project in MGnify’]

-  `alternative names <alternative_names.md>`__ - A list of alternative
   names used to refer to the entity. The distinction between name and
   alternative names is application-specific.
-  `alternative titles <alternative_titles.md>`__ - A list of
   alternative titles for the entity. The distinction between title and
   alternative titles is application-specific.
-  `assembly identifiers <assembly_identifiers.md>`__

   -  `INSDC assembly identifiers <INSDC_assembly_identifiers.md>`__

-  `attribute <attribute.md>`__ - A attribute of a biosample. Examples:
   depth, habitat, material. For NMDC, attributes SHOULD be mapped to
   terms within a MIxS template

   -  `display order <display_order.md>`__ - When rendering information,
      this attribute to specify the order in which the information
      should be rendered.
   -  `execution resource <execution_resource.md>`__ - Example:
      NERSC-Cori

      -  `workflow execution activity➞execution
         resource <workflow_execution_activity_execution_resource.md>`__

   -  `file size bytes <file_size_bytes.md>`__ - Size of the file in
      bytes
   -  `git url <git_url.md>`__ - Example:
      https://github.com/microbiomedata/mg_annotation/releases/tag/0.1

      -  `workflow execution activity➞git
         url <workflow_execution_activity_git_url.md>`__

   -  `md5 checksum <md5_checksum.md>`__ - MD5 checksum of file
      (pre-compressed)
   -  `url <url.md>`__

-  `best protein <best_protein.md>`__ - the specific protein identifier
   most correctly associated with the peptide sequence

   -  `peptide quantification➞best
      protein <peptide_quantification_best_protein.md>`__
   -  `protein quantification➞best
      protein <protein_quantification_best_protein.md>`__ - the specific
      protein identifier most correctly grouped to its associated
      peptide sequences

-  `binned contig num <binned_contig_num.md>`__
-  `chemical formula <chemical_formula.md>`__ - A generic grouping for
   miolecular formulae and empirican formulae

   -  `chemical entity➞chemical
      formula <chemical_entity_chemical_formula.md>`__

-  `description <description.md>`__ - a human-readable description of a
   thing
-  `email <email.md>`__ - An email address for an entity such as a
   person. This should be the primarly email address used.
-  `ended at time <ended_at_time.md>`__

   -  `workflow execution activity➞ended at
      time <workflow_execution_activity_ended_at_time.md>`__

-  `has boolean value <has_boolean_value.md>`__ - Links a quantity value
   to a boolean
-  `has calibration <has_calibration.md>`__ - TODO: Yuri to fill in

   -  `metabolomics analysis activity➞has
      calibration <metabolomics_analysis_activity_has_calibration.md>`__
   -  `nom analysis activity➞has
      calibration <nom_analysis_activity_has_calibration.md>`__ - A
      reference to a file that holds calibration information.

-  `has input <has_input.md>`__ - An input to a process.

   -  `workflow execution activity➞has
      input <workflow_execution_activity_has_input.md>`__

      -  `read QC analysis activity➞has
         input <read_QC_analysis_activity_has_input.md>`__

-  `has metabolite
   quantifications <has_metabolite_quantifications.md>`__

   -  `metabolomics analysis activity➞has metabolite
      quantifications <metabolomics_analysis_activity_has_metabolite_quantifications.md>`__

-  `has numeric value <has_numeric_value.md>`__ - Links a quantity value
   to a number

   -  `has maximum numeric value <has_maximum_numeric_value.md>`__ - The
      maximum value part, expressed as number, of the quantity value
      when the value covers a range.
   -  `has minimum numeric value <has_minimum_numeric_value.md>`__ - The
      minimum value part, expressed as number, of the quantity value
      when the value covers a range.
   -  `quantity value➞has numeric
      value <quantity_value_has_numeric_value.md>`__ - The number part
      of the quantity

-  `has output <has_output.md>`__ - An output biosample to a processing
   step

   -  `workflow execution activity➞has
      output <workflow_execution_activity_has_output.md>`__

      -  `read QC analysis activity➞has
         output <read_QC_analysis_activity_has_output.md>`__

-  `has peptide quantifications <has_peptide_quantifications.md>`__

   -  `metaproteomics analysis activity➞has peptide
      quantifications <metaproteomics_analysis_activity_has_peptide_quantifications.md>`__

-  `has raw value <has_raw_value.md>`__ - The value that was specified
   for an annotation in raw form, i.e. a string. E.g. “2 cm” or “2-4 cm”

   -  `geolocation value➞has raw
      value <geolocation_value_has_raw_value.md>`__ - The raw value for
      a geolocation should follow {lat} {long}
   -  `person value➞has raw value <person_value_has_raw_value.md>`__ -
      The full name of the Investgator in format FIRST LAST.
   -  `quantity value➞has raw value <quantity_value_has_raw_value.md>`__
      - Unnormalized atomic string representation, should in syntax
      {number} {unit}

-  `has unit <has_unit.md>`__ - Links a quantity value to a unit

   -  `quantity value➞has unit <quantity_value_has_unit.md>`__ - The
      unit of the quantity

-  `highest similarity score <highest_similarity_score.md>`__ - TODO:
   Yuri to fill in

   -  `metabolite quantification➞highest similarity
      score <metabolite_quantification_highest_similarity_score.md>`__

-  `id <id.md>`__ - A unique identifier for a thing. Must be either a
   CURIE shorthand for a URI or a complete URI

   -  `person➞id <person_id.md>`__ - Should be an ORCID. Specify in
      CURIE format. E.g ORCID:0000-1111-…

-  `inchi <inchi.md>`__

   -  `chemical entity➞inchi <chemical_entity_inchi.md>`__

-  `inchi key <inchi_key.md>`__

   -  `chemical entity➞inchi key <chemical_entity_inchi_key.md>`__

-  `input contig num <input_contig_num.md>`__
-  `input_read_bases <input_read_bases.md>`__ - TODO

   -  `read QC analysis
      activity➞input_read_bases <read_QC_analysis_activity_input_read_bases.md>`__

-  `keywords <keywords.md>`__ - A list of keywords that used to
   associate the entity. Keywords SHOULD come from controlled
   vocabularies, including MESH, ENVO.
-  `language <language.md>`__ - Should use ISO 639-1 code e.g. “en”,
   “fr”
-  `latitude <latitude.md>`__ - latitude
-  `longitude <longitude.md>`__ - longitude
-  `lowDepth contig num <lowDepth_contig_num.md>`__
-  `➞bin name <mAGBin__bin_name.md>`__
-  `➞bin quality <mAGBin__bin_quality.md>`__
-  `➞completeness <mAGBin__completeness.md>`__
-  `➞contamination <mAGBin__contamination.md>`__
-  `➞gene count <mAGBin__gene_count.md>`__
-  `➞gtdbtk class <mAGBin__gtdbtk_class.md>`__
-  `➞gtdbtk domain <mAGBin__gtdbtk_domain.md>`__
-  `➞gtdbtk family <mAGBin__gtdbtk_family.md>`__
-  `➞gtdbtk genus <mAGBin__gtdbtk_genus.md>`__
-  `➞gtdbtk order <mAGBin__gtdbtk_order.md>`__
-  `➞gtdbtk phylum <mAGBin__gtdbtk_phylum.md>`__
-  `➞gtdbtk species <mAGBin__gtdbtk_species.md>`__
-  `➞num 16s <mAGBin__num_16s.md>`__
-  `➞num 23s <mAGBin__num_23s.md>`__
-  `➞num 5s <mAGBin__num_5s.md>`__
-  `➞num tRNA <mAGBin__num_tRNA.md>`__
-  `➞number of contig <mAGBin__number_of_contig.md>`__
-  `mags list <mags_list.md>`__
-  `metabolite quantified <metabolite_quantified.md>`__ - the specific
   metabolite identifier

   -  `metabolite quantification➞metabolite
      quantified <metabolite_quantification_metabolite_quantified.md>`__

-  `metagenome assembly parameter <metagenome_assembly_parameter.md>`__

   -  `asm_score <asm_score.md>`__ - A score for comparing metagenomic
      assembly quality from same sample.
   -  `contig_bp <contig_bp.md>`__ - Total size in bp of all contigs.
   -  `contigs <contigs.md>`__ - The sum of the (length*log(length)) of
      all contigs, times some constant. Increase the contiguity, the
      score will increase
   -  `ctg_L50 <ctg_L50.md>`__ - Given a set of contigs, the L50 is
      defined as the sequence length of the shortest contig at 50% of
      the total genome length.
   -  `ctg_L90 <ctg_L90.md>`__ - The L90 statistic is less than or equal
      to the L50 statistic; it is the length for which the collection of
      all contigs of that length or longer contains at least 90% of the
      sum of the lengths of all contigs.
   -  `ctg_N50 <ctg_N50.md>`__ - Given a set of contigs, each with its
      own length, the N50 count is defined as the smallest number of
      contigs whose length sum makes up half of genome size.
   -  `ctg_N90 <ctg_N90.md>`__ - Given a set of contigs, each with its
      own length, the N90 count is defined as the smallest number of
      contigs whose length sum makes up 90% of genome size.
   -  `ctg_logsum <ctg_logsum.md>`__ - Maximum contig length.
   -  `ctg_max <ctg_max.md>`__ - Maximum contig length.
   -  `ctg_powsum <ctg_powsum.md>`__ - Powersum of all contigs is the
      same as logsum except that it uses the sum of (length*(length^P))
      for some power P (default P=0.25).
   -  `gap_pct <gap_pct.md>`__ - The gap size percentage of all
      scaffolds.
   -  `gc_avg <gc_avg.md>`__ - Average of GC content of all contigs.
   -  `gc_std <gc_std.md>`__ - Standard deviation of GC content of all
      contigs.
   -  `num_aligned_reads <num_aligned_reads.md>`__ - The sequence count
      number of input reads aligned to assembled contigs.
   -  `num_input_reads <num_input_reads.md>`__ - The sequence count
      number of input reads for assembly.
   -  `scaf_L50 <scaf_L50.md>`__ - Given a set of scaffolds, the L50 is
      defined as the sequence length of the shortest scaffold at 50% of
      the total genome length.
   -  `scaf_L90 <scaf_L90.md>`__ - The L90 statistic is less than or
      equal to the L50 statistic; it is the length for which the
      collection of all scaffolds of that length or longer contains at
      least 90% of the sum of the lengths of all scaffolds.
   -  `scaf_N50 <scaf_N50.md>`__ - Given a set of scaffolds, each with
      its own length, the N50 count is defined as the smallest number of
      scaffolds whose length sum makes up half of genome size.
   -  `scaf_N90 <scaf_N90.md>`__ - Given a set of scaffolds, each with
      its own length, the N90 count is defined as the smallest number of
      scaffolds whose length sum makes up 90% of genome size.
   -  `scaf_bp <scaf_bp.md>`__ - Total size in bp of all scaffolds.
   -  `scaf_l_gt50K <scaf_l_gt50K.md>`__ - Total size in bp of all
      scaffolds greater than 50 KB.
   -  `scaf_logsum <scaf_logsum.md>`__ - The sum of the
      (length*log(length)) of all scaffolds, times some constant.
      Increase the contiguity, the score will increase
   -  `scaf_max <scaf_max.md>`__ - Maximum scaffold length.
   -  `scaf_n_gt50K <scaf_n_gt50K.md>`__ - Total sequence count of
      scaffolds greater than 50 KB.
   -  `scaf_pct_gt50K <scaf_pct_gt50K.md>`__ - Total sequence size
      percentage of scaffolds greater than 50 KB.
   -  `scaf_powsum <scaf_powsum.md>`__ - Powersum of all scaffolds is
      the same as logsum except that it uses the sum of
      (length*(length^P)) for some power P (default P=0.25).
   -  `scaffolds <scaffolds.md>`__ - Total sequence count of all
      scaffolds.

-  `min_q_value <min_q_value.md>`__ - smallest Q-Value associated with
   the peptide sequence as provided by MSGFPlus tool

   -  `peptide
      quantification➞min_q_value <peptide_quantification_min_q_value.md>`__

-  `name <name.md>`__ - A human readable label for an entity

   -  `person value➞name <person_value_name.md>`__ - The full name of
      the Investgator. It should follow the format FIRST [MIDDLE NAME\|
      MIDDLE INITIAL] LAST, where MIDDLE NAME\| MIDDLE INITIAL is
      optional.

-  `objective <objective.md>`__ - The scientific objectives associated
   with the entity. It SHOULD correspond to scientific norms for
   objectives field in a structured abstract.
-  `orcid <orcid.md>`__ - The ORICD of a person.
-  `output_read_bases <output_read_bases.md>`__ - TODO

   -  `read QC analysis
      activity➞output_read_bases <read_QC_analysis_activity_output_read_bases.md>`__

-  `part of <part_of.md>`__ - Links a resource to another resource that
   either logically or physically includes it.
-  `peptide sequence <peptide_sequence.md>`__

   -  `peptide quantification➞peptide
      sequence <peptide_quantification_peptide_sequence.md>`__

-  `peptide_sequence_count <peptide_sequence_count.md>`__ - count of
   peptide sequences grouped to the best_protein

   -  `protein
      quantification➞peptide_sequence_count <protein_quantification_peptide_sequence_count.md>`__

-  `peptide_spectral_count <peptide_spectral_count.md>`__ - sum of
   filter passing MS2 spectra associated with the peptide sequence
   within a given LC-MS/MS data file

   -  `peptide
      quantification➞peptide_spectral_count <peptide_quantification_peptide_spectral_count.md>`__

-  `peptide_sum_masic_abundance <peptide_sum_masic_abundance.md>`__ -
   combined MS1 extracted ion chromatograms derived from MS2 spectra
   associated with the peptide sequence from a given LC-MS/MS data file
   using the MASIC tool

   -  `peptide
      quantification➞peptide_sum_masic_abundance <peptide_quantification_peptide_sum_masic_abundance.md>`__

-  `profile image url <profile_image_url.md>`__ - A url that points to
   an image of a person.
-  `protein_spectral_count <protein_spectral_count.md>`__ - sum of
   filter passing MS2 spectra associated with the best protein within a
   given LC-MS/MS data file

   -  `protein
      quantification➞protein_spectral_count <protein_quantification_protein_spectral_count.md>`__

-  `protein_sum_masic_abundance <protein_sum_masic_abundance.md>`__ -
   combined MS1 extracted ion chromatograms derived from MS2 spectra
   associated with the best protein from a given LC-MS/MS data file
   using the MASIC tool

   -  `protein
      quantification➞protein_sum_masic_abundance <protein_quantification_protein_sum_masic_abundance.md>`__

-  `publications <publications.md>`__ - A list of publications that are
   assocatiated with the entity. The publicatons SHOULD be given using
   an identifier, such as a DOI or Pubmed ID, if possible.
-  `read QC analysis statistic <read_QC_analysis_statistic.md>`__

   -  `input base count <input_base_count.md>`__ - The nucleotide base
      count number of input reads for QC analysis.
   -  `input read count <input_read_count.md>`__ - The sequence count
      number of input reads for QC analysis.
   -  `output base count <output_base_count.md>`__ - After QC analysis
      nucleotide base count number.
   -  `output read count <output_read_count.md>`__ - After QC analysis
      sequence count number.

-  `smiles <smiles.md>`__ - A string encoding of a molecular graph, no
   chiral or isotopic information. There are usually a large number of
   valid SMILES which represent a given structure. For example, CCO, OCC
   and C(O)C all specify the structure of ethanol.

   -  `chemical entity➞smiles <chemical_entity_smiles.md>`__

-  `started at time <started_at_time.md>`__

   -  `workflow execution activity➞started at
      time <workflow_execution_activity_started_at_time.md>`__

-  `term <term.md>`__ - pointer to an ontology class
-  `title <title.md>`__ - A name given to the entity that differs from
   the name/label programatically assigned to it. For example, when
   extracting study information for GOLD, the GOLD system has assigned a
   name/label. However, for display purposes, we may also wish the
   capture the title of the proposal that was used to fund the study.
-  `too short contig num <too_short_contig_num.md>`__
-  `type <type.md>`__ - An optional string that specifies the type
   object. This is used to allow for searches for different kinds of
   objects.

   -  `attribute value➞type <attribute_value_type.md>`__ - An optional
      string that specified the type of object.
   -  `workflow execution
      activity➞type <workflow_execution_activity_type.md>`__

-  `unbinned contig num <unbinned_contig_num.md>`__
-  `used <used.md>`__

   -  `metabolomics analysis
      activity➞used <metabolomics_analysis_activity_used.md>`__ - The
      instrument used to collect the data used in the analysis
   -  `metaproteomics analysis
      activity➞used <metaproteomics_analysis_activity_used.md>`__ - The
      instrument used to collect the data used in the analysis
   -  `nom analysis activity➞used <nom_analysis_activity_used.md>`__ -
      The instrument used to collect the data used in the analysis

-  `was associated with <was_associated_with.md>`__

   -  `workflow execution activity➞was associated
      with <workflow_execution_activity_was_associated_with.md>`__ - the
      agent/entity associated with the generation of the file

-  `was generated by <was_generated_by.md>`__
-  `was informed by <was_informed_by.md>`__

   -  `workflow execution activity➞was informed
      by <workflow_execution_activity_was_informed_by.md>`__

-  `websites <websites.md>`__ - A list of websites that are assocatiated
   with the entity.

.. _enums-9:

Enums
-----

Subsets
-------

-  `Environment <Environment.md>`__
-  `Investigation <Investigation.md>`__
-  `MixsExtension <MixsExtension.md>`__
-  `NucleicAcidSequenceSource <NucleicAcidSequenceSource.md>`__
-  `Sequencing <Sequencing.md>`__
-  `Soil <Soil.md>`__
-  `WorkflowSubset <WorkflowSubset.md>`__ - Subset consisting of just
   the workflow exectution activities

.. _types-9:

Types
-----

Built in
~~~~~~~~

-  **Bool**
-  **Decimal**
-  **ElementIdentifier**
-  **NCName**
-  **NodeIdentifier**
-  **URI**
-  **URIorCURIE**
-  **XSDDate**
-  **XSDDateTime**
-  **XSDTime**
-  **float**
-  **int**
-  **str**

Defined
~~~~~~~

-  `Boolean <types/Boolean.md>`__ (**Bool**) - A binary (true or false)
   value
-  `Bytes <types/Bytes.md>`__ (**int**) - An integer value that
   corresponds to a size in bytes
-  `Date <types/Date.md>`__ (**XSDDate**) - a date (year, month and day)
   in an idealized calendar
-  `Datetime <types/Datetime.md>`__ (**XSDDateTime**) - The combination
   of a date and time
-  `Decimal <types/Decimal.md>`__ (**Decimal**) - A real number with
   arbitrary precision that conforms to the xsd:decimal specification
-  `DecimalDegree <types/DecimalDegree.md>`__ (**float**) - A decimal
   degree expresses latitude or longitude as decimal fractions.
-  `Double <types/Double.md>`__ (**float**) - A real number that
   conforms to the xsd:double specification
-  `ExternalIdentifier <types/ExternalIdentifier.md>`__
   (`Uriorcurie <types/Uriorcurie.md>`__) - A CURIE representing an
   external identifier
-  `Float <types/Float.md>`__ (**float**) - A real number that conforms
   to the xsd:float specification
-  `Integer <types/Integer.md>`__ (**int**) - An integer
-  `LanguageCode <types/LanguageCode.md>`__ (**str**) - A language code
   conforming to ISO_639-1
-  `Ncname <types/Ncname.md>`__ (**NCName**) - Prefix part of CURIE
-  `Nodeidentifier <types/Nodeidentifier.md>`__ (**NodeIdentifier**) - A
   URI, CURIE or BNODE that represents a node in a model.
-  `Objectidentifier <types/Objectidentifier.md>`__
   (**ElementIdentifier**) - A URI or CURIE that represents an object in
   the model.
-  `String <types/String.md>`__ (**str**) - A character string
-  `Time <types/Time.md>`__ (**XSDTime**) - A time object represents a
   (local) time of day, independent of any particular day
-  `Unit <types/Unit.md>`__ (**str**)
-  `Uri <types/Uri.md>`__ (**URI**) - a complete URI
-  `Uriorcurie <types/Uriorcurie.md>`__ (**URIorCURIE**) - a URI or a
   CURIE

Slot: indoor_space
==================

“A distinguishable space within a structure, the purpose for which
discrete areas of a building is used”

URI: `nmdc:indoor_space <https://microbiomedata/meta/indoor_space>`__

.. _domain-and-range-387:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-424:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-394:

Children
--------

.. _used-by-387:

Used by
-------

.. _other-properties-341:

Other properties
----------------

============= =================
**Aliases:**   indoor space
**Mappings:**  mixs:indoor_space
============= =================

Slot: indoor_surf
=================

Type of indoor surface

URI: `nmdc:indoor_surf <https://microbiomedata/meta/indoor_surf>`__

.. _domain-and-range-388:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-425:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-395:

Children
--------

.. _used-by-388:

Used by
-------

.. _other-properties-342:

Other properties
----------------

============= ================
**Aliases:**   indoor surface
**Mappings:**  mixs:indoor_surf
============= ================

Slot: indust_eff_percent
========================

Percentage of industrial effluents received by wastewater treatment
plant

URI:
`nmdc:indust_eff_percent <https://microbiomedata/meta/indust_eff_percent>`__

.. _domain-and-range-389:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-426:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-396:

Children
--------

.. _used-by-389:

Used by
-------

.. _other-properties-343:

Other properties
----------------

============= ===========================
**Aliases:**   industrial effluent percent
**Mappings:**  mixs:indust_eff_percent
============= ===========================

Slot: inorg_particles
=====================

“Concentration of particles such as sand, grit, metal particles,
ceramics, etc.; can include multiple particles”

URI:
`nmdc:inorg_particles <https://microbiomedata/meta/inorg_particles>`__

.. _domain-and-range-390:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-427:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-397:

Children
--------

.. _used-by-390:

Used by
-------

.. _other-properties-344:

Other properties
----------------

============= ====================
**Aliases:**   inorganic particles
**Mappings:**  mixs:inorg_particles
============= ====================

Slot: input base count
======================

The nucleotide base count number of input reads for QC analysis.

URI:
https://microbiomedata/schema/workflow_execution_activity/input_base_count

.. _domain-and-range-391:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-428:

Parents
-------

-  is_a: `read QC analysis statistic <read_QC_analysis_statistic.md>`__

.. _children-398:

Children
--------

.. _used-by-391:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

Slot: input contig num
======================

URI:
https://microbiomedata/schema/workflow_execution_activity/input_contig_num

.. _domain-and-range-392:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-429:

Parents
-------

.. _children-399:

Children
--------

.. _used-by-392:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

Slot: input_read_bases
======================

TODO

URI:
https://microbiomedata/schema/workflow_execution_activity/input_read_bases

.. _domain-and-range-393:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-430:

Parents
-------

.. _children-400:

Children
--------

-  `read QC analysis
   activity➞input_read_bases <read_QC_analysis_activity_input_read_bases.md>`__

.. _used-by-393:

Used by
-------

Slot: input read count
======================

The sequence count number of input reads for QC analysis.

URI:
https://microbiomedata/schema/workflow_execution_activity/input_read_count

.. _domain-and-range-394:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-431:

Parents
-------

-  is_a: `read QC analysis statistic <read_QC_analysis_statistic.md>`__

.. _children-401:

Children
--------

.. _used-by-394:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

Slot: inside_lux
================

The recorded value at sampling time (power density)

URI: `nmdc:inside_lux <https://microbiomedata/meta/inside_lux>`__

.. _domain-and-range-395:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-432:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-402:

Children
--------

.. _used-by-395:

Used by
-------

.. _other-properties-345:

Other properties
----------------

============= ================
**Aliases:**   inside lux light
**Mappings:**  mixs:inside_lux
============= ================

Slot: instrument_name
=====================

The name of the instrument that was used for processing the sample.

URI:
`nmdc:instrument_name <https://microbiomedata/meta/instrument_name>`__

.. _domain-and-range-396:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-433:

Parents
-------

.. _children-403:

Children
--------

.. _used-by-396:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

Slot: int_wall_cond
===================

The physical condition of the wall at the time of sampling; photos or
video preferred; use drawings to indicate location of damaged areas

URI: `nmdc:int_wall_cond <https://microbiomedata/meta/int_wall_cond>`__

.. _domain-and-range-397:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-434:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-404:

Children
--------

.. _used-by-397:

Used by
-------

.. _other-properties-346:

Other properties
----------------

============= =======================
**Aliases:**   interior wall condition
**Mappings:**  mixs:int_wall_cond
============= =======================

Slot: investigation_type
========================

“Nucleic Acid Sequence Report is the root element of all MIGS/MIMS
compliant reports as standardized by Genomic Standards Consortium. This
field is either eukaryote,bacteria,virus,plasmid,organelle,
metagenome,mimarks-survey, mimarks-specimen, metatranscriptome, single
amplified genome, metagenome-assembled genome, or uncultivated viral
genome”

URI:
`nmdc:investigation_type <https://microbiomedata/meta/investigation_type>`__

.. _domain-and-range-398:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-435:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-405:

Children
--------

.. _used-by-398:

Used by
-------

.. _other-properties-347:

Other properties
----------------

=============== =======================
**Aliases:**     investigation type
**Mappings:**    mixs:investigation_type
**In Subsets:**  investigation
=============== =======================

Slot: is balanced
=================

URI: `nmdc:is_balanced <https://microbiomedata/meta/is_balanced>`__

.. _domain-and-range-399:

Domain and Range
----------------

None → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-436:

Parents
-------

.. _children-406:

Children
--------

-  `reaction➞is balanced <reaction_is_balanced.md>`__

.. _used-by-399:

Used by
-------

Slot: is diastereoselective
===========================

URI:
`nmdc:is_diastereoselective <https://microbiomedata/meta/is_diastereoselective>`__

.. _domain-and-range-400:

Domain and Range
----------------

None → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-437:

Parents
-------

.. _children-407:

Children
--------

-  `reaction➞is
   diastereoselective <reaction_is_diastereoselective.md>`__

.. _used-by-400:

Used by
-------

Slot: is fully characterized
============================

False if includes R-groups

URI:
`nmdc:is_fully_characterized <https://microbiomedata/meta/is_fully_characterized>`__

.. _domain-and-range-401:

Domain and Range
----------------

None → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-438:

Parents
-------

.. _children-408:

Children
--------

-  `reaction➞is fully
   characterized <reaction_is_fully_characterized.md>`__

.. _used-by-401:

Used by
-------

Slot: is stereo
===============

URI: `nmdc:is_stereo <https://microbiomedata/meta/is_stereo>`__

.. _domain-and-range-402:

Domain and Range
----------------

None → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-439:

Parents
-------

.. _children-409:

Children
--------

-  `reaction➞is stereo <reaction_is_stereo.md>`__

.. _used-by-402:

Used by
-------

Slot: is transport
==================

URI: `nmdc:is_transport <https://microbiomedata/meta/is_transport>`__

.. _domain-and-range-403:

Domain and Range
----------------

None → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-440:

Parents
-------

.. _children-410:

Children
--------

-  `reaction➞is transport <reaction_is_transport.md>`__

.. _used-by-403:

Used by
-------

Slot: isol_growth_condt
=======================

“Publication reference in the form of pubmed ID (pmid), digital object
identifier (doi) or url for isolation and growth condition
specifications of the organism/material”

URI:
`nmdc:isol_growth_condt <https://microbiomedata/meta/isol_growth_condt>`__

.. _domain-and-range-404:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-441:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-411:

Children
--------

.. _used-by-404:

Used by
-------

.. _other-properties-348:

Other properties
----------------

=============== ==============================
**Aliases:**     isolation and growth condition
**Mappings:**    mixs:isol_growth_condt
**In Subsets:**  nucleic acid sequence source
=============== ==============================

Slot: iw_bt_date_well
=====================

Injection water breakthrough date per well following a secondary and/or
tertiary recovery

URI:
`nmdc:iw_bt_date_well <https://microbiomedata/meta/iw_bt_date_well>`__

.. _domain-and-range-405:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-442:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-412:

Children
--------

.. _used-by-405:

Used by
-------

.. _other-properties-349:

Other properties
----------------

============= ==================================================
**Aliases:**   injection water breakthrough date of specific well
**Mappings:**  mixs:iw_bt_date_well
============= ==================================================

Slot: iwf
=========

Proportion of the produced fluids derived from injected water at the
time of sampling. (e.g. 87%)

URI: `nmdc:iwf <https://microbiomedata/meta/iwf>`__

.. _domain-and-range-406:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-443:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-413:

Children
--------

.. _used-by-406:

Used by
-------

.. _other-properties-350:

Other properties
----------------

============= ========================
**Aliases:**   injection water fraction
**Mappings:**  mixs:iwf
============= ========================

Slot: keywords
==============

A list of keywords that used to associate the entity. Keywords SHOULD
come from controlled vocabularies, including MESH, ENVO.

URI: https://microbiomedata/schema/workflow_execution_activity/keywords

.. _domain-and-range-407:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-444:

Parents
-------

.. _children-414:

Children
--------

.. _used-by-407:

Used by
-------

.. _other-properties-351:

Other properties
----------------

============= ===============
**Mappings:**  dcterms:subject
============= ===============

Slot: kidney_disord
===================

History of kidney disorders; can include multiple disorders

URI: `nmdc:kidney_disord <https://microbiomedata/meta/kidney_disord>`__

.. _domain-and-range-408:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-445:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-415:

Children
--------

.. _used-by-408:

Used by
-------

.. _other-properties-352:

Other properties
----------------

============= =====================
**Aliases:**   urine/kidney disorder
**Mappings:**  mixs:kidney_disord
============= =====================

Slot: language
==============

Should use ISO 639-1 code e.g. “en”, “fr”

URI: https://microbiomedata/schema/workflow_execution_activity/language

.. _domain-and-range-409:

Domain and Range
----------------

None → 0..1 `LanguageCode <types/LanguageCode.md>`__

.. _parents-446:

Parents
-------

.. _children-416:

Children
--------

.. _used-by-409:

Used by
-------

-  `TextValue <TextValue.md>`__

Slot: last_clean
================

“The last time the floor was cleaned (swept, mopped, vacuumed)”

URI: `nmdc:last_clean <https://microbiomedata/meta/last_clean>`__

.. _domain-and-range-410:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-447:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-417:

Children
--------

.. _used-by-410:

Used by
-------

.. _other-properties-353:

Other properties
----------------

============= ===============================
**Aliases:**   last time swept/mopped/vacuumed
**Mappings:**  mixs:last_clean
============= ===============================

.. _slot-lat_lon-1:

Slot: lat_lon
=============

The geographical origin of the sample as defined by latitude and
longitude. The values should be reported in decimal degrees and in WGS84
system

URI: `nmdc:lat_lon <https://microbiomedata/meta/lat_lon>`__

.. _domain-and-range-411:

Domain and Range
----------------

None → 0..1 `GeolocationValue <GeolocationValue.md>`__

.. _parents-448:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-418:

Children
--------

-  `biosample➞lat_lon <biosample_lat_lon.md>`__

.. _used-by-411:

Used by
-------

.. _other-properties-354:

Other properties
----------------

=============== ============================================
**Aliases:**     geographic location (latitude and longitude)
**Mappings:**    mixs:lat_lon
**In Subsets:**  environment
=============== ============================================

Slot: latitude
==============

latitude

URI: https://microbiomedata/schema/workflow_execution_activity/latitude

.. _domain-and-range-412:

Domain and Range
----------------

`GeolocationValue <GeolocationValue.md>`__ → 0..1
`DecimalDegree <types/DecimalDegree.md>`__

.. _parents-449:

Parents
-------

.. _children-419:

Children
--------

.. _used-by-412:

Used by
-------

-  `GeolocationValue <GeolocationValue.md>`__

.. _other-properties-355:

Other properties
----------------

============= ===============
**Mappings:**  wgs:lat
\              schema:latitude
============= ===============

Slot: left participants
=======================

URI:
`nmdc:left_participants <https://microbiomedata/meta/left_participants>`__

.. _domain-and-range-413:

Domain and Range
----------------

None → 0..\* `ReactionParticipant <ReactionParticipant.md>`__

.. _parents-450:

Parents
-------

-  is_a: `has participants <has_participants.md>`__

.. _children-420:

Children
--------

.. _used-by-413:

Used by
-------

Slot: lib_layout
================

“Specify whether to expect single, paired, or other configuration of
reads”

URI: `nmdc:lib_layout <https://microbiomedata/meta/lib_layout>`__

.. _domain-and-range-414:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-451:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-421:

Children
--------

.. _used-by-414:

Used by
-------

.. _other-properties-356:

Other properties
----------------

=============== ===============
**Aliases:**     library layout
**Mappings:**    mixs:lib_layout
**In Subsets:**  sequencing
=============== ===============

Slot: lib_reads_seqd
====================

Total number of clones sequenced from the library

URI:
`nmdc:lib_reads_seqd <https://microbiomedata/meta/lib_reads_seqd>`__

.. _domain-and-range-415:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-452:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-422:

Children
--------

.. _used-by-415:

Used by
-------

.. _other-properties-357:

Other properties
----------------

=============== =======================
**Aliases:**     library reads sequenced
**Mappings:**    mixs:lib_reads_seqd
**In Subsets:**  sequencing
=============== =======================

Slot: lib_screen
================

Specific enrichment or screening methods applied before and/or after
creating libraries

URI: `nmdc:lib_screen <https://microbiomedata/meta/lib_screen>`__

.. _domain-and-range-416:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-453:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-423:

Children
--------

.. _used-by-416:

Used by
-------

.. _other-properties-358:

Other properties
----------------

=============== ==========================
**Aliases:**     library screening strategy
**Mappings:**    mixs:lib_screen
**In Subsets:**  sequencing
=============== ==========================

Slot: lib_size
==============

Total number of clones in the library prepared for the project

URI: `nmdc:lib_size <https://microbiomedata/meta/lib_size>`__

.. _domain-and-range-417:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-454:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-424:

Children
--------

.. _used-by-417:

Used by
-------

.. _other-properties-359:

Other properties
----------------

=============== =============
**Aliases:**     library size
**Mappings:**    mixs:lib_size
**In Subsets:**  sequencing
=============== =============

Slot: lib_vector
================

Cloning vector type(s) used in construction of libraries

URI: `nmdc:lib_vector <https://microbiomedata/meta/lib_vector>`__

.. _domain-and-range-418:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-455:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-425:

Children
--------

.. _used-by-418:

Used by
-------

.. _other-properties-360:

Other properties
----------------

=============== ===============
**Aliases:**     library vector
**Mappings:**    mixs:lib_vector
**In Subsets:**  sequencing
=============== ===============

Slot: light_intensity
=====================

Measurement of light intensity

URI:
`nmdc:light_intensity <https://microbiomedata/meta/light_intensity>`__

.. _domain-and-range-419:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-456:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-426:

Children
--------

.. _used-by-419:

Used by
-------

.. _other-properties-361:

Other properties
----------------

============= ====================
**Aliases:**   light intensity
**Mappings:**  mixs:light_intensity
============= ====================

Slot: light_regm
================

“Information about treatment(s) involving exposure to light, including
both light intensity and quality.”

URI: `nmdc:light_regm <https://microbiomedata/meta/light_regm>`__

.. _domain-and-range-420:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-457:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-427:

Children
--------

.. _used-by-420:

Used by
-------

.. _other-properties-362:

Other properties
----------------

============= ===============
**Aliases:**   light regimen
**Mappings:**  mixs:light_regm
============= ===============

Slot: light_type
================

“Application of light to achieve some practical or aesthetic effect.
Lighting includes the use of both artificial light sources such as lamps
and light fixtures, as well as natural illumination by capturing
daylight. Can also include absence of light”

URI: `nmdc:light_type <https://microbiomedata/meta/light_type>`__

.. _domain-and-range-421:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-458:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-428:

Children
--------

.. _used-by-421:

Used by
-------

.. _other-properties-363:

Other properties
----------------

============= ===============
**Aliases:**   light type
**Mappings:**  mixs:light_type
============= ===============

Slot: link_addit_analys
=======================

Link to additional analysis results performed on the sample

URI:
`nmdc:link_addit_analys <https://microbiomedata/meta/link_addit_analys>`__

.. _domain-and-range-422:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-459:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-429:

Children
--------

.. _used-by-422:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-364:

Other properties
----------------

============= ============================
**Aliases:**   links to additional analysis
**Mappings:**  mixs:link_addit_analys
============= ============================

Slot: link_class_info
=====================

Link to digitized soil maps or other soil classification information

URI:
`nmdc:link_class_info <https://microbiomedata/meta/link_class_info>`__

.. _domain-and-range-423:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-460:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-430:

Children
--------

.. _used-by-423:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-365:

Other properties
----------------

============= ==================================
**Aliases:**   link to classification information
**Mappings:**  mixs:link_class_info
============= ==================================

Slot: link_climate_info
=======================

Link to climate resource

URI:
`nmdc:link_climate_info <https://microbiomedata/meta/link_climate_info>`__

.. _domain-and-range-424:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-461:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-431:

Children
--------

.. _used-by-424:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-366:

Other properties
----------------

============= ===========================
**Aliases:**   link to climate information
**Mappings:**  mixs:link_climate_info
============= ===========================

Slot: lithology
===============

“Hydrocarbon resource main lithology (Additional information:
http://petrowiki.org/Lithology_and_rock_type_determination). If”“other””
is specified, please propose entry in ““additional info”” field”

URI: `nmdc:lithology <https://microbiomedata/meta/lithology>`__

.. _domain-and-range-425:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-462:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-432:

Children
--------

.. _used-by-425:

Used by
-------

.. _other-properties-367:

Other properties
----------------

============= ==============
**Aliases:**   lithology
**Mappings:**  mixs:lithology
============= ==============

Slot: liver_disord
==================

History of liver disorders; can include multiple disorders

URI: `nmdc:liver_disord <https://microbiomedata/meta/liver_disord>`__

.. _domain-and-range-426:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-463:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-433:

Children
--------

.. _used-by-426:

Used by
-------

.. _other-properties-368:

Other properties
----------------

============= =================
**Aliases:**   liver disorder
**Mappings:**  mixs:liver_disord
============= =================

Slot: local_class
=================

Soil classification based on local soil classification system

URI: `nmdc:local_class <https://microbiomedata/meta/local_class>`__

.. _domain-and-range-427:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-464:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-434:

Children
--------

.. _used-by-427:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-369:

Other properties
----------------

============= ===================================
**Aliases:**   soil_taxonomic/local classification
**Mappings:**  mixs:local_class
============= ===================================

Slot: local_class_meth
======================

Reference or method used in determining the local soil classification

URI:
`nmdc:local_class_meth <https://microbiomedata/meta/local_class_meth>`__

.. _domain-and-range-428:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-465:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-435:

Children
--------

.. _used-by-428:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-370:

Other properties
----------------

============= ==========================================
**Aliases:**   soil_taxonomic/local classification method
**Mappings:**  mixs:local_class_meth
============= ==========================================

Slot: location
==============

URI: `nmdc:location <https://microbiomedata/meta/location>`__

.. _domain-and-range-429:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-466:

Parents
-------

.. _children-436:

Children
--------

.. _used-by-429:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: longitude
===============

longitude

URI: https://microbiomedata/schema/workflow_execution_activity/longitude

.. _domain-and-range-430:

Domain and Range
----------------

`GeolocationValue <GeolocationValue.md>`__ → 0..1
`DecimalDegree <types/DecimalDegree.md>`__

.. _parents-467:

Parents
-------

.. _children-437:

Children
--------

.. _used-by-430:

Used by
-------

-  `GeolocationValue <GeolocationValue.md>`__

.. _other-properties-371:

Other properties
----------------

============= ================
**Mappings:**  wgs:long
\              schema:longitude
============= ================

Slot: lowDepth contig num
=========================

URI:
https://microbiomedata/schema/workflow_execution_activity/lowDepth_contig_num

.. _domain-and-range-431:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-468:

Parents
-------

.. _children-438:

Children
--------

.. _used-by-431:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

Slot: bin name
==============

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_name](https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_name)

.. _domain-and-range-432:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-469:

Parents
-------

.. _children-439:

Children
--------

.. _used-by-432:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: bin quality
=================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_quality](https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_quality)

.. _domain-and-range-433:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-470:

Parents
-------

.. _children-440:

Children
--------

.. _used-by-433:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: completeness
==================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__completeness](https://microbiomedata/schema/workflow_execution_activity/mAGBin__completeness)

.. _domain-and-range-434:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-471:

Parents
-------

.. _children-441:

Children
--------

.. _used-by-434:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: contamination
===================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__contamination](https://microbiomedata/schema/workflow_execution_activity/mAGBin__contamination)

.. _domain-and-range-435:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-472:

Parents
-------

.. _children-442:

Children
--------

.. _used-by-435:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gene count
================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gene_count](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gene_count)

.. _domain-and-range-436:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-473:

Parents
-------

.. _children-443:

Children
--------

.. _used-by-436:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk class
==================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_class](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_class)

.. _domain-and-range-437:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-474:

Parents
-------

.. _children-444:

Children
--------

.. _used-by-437:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk domain
===================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_domain](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_domain)

.. _domain-and-range-438:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-475:

Parents
-------

.. _children-445:

Children
--------

.. _used-by-438:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk family
===================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_family](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_family)

.. _domain-and-range-439:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-476:

Parents
-------

.. _children-446:

Children
--------

.. _used-by-439:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk genus
==================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_genus](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_genus)

.. _domain-and-range-440:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-477:

Parents
-------

.. _children-447:

Children
--------

.. _used-by-440:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk order
==================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_order](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_order)

.. _domain-and-range-441:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-478:

Parents
-------

.. _children-448:

Children
--------

.. _used-by-441:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk phylum
===================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_phylum](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_phylum)

.. _domain-and-range-442:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-479:

Parents
-------

.. _children-449:

Children
--------

.. _used-by-442:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: gtdbtk species
====================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_species](https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_species)

.. _domain-and-range-443:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-480:

Parents
-------

.. _children-450:

Children
--------

.. _used-by-443:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: num 16s
=============

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_16s](https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_16s)

.. _domain-and-range-444:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-481:

Parents
-------

.. _children-451:

Children
--------

.. _used-by-444:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: num 23s
=============

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_23s](https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_23s)

.. _domain-and-range-445:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-482:

Parents
-------

.. _children-452:

Children
--------

.. _used-by-445:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: num 5s
============

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_5s](https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_5s)

.. _domain-and-range-446:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-483:

Parents
-------

.. _children-453:

Children
--------

.. _used-by-446:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: num tRNA
==============

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_tRNA](https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_tRNA)

.. _domain-and-range-447:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-484:

Parents
-------

.. _children-454:

Children
--------

.. _used-by-447:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: number of contig
======================

URI:
[https://microbiomedata/schema/workflow_execution_activity/mAGBin__number_of_contig](https://microbiomedata/schema/workflow_execution_activity/mAGBin__number_of_contig)

.. _domain-and-range-448:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-485:

Parents
-------

.. _children-455:

Children
--------

.. _used-by-448:

Used by
-------

-  `MAGBin <MAGBin.md>`__

Slot: mag_cov_software
======================

Tool(s) used to determine the genome coverage if coverage is used as a
binning parameter in the extraction of genomes from metagenomic datasets

URI:
`nmdc:mag_cov_software <https://microbiomedata/meta/mag_cov_software>`__

.. _domain-and-range-449:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-486:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-456:

Children
--------

.. _used-by-449:

Used by
-------

.. _other-properties-372:

Other properties
----------------

=============== =====================
**Aliases:**     MAG coverage software
**Mappings:**    mixs:mag_cov_software
**In Subsets:**  sequencing
=============== =====================

Slot: magnesium
===============

Concentration of magnesium in the sample

URI: `nmdc:magnesium <https://microbiomedata/meta/magnesium>`__

.. _domain-and-range-450:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-487:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-457:

Children
--------

.. _used-by-450:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-373:

Other properties
----------------

============= ==============
**Aliases:**   magnesium
**Mappings:**  mixs:magnesium
============= ==============

Slot: mags activity set
=======================

This property links a database object to the set of MAGs analysis
activities.

URI:
`nmdc:mags_activity_set <https://microbiomedata/meta/mags_activity_set>`__

.. _domain-and-range-451:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

.. _parents-488:

Parents
-------

.. _children-458:

Children
--------

.. _used-by-451:

Used by
-------

-  `Database <Database.md>`__

Slot: mags list
===============

URI: https://microbiomedata/schema/workflow_execution_activity/mags_list

.. _domain-and-range-452:

Domain and Range
----------------

None → 0..\* `MAGBin <MAGBin.md>`__

.. _parents-489:

Parents
-------

.. _children-459:

Children
--------

.. _used-by-452:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

Slot: maternal_health_stat
==========================

Specification of the maternal health status

URI:
`nmdc:maternal_health_stat <https://microbiomedata/meta/maternal_health_stat>`__

.. _domain-and-range-453:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-490:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-460:

Children
--------

.. _used-by-453:

Used by
-------

.. _other-properties-374:

Other properties
----------------

============= =====================================
**Aliases:**   amniotic fluid/maternal health status
**Mappings:**  mixs:maternal_health_stat
============= =====================================

Slot: max_occup
===============

The maximum amount of people allowed in the indoor environment

URI: `nmdc:max_occup <https://microbiomedata/meta/max_occup>`__

.. _domain-and-range-454:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-491:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-461:

Children
--------

.. _used-by-454:

Used by
-------

.. _other-properties-375:

Other properties
----------------

============= =================
**Aliases:**   maximum occupancy
**Mappings:**  mixs:max_occup
============= =================

Slot: md5 checksum
==================

MD5 checksum of file (pre-compressed)

URI:
https://microbiomedata/schema/workflow_execution_activity/md5_checksum

.. _domain-and-range-455:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-492:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-462:

Children
--------

.. _used-by-455:

Used by
-------

Slot: mean_frict_vel
====================

Measurement of mean friction velocity

URI:
`nmdc:mean_frict_vel <https://microbiomedata/meta/mean_frict_vel>`__

.. _domain-and-range-456:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-493:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-463:

Children
--------

.. _used-by-456:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-376:

Other properties
----------------

============= ======================
**Aliases:**   mean friction velocity
**Mappings:**  mixs:mean_frict_vel
============= ======================

Slot: mean_peak_frict_vel
=========================

Measurement of mean peak friction velocity

URI:
`nmdc:mean_peak_frict_vel <https://microbiomedata/meta/mean_peak_frict_vel>`__

.. _domain-and-range-457:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-494:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-464:

Children
--------

.. _used-by-457:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-377:

Other properties
----------------

============= ===========================
**Aliases:**   mean peak friction velocity
**Mappings:**  mixs:mean_peak_frict_vel
============= ===========================

Slot: mech_struc
================

mechanical structure: a moving structure

URI: `nmdc:mech_struc <https://microbiomedata/meta/mech_struc>`__

.. _domain-and-range-458:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-495:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-465:

Children
--------

.. _used-by-458:

Used by
-------

.. _other-properties-378:

Other properties
----------------

============= ====================
**Aliases:**   mechanical structure
**Mappings:**  mixs:mech_struc
============= ====================

Slot: mechanical_damage
=======================

Information about any mechanical damage exerted on the plant; can
include multiple damages and sites

URI:
`nmdc:mechanical_damage <https://microbiomedata/meta/mechanical_damage>`__

.. _domain-and-range-459:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-496:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-466:

Children
--------

.. _used-by-459:

Used by
-------

.. _other-properties-379:

Other properties
----------------

============= ======================
**Aliases:**   mechanical damage
**Mappings:**  mixs:mechanical_damage
============= ======================

Slot: medic_hist_perform
========================

Whether full medical history was collected

URI:
`nmdc:medic_hist_perform <https://microbiomedata/meta/medic_hist_perform>`__

.. _domain-and-range-460:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-497:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-467:

Children
--------

.. _used-by-460:

Used by
-------

.. _other-properties-380:

Other properties
----------------

============= =========================
**Aliases:**   medical history performed
**Mappings:**  mixs:medic_hist_perform
============= =========================

Slot: menarche
==============

Date of most recent menstruation

URI: `nmdc:menarche <https://microbiomedata/meta/menarche>`__

.. _domain-and-range-461:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-498:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-468:

Children
--------

.. _used-by-461:

Used by
-------

.. _other-properties-381:

Other properties
----------------

============= =============
**Aliases:**   menarche
**Mappings:**  mixs:menarche
============= =============

Slot: menopause
===============

Date of onset of menopause

URI: `nmdc:menopause <https://microbiomedata/meta/menopause>`__

.. _domain-and-range-462:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-499:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-469:

Children
--------

.. _used-by-462:

Used by
-------

.. _other-properties-382:

Other properties
----------------

============= ==============
**Aliases:**   menopause
**Mappings:**  mixs:menopause
============= ==============

.. _slot-highest-similarity-score-1:

Slot: highest similarity score
==============================

TODO: Yuri to fill in

URI:
https://microbiomedata/schema/workflow_execution_activity/metabolite_quantification_highest_similarity_score

.. _domain-and-range-463:

Domain and Range
----------------

`MetaboliteQuantification <MetaboliteQuantification.md>`__ → 0..1
`Float <types/Float.md>`__

.. _parents-500:

Parents
-------

-  is_a: `highest similarity score <highest_similarity_score.md>`__

.. _children-470:

Children
--------

.. _used-by-463:

Used by
-------

-  `MetaboliteQuantification <MetaboliteQuantification.md>`__

Slot: metabolite quantified
===========================

the specific metabolite identifier

URI:
https://microbiomedata/schema/workflow_execution_activity/metabolite_quantification_metabolite_quantified

.. _domain-and-range-464:

Domain and Range
----------------

`MetaboliteQuantification <MetaboliteQuantification.md>`__ → 0..1
`ChemicalEntity <ChemicalEntity.md>`__

.. _parents-501:

Parents
-------

-  is_a: `metabolite quantified <metabolite_quantified.md>`__

.. _children-471:

Children
--------

.. _used-by-464:

Used by
-------

-  `MetaboliteQuantification <MetaboliteQuantification.md>`__

.. _slot-metabolite-quantified-1:

Slot: metabolite quantified
===========================

the specific metabolite identifier

URI:
https://microbiomedata/schema/workflow_execution_activity/metabolite_quantified

.. _domain-and-range-465:

Domain and Range
----------------

None → 0..1 `ChemicalEntity <ChemicalEntity.md>`__

.. _parents-502:

Parents
-------

.. _children-472:

Children
--------

-  `metabolite quantification➞metabolite
   quantified <metabolite_quantification_metabolite_quantified.md>`__

.. _used-by-465:

Used by
-------

.. _slot-has-calibration-1:

Slot: has calibration
=====================

TODO: Yuri to fill in

URI:
https://microbiomedata/schema/workflow_execution_activity/metabolomics_analysis_activity_has_calibration

.. _domain-and-range-466:

Domain and Range
----------------

`MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__ →
0..1 `String <types/String.md>`__

.. _parents-503:

Parents
-------

-  is_a: `has calibration <has_calibration.md>`__

.. _children-473:

Children
--------

.. _used-by-466:

Used by
-------

-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__

.. _slot-has-metabolite-quantifications-1:

Slot: has metabolite quantifications
====================================

URI:
https://microbiomedata/schema/workflow_execution_activity/metabolomics_analysis_activity_has_metabolite_quantifications

.. _domain-and-range-467:

Domain and Range
----------------

`MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__ →
0..\* `MetaboliteQuantification <MetaboliteQuantification.md>`__

.. _parents-504:

Parents
-------

-  is_a: `has metabolite
   quantifications <has_metabolite_quantifications.md>`__

.. _children-474:

Children
--------

.. _used-by-467:

Used by
-------

-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__

Slot: metabolomics analysis activity set
========================================

This property links a database object to the set of metabolomics
analysis activities.

URI:
`nmdc:metabolomics_analysis_activity_set <https://microbiomedata/meta/metabolomics_analysis_activity_set>`__

.. _domain-and-range-468:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__

.. _parents-505:

Parents
-------

.. _children-475:

Children
--------

.. _used-by-468:

Used by
-------

-  `Database <Database.md>`__

Slot: used
==========

The instrument used to collect the data used in the analysis

URI:
https://microbiomedata/schema/workflow_execution_activity/metabolomics_analysis_activity_used

.. _domain-and-range-469:

Domain and Range
----------------

`MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__ →
0..1 `Instrument <Instrument.md>`__

.. _parents-506:

Parents
-------

-  is_a: `used <used.md>`__

.. _children-476:

Children
--------

.. _used-by-469:

Used by
-------

-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__

.. _other-properties-383:

Other properties
----------------

============= =========
**Mappings:**  prov:used
============= =========

Slot: metagenome annotation activity set
========================================

This property links a database object to the set of metagenome
annotation activities.

URI:
`nmdc:metagenome_annotation_activity_set <https://microbiomedata/meta/metagenome_annotation_activity_set>`__

.. _domain-and-range-470:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__

.. _parents-507:

Parents
-------

.. _children-477:

Children
--------

.. _used-by-470:

Used by
-------

-  `Database <Database.md>`__

Slot: metagenome assembly parameter
===================================

URI:
https://microbiomedata/schema/workflow_execution_activity/metagenome_assembly_parameter

.. _domain-and-range-471:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-508:

Parents
-------

.. _children-478:

Children
--------

-  `asm_score <asm_score.md>`__
-  `contig_bp <contig_bp.md>`__
-  `contigs <contigs.md>`__
-  `ctg_L50 <ctg_L50.md>`__
-  `ctg_L90 <ctg_L90.md>`__
-  `ctg_N50 <ctg_N50.md>`__
-  `ctg_N90 <ctg_N90.md>`__
-  `ctg_logsum <ctg_logsum.md>`__
-  `ctg_max <ctg_max.md>`__
-  `ctg_powsum <ctg_powsum.md>`__
-  `gap_pct <gap_pct.md>`__
-  `gc_avg <gc_avg.md>`__
-  `gc_std <gc_std.md>`__
-  `num_aligned_reads <num_aligned_reads.md>`__
-  `num_input_reads <num_input_reads.md>`__
-  `scaf_L50 <scaf_L50.md>`__
-  `scaf_L90 <scaf_L90.md>`__
-  `scaf_N50 <scaf_N50.md>`__
-  `scaf_N90 <scaf_N90.md>`__
-  `scaf_bp <scaf_bp.md>`__
-  `scaf_l_gt50K <scaf_l_gt50K.md>`__
-  `scaf_logsum <scaf_logsum.md>`__
-  `scaf_max <scaf_max.md>`__
-  `scaf_n_gt50K <scaf_n_gt50K.md>`__
-  `scaf_pct_gt50K <scaf_pct_gt50K.md>`__
-  `scaf_powsum <scaf_powsum.md>`__
-  `scaffolds <scaffolds.md>`__

.. _used-by-471:

Used by
-------

Slot: metagenome assembly set
=============================

This property links a database object to the set of metagenome assembly
activities.

URI:
`nmdc:metagenome_assembly_set <https://microbiomedata/meta/metagenome_assembly_set>`__

.. _domain-and-range-472:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`MetagenomeAssembly <MetagenomeAssembly.md>`__

.. _parents-509:

Parents
-------

.. _children-479:

Children
--------

.. _used-by-472:

Used by
-------

-  `Database <Database.md>`__

.. _slot-has-peptide-quantifications-1:

Slot: has peptide quantifications
=================================

URI:
https://microbiomedata/schema/workflow_execution_activity/metaproteomics_analysis_activity_has_peptide_quantifications

.. _domain-and-range-473:

Domain and Range
----------------

`MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__ →
0..\* `PeptideQuantification <PeptideQuantification.md>`__

.. _parents-510:

Parents
-------

-  is_a: `has peptide
   quantifications <has_peptide_quantifications.md>`__

.. _children-480:

Children
--------

.. _used-by-473:

Used by
-------

-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__

Slot: metaproteomics analysis activity set
==========================================

This property links a database object to the set of metaproteomics
analysis activities.

URI:
`nmdc:metaproteomics_analysis_activity_set <https://microbiomedata/meta/metaproteomics_analysis_activity_set>`__

.. _domain-and-range-474:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__

.. _parents-511:

Parents
-------

.. _children-481:

Children
--------

.. _used-by-474:

Used by
-------

-  `Database <Database.md>`__

.. _slot-used-1:

Slot: used
==========

The instrument used to collect the data used in the analysis

URI:
https://microbiomedata/schema/workflow_execution_activity/metaproteomics_analysis_activity_used

.. _domain-and-range-475:

Domain and Range
----------------

`MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__ →
0..1 `Instrument <Instrument.md>`__

.. _parents-512:

Parents
-------

-  is_a: `used <used.md>`__

.. _children-482:

Children
--------

.. _used-by-475:

Used by
-------

-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__

.. _other-properties-384:

Other properties
----------------

============= =========
**Mappings:**  prov:used
============= =========

Slot: metatranscriptome activity set
====================================

This property links a database object to the set of metatranscriptome
analysis activities.

URI:
`nmdc:metatranscriptome_activity_set <https://microbiomedata/meta/metatranscriptome_activity_set>`__

.. _domain-and-range-476:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__

.. _parents-513:

Parents
-------

.. _children-483:

Children
--------

.. _used-by-476:

Used by
-------

-  `Database <Database.md>`__

Slot: methane
=============

Methane (gas) amount or concentration at the time of sampling

URI: `nmdc:methane <https://microbiomedata/meta/methane>`__

.. _domain-and-range-477:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-514:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-484:

Children
--------

.. _used-by-477:

Used by
-------

.. _other-properties-385:

Other properties
----------------

============= ============
**Aliases:**   methane
**Mappings:**  mixs:methane
============= ============

Slot: microbial_biomass
=======================

“The part of the organic matter in the soil that constitutes living
microorganisms smaller than 5-10 micrometer. If you keep this, you would
need to have correction factors used for conversion to the final units”

URI:
`nmdc:microbial_biomass <https://microbiomedata/meta/microbial_biomass>`__

.. _domain-and-range-478:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-515:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-485:

Children
--------

.. _used-by-478:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-386:

Other properties
----------------

============= ======================
**Aliases:**   microbial biomass
**Mappings:**  mixs:microbial_biomass
============= ======================

Slot: microbial_biomass_meth
============================

Reference or method used in determining microbial biomass

URI:
`nmdc:microbial_biomass_meth <https://microbiomedata/meta/microbial_biomass_meth>`__

.. _domain-and-range-479:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-516:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-486:

Children
--------

.. _used-by-479:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-387:

Other properties
----------------

============= ===========================
**Aliases:**   microbial biomass method
**Mappings:**  mixs:microbial_biomass_meth
============= ===========================

Slot: mid
=========

“Molecular barcodes, called Multiplex Identifiers (MIDs), that are used
to specifically tag unique samples in a sequencing run. Sequence should
be reported in uppercase letters”

URI: `nmdc:mid <https://microbiomedata/meta/mid>`__

.. _domain-and-range-480:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-517:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-487:

Children
--------

.. _used-by-480:

Used by
-------

.. _other-properties-388:

Other properties
----------------

=============== =====================
**Aliases:**     multiplex identifiers
**Mappings:**    mixs:mid
**In Subsets:**  sequencing
=============== =====================

Slot: min_q_value
=================

smallest Q-Value associated with the peptide sequence as provided by
MSGFPlus tool

URI:
https://microbiomedata/schema/workflow_execution_activity/min_q_value

.. _domain-and-range-481:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-518:

Parents
-------

.. _children-488:

Children
--------

-  `peptide
   quantification➞min_q_value <peptide_quantification_min_q_value.md>`__

.. _used-by-481:

Used by
-------

.. _other-properties-389:

Other properties
----------------

============= =============================
**See also:**  `OBI:0001442 <OBI:0001442>`__
============= =============================

Slot: mineral_nutr_regm
=======================

“Information about treatment involving the use of mineral supplements;
should include the name of mineral nutrient, amount administered,
treatment regimen including how many times the treatment was repeated,
how long each treatment lasted, and the start and end time of the entire
treatment; can include multiple mineral nutrient regimens”

URI:
`nmdc:mineral_nutr_regm <https://microbiomedata/meta/mineral_nutr_regm>`__

.. _domain-and-range-482:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-519:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-489:

Children
--------

.. _used-by-482:

Used by
-------

.. _other-properties-390:

Other properties
----------------

============= ========================
**Aliases:**   mineral nutrient regimen
**Mappings:**  mixs:mineral_nutr_regm
============= ========================

Slot: misc_param
================

“Any other measurement performed or parameter collected, that is not
listed here”

URI: `nmdc:misc_param <https://microbiomedata/meta/misc_param>`__

.. _domain-and-range-483:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-520:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-490:

Children
--------

.. _used-by-483:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-391:

Other properties
----------------

============= =======================
**Aliases:**   miscellaneous parameter
**Mappings:**  mixs:misc_param
============= =======================

Slot: mixs_url
==============

URI: `nmdc:mixs_url <https://microbiomedata/meta/mixs_url>`__

.. _domain-and-range-484:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-521:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-491:

Children
--------

.. _used-by-484:

Used by
-------

.. _other-properties-392:

Other properties
----------------

=============== =============================
**Aliases:**     relevant electronic resources
**Mappings:**    mixs:mixs_url
**In Subsets:**  sequencing
=============== =============================

Slot: mod_date
==============

The last date on which the database information was modified.

URI: `nmdc:mod_date <https://microbiomedata/meta/mod_date>`__

.. _domain-and-range-485:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-522:

Parents
-------

.. _children-492:

Children
--------

.. _used-by-485:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `OmicsProcessing <OmicsProcessing.md>`__

Slot: n_alkanes
===============

Concentration of n-alkanes; can include multiple n-alkanes

URI: `nmdc:n_alkanes <https://microbiomedata/meta/n_alkanes>`__

.. _domain-and-range-486:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-523:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-493:

Children
--------

.. _used-by-486:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-393:

Other properties
----------------

============= ==============
**Aliases:**   n-alkanes
**Mappings:**  mixs:n_alkanes
============= ==============

.. _slot-name-1:

Slot: name
==========

A human readable label for an entity

URI: https://microbiomedata/schema/workflow_execution_activity/name

.. _domain-and-range-487:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-524:

Parents
-------

.. _children-494:

Children
--------

-  `person value➞name <person_value_name.md>`__

.. _used-by-487:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `Activity <Activity.md>`__
-  `ChemicalEntity <ChemicalEntity.md>`__
-  `EnvironmentalMaterialTerm <EnvironmentalMaterialTerm.md>`__
-  `GeneProduct <GeneProduct.md>`__
-  `Instrument <Instrument.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NamedThing <NamedThing.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `OntologyClass <OntologyClass.md>`__
-  `Person <Person.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

Slot: ncbi_project_name
=======================

URI:
`nmdc:ncbi_project_name <https://microbiomedata/meta/ncbi_project_name>`__

.. _domain-and-range-488:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-525:

Parents
-------

.. _children-495:

Children
--------

.. _used-by-488:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

Slot: ncbi_taxonomy_name
========================

URI:
`nmdc:ncbi_taxonomy_name <https://microbiomedata/meta/ncbi_taxonomy_name>`__

.. _domain-and-range-489:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-526:

Parents
-------

.. _children-496:

Children
--------

.. _used-by-489:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: nitrate
=============

Concentration of nitrate in the sample

URI: `nmdc:nitrate <https://microbiomedata/meta/nitrate>`__

.. _domain-and-range-490:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-527:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-497:

Children
--------

.. _used-by-490:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-394:

Other properties
----------------

============= ============
**Aliases:**   nitrate
**Mappings:**  mixs:nitrate
============= ============

Slot: nitrite
=============

Concentration of nitrite in the sample

URI: `nmdc:nitrite <https://microbiomedata/meta/nitrite>`__

.. _domain-and-range-491:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-528:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-498:

Children
--------

.. _used-by-491:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-395:

Other properties
----------------

============= ============
**Aliases:**   nitrite
**Mappings:**  mixs:nitrite
============= ============

Slot: nitro
===========

Concentration of nitrogen (total)

URI: `nmdc:nitro <https://microbiomedata/meta/nitro>`__

.. _domain-and-range-492:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-529:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-499:

Children
--------

.. _used-by-492:

Used by
-------

.. _other-properties-396:

Other properties
----------------

============= ==========
**Aliases:**   nitrogen
**Mappings:**  mixs:nitro
============= ==========

.. _slot-nmdc-schema-version-1:

Slot: nmdc schema version
=========================

TODO

URI:
`nmdc:nmdc_schema_version <https://microbiomedata/meta/nmdc_schema_version>`__

.. _domain-and-range-493:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-530:

Parents
-------

.. _children-500:

Children
--------

-  `database➞nmdc schema version <database_nmdc_schema_version.md>`__

.. _used-by-493:

Used by
-------

.. _slot-has-calibration-2:

Slot: has calibration
=====================

A reference to a file that holds calibration information.

URI:
https://microbiomedata/schema/workflow_execution_activity/nom_analysis_activity_has_calibration

.. _domain-and-range-494:

Domain and Range
----------------

`NomAnalysisActivity <NomAnalysisActivity.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-531:

Parents
-------

-  is_a: `has calibration <has_calibration.md>`__

.. _children-501:

Children
--------

.. _used-by-494:

Used by
-------

-  `NomAnalysisActivity <NomAnalysisActivity.md>`__

Slot: nom analysis activity set
===============================

This property links a database object to the set of natural organic
matter (NOM) analysis activities.

URI:
`nmdc:nom_analysis_activity_set <https://microbiomedata/meta/nom_analysis_activity_set>`__

.. _domain-and-range-495:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`NomAnalysisActivity <NomAnalysisActivity.md>`__

.. _parents-532:

Parents
-------

.. _children-502:

Children
--------

.. _used-by-495:

Used by
-------

-  `Database <Database.md>`__

.. _slot-used-2:

Slot: used
==========

The instrument used to collect the data used in the analysis

URI:
https://microbiomedata/schema/workflow_execution_activity/nom_analysis_activity_used

.. _domain-and-range-496:

Domain and Range
----------------

`NomAnalysisActivity <NomAnalysisActivity.md>`__ → 0..1
`Instrument <Instrument.md>`__

.. _parents-533:

Parents
-------

-  is_a: `used <used.md>`__

.. _children-503:

Children
--------

.. _used-by-496:

Used by
-------

-  `NomAnalysisActivity <NomAnalysisActivity.md>`__

.. _other-properties-397:

Other properties
----------------

============= =========
**Mappings:**  prov:used
============= =========

Slot: non_mineral_nutr_regm
===========================

“Information about treatment involving the exposure of plant to
non-mineral nutrient such as oxygen, hydrogen or carbon; should include
the name of non-mineral nutrient, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple non-mineral nutrient regimens”

URI:
`nmdc:non_mineral_nutr_regm <https://microbiomedata/meta/non_mineral_nutr_regm>`__

.. _domain-and-range-497:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-534:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-504:

Children
--------

.. _used-by-497:

Used by
-------

.. _other-properties-398:

Other properties
----------------

============= ============================
**Aliases:**   non-mineral nutrient regimen
**Mappings:**  mixs:non_mineral_nutr_regm
============= ============================

Slot: nose_mouth_teeth_throat_disord
====================================

History of nose/mouth/teeth/throat disorders; can include multiple
disorders

URI:
`nmdc:nose_mouth_teeth_throat_disord <https://microbiomedata/meta/nose_mouth_teeth_throat_disord>`__

.. _domain-and-range-498:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-535:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-505:

Children
--------

.. _used-by-498:

Used by
-------

.. _other-properties-399:

Other properties
----------------

============= ===================================
**Aliases:**   nose/mouth/teeth/throat disorder
**Mappings:**  mixs:nose_mouth_teeth_throat_disord
============= ===================================

Slot: nose_throat_disord
========================

History of nose-throat disorders; can include multiple disorders

URI:
`nmdc:nose_throat_disord <https://microbiomedata/meta/nose_throat_disord>`__

.. _domain-and-range-499:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-536:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-506:

Children
--------

.. _used-by-499:

Used by
-------

.. _other-properties-400:

Other properties
----------------

============= =========================
**Aliases:**   lung/nose-throat disorder
**Mappings:**  mixs:nose_throat_disord
============= =========================

Slot: nucl_acid_amp
===================

“A link to a literature reference, electronic resource or a standard
operating procedure (SOP), that describes the enzymatic amplification
(PCR, TMA, NASBA) of specific nucleic acids”

URI: `nmdc:nucl_acid_amp <https://microbiomedata/meta/nucl_acid_amp>`__

.. _domain-and-range-500:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-537:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-507:

Children
--------

.. _used-by-500:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-401:

Other properties
----------------

=============== ==========================
**Aliases:**     nucleic acid amplification
**Mappings:**    mixs:nucl_acid_amp
**In Subsets:**  sequencing
=============== ==========================

Slot: nucl_acid_ext
===================

“A link to a literature reference, electronic resource or a standard
operating procedure (SOP), that describes the material separation to
recover the nucleic acid fraction from a sample”

URI: `nmdc:nucl_acid_ext <https://microbiomedata/meta/nucl_acid_ext>`__

.. _domain-and-range-501:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-538:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-508:

Children
--------

.. _used-by-501:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-402:

Other properties
----------------

=============== =======================
**Aliases:**     nucleic acid extraction
**Mappings:**    mixs:nucl_acid_ext
**In Subsets:**  sequencing
=============== =======================

Slot: num_aligned_reads
=======================

The sequence count number of input reads aligned to assembled contigs.

URI:
https://microbiomedata/schema/workflow_execution_activity/num_aligned_reads

.. _domain-and-range-502:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-539:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-509:

Children
--------

.. _used-by-502:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: num_input_reads
=====================

The sequence count number of input reads for assembly.

URI:
https://microbiomedata/schema/workflow_execution_activity/num_input_reads

.. _domain-and-range-503:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-540:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-510:

Children
--------

.. _used-by-503:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: num_replicons
===================

“Reports the number of replicons in a nuclear genome of eukaryotes, in
the genome of a bacterium or archaea or the number of segments in a
segmented virus. Always applied to the haploid chromosome count of a
eukaryote”

URI: `nmdc:num_replicons <https://microbiomedata/meta/num_replicons>`__

.. _domain-and-range-504:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-541:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-511:

Children
--------

.. _used-by-504:

Used by
-------

.. _other-properties-403:

Other properties
----------------

=============== ============================
**Aliases:**     number of replicons
**Mappings:**    mixs:num_replicons
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: number_contig
===================

“Total number of contigs in the cleaned/submitted assembly that makes up
a given genome, SAG, MAG, or UViG”

URI: `nmdc:number_contig <https://microbiomedata/meta/number_contig>`__

.. _domain-and-range-505:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-542:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-512:

Children
--------

.. _used-by-505:

Used by
-------

.. _other-properties-404:

Other properties
----------------

=============== ==================
**Aliases:**     number of contigs
**Mappings:**    mixs:number_contig
**In Subsets:**  sequencing
=============== ==================

Slot: number_pets
=================

The number of pets residing in the sampled space

URI: `nmdc:number_pets <https://microbiomedata/meta/number_pets>`__

.. _domain-and-range-506:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-543:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-513:

Children
--------

.. _used-by-506:

Used by
-------

.. _other-properties-405:

Other properties
----------------

============= ================
**Aliases:**   number of pets
**Mappings:**  mixs:number_pets
============= ================

Slot: number_plants
===================

The number of plant(s) in the sampling space

URI: `nmdc:number_plants <https://microbiomedata/meta/number_plants>`__

.. _domain-and-range-507:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-544:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-514:

Children
--------

.. _used-by-507:

Used by
-------

.. _other-properties-406:

Other properties
----------------

============= =====================
**Aliases:**   number of houseplants
**Mappings:**  mixs:number_plants
============= =====================

Slot: number_resident
=====================

The number of individuals currently occupying in the sampling location

URI:
`nmdc:number_resident <https://microbiomedata/meta/number_resident>`__

.. _domain-and-range-508:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-545:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-515:

Children
--------

.. _used-by-508:

Used by
-------

.. _other-properties-407:

Other properties
----------------

============= ====================
**Aliases:**   number of residents
**Mappings:**  mixs:number_resident
============= ====================

Slot: object set
================

Applies to a property that links a database object to a set of objects.
This is necessary in a json document to provide context for a list, and
to allow for a single json object that combines multiple object types

URI: `nmdc:object_set <https://microbiomedata/meta/object_set>`__

.. _domain-and-range-509:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\* `String <types/String.md>`__

.. _parents-546:

Parents
-------

.. _children-516:

Children
--------

.. _used-by-509:

Used by
-------

Slot: objective
===============

The scientific objectives associated with the entity. It SHOULD
correspond to scientific norms for objectives field in a structured
abstract.

URI: https://microbiomedata/schema/workflow_execution_activity/objective

.. _domain-and-range-510:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-547:

Parents
-------

.. _children-517:

Children
--------

.. _used-by-510:

Used by
-------

.. _other-properties-408:

Other properties
----------------

============= ==========
**Mappings:**  sio:000337
============= ==========

Slot: occup_density_samp
========================

Average number of occupants at time of sampling per square footage

URI:
`nmdc:occup_density_samp <https://microbiomedata/meta/occup_density_samp>`__

.. _domain-and-range-511:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-548:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-518:

Children
--------

.. _used-by-511:

Used by
-------

.. _other-properties-409:

Other properties
----------------

============= ============================
**Aliases:**   occupant density at sampling
**Mappings:**  mixs:occup_density_samp
============= ============================

Slot: occup_document
====================

The type of documentation of occupancy

URI:
`nmdc:occup_document <https://microbiomedata/meta/occup_document>`__

.. _domain-and-range-512:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-549:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-519:

Children
--------

.. _used-by-512:

Used by
-------

.. _other-properties-410:

Other properties
----------------

============= =======================
**Aliases:**   occupancy documentation
**Mappings:**  mixs:occup_document
============= =======================

Slot: occup_samp
================

Number of occupants present at time of sample within the given space

URI: `nmdc:occup_samp <https://microbiomedata/meta/occup_samp>`__

.. _domain-and-range-513:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-550:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-520:

Children
--------

.. _used-by-513:

Used by
-------

.. _other-properties-411:

Other properties
----------------

============= =====================
**Aliases:**   occupancy at sampling
**Mappings:**  mixs:occup_samp
============= =====================

.. _slot-has-input-2:

Slot: has input
===============

An input to a process.

URI:
`nmdc:omics_processing_has_input <https://microbiomedata/meta/omics_processing_has_input>`__

.. _domain-and-range-514:

Domain and Range
----------------

`OmicsProcessing <OmicsProcessing.md>`__ → 1..\*
`Biosample <Biosample.md>`__

.. _parents-551:

Parents
-------

-  is_a: `biosample processing➞has
   input <biosample_processing_has_input.md>`__

.. _children-521:

Children
--------

.. _used-by-514:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-412:

Other properties
----------------

============ =====
**Aliases:**  input
============ =====

Slot: omics processing identifiers
==================================

URI:
https://microbiomedata/schema/workflow_execution_activity/omics_processing_identifiers

.. _domain-and-range-515:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-552:

Parents
-------

-  is_a: `external database
   identifiers <external_database_identifiers.md>`__

.. _children-522:

Children
--------

.. _used-by-515:

Used by
-------

Slot: omics processing set
==========================

This property links a database object to the set of omics processings
within it.

URI:
`nmdc:omics_processing_set <https://microbiomedata/meta/omics_processing_set>`__

.. _domain-and-range-516:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`OmicsProcessing <OmicsProcessing.md>`__

.. _parents-553:

Parents
-------

.. _children-523:

Children
--------

.. _used-by-516:

Used by
-------

-  `Database <Database.md>`__

Slot: omics_type
================

URI: `nmdc:omics_type <https://microbiomedata/meta/omics_type>`__

.. _domain-and-range-517:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-554:

Parents
-------

.. _children-524:

Children
--------

.. _used-by-517:

Used by
-------

Slot: orcid
===========

The ORICD of a person.

URI: https://microbiomedata/schema/workflow_execution_activity/orcid

.. _domain-and-range-518:

Domain and Range
----------------

`PersonValue <PersonValue.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-555:

Parents
-------

.. _children-525:

Children
--------

.. _used-by-518:

Used by
-------

-  `PersonValue <PersonValue.md>`__

Slot: org_carb
==============

Concentration of organic carbon

URI: `nmdc:org_carb <https://microbiomedata/meta/org_carb>`__

.. _domain-and-range-519:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-556:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-526:

Children
--------

.. _used-by-519:

Used by
-------

.. _other-properties-413:

Other properties
----------------

============= ==============
**Aliases:**   organic carbon
**Mappings:**  mixs:org_carb
============= ==============

Slot: org_matter
================

Concentration of organic matter

URI: `nmdc:org_matter <https://microbiomedata/meta/org_matter>`__

.. _domain-and-range-520:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-557:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-527:

Children
--------

.. _used-by-520:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-414:

Other properties
----------------

============= ===============
**Aliases:**   organic matter
**Mappings:**  mixs:org_matter
============= ===============

Slot: org_nitro
===============

Concentration of organic nitrogen

URI: `nmdc:org_nitro <https://microbiomedata/meta/org_nitro>`__

.. _domain-and-range-521:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-558:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-528:

Children
--------

.. _used-by-521:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-415:

Other properties
----------------

============= ================
**Aliases:**   organic nitrogen
**Mappings:**  mixs:org_nitro
============= ================

Slot: org_particles
===================

“Concentration of particles such as faeces, hairs, food, vomit, paper
fibers, plant material, humus, etc.”

URI: `nmdc:org_particles <https://microbiomedata/meta/org_particles>`__

.. _domain-and-range-522:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-559:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-529:

Children
--------

.. _used-by-522:

Used by
-------

.. _other-properties-416:

Other properties
----------------

============= ==================
**Aliases:**   organic particles
**Mappings:**  mixs:org_particles
============= ==================

Slot: organism_count
====================

“Total cell count of any organism (or group of organisms) per gram,
volume or area of sample, should include name of organism followed by
count. The method that was used for the enumeration (e.g. qPCR, atp,
mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7
cells per ml; qpcr)”

URI:
`nmdc:organism_count <https://microbiomedata/meta/organism_count>`__

.. _domain-and-range-523:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-560:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-530:

Children
--------

.. _used-by-523:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-417:

Other properties
----------------

============= ===================
**Aliases:**   organism count
**Mappings:**  mixs:organism_count
============= ===================

Slot: organism_count_qpcr_info
==============================

“If qpcr was used for the cell count, the target gene name, the primer
sequence and the cycling conditions should also be provided. (Example:
16S rrna; FWD:ACGTAGCTATGACGT REV:GTGCTAGTCGAGTAC; initial
denaturation:90C_5min; denaturation:90C_2min; annealing:52C_30 sec;
elongation:72C_30 sec; 90 C for 1 min; final elongation:72C_5min; 30
cycles)”

URI:
`nmdc:organism_count_qpcr_info <https://microbiomedata/meta/organism_count_qpcr_info>`__

.. _domain-and-range-524:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-561:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-531:

Children
--------

.. _used-by-524:

Used by
-------

.. _other-properties-418:

Other properties
----------------

============= ===============================
**Aliases:**   organism count qPCR information
**Mappings:**  mixs:organism_count_qpcr_info
============= ===============================

Slot: output base count
=======================

After QC analysis nucleotide base count number.

URI:
https://microbiomedata/schema/workflow_execution_activity/output_base_count

.. _domain-and-range-525:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-562:

Parents
-------

-  is_a: `read QC analysis statistic <read_QC_analysis_statistic.md>`__

.. _children-532:

Children
--------

.. _used-by-525:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

Slot: output_read_bases
=======================

TODO

URI:
https://microbiomedata/schema/workflow_execution_activity/output_read_bases

.. _domain-and-range-526:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-563:

Parents
-------

.. _children-533:

Children
--------

-  `read QC analysis
   activity➞output_read_bases <read_QC_analysis_activity_output_read_bases.md>`__

.. _used-by-526:

Used by
-------

Slot: output read count
=======================

After QC analysis sequence count number.

URI:
https://microbiomedata/schema/workflow_execution_activity/output_read_count

.. _domain-and-range-527:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-564:

Parents
-------

-  is_a: `read QC analysis statistic <read_QC_analysis_statistic.md>`__

.. _children-534:

Children
--------

.. _used-by-527:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

Slot: owc_tvdss
===============

Depth of the original oil water contact (OWC) zone (average) (m TVDSS)

URI: `nmdc:owc_tvdss <https://microbiomedata/meta/owc_tvdss>`__

.. _domain-and-range-528:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-565:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-535:

Children
--------

.. _used-by-528:

Used by
-------

.. _other-properties-419:

Other properties
----------------

============= =======================
**Aliases:**   oil water contact depth
**Mappings:**  mixs:owc_tvdss
============= =======================

Slot: oxy_stat_samp
===================

Oxygenation status of sample

URI: `nmdc:oxy_stat_samp <https://microbiomedata/meta/oxy_stat_samp>`__

.. _domain-and-range-529:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-566:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-536:

Children
--------

.. _used-by-529:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-420:

Other properties
----------------

============= ============================
**Aliases:**   oxygenation status of sample
**Mappings:**  mixs:oxy_stat_samp
============= ============================

Slot: oxygen
============

Oxygen (gas) amount or concentration at the time of sampling

URI: `nmdc:oxygen <https://microbiomedata/meta/oxygen>`__

.. _domain-and-range-530:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-567:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-537:

Children
--------

.. _used-by-530:

Used by
-------

.. _other-properties-421:

Other properties
----------------

============= ===========
**Aliases:**   oxygen
**Mappings:**  mixs:oxygen
============= ===========

Slot: part of
=============

Links a resource to another resource that either logically or physically
includes it.

URI: https://microbiomedata/schema/workflow_execution_activity/part_of

.. _domain-and-range-531:

Domain and Range
----------------

`NamedThing <NamedThing.md>`__ → 0..\* `NamedThing <NamedThing.md>`__

.. _parents-568:

Parents
-------

.. _children-538:

Children
--------

.. _used-by-531:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-422:

Other properties
----------------

============= ================
**Aliases:**   is part of
**Mappings:**  dcterms:isPartOf
============= ================

Slot: part_org_carb
===================

Concentration of particulate organic carbon

URI: `nmdc:part_org_carb <https://microbiomedata/meta/part_org_carb>`__

.. _domain-and-range-532:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-569:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-539:

Children
--------

.. _used-by-532:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-423:

Other properties
----------------

============= ==========================
**Aliases:**   particulate organic carbon
**Mappings:**  mixs:part_org_carb
============= ==========================

Slot: part_org_nitro
====================

Concentration of particulate organic nitrogen

URI:
`nmdc:part_org_nitro <https://microbiomedata/meta/part_org_nitro>`__

.. _domain-and-range-533:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-570:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-540:

Children
--------

.. _used-by-533:

Used by
-------

.. _other-properties-424:

Other properties
----------------

============= ============================
**Aliases:**   particulate organic nitrogen
**Mappings:**  mixs:part_org_nitro
============= ============================

Slot: particle_class
====================

“Particles are classified, based on their size, into six general
categories:clay, silt, sand, gravel, cobbles, and boulders; should
include amount of particle preceded by the name of the particle type;
can include multiple values”

URI:
`nmdc:particle_class <https://microbiomedata/meta/particle_class>`__

.. _domain-and-range-534:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-571:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-541:

Children
--------

.. _used-by-534:

Used by
-------

.. _other-properties-425:

Other properties
----------------

============= =======================
**Aliases:**   particle classification
**Mappings:**  mixs:particle_class
============= =======================

Slot: pathogenicity
===================

To what is the entity pathogenic

URI: `nmdc:pathogenicity <https://microbiomedata/meta/pathogenicity>`__

.. _domain-and-range-535:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-572:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-542:

Children
--------

.. _used-by-535:

Used by
-------

.. _other-properties-426:

Other properties
----------------

=============== ============================
**Aliases:**     known pathogenicity
**Mappings:**    mixs:pathogenicity
**In Subsets:**  nucleic acid sequence source
=============== ============================

.. _slot-has_part-1:

Slot: has_part
==============

A pathway can be broken down to a series of reaction step

URI:
`nmdc:pathway_has_part <https://microbiomedata/meta/pathway_has_part>`__

.. _domain-and-range-536:

Domain and Range
----------------

`Pathway <Pathway.md>`__ → 0..\* `Reaction <Reaction.md>`__

.. _parents-573:

Parents
-------

-  is_a: `has_part <has_part.md>`__

.. _children-543:

Children
--------

.. _used-by-536:

Used by
-------

-  `Pathway <Pathway.md>`__

Slot: pcr_cond
==============

Description of reaction conditions and components of PCR in the form of
‘initial denaturation:94degC_1.5min; annealing=…’

URI: `nmdc:pcr_cond <https://microbiomedata/meta/pcr_cond>`__

.. _domain-and-range-537:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-574:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-544:

Children
--------

.. _used-by-537:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-427:

Other properties
----------------

=============== ==============
**Aliases:**     pcr conditions
**Mappings:**    mixs:pcr_cond
**In Subsets:**  sequencing
=============== ==============

Slot: pcr_primers
=================

“PCR primers that were used to amplify the sequence of the targeted
gene, locus or subfragment. This field should contain all the primers
used for a single PCR reaction if multiple forward or reverse primers
are present in a single PCR reaction. The primer sequence should be
reported in uppercase letters”

URI: `nmdc:pcr_primers <https://microbiomedata/meta/pcr_primers>`__

.. _domain-and-range-538:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-575:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-545:

Children
--------

.. _used-by-538:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-428:

Other properties
----------------

=============== ================
**Aliases:**     pcr primers
**Mappings:**    mixs:pcr_primers
**In Subsets:**  sequencing
=============== ================

.. _slot-all-proteins-1:

Slot: all proteins
==================

the list of protein identifiers that are associated with the peptide
sequence

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_all_proteins

.. _domain-and-range-539:

Domain and Range
----------------

`PeptideQuantification <PeptideQuantification.md>`__ → 0..\*
`GeneProduct <GeneProduct.md>`__

.. _parents-576:

Parents
-------

-  is_a: `all proteins <all_proteins.md>`__

.. _children-546:

Children
--------

.. _used-by-539:

Used by
-------

-  `PeptideQuantification <PeptideQuantification.md>`__

.. _slot-best-protein-1:

Slot: best protein
==================

the specific protein identifier most correctly associated with the
peptide sequence

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_best_protein

.. _domain-and-range-540:

Domain and Range
----------------

`PeptideQuantification <PeptideQuantification.md>`__ → 0..1
`GeneProduct <GeneProduct.md>`__

.. _parents-577:

Parents
-------

-  is_a: `best protein <best_protein.md>`__

.. _children-547:

Children
--------

.. _used-by-540:

Used by
-------

-  `PeptideQuantification <PeptideQuantification.md>`__

.. _slot-min_q_value-1:

Slot: min_q_value
=================

smallest Q-Value associated with the peptide sequence as provided by
MSGFPlus tool

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_min_q_value

.. _domain-and-range-541:

Domain and Range
----------------

`PeptideQuantification <PeptideQuantification.md>`__ → 0..1
`Float <types/Float.md>`__

.. _parents-578:

Parents
-------

-  is_a: `min_q_value <min_q_value.md>`__

.. _children-548:

Children
--------

.. _used-by-541:

Used by
-------

-  `PeptideQuantification <PeptideQuantification.md>`__

.. _other-properties-429:

Other properties
----------------

============= =============================
**See also:**  `OBI:0001442 <OBI:0001442>`__
============= =============================

Slot: peptide sequence
======================

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_peptide_sequence

.. _domain-and-range-542:

Domain and Range
----------------

`PeptideQuantification <PeptideQuantification.md>`__ → 0..1
`String <types/String.md>`__

.. _parents-579:

Parents
-------

-  is_a: `peptide sequence <peptide_sequence.md>`__

.. _children-549:

Children
--------

.. _used-by-542:

Used by
-------

-  `PeptideQuantification <PeptideQuantification.md>`__

Slot: peptide_spectral_count
============================

sum of filter passing MS2 spectra associated with the peptide sequence
within a given LC-MS/MS data file

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_peptide_spectral_count

.. _domain-and-range-543:

Domain and Range
----------------

`PeptideQuantification <PeptideQuantification.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-580:

Parents
-------

-  is_a: `peptide_spectral_count <peptide_spectral_count.md>`__

.. _children-550:

Children
--------

.. _used-by-543:

Used by
-------

-  `PeptideQuantification <PeptideQuantification.md>`__

Slot: peptide_sum_masic_abundance
=================================

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the peptide sequence from a given LC-MS/MS data file
using the MASIC tool

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_peptide_sum_masic_abundance

.. _domain-and-range-544:

Domain and Range
----------------

`PeptideQuantification <PeptideQuantification.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-581:

Parents
-------

-  is_a:
   `peptide_sum_masic_abundance <peptide_sum_masic_abundance.md>`__

.. _children-551:

Children
--------

.. _used-by-544:

Used by
-------

-  `PeptideQuantification <PeptideQuantification.md>`__

.. _slot-peptide-sequence-1:

Slot: peptide sequence
======================

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_sequence

.. _domain-and-range-545:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-582:

Parents
-------

.. _children-552:

Children
--------

-  `peptide quantification➞peptide
   sequence <peptide_quantification_peptide_sequence.md>`__

.. _used-by-545:

Used by
-------

Slot: peptide_sequence_count
============================

count of peptide sequences grouped to the best_protein

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_sequence_count

.. _domain-and-range-546:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-583:

Parents
-------

.. _children-553:

Children
--------

-  `protein
   quantification➞peptide_sequence_count <protein_quantification_peptide_sequence_count.md>`__

.. _used-by-546:

Used by
-------

.. _slot-peptide_spectral_count-1:

Slot: peptide_spectral_count
============================

sum of filter passing MS2 spectra associated with the peptide sequence
within a given LC-MS/MS data file

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_spectral_count

.. _domain-and-range-547:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-584:

Parents
-------

.. _children-554:

Children
--------

-  `peptide
   quantification➞peptide_spectral_count <peptide_quantification_peptide_spectral_count.md>`__

.. _used-by-547:

Used by
-------

.. _slot-peptide_sum_masic_abundance-1:

Slot: peptide_sum_masic_abundance
=================================

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the peptide sequence from a given LC-MS/MS data file
using the MASIC tool

URI:
https://microbiomedata/schema/workflow_execution_activity/peptide_sum_masic_abundance

.. _domain-and-range-548:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-585:

Parents
-------

.. _children-555:

Children
--------

-  `peptide
   quantification➞peptide_sum_masic_abundance <peptide_quantification_peptide_sum_masic_abundance.md>`__

.. _used-by-548:

Used by
-------

Slot: permeability
==================

Measure of the ability of a hydrocarbon resource to allow fluids to pass
through it. (Additional information:
https://en.wikipedia.org/wiki/Permeability_(earth_sciences))

URI: `nmdc:permeability <https://microbiomedata/meta/permeability>`__

.. _domain-and-range-549:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-586:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-556:

Children
--------

.. _used-by-549:

Used by
-------

.. _other-properties-430:

Other properties
----------------

============= =================
**Aliases:**   permeability
**Mappings:**  mixs:permeability
============= =================

.. _slot-id-1:

Slot: id
========

Should be an ORCID. Specify in CURIE format. E.g ORCID:0000-1111-…

URI: https://microbiomedata/schema/workflow_execution_activity/person_id

.. _domain-and-range-550:

Domain and Range
----------------

`Person <Person.md>`__ → 1..1 `String <types/String.md>`__

.. _parents-587:

Parents
-------

-  is_a: `id <id.md>`__

.. _children-557:

Children
--------

.. _used-by-550:

Used by
-------

-  `Person <Person.md>`__

.. _slot-has-raw-value-2:

Slot: has raw value
===================

The full name of the Investgator in format FIRST LAST.

URI:
https://microbiomedata/schema/workflow_execution_activity/person_value_has_raw_value

.. _domain-and-range-551:

Domain and Range
----------------

`PersonValue <PersonValue.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-588:

Parents
-------

-  is_a: `has raw value <has_raw_value.md>`__

.. _children-558:

Children
--------

.. _used-by-551:

Used by
-------

-  `PersonValue <PersonValue.md>`__

.. _slot-name-2:

Slot: name
==========

The full name of the Investgator. It should follow the format FIRST
[MIDDLE NAME\| MIDDLE INITIAL] LAST, where MIDDLE NAME\| MIDDLE INITIAL
is optional.

URI:
https://microbiomedata/schema/workflow_execution_activity/person_value_name

.. _domain-and-range-552:

Domain and Range
----------------

`PersonValue <PersonValue.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-589:

Parents
-------

-  is_a: `name <name.md>`__

.. _children-559:

Children
--------

.. _used-by-552:

Used by
-------

-  `PersonValue <PersonValue.md>`__

Slot: perturbation
==================

“Type of perturbation, e.g. chemical administration, physical
disturbance, etc., coupled with perturbation regimen including how many
times the perturbation was repeated, how long each perturbation lasted,
and the start and end time of the entire perturbation period; can
include multiple perturbation types”

URI: `nmdc:perturbation <https://microbiomedata/meta/perturbation>`__

.. _domain-and-range-553:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-590:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-560:

Children
--------

.. _used-by-553:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-431:

Other properties
----------------

============= =================
**Aliases:**   perturbation
**Mappings:**  mixs:perturbation
============= =================

Slot: pesticide_regm
====================

“Information about treatment involving use of insecticides; should
include the name of pesticide, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple pesticide regimens”

URI:
`nmdc:pesticide_regm <https://microbiomedata/meta/pesticide_regm>`__

.. _domain-and-range-554:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-591:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-561:

Children
--------

.. _used-by-554:

Used by
-------

.. _other-properties-432:

Other properties
----------------

============= ===================
**Aliases:**   pesticide regimen
**Mappings:**  mixs:pesticide_regm
============= ===================

Slot: pet_farm_animal
=====================

“Specification of presence of pets or farm animals in the environment of
subject, if yes the animals should be specified; can include multiple
animals present”

URI:
`nmdc:pet_farm_animal <https://microbiomedata/meta/pet_farm_animal>`__

.. _domain-and-range-555:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-592:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-562:

Children
--------

.. _used-by-555:

Used by
-------

.. _other-properties-433:

Other properties
----------------

============= ================================
**Aliases:**   presence of pets or farm animals
**Mappings:**  mixs:pet_farm_animal
============= ================================

Slot: petroleum_hydrocarb
=========================

Concentration of petroleum hydrocarbon

URI:
`nmdc:petroleum_hydrocarb <https://microbiomedata/meta/petroleum_hydrocarb>`__

.. _domain-and-range-556:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-593:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-563:

Children
--------

.. _used-by-556:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-434:

Other properties
----------------

============= ========================
**Aliases:**   petroleum hydrocarbon
**Mappings:**  mixs:petroleum_hydrocarb
============= ========================

Slot: ph
========

“Ph measurement of the sample, or liquid portion of sample, or aqueous
phase of the fluid”

URI: `nmdc:ph <https://microbiomedata/meta/ph>`__

.. _domain-and-range-557:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-594:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-564:

Children
--------

.. _used-by-557:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-435:

Other properties
----------------

============= =======
**Aliases:**   pH
**Mappings:**  mixs:ph
============= =======

Slot: ph_meth
=============

Reference or method used in determining ph

URI: `nmdc:ph_meth <https://microbiomedata/meta/ph_meth>`__

.. _domain-and-range-558:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-595:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-565:

Children
--------

.. _used-by-558:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-436:

Other properties
----------------

============= ============
**Aliases:**   pH method
**Mappings:**  mixs:ph_meth
============= ============

Slot: ph_regm
=============

“Information about treatment involving exposure of plants to varying
levels of ph of the growth media, treatment regimen including how many
times the treatment was repeated, how long each treatment lasted, and
the start and end time of the entire treatment; can include multiple
regimen”

URI: `nmdc:ph_regm <https://microbiomedata/meta/ph_regm>`__

.. _domain-and-range-559:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-596:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-566:

Children
--------

.. _used-by-559:

Used by
-------

.. _other-properties-437:

Other properties
----------------

============= ============
**Aliases:**   pH regimen
**Mappings:**  mixs:ph_regm
============= ============

Slot: phaeopigments
===================

Concentration of phaeopigments; can include multiple phaeopigments

URI: `nmdc:phaeopigments <https://microbiomedata/meta/phaeopigments>`__

.. _domain-and-range-560:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-597:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-567:

Children
--------

.. _used-by-560:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-438:

Other properties
----------------

============= ==================
**Aliases:**   phaeopigments
**Mappings:**  mixs:phaeopigments
============= ==================

.. _slot-phase-1:

Slot: phase
===========

The phase for a coding sequence entity. For example, phase of a CDS as
represented in a GFF3 with a value of 0, 1 or 2.

URI: `nmdc:phase <https://microbiomedata/meta/phase>`__

.. _domain-and-range-561:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-598:

Parents
-------

.. _children-568:

Children
--------

-  `genome feature➞phase <genome_feature_phase.md>`__

.. _used-by-561:

Used by
-------

.. _other-properties-439:

Other properties
----------------

=================== =============
**Exact Mappings:**  biolink:phase
=================== =============

Slot: phosphate
===============

Concentration of phosphate

URI: `nmdc:phosphate <https://microbiomedata/meta/phosphate>`__

.. _domain-and-range-562:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-599:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-569:

Children
--------

.. _used-by-562:

Used by
-------

.. _other-properties-440:

Other properties
----------------

============= ==============
**Aliases:**   phosphate
**Mappings:**  mixs:phosphate
============= ==============

Slot: phosplipid_fatt_acid
==========================

Concentration of phospholipid fatty acids; can include multiple values

URI:
`nmdc:phosplipid_fatt_acid <https://microbiomedata/meta/phosplipid_fatt_acid>`__

.. _domain-and-range-563:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-600:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-570:

Children
--------

.. _used-by-563:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-441:

Other properties
----------------

============= =========================
**Aliases:**   phospholipid fatty acid
**Mappings:**  mixs:phosplipid_fatt_acid
============= =========================

Slot: photon_flux
=================

Measurement of photon flux

URI: `nmdc:photon_flux <https://microbiomedata/meta/photon_flux>`__

.. _domain-and-range-564:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-601:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-571:

Children
--------

.. _used-by-564:

Used by
-------

.. _other-properties-442:

Other properties
----------------

============= ================
**Aliases:**   photon flux
**Mappings:**  mixs:photon_flux
============= ================

Slot: plant_growth_med
======================

“Specification of the media for growing the plants or tissue cultured
samples, e.g. soil, aeroponic, hydroponic, in vitro solid culture
medium, in vitro liquid culture medium. Recommended value is a specific
value from EO:plant growth medium (follow this link for terms
http://purl.obolibrary.org/obo/EO_0007147) or other controlled
vocabulary”

URI:
`nmdc:plant_growth_med <https://microbiomedata/meta/plant_growth_med>`__

.. _domain-and-range-565:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-602:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-572:

Children
--------

.. _used-by-565:

Used by
-------

.. _other-properties-443:

Other properties
----------------

============= =====================
**Aliases:**   plant growth medium
**Mappings:**  mixs:plant_growth_med
============= =====================

Slot: plant_product
===================

“Substance produced by the plant, where the sample was obtained from”

URI: `nmdc:plant_product <https://microbiomedata/meta/plant_product>`__

.. _domain-and-range-566:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-603:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-573:

Children
--------

.. _used-by-566:

Used by
-------

.. _other-properties-444:

Other properties
----------------

============= ==================
**Aliases:**   plant product
**Mappings:**  mixs:plant_product
============= ==================

Slot: plant_sex
===============

“Sex of the reproductive parts on the whole plant, e.g. pistillate,
staminate, monoecieous, hermaphrodite.”

URI: `nmdc:plant_sex <https://microbiomedata/meta/plant_sex>`__

.. _domain-and-range-567:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-604:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-574:

Children
--------

.. _used-by-567:

Used by
-------

.. _other-properties-445:

Other properties
----------------

============= ==============
**Aliases:**   plant sex
**Mappings:**  mixs:plant_sex
============= ==============

Slot: plant_struc
=================

“Name of plant structure the sample was obtained from; for Plant
Ontology (PO) (v releases/2017-12-14) terms, see
http://purl.bioontology.org/ontology/PO, e.g. petiole epidermis
(PO_0000051). If an individual flower is sampled, the sex of it can be
recorded here.”

URI: `nmdc:plant_struc <https://microbiomedata/meta/plant_struc>`__

.. _domain-and-range-568:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-605:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-575:

Children
--------

.. _used-by-568:

Used by
-------

.. _other-properties-446:

Other properties
----------------

============= ================
**Aliases:**   plant structure
**Mappings:**  mixs:plant_struc
============= ================

Slot: ploidy
============

“The ploidy level of the genome (e.g. allopolyploid, haploid, diploid,
triploid, tetraploid). It has implications for the downstream study of
duplicated gene and regions of the genomes (and perhaps for difficulties
in assembly). For terms, please select terms listed under class ploidy
(PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser
of PATO (v 2018-03-27) please refer to
http://purl.bioontology.org/ontology/PATO”

URI: `nmdc:ploidy <https://microbiomedata/meta/ploidy>`__

.. _domain-and-range-569:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-606:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-576:

Children
--------

.. _used-by-569:

Used by
-------

.. _other-properties-447:

Other properties
----------------

=============== ============================
**Aliases:**     ploidy
**Mappings:**    mixs:ploidy
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: pollutants
================

“Pollutant types and, amount or concentrations measured at the time of
sampling; can report multiple pollutants by entering numeric values
preceded by name of pollutant”

URI: `nmdc:pollutants <https://microbiomedata/meta/pollutants>`__

.. _domain-and-range-570:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-607:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-577:

Children
--------

.. _used-by-570:

Used by
-------

.. _other-properties-448:

Other properties
----------------

============= ===============
**Aliases:**   pollutants
**Mappings:**  mixs:pollutants
============= ===============

Slot: pool_dna_extracts
=======================

“Indicate whether multiple DNA extractions were mixed. If the answer
yes, the number of extracts that were pooled should be given”

URI:
`nmdc:pool_dna_extracts <https://microbiomedata/meta/pool_dna_extracts>`__

.. _domain-and-range-571:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-608:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-578:

Children
--------

.. _used-by-571:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-449:

Other properties
----------------

============= =================================
**Aliases:**   pooling of DNA extracts (if done)
**Mappings:**  mixs:pool_dna_extracts
============= =================================

Slot: porosity
==============

Porosity of deposited sediment is volume of voids divided by the total
volume of sample

URI: `nmdc:porosity <https://microbiomedata/meta/porosity>`__

.. _domain-and-range-572:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-609:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-579:

Children
--------

.. _used-by-572:

Used by
-------

.. _other-properties-450:

Other properties
----------------

============= =============
**Aliases:**   porosity
**Mappings:**  mixs:porosity
============= =============

Slot: potassium
===============

Concentration of potassium in the sample

URI: `nmdc:potassium <https://microbiomedata/meta/potassium>`__

.. _domain-and-range-573:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-610:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-580:

Children
--------

.. _used-by-573:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-451:

Other properties
----------------

============= ==============
**Aliases:**   potassium
**Mappings:**  mixs:potassium
============= ==============

Slot: pour_point
================

“Temperature at which a liquid becomes semi solid and loses its flow
characteristics. In crude oil a highB pour pointB is generally
associated with a high paraffin content, typically found in crude
deriving from a larger proportion of plant material. (soure:
https://en.wikipedia.org/wiki/pour_point)”

URI: `nmdc:pour_point <https://microbiomedata/meta/pour_point>`__

.. _domain-and-range-574:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-611:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-581:

Children
--------

.. _used-by-574:

Used by
-------

.. _other-properties-452:

Other properties
----------------

============= ===============
**Aliases:**   pour point
**Mappings:**  mixs:pour_point
============= ===============

Slot: pre_treatment
===================

The process of pre-treatment removes materials that can be easily
collected from the raw wastewater

URI: `nmdc:pre_treatment <https://microbiomedata/meta/pre_treatment>`__

.. _domain-and-range-575:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-612:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-582:

Children
--------

.. _used-by-575:

Used by
-------

.. _other-properties-453:

Other properties
----------------

============= ==================
**Aliases:**   pre-treatment
**Mappings:**  mixs:pre_treatment
============= ==================

Slot: pred_genome_struc
=======================

Expected structure of the viral genome

URI:
`nmdc:pred_genome_struc <https://microbiomedata/meta/pred_genome_struc>`__

.. _domain-and-range-576:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-613:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-583:

Children
--------

.. _used-by-576:

Used by
-------

.. _other-properties-454:

Other properties
----------------

=============== ==========================
**Aliases:**     predicted genome structure
**Mappings:**    mixs:pred_genome_struc
**In Subsets:**  sequencing
=============== ==========================

Slot: pred_genome_type
======================

Type of genome predicted for the UViG

URI:
`nmdc:pred_genome_type <https://microbiomedata/meta/pred_genome_type>`__

.. _domain-and-range-577:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-614:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-584:

Children
--------

.. _used-by-577:

Used by
-------

.. _other-properties-455:

Other properties
----------------

=============== =====================
**Aliases:**     predicted genome type
**Mappings:**    mixs:pred_genome_type
**In Subsets:**  sequencing
=============== =====================

Slot: pregnancy
===============

Date due of pregnancy

URI: `nmdc:pregnancy <https://microbiomedata/meta/pregnancy>`__

.. _domain-and-range-578:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-615:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-585:

Children
--------

.. _used-by-578:

Used by
-------

.. _other-properties-456:

Other properties
----------------

============= ==============
**Aliases:**   pregnancy
**Mappings:**  mixs:pregnancy
============= ==============

Slot: pres_animal
=================

The number and type of animals present in the sampling space

URI: `nmdc:pres_animal <https://microbiomedata/meta/pres_animal>`__

.. _domain-and-range-579:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-616:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-586:

Children
--------

.. _used-by-579:

Used by
-------

.. _other-properties-457:

Other properties
----------------

============= =====================================
**Aliases:**   presence of pets, animals, or insects
**Mappings:**  mixs:pres_animal
============= =====================================

Slot: pressure
==============

“Pressure to which the sample is subject to, in atmospheres”

URI: `nmdc:pressure <https://microbiomedata/meta/pressure>`__

.. _domain-and-range-580:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-617:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-587:

Children
--------

.. _used-by-580:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-458:

Other properties
----------------

============= =============
**Aliases:**   pressure
**Mappings:**  mixs:pressure
============= =============

Slot: previous_land_use
=======================

Previous land use and dates

URI:
`nmdc:previous_land_use <https://microbiomedata/meta/previous_land_use>`__

.. _domain-and-range-581:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-618:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-588:

Children
--------

.. _used-by-581:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-459:

Other properties
----------------

============= =========================
**Aliases:**   history/previous land use
**Mappings:**  mixs:previous_land_use
============= =========================

Slot: previous_land_use_meth
============================

Reference or method used in determining previous land use and dates

URI:
`nmdc:previous_land_use_meth <https://microbiomedata/meta/previous_land_use_meth>`__

.. _domain-and-range-582:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-619:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-589:

Children
--------

.. _used-by-582:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-460:

Other properties
----------------

============= ================================
**Aliases:**   history/previous land use method
**Mappings:**  mixs:previous_land_use_meth
============= ================================

Slot: primary_prod
==================

“Measurement of primary production, generally measured as isotope
uptake”

URI: `nmdc:primary_prod <https://microbiomedata/meta/primary_prod>`__

.. _domain-and-range-583:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-620:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-590:

Children
--------

.. _used-by-583:

Used by
-------

.. _other-properties-461:

Other properties
----------------

============= ==================
**Aliases:**   primary production
**Mappings:**  mixs:primary_prod
============= ==================

Slot: primary_treatment
=======================

The process to produce both a generally homogeneous liquid capable of
being treated biologically and a sludge that can be separately treated
or processed

URI:
`nmdc:primary_treatment <https://microbiomedata/meta/primary_treatment>`__

.. _domain-and-range-584:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-621:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-591:

Children
--------

.. _used-by-584:

Used by
-------

.. _other-properties-462:

Other properties
----------------

============= ======================
**Aliases:**   primary treatment
**Mappings:**  mixs:primary_treatment
============= ======================

Slot: principal investigator
============================

represents the PI

URI:
`nmdc:principal_investigator <https://microbiomedata/meta/principal_investigator>`__

.. _domain-and-range-585:

Domain and Range
----------------

None → 0..1 `PersonValue <PersonValue.md>`__

.. _parents-622:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-592:

Children
--------

.. _used-by-585:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__
-  `Study <Study.md>`__

.. _other-properties-463:

Other properties
----------------

============ ==
**Aliases:**  PI
============ ==

Slot: processing_institution
============================

The organization that processed the sample.

URI:
`nmdc:processing_institution <https://microbiomedata/meta/processing_institution>`__

.. _domain-and-range-586:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-623:

Parents
-------

.. _children-593:

Children
--------

.. _used-by-586:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

Slot: prod_rate
===============

Oil and/or gas production rates per well (e.g. 524 m3 / day)

URI: `nmdc:prod_rate <https://microbiomedata/meta/prod_rate>`__

.. _domain-and-range-587:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-624:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-594:

Children
--------

.. _used-by-587:

Used by
-------

.. _other-properties-464:

Other properties
----------------

============= ===============
**Aliases:**   production rate
**Mappings:**  mixs:prod_rate
============= ===============

Slot: prod_start_date
=====================

Date of field’s first production

URI:
`nmdc:prod_start_date <https://microbiomedata/meta/prod_start_date>`__

.. _domain-and-range-588:

Domain and Range
----------------

None → 0..1 `TimestampValue <TimestampValue.md>`__

.. _parents-625:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-595:

Children
--------

.. _used-by-588:

Used by
-------

.. _other-properties-465:

Other properties
----------------

============= =====================
**Aliases:**   production start date
**Mappings:**  mixs:prod_start_date
============= =====================

Slot: profile image url
=======================

A url that points to an image of a person.

URI:
https://microbiomedata/schema/workflow_execution_activity/profile_image_url

.. _domain-and-range-589:

Domain and Range
----------------

`PersonValue <PersonValue.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-626:

Parents
-------

.. _children-596:

Children
--------

.. _used-by-589:

Used by
-------

-  `PersonValue <PersonValue.md>`__

Slot: profile_position
======================

Cross-sectional position in the hillslope where sample was
collected.sample area position in relation to surrounding areas

URI:
`nmdc:profile_position <https://microbiomedata/meta/profile_position>`__

.. _domain-and-range-590:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-627:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-597:

Children
--------

.. _used-by-590:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-466:

Other properties
----------------

============= =====================
**Aliases:**   profile position
**Mappings:**  mixs:profile_position
============= =====================

Slot: project_name
==================

Name of the project within which the sequencing was organized

URI: `nmdc:project_name <https://microbiomedata/meta/project_name>`__

.. _domain-and-range-591:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-628:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-598:

Children
--------

.. _used-by-591:

Used by
-------

.. _other-properties-467:

Other properties
----------------

=============== =================
**Aliases:**     project name
**Mappings:**    mixs:project_name
**In Subsets:**  investigation
=============== =================

Slot: propagation
=================

“This field is specific to different taxa. For phages: lytic/lysogenic,
for plasmids: incompatibility group, for eukaryotes: sexual/asexual
(Note: there is the strong opinion to name phage propagation obligately
lytic or temperate, therefore we also give this choice”

URI: `nmdc:propagation <https://microbiomedata/meta/propagation>`__

.. _domain-and-range-592:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-629:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-599:

Children
--------

.. _used-by-592:

Used by
-------

.. _other-properties-468:

Other properties
----------------

=============== ============================
**Aliases:**     propagation
**Mappings:**    mixs:propagation
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: proport_woa_temperature
=============================

URI:
`nmdc:proport_woa_temperature <https://microbiomedata/meta/proport_woa_temperature>`__

.. _domain-and-range-593:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-630:

Parents
-------

.. _children-600:

Children
--------

.. _used-by-593:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _slot-all-proteins-2:

Slot: all proteins
==================

the grouped list of protein identifiers associated with the peptide
sequences that were grouped to a best protein

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_quantification_all_proteins

.. _domain-and-range-594:

Domain and Range
----------------

`ProteinQuantification <ProteinQuantification.md>`__ → 0..\*
`GeneProduct <GeneProduct.md>`__

.. _parents-631:

Parents
-------

-  is_a: `all proteins <all_proteins.md>`__

.. _children-601:

Children
--------

.. _used-by-594:

Used by
-------

-  `ProteinQuantification <ProteinQuantification.md>`__

.. _slot-best-protein-2:

Slot: best protein
==================

the specific protein identifier most correctly grouped to its associated
peptide sequences

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_quantification_best_protein

.. _domain-and-range-595:

Domain and Range
----------------

`ProteinQuantification <ProteinQuantification.md>`__ → 0..1
`GeneProduct <GeneProduct.md>`__

.. _parents-632:

Parents
-------

-  is_a: `best protein <best_protein.md>`__

.. _children-602:

Children
--------

.. _used-by-595:

Used by
-------

-  `ProteinQuantification <ProteinQuantification.md>`__

.. _slot-peptide_sequence_count-1:

Slot: peptide_sequence_count
============================

count of peptide sequences grouped to the best_protein

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_quantification_peptide_sequence_count

.. _domain-and-range-596:

Domain and Range
----------------

`ProteinQuantification <ProteinQuantification.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-633:

Parents
-------

-  is_a: `peptide_sequence_count <peptide_sequence_count.md>`__

.. _children-603:

Children
--------

.. _used-by-596:

Used by
-------

-  `ProteinQuantification <ProteinQuantification.md>`__

Slot: protein_spectral_count
============================

sum of filter passing MS2 spectra associated with the best protein
within a given LC-MS/MS data file

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_quantification_protein_spectral_count

.. _domain-and-range-597:

Domain and Range
----------------

`ProteinQuantification <ProteinQuantification.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-634:

Parents
-------

-  is_a: `protein_spectral_count <protein_spectral_count.md>`__

.. _children-604:

Children
--------

.. _used-by-597:

Used by
-------

-  `ProteinQuantification <ProteinQuantification.md>`__

Slot: protein_sum_masic_abundance
=================================

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the best protein from a given LC-MS/MS data file using
the MASIC tool

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_quantification_protein_sum_masic_abundance

.. _domain-and-range-598:

Domain and Range
----------------

`ProteinQuantification <ProteinQuantification.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-635:

Parents
-------

-  is_a:
   `protein_sum_masic_abundance <protein_sum_masic_abundance.md>`__

.. _children-605:

Children
--------

.. _used-by-598:

Used by
-------

-  `ProteinQuantification <ProteinQuantification.md>`__

.. _slot-protein_spectral_count-1:

Slot: protein_spectral_count
============================

sum of filter passing MS2 spectra associated with the best protein
within a given LC-MS/MS data file

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_spectral_count

.. _domain-and-range-599:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-636:

Parents
-------

.. _children-606:

Children
--------

-  `protein
   quantification➞protein_spectral_count <protein_quantification_protein_spectral_count.md>`__

.. _used-by-599:

Used by
-------

.. _slot-protein_sum_masic_abundance-1:

Slot: protein_sum_masic_abundance
=================================

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the best protein from a given LC-MS/MS data file using
the MASIC tool

URI:
https://microbiomedata/schema/workflow_execution_activity/protein_sum_masic_abundance

.. _domain-and-range-600:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-637:

Parents
-------

.. _children-607:

Children
--------

-  `protein
   quantification➞protein_sum_masic_abundance <protein_quantification_protein_sum_masic_abundance.md>`__

.. _used-by-600:

Used by
-------

Slot: publications
==================

A list of publications that are assocatiated with the entity. The
publicatons SHOULD be given using an identifier, such as a DOI or Pubmed
ID, if possible.

URI:
https://microbiomedata/schema/workflow_execution_activity/publications

.. _domain-and-range-601:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-638:

Parents
-------

.. _children-608:

Children
--------

.. _used-by-601:

Used by
-------

Slot: pulmonary_disord
======================

History of pulmonary disorders; can include multiple disorders

URI:
`nmdc:pulmonary_disord <https://microbiomedata/meta/pulmonary_disord>`__

.. _domain-and-range-602:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-639:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-609:

Children
--------

.. _used-by-602:

Used by
-------

.. _other-properties-469:

Other properties
----------------

============= =======================
**Aliases:**   lung/pulmonary disorder
**Mappings:**  mixs:pulmonary_disord
============= =======================

Slot: quad_pos
==============

The quadrant position of the sampling room within the building

URI: `nmdc:quad_pos <https://microbiomedata/meta/quad_pos>`__

.. _domain-and-range-603:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-640:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-610:

Children
--------

.. _used-by-603:

Used by
-------

.. _other-properties-470:

Other properties
----------------

============= =================
**Aliases:**   quadrant position
**Mappings:**  mixs:quad_pos
============= =================

.. _slot-has-numeric-value-1:

Slot: has numeric value
=======================

The number part of the quantity

URI:
https://microbiomedata/schema/workflow_execution_activity/quantity_value_has_numeric_value

.. _domain-and-range-604:

Domain and Range
----------------

`QuantityValue <QuantityValue.md>`__ → 0..1 `Double <types/Double.md>`__

.. _parents-641:

Parents
-------

-  is_a: `has numeric value <has_numeric_value.md>`__

.. _children-611:

Children
--------

.. _used-by-604:

Used by
-------

-  `QuantityValue <QuantityValue.md>`__

.. _other-properties-471:

Other properties
----------------

============= =================
**Mappings:**  qud:quantityValue
\              schema:value
============= =================

.. _slot-has-raw-value-3:

Slot: has raw value
===================

Unnormalized atomic string representation, should in syntax {number}
{unit}

URI:
https://microbiomedata/schema/workflow_execution_activity/quantity_value_has_raw_value

.. _domain-and-range-605:

Domain and Range
----------------

`QuantityValue <QuantityValue.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-642:

Parents
-------

-  is_a: `has raw value <has_raw_value.md>`__

.. _children-612:

Children
--------

.. _used-by-605:

Used by
-------

-  `QuantityValue <QuantityValue.md>`__

.. _slot-has-unit-1:

Slot: has unit
==============

The unit of the quantity

URI:
https://microbiomedata/schema/workflow_execution_activity/quantity_value_has_unit

.. _domain-and-range-606:

Domain and Range
----------------

`QuantityValue <QuantityValue.md>`__ → 0..1 `Unit <types/Unit.md>`__

.. _parents-643:

Parents
-------

-  is_a: `has unit <has_unit.md>`__

.. _children-613:

Children
--------

.. _used-by-606:

Used by
-------

-  `QuantityValue <QuantityValue.md>`__

.. _other-properties-472:

Other properties
----------------

============= ===============
**Aliases:**   scale
**Mappings:**  qud:unit
\              schema:unitCode
============= ===============

Slot: radiation_regm
====================

“Information about treatment involving exposure of plant or a plant part
to a particular radiation regimen; should include the radiation type,
amount or intensity administered, treatment regimen including how many
times the treatment was repeated, how long each treatment lasted, and
the start and end time of the entire treatment; can include multiple
radiation regimens”

URI:
`nmdc:radiation_regm <https://microbiomedata/meta/radiation_regm>`__

.. _domain-and-range-607:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-644:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-614:

Children
--------

.. _used-by-607:

Used by
-------

.. _other-properties-473:

Other properties
----------------

============= ===================
**Aliases:**   radiation regimen
**Mappings:**  mixs:radiation_regm
============= ===================

Slot: rainfall_regm
===================

“Information about treatment involving an exposure to a given amount of
rainfall, treatment regimen including how many times the treatment was
repeated, how long each treatment lasted, and the start and end time of
the entire treatment; can include multiple regimens”

URI: `nmdc:rainfall_regm <https://microbiomedata/meta/rainfall_regm>`__

.. _domain-and-range-608:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-645:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-615:

Children
--------

.. _used-by-608:

Used by
-------

.. _other-properties-474:

Other properties
----------------

============= ==================
**Aliases:**   rainfall regimen
**Mappings:**  mixs:rainfall_regm
============= ==================

.. _slot-direction-1:

Slot: direction
===============

One of l->r, r->l, bidirectional, neutral

URI:
`nmdc:reaction_direction <https://microbiomedata/meta/reaction_direction>`__

.. _domain-and-range-609:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-646:

Parents
-------

-  is_a: `direction <direction.md>`__

.. _children-616:

Children
--------

.. _used-by-609:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-is-balanced-1:

Slot: is balanced
=================

URI:
`nmdc:reaction_is_balanced <https://microbiomedata/meta/reaction_is_balanced>`__

.. _domain-and-range-610:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-647:

Parents
-------

-  is_a: `is balanced <is_balanced.md>`__

.. _children-617:

Children
--------

.. _used-by-610:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-is-diastereoselective-1:

Slot: is diastereoselective
===========================

URI:
`nmdc:reaction_is_diastereoselective <https://microbiomedata/meta/reaction_is_diastereoselective>`__

.. _domain-and-range-611:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-648:

Parents
-------

-  is_a: `is diastereoselective <is_diastereoselective.md>`__

.. _children-618:

Children
--------

.. _used-by-611:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-is-fully-characterized-1:

Slot: is fully characterized
============================

False if includes R-groups

URI:
`nmdc:reaction_is_fully_characterized <https://microbiomedata/meta/reaction_is_fully_characterized>`__

.. _domain-and-range-612:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-649:

Parents
-------

-  is_a: `is fully characterized <is_fully_characterized.md>`__

.. _children-619:

Children
--------

.. _used-by-612:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-is-stereo-1:

Slot: is stereo
===============

URI:
`nmdc:reaction_is_stereo <https://microbiomedata/meta/reaction_is_stereo>`__

.. _domain-and-range-613:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-650:

Parents
-------

-  is_a: `is stereo <is_stereo.md>`__

.. _children-620:

Children
--------

.. _used-by-613:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-is-transport-1:

Slot: is transport
==================

URI:
`nmdc:reaction_is_transport <https://microbiomedata/meta/reaction_is_transport>`__

.. _domain-and-range-614:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `Boolean <types/Boolean.md>`__

.. _parents-651:

Parents
-------

-  is_a: `is transport <is_transport.md>`__

.. _children-621:

Children
--------

.. _used-by-614:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-left-participants-1:

Slot: left participants
=======================

URI:
`nmdc:reaction_left_participants <https://microbiomedata/meta/reaction_left_participants>`__

.. _domain-and-range-615:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..\*
`ReactionParticipant <ReactionParticipant.md>`__

.. _parents-652:

Parents
-------

-  is_a: `has participants <has_participants.md>`__

.. _children-622:

Children
--------

.. _used-by-615:

Used by
-------

-  `Reaction <Reaction.md>`__

.. _slot-chemical-1:

Slot: chemical
==============

URI:
`nmdc:reaction_participant_chemical <https://microbiomedata/meta/reaction_participant_chemical>`__

.. _domain-and-range-616:

Domain and Range
----------------

`ReactionParticipant <ReactionParticipant.md>`__ → 0..1
`ChemicalEntity <ChemicalEntity.md>`__

.. _parents-653:

Parents
-------

-  is_a: `chemical <chemical.md>`__

.. _children-623:

Children
--------

.. _used-by-616:

Used by
-------

-  `ReactionParticipant <ReactionParticipant.md>`__

Slot: stoichiometry
===================

URI:
`nmdc:reaction_participant_stoichiometry <https://microbiomedata/meta/reaction_participant_stoichiometry>`__

.. _domain-and-range-617:

Domain and Range
----------------

`ReactionParticipant <ReactionParticipant.md>`__ → 0..1
`Integer <types/Integer.md>`__

.. _parents-654:

Parents
-------

-  is_a: `stoichiometry <stoichiometry.md>`__

.. _children-624:

Children
--------

.. _used-by-617:

Used by
-------

-  `ReactionParticipant <ReactionParticipant.md>`__

Slot: right participants
========================

URI:
`nmdc:reaction_right_participants <https://microbiomedata/meta/reaction_right_participants>`__

.. _domain-and-range-618:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..\*
`ReactionParticipant <ReactionParticipant.md>`__

.. _parents-655:

Parents
-------

-  is_a: `has participants <has_participants.md>`__

.. _children-625:

Children
--------

.. _used-by-618:

Used by
-------

-  `Reaction <Reaction.md>`__

Slot: smarts string
===================

URI:
`nmdc:reaction_smarts_string <https://microbiomedata/meta/reaction_smarts_string>`__

.. _domain-and-range-619:

Domain and Range
----------------

`Reaction <Reaction.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-656:

Parents
-------

-  is_a: `smarts string <smarts_string.md>`__

.. _children-626:

Children
--------

.. _used-by-619:

Used by
-------

-  `Reaction <Reaction.md>`__

Slot: reactor_type
==================

“Anaerobic digesters can be designed and engineered to operate using a
number of different process configurations, as batch or continuous,
mesophilic, high solid or low solid, and single stage or multistage”

URI: `nmdc:reactor_type <https://microbiomedata/meta/reactor_type>`__

.. _domain-and-range-620:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-657:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-627:

Children
--------

.. _used-by-620:

Used by
-------

.. _other-properties-475:

Other properties
----------------

============= =================
**Aliases:**   reactor type
**Mappings:**  mixs:reactor_type
============= =================

.. _slot-has-input-3:

Slot: has input
===============

An input to a process.

URI:
https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_has_input

.. _domain-and-range-621:

Domain and Range
----------------

`ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__ → 1..\*
`NamedThing <NamedThing.md>`__

.. _parents-658:

Parents
-------

-  is_a: `workflow execution activity➞has
   input <workflow_execution_activity_has_input.md>`__

.. _children-628:

Children
--------

.. _used-by-621:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

.. _other-properties-476:

Other properties
----------------

============ =====
**Aliases:**  input
============ =====

.. _slot-has-output-1:

Slot: has output
================

An output biosample to a processing step

URI:
https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_has_output

.. _domain-and-range-622:

Domain and Range
----------------

`ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__ → 1..\*
`NamedThing <NamedThing.md>`__

.. _parents-659:

Parents
-------

-  is_a: `workflow execution activity➞has
   output <workflow_execution_activity_has_output.md>`__

.. _children-629:

Children
--------

.. _used-by-622:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

.. _other-properties-477:

Other properties
----------------

============ ======
**Aliases:**  output
============ ======

.. _slot-input_read_bases-1:

Slot: input_read_bases
======================

TODO

URI:
https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_input_read_bases

.. _domain-and-range-623:

Domain and Range
----------------

`ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__ → 0..1
`Float <types/Float.md>`__

.. _parents-660:

Parents
-------

-  is_a: `input_read_bases <input_read_bases.md>`__

.. _children-630:

Children
--------

.. _used-by-623:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

.. _slot-output_read_bases-1:

Slot: output_read_bases
=======================

TODO

URI:
https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_output_read_bases

.. _domain-and-range-624:

Domain and Range
----------------

`ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__ → 0..1
`Float <types/Float.md>`__

.. _parents-661:

Parents
-------

-  is_a: `output_read_bases <output_read_bases.md>`__

.. _children-631:

Children
--------

.. _used-by-624:

Used by
-------

-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

Slot: read QC analysis activity set
===================================

This property links a database object to the set of read QC analysis
activities.

URI:
`nmdc:read_QC_analysis_activity_set <https://microbiomedata/meta/read_QC_analysis_activity_set>`__

.. _domain-and-range-625:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__

.. _parents-662:

Parents
-------

.. _children-632:

Children
--------

.. _used-by-625:

Used by
-------

-  `Database <Database.md>`__

Slot: read QC analysis statistic
================================

URI:
https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_statistic

.. _domain-and-range-626:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-663:

Parents
-------

.. _children-633:

Children
--------

-  `input base count <input_base_count.md>`__
-  `input read count <input_read_count.md>`__
-  `output base count <output_base_count.md>`__
-  `output read count <output_read_count.md>`__

.. _used-by-626:

Used by
-------

Slot: read based analysis activity set
======================================

This property links a database object to the set of read based analysis
activities.

URI:
`nmdc:read_based_analysis_activity_set <https://microbiomedata/meta/read_based_analysis_activity_set>`__

.. _domain-and-range-627:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\*
`ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__

.. _parents-664:

Parents
-------

.. _children-634:

Children
--------

.. _used-by-627:

Used by
-------

-  `Database <Database.md>`__

Slot: reassembly_bin
====================

Has an assembly been performed on a genome bin extracted from a
metagenomic assembly?

URI:
`nmdc:reassembly_bin <https://microbiomedata/meta/reassembly_bin>`__

.. _domain-and-range-628:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-665:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-635:

Children
--------

.. _used-by-628:

Used by
-------

.. _other-properties-478:

Other properties
----------------

=============== =======================
**Aliases:**     reassembly post binning
**Mappings:**    mixs:reassembly_bin
**In Subsets:**  sequencing
=============== =======================

Slot: redox_potential
=====================

“Redox potential, measured relative to a hydrogen cell, indicating
oxidation or reduction potential”

URI:
`nmdc:redox_potential <https://microbiomedata/meta/redox_potential>`__

.. _domain-and-range-629:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-666:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-636:

Children
--------

.. _used-by-629:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-479:

Other properties
----------------

============= ====================
**Aliases:**   redox potential
**Mappings:**  mixs:redox_potential
============= ====================

Slot: ref_biomaterial
=====================

“Primary publication if isolated before genome publication; otherwise,
primary genome report”

URI:
`nmdc:ref_biomaterial <https://microbiomedata/meta/ref_biomaterial>`__

.. _domain-and-range-630:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-667:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-637:

Children
--------

.. _used-by-630:

Used by
-------

.. _other-properties-480:

Other properties
----------------

=============== ============================
**Aliases:**     reference for biomaterial
**Mappings:**    mixs:ref_biomaterial
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: ref_db
============

“List of database(s) used for ORF annotation, along with version number
and reference to website or publication”

URI: `nmdc:ref_db <https://microbiomedata/meta/ref_db>`__

.. _domain-and-range-631:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-668:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-638:

Children
--------

.. _used-by-631:

Used by
-------

.. _other-properties-481:

Other properties
----------------

=============== =====================
**Aliases:**     reference database(s)
**Mappings:**    mixs:ref_db
**In Subsets:**  sequencing
=============== =====================

Slot: rel_air_humidity
======================

“Partial vapor and air pressure, density of the vapor and air, or by the
actual mass of the vapor and air”

URI:
`nmdc:rel_air_humidity <https://microbiomedata/meta/rel_air_humidity>`__

.. _domain-and-range-632:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-669:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-639:

Children
--------

.. _used-by-632:

Used by
-------

.. _other-properties-482:

Other properties
----------------

============= =====================
**Aliases:**   relative air humidity
**Mappings:**  mixs:rel_air_humidity
============= =====================

Slot: rel_humidity_out
======================

The recorded outside relative humidity value at the time of sampling

URI:
`nmdc:rel_humidity_out <https://microbiomedata/meta/rel_humidity_out>`__

.. _domain-and-range-633:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-670:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-640:

Children
--------

.. _used-by-633:

Used by
-------

.. _other-properties-483:

Other properties
----------------

============= =========================
**Aliases:**   outside relative humidity
**Mappings:**  mixs:rel_humidity_out
============= =========================

Slot: rel_samp_loc
==================

The sampling location within the train car

URI: `nmdc:rel_samp_loc <https://microbiomedata/meta/rel_samp_loc>`__

.. _domain-and-range-634:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-671:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-641:

Children
--------

.. _used-by-634:

Used by
-------

.. _other-properties-484:

Other properties
----------------

============= ==========================
**Aliases:**   relative sampling location
**Mappings:**  mixs:rel_samp_loc
============= ==========================

Slot: rel_to_oxygen
===================

“Is this organism an aerobe, anaerobe? Please note that aerobic and
anaerobic are valid descriptors for microbial environments”

URI: `nmdc:rel_to_oxygen <https://microbiomedata/meta/rel_to_oxygen>`__

.. _domain-and-range-635:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-672:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-642:

Children
--------

.. _used-by-635:

Used by
-------

.. _other-properties-485:

Other properties
----------------

=============== ============================
**Aliases:**     relationship to oxygen
**Mappings:**    mixs:rel_to_oxygen
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: relevant protocols
========================

URI:
`nmdc:relevant_protocols <https://microbiomedata/meta/relevant_protocols>`__

.. _domain-and-range-636:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-673:

Parents
-------

.. _children-643:

Children
--------

.. _used-by-636:

Used by
-------

-  `Study <Study.md>`__

Slot: reservoir
===============

Name of the reservoir (e.g. Carapebus)

URI: `nmdc:reservoir <https://microbiomedata/meta/reservoir>`__

.. _domain-and-range-637:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-674:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-644:

Children
--------

.. _used-by-637:

Used by
-------

.. _other-properties-486:

Other properties
----------------

============= ==============
**Aliases:**   reservoir name
**Mappings:**  mixs:reservoir
============= ==============

Slot: resins_pc
===============

“Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)”

URI: `nmdc:resins_pc <https://microbiomedata/meta/resins_pc>`__

.. _domain-and-range-638:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-675:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-645:

Children
--------

.. _used-by-638:

Used by
-------

.. _other-properties-487:

Other properties
----------------

============= ==============
**Aliases:**   resins wt%
**Mappings:**  mixs:resins_pc
============= ==============

Slot: resp_part_matter
======================

“Concentration of substances that remain suspended in the air, and
comprise mixtures of organic and inorganic substances (PM10 and PM2.5);
can report multiple PM’s by entering numeric values preceded by name of
PM”

URI:
`nmdc:resp_part_matter <https://microbiomedata/meta/resp_part_matter>`__

.. _domain-and-range-639:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-676:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-646:

Children
--------

.. _used-by-639:

Used by
-------

.. _other-properties-488:

Other properties
----------------

============= =============================
**Aliases:**   respirable particulate matter
**Mappings:**  mixs:resp_part_matter
============= =============================

.. _slot-right-participants-1:

Slot: right participants
========================

URI:
`nmdc:right_participants <https://microbiomedata/meta/right_participants>`__

.. _domain-and-range-640:

Domain and Range
----------------

None → 0..\* `ReactionParticipant <ReactionParticipant.md>`__

.. _parents-677:

Parents
-------

-  is_a: `has participants <has_participants.md>`__

.. _children-647:

Children
--------

.. _used-by-640:

Used by
-------

Slot: room_air_exch_rate
========================

The rate at which outside air replaces indoor air in a given space

URI:
`nmdc:room_air_exch_rate <https://microbiomedata/meta/room_air_exch_rate>`__

.. _domain-and-range-641:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-678:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-648:

Children
--------

.. _used-by-641:

Used by
-------

.. _other-properties-489:

Other properties
----------------

============= =======================
**Aliases:**   room air exchange rate
**Mappings:**  mixs:room_air_exch_rate
============= =======================

Slot: room_architec_element
===========================

“The unique details and component parts that, together, form the
architecture of a distinguisahable space within a built structure”

URI:
`nmdc:room_architec_element <https://microbiomedata/meta/room_architec_element>`__

.. _domain-and-range-642:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-679:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-649:

Children
--------

.. _used-by-642:

Used by
-------

.. _other-properties-490:

Other properties
----------------

============= ===========================
**Aliases:**   room architectural elements
**Mappings:**  mixs:room_architec_element
============= ===========================

Slot: room_condt
================

The condition of the room at the time of sampling

URI: `nmdc:room_condt <https://microbiomedata/meta/room_condt>`__

.. _domain-and-range-643:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-680:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-650:

Children
--------

.. _used-by-643:

Used by
-------

.. _other-properties-491:

Other properties
----------------

============= ===============
**Aliases:**   room condition
**Mappings:**  mixs:room_condt
============= ===============

Slot: room_connected
====================

List of rooms connected to the sampling room by a doorway

URI:
`nmdc:room_connected <https://microbiomedata/meta/room_connected>`__

.. _domain-and-range-644:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-681:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-651:

Children
--------

.. _used-by-644:

Used by
-------

.. _other-properties-492:

Other properties
----------------

============= ============================
**Aliases:**   rooms connected by a doorway
**Mappings:**  mixs:room_connected
============= ============================

Slot: room_count
================

The total count of rooms in the built structure including all room types

URI: `nmdc:room_count <https://microbiomedata/meta/room_count>`__

.. _domain-and-range-645:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-682:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-652:

Children
--------

.. _used-by-645:

Used by
-------

.. _other-properties-493:

Other properties
----------------

============= ===============
**Aliases:**   room count
**Mappings:**  mixs:room_count
============= ===============

Slot: room_dim
==============

“The length, width and height of sampling room”

URI: `nmdc:room_dim <https://microbiomedata/meta/room_dim>`__

.. _domain-and-range-646:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-683:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-653:

Children
--------

.. _used-by-646:

Used by
-------

.. _other-properties-494:

Other properties
----------------

============= ===============
**Aliases:**   room dimensions
**Mappings:**  mixs:room_dim
============= ===============

Slot: room_door_dist
====================

Distance between doors (meters) in the hallway between the sampling room
and adjacent rooms

URI:
`nmdc:room_door_dist <https://microbiomedata/meta/room_door_dist>`__

.. _domain-and-range-647:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-684:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-654:

Children
--------

.. _used-by-647:

Used by
-------

.. _other-properties-495:

Other properties
----------------

============= ===================
**Aliases:**   room door distance
**Mappings:**  mixs:room_door_dist
============= ===================

Slot: room_door_share
=====================

“List of room(s) (room number, room name) sharing a door with the
sampling room”

URI:
`nmdc:room_door_share <https://microbiomedata/meta/room_door_share>`__

.. _domain-and-range-648:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-685:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-655:

Children
--------

.. _used-by-648:

Used by
-------

.. _other-properties-496:

Other properties
----------------

============= ==========================================
**Aliases:**   rooms that share a door with sampling room
**Mappings:**  mixs:room_door_share
============= ==========================================

Slot: room_hallway
==================

“List of room(s) (room number, room name) located in the same hallway as
sampling room”

URI: `nmdc:room_hallway <https://microbiomedata/meta/room_hallway>`__

.. _domain-and-range-649:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-686:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-656:

Children
--------

.. _used-by-649:

Used by
-------

.. _other-properties-497:

Other properties
----------------

============= ==================================
**Aliases:**   rooms that are on the same hallway
**Mappings:**  mixs:room_hallway
============= ==================================

Slot: room_loc
==============

The position of the room within the building

URI: `nmdc:room_loc <https://microbiomedata/meta/room_loc>`__

.. _domain-and-range-650:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-687:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-657:

Children
--------

.. _used-by-650:

Used by
-------

.. _other-properties-498:

Other properties
----------------

============= =========================
**Aliases:**   room location in building
**Mappings:**  mixs:room_loc
============= =========================

Slot: room_moist_damage_hist
============================

The history of moisture damage or mold in the past 12 months. Number of
events of moisture damage or mold observed

URI:
`nmdc:room_moist_damage_hist <https://microbiomedata/meta/room_moist_damage_hist>`__

.. _domain-and-range-651:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-688:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-658:

Children
--------

.. _used-by-651:

Used by
-------

.. _other-properties-499:

Other properties
----------------

============= ====================================
**Aliases:**   room moisture damage or mold history
**Mappings:**  mixs:room_moist_damage_hist
============= ====================================

Slot: room_net_area
===================

The net floor area of sampling room. Net area excludes wall thicknesses

URI: `nmdc:room_net_area <https://microbiomedata/meta/room_net_area>`__

.. _domain-and-range-652:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-689:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-659:

Children
--------

.. _used-by-652:

Used by
-------

.. _other-properties-500:

Other properties
----------------

============= ==================
**Aliases:**   room net area
**Mappings:**  mixs:room_net_area
============= ==================

Slot: room_occup
================

Count of room occupancy at time of sampling

URI: `nmdc:room_occup <https://microbiomedata/meta/room_occup>`__

.. _domain-and-range-653:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-690:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-660:

Children
--------

.. _used-by-653:

Used by
-------

.. _other-properties-501:

Other properties
----------------

============= ===============
**Aliases:**   room occupancy
**Mappings:**  mixs:room_occup
============= ===============

Slot: room_samp_pos
===================

The horizontal sampling position in the room relative to architectural
elements

URI: `nmdc:room_samp_pos <https://microbiomedata/meta/room_samp_pos>`__

.. _domain-and-range-654:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-691:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-661:

Children
--------

.. _used-by-654:

Used by
-------

.. _other-properties-502:

Other properties
----------------

============= ======================
**Aliases:**   room sampling position
**Mappings:**  mixs:room_samp_pos
============= ======================

Slot: room_type
===============

The main purpose or activity of the sampling room. A room is any
distinguishable space within a structure

URI: `nmdc:room_type <https://microbiomedata/meta/room_type>`__

.. _domain-and-range-655:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-692:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-662:

Children
--------

.. _used-by-655:

Used by
-------

.. _other-properties-503:

Other properties
----------------

============= ==============
**Aliases:**   room type
**Mappings:**  mixs:room_type
============= ==============

Slot: room_vol
==============

Volume of sampling room

URI: `nmdc:room_vol <https://microbiomedata/meta/room_vol>`__

.. _domain-and-range-656:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-693:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-663:

Children
--------

.. _used-by-656:

Used by
-------

.. _other-properties-504:

Other properties
----------------

============= =============
**Aliases:**   room volume
**Mappings:**  mixs:room_vol
============= =============

Slot: room_wall_share
=====================

“List of room(s) (room number, room name) sharing a wall with the
sampling room”

URI:
`nmdc:room_wall_share <https://microbiomedata/meta/room_wall_share>`__

.. _domain-and-range-657:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-694:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-664:

Children
--------

.. _used-by-657:

Used by
-------

.. _other-properties-505:

Other properties
----------------

============= ==========================================
**Aliases:**   rooms that share a wall with sampling room
**Mappings:**  mixs:room_wall_share
============= ==========================================

Slot: room_window_count
=======================

Number of windows in the room

URI:
`nmdc:room_window_count <https://microbiomedata/meta/room_window_count>`__

.. _domain-and-range-658:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-695:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-665:

Children
--------

.. _used-by-658:

Used by
-------

.. _other-properties-506:

Other properties
----------------

============= ======================
**Aliases:**   room window count
**Mappings:**  mixs:room_window_count
============= ======================

Slot: root_cond
===============

“Relevant rooting conditions such as field plot size, sowing density,
container dimensions, number of plants per container.”

URI: `nmdc:root_cond <https://microbiomedata/meta/root_cond>`__

.. _domain-and-range-659:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-696:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-666:

Children
--------

.. _used-by-659:

Used by
-------

.. _other-properties-507:

Other properties
----------------

============= ==================
**Aliases:**   rooting conditions
**Mappings:**  mixs:root_cond
============= ==================

Slot: root_med_carbon
=====================

Source of organic carbon in the culture rooting medium; e.g. sucrose.

URI:
`nmdc:root_med_carbon <https://microbiomedata/meta/root_med_carbon>`__

.. _domain-and-range-660:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-697:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-667:

Children
--------

.. _used-by-660:

Used by
-------

.. _other-properties-508:

Other properties
----------------

============= =====================
**Aliases:**   rooting medium carbon
**Mappings:**  mixs:root_med_carbon
============= =====================

Slot: root_med_macronutr
========================

“Measurement of the culture rooting medium macronutrients (N,P, K, Ca,
Mg, S); e.g. KH2PO4 (170B mg/L).”

URI:
`nmdc:root_med_macronutr <https://microbiomedata/meta/root_med_macronutr>`__

.. _domain-and-range-661:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-698:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-668:

Children
--------

.. _used-by-661:

Used by
-------

.. _other-properties-509:

Other properties
----------------

============= =============================
**Aliases:**   rooting medium macronutrients
**Mappings:**  mixs:root_med_macronutr
============= =============================

Slot: root_med_micronutr
========================

“Measurement of the culture rooting medium micronutrients (Fe, Mn, Zn,
B, Cu, Mo); e.g. H3BO3 (6.2B mg/L).”

URI:
`nmdc:root_med_micronutr <https://microbiomedata/meta/root_med_micronutr>`__

.. _domain-and-range-662:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-699:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-669:

Children
--------

.. _used-by-662:

Used by
-------

.. _other-properties-510:

Other properties
----------------

============= =============================
**Aliases:**   rooting medium micronutrients
**Mappings:**  mixs:root_med_micronutr
============= =============================

Slot: root_med_ph
=================

pH measurement of the culture rooting medium; e.g. 5.5.

URI: `nmdc:root_med_ph <https://microbiomedata/meta/root_med_ph>`__

.. _domain-and-range-663:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-700:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-670:

Children
--------

.. _used-by-663:

Used by
-------

.. _other-properties-511:

Other properties
----------------

============= =================
**Aliases:**   rooting medium pH
**Mappings:**  mixs:root_med_ph
============= =================

Slot: root_med_regl
===================

“Growth regulators in the culture rooting medium such as cytokinins,
auxins, gybberellins, abscisic acid; e.g. 0.5B mg/L NAA.”

URI: `nmdc:root_med_regl <https://microbiomedata/meta/root_med_regl>`__

.. _domain-and-range-664:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-701:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-671:

Children
--------

.. _used-by-664:

Used by
-------

.. _other-properties-512:

Other properties
----------------

============= =========================
**Aliases:**   rooting medium regulators
**Mappings:**  mixs:root_med_regl
============= =========================

Slot: root_med_solid
====================

Specification of the solidifying agent in the culture rooting medium;
e.g. agar.

URI:
`nmdc:root_med_solid <https://microbiomedata/meta/root_med_solid>`__

.. _domain-and-range-665:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-702:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-672:

Children
--------

.. _used-by-665:

Used by
-------

.. _other-properties-513:

Other properties
----------------

============= =========================
**Aliases:**   rooting medium solidifier
**Mappings:**  mixs:root_med_solid
============= =========================

Slot: root_med_suppl
====================

“Organic supplements of the culture rooting medium, such as vitamins,
amino acids, organic acids, antibiotics activated charcoal;
e.g. nicotinic acid (0.5B mg/L).”

URI:
`nmdc:root_med_suppl <https://microbiomedata/meta/root_med_suppl>`__

.. _domain-and-range-666:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-703:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-673:

Children
--------

.. _used-by-666:

Used by
-------

.. _other-properties-514:

Other properties
----------------

============= ==================================
**Aliases:**   rooting medium organic supplements
**Mappings:**  mixs:root_med_suppl
============= ==================================

Slot: salinity
==============

“Salinity is the total concentration of all dissolved salts in a water
sample. While salinity can be measured by a complete chemical analysis,
this method is difficult and time consuming. More often, it is instead
derived from the conductivity measurement. This is known as practical
salinity. These derivations compare the specific conductance of the
sample to a salinity standard such as seawater”

URI: `nmdc:salinity <https://microbiomedata/meta/salinity>`__

.. _domain-and-range-667:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-704:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-674:

Children
--------

.. _used-by-667:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-515:

Other properties
----------------

============= =============
**Aliases:**   salinity
**Mappings:**  mixs:salinity
============= =============

Slot: salinity_category
=======================

Categorcial description of the sample’s salinity. Examples: halophile,
halotolerant, hypersaline, huryhaline

URI:
`nmdc:salinity_category <https://microbiomedata/meta/salinity_category>`__

.. _domain-and-range-668:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-705:

Parents
-------

.. _children-675:

Children
--------

.. _used-by-668:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-516:

Other properties
----------------

============= ========================================================
**See also:**  https://github.com/microbiomedata/nmdc-metadata/pull/297
============= ========================================================

Slot: salinity_meth
===================

Reference or method used in determining salinity

URI: `nmdc:salinity_meth <https://microbiomedata/meta/salinity_meth>`__

.. _domain-and-range-669:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-706:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-676:

Children
--------

.. _used-by-669:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-517:

Other properties
----------------

============= ==========================================
**Aliases:**   extreme_unusual_properties/salinity method
**Mappings:**  mixs:salinity_meth
============= ==========================================

Slot: salt_regm
===============

“Information about treatment involving use of salts as supplement to
liquid and soil growth media; should include the name of salt, amount
administered, treatment regimen including how many times the treatment
was repeated, how long each treatment lasted, and the start and end time
of the entire treatment; can include multiple salt regimens”

URI: `nmdc:salt_regm <https://microbiomedata/meta/salt_regm>`__

.. _domain-and-range-670:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-707:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-677:

Children
--------

.. _used-by-670:

Used by
-------

.. _other-properties-518:

Other properties
----------------

============= ==============
**Aliases:**   salt regimen
**Mappings:**  mixs:salt_regm
============= ==============

Slot: samp_capt_status
======================

Reason for the sample

URI:
`nmdc:samp_capt_status <https://microbiomedata/meta/samp_capt_status>`__

.. _domain-and-range-671:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-708:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-678:

Children
--------

.. _used-by-671:

Used by
-------

.. _other-properties-519:

Other properties
----------------

============= =====================
**Aliases:**   sample capture status
**Mappings:**  mixs:samp_capt_status
============= =====================

Slot: samp_collect_device
=========================

The method or device employed for collecting the sample

URI:
`nmdc:samp_collect_device <https://microbiomedata/meta/samp_collect_device>`__

.. _domain-and-range-672:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-709:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-679:

Children
--------

.. _used-by-672:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-520:

Other properties
----------------

=============== ==================================
**Aliases:**     sample collection device or method
**Mappings:**    mixs:samp_collect_device
**In Subsets:**  nucleic acid sequence source
=============== ==================================

Slot: samp_collection_point
===========================

“Sampling point on the asset were sample was collected (e.g. Wellhead,
storage tank, separator, etc). If”“other”” is specified, please propose
entry in ““additional info”” field”

URI:
`nmdc:samp_collection_point <https://microbiomedata/meta/samp_collection_point>`__

.. _domain-and-range-673:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-710:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-680:

Children
--------

.. _used-by-673:

Used by
-------

.. _other-properties-521:

Other properties
----------------

============= ==========================
**Aliases:**   sample collection point
**Mappings:**  mixs:samp_collection_point
============= ==========================

Slot: samp_dis_stage
====================

“Stage of the disease at the time of sample collection,
e.g. inoculation, penetration, infection, growth and reproduction,
dissemination of pathogen.”

URI:
`nmdc:samp_dis_stage <https://microbiomedata/meta/samp_dis_stage>`__

.. _domain-and-range-674:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-711:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-681:

Children
--------

.. _used-by-674:

Used by
-------

.. _other-properties-522:

Other properties
----------------

============= ====================
**Aliases:**   sample disease stage
**Mappings:**  mixs:samp_dis_stage
============= ====================

Slot: samp_floor
================

“The floor of the building, where the sampling room is located”

URI: `nmdc:samp_floor <https://microbiomedata/meta/samp_floor>`__

.. _domain-and-range-675:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-712:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-682:

Children
--------

.. _used-by-675:

Used by
-------

.. _other-properties-523:

Other properties
----------------

============= ===============
**Aliases:**   sampling floor
**Mappings:**  mixs:samp_floor
============= ===============

Slot: samp_loc_corr_rate
========================

“Metal corrosion rate is the speed of metal deterioration due to
environmental conditions. As environmental conditions change corrosion
rates change accordingly. Therefore, long term corrosion rates are
generally more informative than short term rates and for that reason
they are preferred during reporting. In the case of suspected MIC,
corrosion rate measurements at the time of sampling might provide
insights into the involvement of certain microbial community members in
MIC as well as potential microbial interplays”

URI:
`nmdc:samp_loc_corr_rate <https://microbiomedata/meta/samp_loc_corr_rate>`__

.. _domain-and-range-676:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-713:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-683:

Children
--------

.. _used-by-676:

Used by
-------

.. _other-properties-524:

Other properties
----------------

============= =================================
**Aliases:**   corrosion rate at sample location
**Mappings:**  mixs:samp_loc_corr_rate
============= =================================

Slot: samp_mat_process
======================

“Any processing applied to the sample during or after retrieving the
sample from environment. This field accepts OBI, for a browser of OBI (v
2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI”

URI:
`nmdc:samp_mat_process <https://microbiomedata/meta/samp_mat_process>`__

.. _domain-and-range-677:

Domain and Range
----------------

None → 0..1 `ControlledTermValue <ControlledTermValue.md>`__

.. _parents-714:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-684:

Children
--------

.. _used-by-677:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-525:

Other properties
----------------

=============== ============================
**Aliases:**     sample material processing
**Mappings:**    mixs:samp_mat_process
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: samp_md
=============

“In non deviated well, measured depth is equal to the true vertical
depth, TVD (TVD=TVDSS plus the reference or datum it refers to). In
deviated wells, the MD is the length of trajectory of the borehole
measured from the same reference or datum. Common datums used are ground
level (GL), drilling rig floor (DF), rotary table (RT), kelly bushing
(KB) and mean sea level (MSL). If”“other”” is specified, please propose
entry in ““additional info”” field ”

URI: `nmdc:samp_md <https://microbiomedata/meta/samp_md>`__

.. _domain-and-range-678:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-715:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-685:

Children
--------

.. _used-by-678:

Used by
-------

.. _other-properties-526:

Other properties
----------------

============= =====================
**Aliases:**   sample measured depth
**Mappings:**  mixs:samp_md
============= =====================

Slot: samp_preserv
==================

“Preservative added to the sample (e.g. Rnalater, alcohol, formaldehyde,
etc.). Where appropriate include volume added (e.g. Rnalater; 2 ml)”

URI: `nmdc:samp_preserv <https://microbiomedata/meta/samp_preserv>`__

.. _domain-and-range-679:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-716:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-686:

Children
--------

.. _used-by-679:

Used by
-------

.. _other-properties-527:

Other properties
----------------

============= ============================
**Aliases:**   preservative added to sample
**Mappings:**  mixs:samp_preserv
============= ============================

Slot: samp_room_id
==================

Sampling room number. This ID should be consistent with the designations
on the building floor plans

URI: `nmdc:samp_room_id <https://microbiomedata/meta/samp_room_id>`__

.. _domain-and-range-680:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-717:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-687:

Children
--------

.. _used-by-680:

Used by
-------

.. _other-properties-528:

Other properties
----------------

============= ========================
**Aliases:**   sampling room ID or name
**Mappings:**  mixs:samp_room_id
============= ========================

Slot: samp_salinity
===================

“Salinity is the total concentration of all dissolved salts in a liquid
or solid (in the form of an extract obtained by centrifugation) sample.
While salinity can be measured by a complete chemical analysis, this
method is difficult and time consuming. More often, it is instead
derived from the conductivity measurement. This is known as practical
salinity. These derivations compare the specific conductance of the
sample to a salinity standard such as seawater”

URI: `nmdc:samp_salinity <https://microbiomedata/meta/samp_salinity>`__

.. _domain-and-range-681:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-718:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-688:

Children
--------

.. _used-by-681:

Used by
-------

.. _other-properties-529:

Other properties
----------------

============= ==================
**Aliases:**   sample salinity
**Mappings:**  mixs:samp_salinity
============= ==================

Slot: samp_size
===============

“Amount or size of sample (volume, mass or area) that was collected”

URI: `nmdc:samp_size <https://microbiomedata/meta/samp_size>`__

.. _domain-and-range-682:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-719:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-689:

Children
--------

.. _used-by-682:

Used by
-------

.. _other-properties-530:

Other properties
----------------

=============== ==================================
**Aliases:**     amount or size of sample collected
**Mappings:**    mixs:samp_size
**In Subsets:**  nucleic acid sequence source
=============== ==================================

Slot: samp_sort_meth
====================

“Method by which samples are sorted; open face filter collecting total
suspended particles, prefilter to remove particles larger than X
micrometers in diameter, where common values of X would be 10 and 2.5
full size sorting in a cascade impactor.”

URI:
`nmdc:samp_sort_meth <https://microbiomedata/meta/samp_sort_meth>`__

.. _domain-and-range-683:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-720:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-690:

Children
--------

.. _used-by-683:

Used by
-------

.. _other-properties-531:

Other properties
----------------

============= ==========================
**Aliases:**   sample size sorting method
**Mappings:**  mixs:samp_sort_meth
============= ==========================

Slot: samp_store_dur
====================

Duration for which the sample was stored

URI:
`nmdc:samp_store_dur <https://microbiomedata/meta/samp_store_dur>`__

.. _domain-and-range-684:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-721:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-691:

Children
--------

.. _used-by-684:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-532:

Other properties
----------------

============= =======================
**Aliases:**   sample storage duration
**Mappings:**  mixs:samp_store_dur
============= =======================

Slot: samp_store_loc
====================

“Location at which sample was stored, usually name of a specific
freezer/room”

URI:
`nmdc:samp_store_loc <https://microbiomedata/meta/samp_store_loc>`__

.. _domain-and-range-685:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-722:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-692:

Children
--------

.. _used-by-685:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-533:

Other properties
----------------

============= =======================
**Aliases:**   sample storage location
**Mappings:**  mixs:samp_store_loc
============= =======================

Slot: samp_store_temp
=====================

“Temperature at which sample was stored, e.g. -80 degree Celsius”

URI:
`nmdc:samp_store_temp <https://microbiomedata/meta/samp_store_temp>`__

.. _domain-and-range-686:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-723:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-693:

Children
--------

.. _used-by-686:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-534:

Other properties
----------------

============= ==========================
**Aliases:**   sample storage temperature
**Mappings:**  mixs:samp_store_temp
============= ==========================

Slot: samp_subtype
==================

“Name of sample sub-type. For example if”“sample type”” is ““Produced
Water”” then subtype could be ““Oil Phase”” or ““Water Phase”“.
If”“other”” is specified, please propose entry in ““additional info””
field”

URI: `nmdc:samp_subtype <https://microbiomedata/meta/samp_subtype>`__

.. _domain-and-range-687:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-724:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-694:

Children
--------

.. _used-by-687:

Used by
-------

.. _other-properties-535:

Other properties
----------------

============= =================
**Aliases:**   sample subtype
**Mappings:**  mixs:samp_subtype
============= =================

Slot: samp_time_out
===================

The recent and long term history of outside sampling

URI: `nmdc:samp_time_out <https://microbiomedata/meta/samp_time_out>`__

.. _domain-and-range-688:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-725:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-695:

Children
--------

.. _used-by-688:

Used by
-------

.. _other-properties-536:

Other properties
----------------

============= =====================
**Aliases:**   sampling time outside
**Mappings:**  mixs:samp_time_out
============= =====================

Slot: samp_transport_cond
=========================

Sample transport duration (in days or hrs) and temperature the sample
was exposed to (e.g. 5.5 days; 20 B0C)

URI:
`nmdc:samp_transport_cond <https://microbiomedata/meta/samp_transport_cond>`__

.. _domain-and-range-689:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-726:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-696:

Children
--------

.. _used-by-689:

Used by
-------

.. _other-properties-537:

Other properties
----------------

============= ===========================
**Aliases:**   sample transport conditions
**Mappings:**  mixs:samp_transport_cond
============= ===========================

Slot: samp_tvdss
================

Depth of the sample i.e. The vertical distance between the sea level and
the sampled position in the subsurface. Depth can be reported as an
interval for subsurface samples e.g. 1325.75-1362.25 m

URI: `nmdc:samp_tvdss <https://microbiomedata/meta/samp_tvdss>`__

.. _domain-and-range-690:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-727:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-697:

Children
--------

.. _used-by-690:

Used by
-------

.. _other-properties-538:

Other properties
----------------

============= =================================
**Aliases:**   sample true vertical depth subsea
**Mappings:**  mixs:samp_tvdss
============= =================================

Slot: samp_type
===============

“Type of material (i.e. sample) collected. Includes types like core,
rock trimmings, drill cuttings, piping section, coupon, pigging debris,
solid deposit, produced fluid, produced water, injected water, swabs,
etc. If”“other”” is specified, please propose entry in ““additional
info”” field”

URI: `nmdc:samp_type <https://microbiomedata/meta/samp_type>`__

.. _domain-and-range-691:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-728:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-698:

Children
--------

.. _used-by-691:

Used by
-------

.. _other-properties-539:

Other properties
----------------

============= ==============
**Aliases:**   sample type
**Mappings:**  mixs:samp_type
============= ==============

Slot: samp_vol_we_dna_ext
=========================

“Volume (ml), weight (g) of processed sample, or surface area swabbed
from sample for DNA extraction”

URI:
`nmdc:samp_vol_we_dna_ext <https://microbiomedata/meta/samp_vol_we_dna_ext>`__

.. _domain-and-range-692:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-729:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-699:

Children
--------

.. _used-by-692:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-540:

Other properties
----------------

============= ==========================================
**Aliases:**   sample volume or weight for DNA extraction
**Mappings:**  mixs:samp_vol_we_dna_ext
============= ==========================================

Slot: samp_weather
==================

The weather on the sampling day

URI: `nmdc:samp_weather <https://microbiomedata/meta/samp_weather>`__

.. _domain-and-range-693:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-730:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-700:

Children
--------

.. _used-by-693:

Used by
-------

.. _other-properties-541:

Other properties
----------------

============= ====================
**Aliases:**   sampling day weather
**Mappings:**  mixs:samp_weather
============= ====================

Slot: samp_well_name
====================

Name of the well (e.g. BXA1123) where sample was taken

URI:
`nmdc:samp_well_name <https://microbiomedata/meta/samp_well_name>`__

.. _domain-and-range-694:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-731:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-701:

Children
--------

.. _used-by-694:

Used by
-------

.. _other-properties-542:

Other properties
----------------

============= ===================
**Aliases:**   sample well name
**Mappings:**  mixs:samp_well_name
============= ===================

Slot: sample_collection_day
===========================

URI:
`nmdc:sample_collection_day <https://microbiomedata/meta/sample_collection_day>`__

.. _domain-and-range-695:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-732:

Parents
-------

.. _children-702:

Children
--------

.. _used-by-695:

Used by
-------

Slot: sample_collection_hour
============================

URI:
`nmdc:sample_collection_hour <https://microbiomedata/meta/sample_collection_hour>`__

.. _domain-and-range-696:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-733:

Parents
-------

.. _children-703:

Children
--------

.. _used-by-696:

Used by
-------

Slot: sample_collection_minute
==============================

URI:
`nmdc:sample_collection_minute <https://microbiomedata/meta/sample_collection_minute>`__

.. _domain-and-range-697:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-734:

Parents
-------

.. _children-704:

Children
--------

.. _used-by-697:

Used by
-------

Slot: sample_collection_month
=============================

URI:
`nmdc:sample_collection_month <https://microbiomedata/meta/sample_collection_month>`__

.. _domain-and-range-698:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-735:

Parents
-------

.. _children-705:

Children
--------

.. _used-by-698:

Used by
-------

Slot: sample_collection_site
============================

URI:
`nmdc:sample_collection_site <https://microbiomedata/meta/sample_collection_site>`__

.. _domain-and-range-699:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-736:

Parents
-------

.. _children-706:

Children
--------

.. _used-by-699:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: sample_collection_year
============================

URI:
`nmdc:sample_collection_year <https://microbiomedata/meta/sample_collection_year>`__

.. _domain-and-range-700:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-737:

Parents
-------

.. _children-707:

Children
--------

.. _used-by-700:

Used by
-------

Slot: sample identifiers
========================

URI:
https://microbiomedata/schema/workflow_execution_activity/sample_identifiers

.. _domain-and-range-701:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-738:

Parents
-------

-  is_a: `external database
   identifiers <external_database_identifiers.md>`__

.. _children-708:

Children
--------

-  `GOLD sample identifiers <GOLD_sample_identifiers.md>`__
-  `INSDC biosample identifiers <INSDC_biosample_identifiers.md>`__
-  `INSDC secondary sample
   identifiers <INSDC_secondary_sample_identifiers.md>`__

.. _used-by-701:

Used by
-------

Slot: saturates_pc
==================

“Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)”

URI: `nmdc:saturates_pc <https://microbiomedata/meta/saturates_pc>`__

.. _domain-and-range-702:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-739:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-709:

Children
--------

.. _used-by-702:

Used by
-------

.. _other-properties-543:

Other properties
----------------

============= =================
**Aliases:**   saturates wt%
**Mappings:**  mixs:saturates_pc
============= =================

Slot: scaf_L50
==============

Given a set of scaffolds, the L50 is defined as the sequence length of
the shortest scaffold at 50% of the total genome length.

URI: https://microbiomedata/schema/workflow_execution_activity/scaf_L50

.. _domain-and-range-703:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-740:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-710:

Children
--------

.. _used-by-703:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_L90
==============

The L90 statistic is less than or equal to the L50 statistic; it is the
length for which the collection of all scaffolds of that length or
longer contains at least 90% of the sum of the lengths of all scaffolds.

URI: https://microbiomedata/schema/workflow_execution_activity/scaf_L90

.. _domain-and-range-704:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-741:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-711:

Children
--------

.. _used-by-704:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_N50
==============

Given a set of scaffolds, each with its own length, the N50 count is
defined as the smallest number of scaffolds whose length sum makes up
half of genome size.

URI: https://microbiomedata/schema/workflow_execution_activity/scaf_N50

.. _domain-and-range-705:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-742:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-712:

Children
--------

.. _used-by-705:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_N90
==============

Given a set of scaffolds, each with its own length, the N90 count is
defined as the smallest number of scaffolds whose length sum makes up
90% of genome size.

URI: https://microbiomedata/schema/workflow_execution_activity/scaf_N90

.. _domain-and-range-706:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-743:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-713:

Children
--------

.. _used-by-706:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_bp
=============

Total size in bp of all scaffolds.

URI: https://microbiomedata/schema/workflow_execution_activity/scaf_bp

.. _domain-and-range-707:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-744:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-714:

Children
--------

.. _used-by-707:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_l_gt50K
==================

Total size in bp of all scaffolds greater than 50 KB.

URI:
https://microbiomedata/schema/workflow_execution_activity/scaf_l_gt50K

.. _domain-and-range-708:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-745:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-715:

Children
--------

.. _used-by-708:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_logsum
=================

The sum of the (length*log(length)) of all scaffolds, times some
constant. Increase the contiguity, the score will increase

URI:
https://microbiomedata/schema/workflow_execution_activity/scaf_logsum

.. _domain-and-range-709:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-746:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-716:

Children
--------

.. _used-by-709:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_max
==============

Maximum scaffold length.

URI: https://microbiomedata/schema/workflow_execution_activity/scaf_max

.. _domain-and-range-710:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-747:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-717:

Children
--------

.. _used-by-710:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_n_gt50K
==================

Total sequence count of scaffolds greater than 50 KB.

URI:
https://microbiomedata/schema/workflow_execution_activity/scaf_n_gt50K

.. _domain-and-range-711:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-748:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-718:

Children
--------

.. _used-by-711:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_pct_gt50K
====================

Total sequence size percentage of scaffolds greater than 50 KB.

URI:
https://microbiomedata/schema/workflow_execution_activity/scaf_pct_gt50K

.. _domain-and-range-712:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-749:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-719:

Children
--------

.. _used-by-712:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaf_powsum
=================

Powersum of all scaffolds is the same as logsum except that it uses the
sum of (length*(length^P)) for some power P (default P=0.25).

URI:
https://microbiomedata/schema/workflow_execution_activity/scaf_powsum

.. _domain-and-range-713:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-750:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-720:

Children
--------

.. _used-by-713:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: scaffolds
===============

Total sequence count of all scaffolds.

URI: https://microbiomedata/schema/workflow_execution_activity/scaffolds

.. _domain-and-range-714:

Domain and Range
----------------

None → 0..1 `Float <types/Float.md>`__

.. _parents-751:

Parents
-------

-  is_a: `metagenome assembly
   parameter <metagenome_assembly_parameter.md>`__

.. _children-721:

Children
--------

.. _used-by-714:

Used by
-------

-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__

Slot: season
============

The season when sampling occurred

URI: `nmdc:season <https://microbiomedata/meta/season>`__

.. _domain-and-range-715:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-752:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-722:

Children
--------

.. _used-by-715:

Used by
-------

.. _other-properties-544:

Other properties
----------------

============= ===========
**Aliases:**   season
**Mappings:**  mixs:season
============= ===========

Slot: season_environment
========================

“Treatment involving an exposure to a particular season (e.g. Winter,
summer, rabi, rainy etc.), treatment regimen including how many times
the treatment was repeated, how long each treatment lasted, and the
start and end time of the entire treatment”

URI:
`nmdc:season_environment <https://microbiomedata/meta/season_environment>`__

.. _domain-and-range-716:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-753:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-723:

Children
--------

.. _used-by-716:

Used by
-------

.. _other-properties-545:

Other properties
----------------

============= =======================
**Aliases:**   seasonal environment
**Mappings:**  mixs:season_environment
============= =======================

Slot: season_precpt
===================

“The average of all seasonal precipitation values known, or an estimated
equivalent value derived by such methods as regional indexes or
Isohyetal maps.”

URI: `nmdc:season_precpt <https://microbiomedata/meta/season_precpt>`__

.. _domain-and-range-717:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-754:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-724:

Children
--------

.. _used-by-717:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-546:

Other properties
----------------

============= ===========================
**Aliases:**   mean seasonal precipitation
**Mappings:**  mixs:season_precpt
============= ===========================

Slot: season_temp
=================

Mean seasonal temperature

URI: `nmdc:season_temp <https://microbiomedata/meta/season_temp>`__

.. _domain-and-range-718:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-755:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-725:

Children
--------

.. _used-by-718:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-547:

Other properties
----------------

============= =========================
**Aliases:**   mean seasonal temperature
**Mappings:**  mixs:season_temp
============= =========================

Slot: season_use
================

The seasons the space is occupied

URI: `nmdc:season_use <https://microbiomedata/meta/season_use>`__

.. _domain-and-range-719:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-756:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-726:

Children
--------

.. _used-by-719:

Used by
-------

.. _other-properties-548:

Other properties
----------------

============= ===============
**Aliases:**   seasonal use
**Mappings:**  mixs:season_use
============= ===============

Slot: secondary_treatment
=========================

The process for substantially degrading the biological content of the
sewage

URI:
`nmdc:secondary_treatment <https://microbiomedata/meta/secondary_treatment>`__

.. _domain-and-range-720:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-757:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-727:

Children
--------

.. _used-by-720:

Used by
-------

.. _other-properties-549:

Other properties
----------------

============= ========================
**Aliases:**   secondary treatment
**Mappings:**  mixs:secondary_treatment
============= ========================

Slot: sediment_type
===================

Information about the sediment type based on major constituents

URI: `nmdc:sediment_type <https://microbiomedata/meta/sediment_type>`__

.. _domain-and-range-721:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-758:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-728:

Children
--------

.. _used-by-721:

Used by
-------

.. _other-properties-550:

Other properties
----------------

============= ==================
**Aliases:**   sediment type
**Mappings:**  mixs:sediment_type
============= ==================

Slot: seq_meth
==============

“Sequencing method used; e.g. Sanger, pyrosequencing, ABI-solid”

URI: `nmdc:seq_meth <https://microbiomedata/meta/seq_meth>`__

.. _domain-and-range-722:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-759:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-729:

Children
--------

.. _used-by-722:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-551:

Other properties
----------------

=============== =================
**Aliases:**     sequencing method
**Mappings:**    mixs:seq_meth
**In Subsets:**  sequencing
=============== =================

Slot: seq_quality_check
=======================

“Indicate if the sequence has been called by automatic systems (none) or
undergone a manual editing procedure (e.g. by inspecting the raw data or
chromatograms). Applied only for sequences that are not submitted to
SRA,ENA or DRA”

URI:
`nmdc:seq_quality_check <https://microbiomedata/meta/seq_quality_check>`__

.. _domain-and-range-723:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-760:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-730:

Children
--------

.. _used-by-723:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-552:

Other properties
----------------

=============== ======================
**Aliases:**     sequence quality check
**Mappings:**    mixs:seq_quality_check
**In Subsets:**  sequencing
=============== ======================

.. _slot-seqid-1:

Slot: seqid
===========

The ID of the landmark used to establish the coordinate system for the
current feature.

URI: `nmdc:seqid <https://microbiomedata/meta/seqid>`__

.. _domain-and-range-724:

Domain and Range
----------------

None → 1..1 `String <types/String.md>`__

.. _parents-761:

Parents
-------

.. _children-731:

Children
--------

-  `genome feature➞seqid <genome_feature_seqid.md>`__

.. _used-by-724:

Used by
-------

Slot: sewage_type
=================

Type of wastewater treatment plant as municipial or industrial

URI: `nmdc:sewage_type <https://microbiomedata/meta/sewage_type>`__

.. _domain-and-range-725:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-762:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-732:

Children
--------

.. _used-by-725:

Used by
-------

.. _other-properties-553:

Other properties
----------------

============= ================
**Aliases:**   sewage type
**Mappings:**  mixs:sewage_type
============= ================

Slot: sexual_act
================

Current sexual partner and frequency of sex

URI: `nmdc:sexual_act <https://microbiomedata/meta/sexual_act>`__

.. _domain-and-range-726:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-763:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-733:

Children
--------

.. _used-by-726:

Used by
-------

.. _other-properties-554:

Other properties
----------------

============= ===============
**Aliases:**   sexual activity
**Mappings:**  mixs:sexual_act
============= ===============

Slot: shading_device_cond
=========================

The physical condition of the shading device at the time of sampling

URI:
`nmdc:shading_device_cond <https://microbiomedata/meta/shading_device_cond>`__

.. _domain-and-range-727:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-764:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-734:

Children
--------

.. _used-by-727:

Used by
-------

.. _other-properties-555:

Other properties
----------------

============= ========================
**Aliases:**   shading device condition
**Mappings:**  mixs:shading_device_cond
============= ========================

Slot: shading_device_loc
========================

The location of the shading device in relation to the built structure

URI:
`nmdc:shading_device_loc <https://microbiomedata/meta/shading_device_loc>`__

.. _domain-and-range-728:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-765:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-735:

Children
--------

.. _used-by-728:

Used by
-------

.. _other-properties-556:

Other properties
----------------

============= =======================
**Aliases:**   shading device location
**Mappings:**  mixs:shading_device_loc
============= =======================

Slot: shading_device_mat
========================

The material the shading device is composed of

URI:
`nmdc:shading_device_mat <https://microbiomedata/meta/shading_device_mat>`__

.. _domain-and-range-729:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-766:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-736:

Children
--------

.. _used-by-729:

Used by
-------

.. _other-properties-557:

Other properties
----------------

============= =======================
**Aliases:**   shading device material
**Mappings:**  mixs:shading_device_mat
============= =======================

Slot: shading_device_type
=========================

The type of shading device

URI:
`nmdc:shading_device_type <https://microbiomedata/meta/shading_device_type>`__

.. _domain-and-range-730:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-767:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-737:

Children
--------

.. _used-by-730:

Used by
-------

.. _other-properties-558:

Other properties
----------------

============= ========================
**Aliases:**   shading device type
**Mappings:**  mixs:shading_device_type
============= ========================

Slot: shading_device_water_mold
===============================

Signs of the presence of mold or mildew on the shading device

URI:
`nmdc:shading_device_water_mold <https://microbiomedata/meta/shading_device_water_mold>`__

.. _domain-and-range-731:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-768:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-738:

Children
--------

.. _used-by-731:

Used by
-------

.. _other-properties-559:

Other properties
----------------

============= ==================================
**Aliases:**   shading device signs of water/mold
**Mappings:**  mixs:shading_device_water_mold
============= ==================================

Slot: sieving
=============

Collection design of pooled samples and/or sieve size and amount of
sample sieved

URI: `nmdc:sieving <https://microbiomedata/meta/sieving>`__

.. _domain-and-range-732:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-769:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-739:

Children
--------

.. _used-by-732:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-560:

Other properties
----------------

============= =================================
**Aliases:**   composite design/sieving (if any)
**Mappings:**  mixs:sieving
============= =================================

Slot: silicate
==============

Concentration of silicate

URI: `nmdc:silicate <https://microbiomedata/meta/silicate>`__

.. _domain-and-range-733:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-770:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-740:

Children
--------

.. _used-by-733:

Used by
-------

.. _other-properties-561:

Other properties
----------------

============= =============
**Aliases:**   silicate
**Mappings:**  mixs:silicate
============= =============

Slot: sim_search_meth
=====================

“Tool used to compare ORFs with database, along with version and cutoffs
used”

URI:
`nmdc:sim_search_meth <https://microbiomedata/meta/sim_search_meth>`__

.. _domain-and-range-734:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-771:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-741:

Children
--------

.. _used-by-734:

Used by
-------

.. _other-properties-562:

Other properties
----------------

=============== ========================
**Aliases:**     similarity search method
**Mappings:**    mixs:sim_search_meth
**In Subsets:**  sequencing
=============== ========================

Slot: single_cell_lysis_appr
============================

Method used to free DNA from interior of the cell(s) or particle(s)

URI:
`nmdc:single_cell_lysis_appr <https://microbiomedata/meta/single_cell_lysis_appr>`__

.. _domain-and-range-735:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-772:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-742:

Children
--------

.. _used-by-735:

Used by
-------

.. _other-properties-563:

Other properties
----------------

=============== ============================================
**Aliases:**     single cell or viral particle lysis approach
**Mappings:**    mixs:single_cell_lysis_appr
**In Subsets:**  sequencing
=============== ============================================

Slot: single_cell_lysis_prot
============================

Name of the kit or standard protocol used for cell(s) or particle(s)
lysis

URI:
`nmdc:single_cell_lysis_prot <https://microbiomedata/meta/single_cell_lysis_prot>`__

.. _domain-and-range-736:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-773:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-743:

Children
--------

.. _used-by-736:

Used by
-------

.. _other-properties-564:

Other properties
----------------

=============== ================================================
**Aliases:**     single cell or viral particle lysis kit protocol
**Mappings:**    mixs:single_cell_lysis_prot
**In Subsets:**  sequencing
=============== ================================================

Slot: size_frac
===============

Filtering pore size used in sample preparation

URI: `nmdc:size_frac <https://microbiomedata/meta/size_frac>`__

.. _domain-and-range-737:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-774:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-744:

Children
--------

.. _used-by-737:

Used by
-------

.. _other-properties-565:

Other properties
----------------

=============== ============================
**Aliases:**     size fraction selected
**Mappings:**    mixs:size_frac
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: size_frac_low
===================

Refers to the mesh/pore size used to pre-filter/pre-sort the sample.
Materials larger than the size threshold are excluded from the sample

URI: `nmdc:size_frac_low <https://microbiomedata/meta/size_frac_low>`__

.. _domain-and-range-738:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-775:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-745:

Children
--------

.. _used-by-738:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-566:

Other properties
----------------

============= =============================
**Aliases:**   size-fraction lower threshold
**Mappings:**  mixs:size_frac_low
============= =============================

Slot: size_frac_up
==================

Refers to the mesh/pore size used to retain the sample. Materials
smaller than the size threshold are excluded from the sample

URI: `nmdc:size_frac_up <https://microbiomedata/meta/size_frac_up>`__

.. _domain-and-range-739:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-776:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-746:

Children
--------

.. _used-by-739:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-567:

Other properties
----------------

============= =============================
**Aliases:**   size-fraction upper threshold
**Mappings:**  mixs:size_frac_up
============= =============================

Slot: slope_aspect
==================

“The direction a slope faces. While looking down a slope use a compass
to record the direction you are facing (direction or degrees); e.g., nw
or 315 degrees. This measure provides an indication of sun and wind
exposure that will influence soil temperature and evapotranspiration.”

URI: `nmdc:slope_aspect <https://microbiomedata/meta/slope_aspect>`__

.. _domain-and-range-740:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-777:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-747:

Children
--------

.. _used-by-740:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-568:

Other properties
----------------

============= =================
**Aliases:**   slope aspect
**Mappings:**  mixs:slope_aspect
============= =================

Slot: slope_gradient
====================

Commonly called ‘slope’. The angle between ground surface and a
horizontal line (in percent). This is the direction that overland water
would flow. This measure is usually taken with a hand level meter or
clinometer

URI:
`nmdc:slope_gradient <https://microbiomedata/meta/slope_gradient>`__

.. _domain-and-range-741:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-778:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-748:

Children
--------

.. _used-by-741:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-569:

Other properties
----------------

============= ===================
**Aliases:**   slope gradient
**Mappings:**  mixs:slope_gradient
============= ===================

Slot: sludge_retent_time
========================

The time activated sludge remains in reactor

URI:
`nmdc:sludge_retent_time <https://microbiomedata/meta/sludge_retent_time>`__

.. _domain-and-range-742:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-779:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-749:

Children
--------

.. _used-by-742:

Used by
-------

.. _other-properties-570:

Other properties
----------------

============= =======================
**Aliases:**   sludge retention time
**Mappings:**  mixs:sludge_retent_time
============= =======================

.. _slot-smarts-string-1:

Slot: smarts string
===================

URI: `nmdc:smarts_string <https://microbiomedata/meta/smarts_string>`__

.. _domain-and-range-743:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-780:

Parents
-------

.. _children-750:

Children
--------

-  `reaction➞smarts string <reaction_smarts_string.md>`__

.. _used-by-743:

Used by
-------

.. _slot-smiles-1:

Slot: smiles
============

A string encoding of a molecular graph, no chiral or isotopic
information. There are usually a large number of valid SMILES which
represent a given structure. For example, CCO, OCC and C(O)C all specify
the structure of ethanol.

URI: https://microbiomedata/schema/workflow_execution_activity/smiles

.. _domain-and-range-744:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-781:

Parents
-------

.. _children-751:

Children
--------

-  `chemical entity➞smiles <chemical_entity_smiles.md>`__

.. _used-by-744:

Used by
-------

Slot: smoker
============

Specification of smoking status

URI: `nmdc:smoker <https://microbiomedata/meta/smoker>`__

.. _domain-and-range-745:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-782:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-752:

Children
--------

.. _used-by-745:

Used by
-------

.. _other-properties-571:

Other properties
----------------

============= ===========
**Aliases:**   smoker
**Mappings:**  mixs:smoker
============= ===========

Slot: sodium
============

Sodium concentration in the sample

URI: `nmdc:sodium <https://microbiomedata/meta/sodium>`__

.. _domain-and-range-746:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-783:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-753:

Children
--------

.. _used-by-746:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-572:

Other properties
----------------

============= ===========
**Aliases:**   sodium
**Mappings:**  mixs:sodium
============= ===========

Slot: soil_annual_season_temp
=============================

URI:
`nmdc:soil_annual_season_temp <https://microbiomedata/meta/soil_annual_season_temp>`__

.. _domain-and-range-747:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-784:

Parents
-------

.. _children-754:

Children
--------

.. _used-by-747:

Used by
-------

Slot: soil_type
===============

Soil series name or other lower-level classification

URI: `nmdc:soil_type <https://microbiomedata/meta/soil_type>`__

.. _domain-and-range-748:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-785:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-755:

Children
--------

.. _used-by-748:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-573:

Other properties
----------------

============= ==============
**Aliases:**   soil type
**Mappings:**  mixs:soil_type
============= ==============

Slot: soil_type_meth
====================

Reference or method used in determining soil series name or other
lower-level classification

URI:
`nmdc:soil_type_meth <https://microbiomedata/meta/soil_type_meth>`__

.. _domain-and-range-749:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-786:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-756:

Children
--------

.. _used-by-749:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-574:

Other properties
----------------

============= ===================
**Aliases:**   soil type method
**Mappings:**  mixs:soil_type_meth
============= ===================

Slot: solar_irradiance
======================

The amount of solar energy that arrives at a specific area of a surface
during a specific time interval

URI:
`nmdc:solar_irradiance <https://microbiomedata/meta/solar_irradiance>`__

.. _domain-and-range-750:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-787:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-757:

Children
--------

.. _used-by-750:

Used by
-------

.. _other-properties-575:

Other properties
----------------

============= =====================
**Aliases:**   solar irradiance
**Mappings:**  mixs:solar_irradiance
============= =====================

Slot: soluble_inorg_mat
=======================

“Concentration of substances such as ammonia, road-salt, sea-salt,
cyanide, hydrogen sulfide, thiocyanates, thiosulfates, etc.”

URI:
`nmdc:soluble_inorg_mat <https://microbiomedata/meta/soluble_inorg_mat>`__

.. _domain-and-range-751:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-788:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-758:

Children
--------

.. _used-by-751:

Used by
-------

.. _other-properties-576:

Other properties
----------------

============= ==========================
**Aliases:**   soluble inorganic material
**Mappings:**  mixs:soluble_inorg_mat
============= ==========================

Slot: soluble_iron_micromol
===========================

URI:
`nmdc:soluble_iron_micromol <https://microbiomedata/meta/soluble_iron_micromol>`__

.. _domain-and-range-752:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-789:

Parents
-------

.. _children-759:

Children
--------

.. _used-by-752:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: soluble_org_mat
=====================

“Concentration of substances such as urea, fruit sugars, soluble
proteins, drugs, pharmaceuticals, etc.”

URI:
`nmdc:soluble_org_mat <https://microbiomedata/meta/soluble_org_mat>`__

.. _domain-and-range-753:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-790:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-760:

Children
--------

.. _used-by-753:

Used by
-------

.. _other-properties-577:

Other properties
----------------

============= ========================
**Aliases:**   soluble organic material
**Mappings:**  mixs:soluble_org_mat
============= ========================

Slot: soluble_react_phosp
=========================

Concentration of soluble reactive phosphorus

URI:
`nmdc:soluble_react_phosp <https://microbiomedata/meta/soluble_react_phosp>`__

.. _domain-and-range-754:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-791:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-761:

Children
--------

.. _used-by-754:

Used by
-------

.. _other-properties-578:

Other properties
----------------

============= ===========================
**Aliases:**   soluble reactive phosphorus
**Mappings:**  mixs:soluble_react_phosp
============= ===========================

Slot: sop
=========

“Standard operating procedures used in assembly and/or annotation of
genomes, metagenomes or environmental sequences”

URI: `nmdc:sop <https://microbiomedata/meta/sop>`__

.. _domain-and-range-755:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-792:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-762:

Children
--------

.. _used-by-755:

Used by
-------

.. _other-properties-579:

Other properties
----------------

=============== ======================================
**Aliases:**     relevant standard operating procedures
**Mappings:**    mixs:sop
**In Subsets:**  sequencing
=============== ======================================

Slot: sort_tech
===============

Method used to sort/isolate cells or particles of interest

URI: `nmdc:sort_tech <https://microbiomedata/meta/sort_tech>`__

.. _domain-and-range-756:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-793:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-763:

Children
--------

.. _used-by-756:

Used by
-------

.. _other-properties-580:

Other properties
----------------

=============== ==================
**Aliases:**     sorting technology
**Mappings:**    mixs:sort_tech
**In Subsets:**  sequencing
=============== ==================

Slot: source_mat_id
===================

“A unique identifier assigned to a material sample (as defined by
http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a
particular digital record of a material sample) used for extracting
nucleic acids, and subsequent sequencing. The identifier can refer
either to the original material collected or to any derived sub-samples.
The INSDC qualifiers /specimen_voucher, /bio_material, or
/culture_collection may or may not share the same value as the
source_mat_id field. For instance, the /specimen_voucher qualifier and
source_mat_id may both contain ‘UAM:Herps:14’ , referring to both the
specimen voucher and sampled tissue with the same identifier. However,
the /culture_collection qualifier may refer to a value from an initial
culture (e.g. ATCC:11775) while source_mat_id would refer to an
identifier from some derived culture from which the nucleic acids were
extracted (e.g. xatc123 or ark:/2154/R2).”

URI: `nmdc:source_mat_id <https://microbiomedata/meta/source_mat_id>`__

.. _domain-and-range-757:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-794:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-764:

Children
--------

.. _used-by-757:

Used by
-------

.. _other-properties-581:

Other properties
----------------

=============== ============================
**Aliases:**     source material identifiers
**Mappings:**    mixs:source_mat_id
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: source_uvig
=================

Type of dataset from which the UViG was obtained

URI: `nmdc:source_uvig <https://microbiomedata/meta/source_uvig>`__

.. _domain-and-range-758:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-795:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-765:

Children
--------

.. _used-by-758:

Used by
-------

.. _other-properties-582:

Other properties
----------------

=============== ============================
**Aliases:**     source of UViGs
**Mappings:**    mixs:source_uvig
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: space_typ_state
=====================

Customary or normal state of the space

URI:
`nmdc:space_typ_state <https://microbiomedata/meta/space_typ_state>`__

.. _domain-and-range-759:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-796:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-766:

Children
--------

.. _used-by-759:

Used by
-------

.. _other-properties-583:

Other properties
----------------

============= ====================
**Aliases:**   space typical state
**Mappings:**  mixs:space_typ_state
============= ====================

Slot: special_diet
==================

Specification of special diet; can include multiple special diets

URI: `nmdc:special_diet <https://microbiomedata/meta/special_diet>`__

.. _domain-and-range-760:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-797:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-767:

Children
--------

.. _used-by-760:

Used by
-------

.. _other-properties-584:

Other properties
----------------

============= =================
**Aliases:**   special diet
**Mappings:**  mixs:special_diet
============= =================

Slot: specific
==============

“The building specifications. If design is chosen, indicate phase:
conceptual, schematic, design development, construction documents”

URI: `nmdc:specific <https://microbiomedata/meta/specific>`__

.. _domain-and-range-761:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-798:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-768:

Children
--------

.. _used-by-761:

Used by
-------

.. _other-properties-585:

Other properties
----------------

============= ==============
**Aliases:**   specifications
**Mappings:**  mixs:specific
============= ==============

Slot: specific_ecosystem
========================

TODO

URI:
`nmdc:specific_ecosystem <https://microbiomedata/meta/specific_ecosystem>`__

.. _domain-and-range-762:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-799:

Parents
-------

-  is_a: `gold_path_field <gold_path_field.md>`__

.. _children-769:

Children
--------

.. _used-by-762:

Used by
-------

-  `Biosample <Biosample.md>`__
-  `Study <Study.md>`__

Slot: specific_host
===================

“If there is a host involved, please provide its taxid (or environmental
if not actually isolated from the dead or alive host - i.e. a pathogen
could be isolated from a swipe of a bench etc) and report whether it is
a laboratory or natural host)”

URI: `nmdc:specific_host <https://microbiomedata/meta/specific_host>`__

.. _domain-and-range-763:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-800:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-770:

Children
--------

.. _used-by-763:

Used by
-------

.. _other-properties-586:

Other properties
----------------

=============== ============================
**Aliases:**     specific host
**Mappings:**    mixs:specific_host
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: specific_humidity
=======================

“The mass of water vapour in a unit mass of moist air, usually expressed
as grams of vapour per kilogram of air, or, in air conditioning, as
grains per pound.”

URI:
`nmdc:specific_humidity <https://microbiomedata/meta/specific_humidity>`__

.. _domain-and-range-764:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-801:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-771:

Children
--------

.. _used-by-764:

Used by
-------

.. _other-properties-587:

Other properties
----------------

============= ======================
**Aliases:**   specific humidity
**Mappings:**  mixs:specific_humidity
============= ======================

Slot: sr_dep_env
================

“Source rock depositional environment
(https://en.wikipedia.org/wiki/Source_rock). If”“other”” is specified,
please propose entry in ““additional info”” field”

URI: `nmdc:sr_dep_env <https://microbiomedata/meta/sr_dep_env>`__

.. _domain-and-range-765:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-802:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-772:

Children
--------

.. _used-by-765:

Used by
-------

.. _other-properties-588:

Other properties
----------------

============= ====================================
**Aliases:**   source rock depositional environment
**Mappings:**  mixs:sr_dep_env
============= ====================================

Slot: sr_geol_age
=================

“Geological age of source rock (Additional info:
https://en.wikipedia.org/wiki/Period_(geology)). If”“other”” is
specified, please propose entry in ““additional info”” field”

URI: `nmdc:sr_geol_age <https://microbiomedata/meta/sr_geol_age>`__

.. _domain-and-range-766:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-803:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-773:

Children
--------

.. _used-by-766:

Used by
-------

.. _other-properties-589:

Other properties
----------------

============= ==========================
**Aliases:**   source rock geological age
**Mappings:**  mixs:sr_geol_age
============= ==========================

Slot: sr_kerog_type
===================

“Origin of kerogen. Type I: Algal (aquatic), Type II: planktonic and
soft plant material (aquatic or terrestrial), Type III: terrestrial
woody/ fibrous plant material (terrestrial), Type IV: oxidized recycled
woody debris (terrestrial) (additional information:
https://en.wikipedia.org/wiki/Kerogen). If”“other”” is specified, please
propose entry in ““additional info”” field”

URI: `nmdc:sr_kerog_type <https://microbiomedata/meta/sr_kerog_type>`__

.. _domain-and-range-767:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-804:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-774:

Children
--------

.. _used-by-767:

Used by
-------

.. _other-properties-590:

Other properties
----------------

============= ========================
**Aliases:**   source rock kerogen type
**Mappings:**  mixs:sr_kerog_type
============= ========================

Slot: sr_lithology
==================

“Lithology of source rock (https://en.wikipedia.org/wiki/Source_rock).
If”“other”” is specified, please propose entry in ““additional info””
field”

URI: `nmdc:sr_lithology <https://microbiomedata/meta/sr_lithology>`__

.. _domain-and-range-768:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-805:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-775:

Children
--------

.. _used-by-768:

Used by
-------

.. _other-properties-591:

Other properties
----------------

============= =====================
**Aliases:**   source rock lithology
**Mappings:**  mixs:sr_lithology
============= =====================

Slot: standing_water_regm
=========================

“Treatment involving an exposure to standing water during a plant’s life
span, types can be flood water or standing water, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple regimens”

URI:
`nmdc:standing_water_regm <https://microbiomedata/meta/standing_water_regm>`__

.. _domain-and-range-769:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-806:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-776:

Children
--------

.. _used-by-769:

Used by
-------

.. _other-properties-592:

Other properties
----------------

============= ========================
**Aliases:**   standing water regimen
**Mappings:**  mixs:standing_water_regm
============= ========================

.. _slot-start-1:

Slot: start
===========

The start of the feature in positive 1-based integer coordinates

URI: `nmdc:start <https://microbiomedata/meta/start>`__

.. _domain-and-range-770:

Domain and Range
----------------

None → 1..1 `Integer <types/Integer.md>`__

.. _parents-807:

Parents
-------

-  is_a: `gff coordinate <gff_coordinate.md>`__

.. _children-777:

Children
--------

.. _used-by-770:

Used by
-------

.. _other-properties-593:

Other properties
----------------

=================== ==================================
**Close Mappings:**  biolink:start_interbase_coordinate
=================== ==================================

Slot: started at time
=====================

URI:
https://microbiomedata/schema/workflow_execution_activity/started_at_time

.. _domain-and-range-771:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-808:

Parents
-------

.. _children-778:

Children
--------

-  `workflow execution activity➞started at
   time <workflow_execution_activity_started_at_time.md>`__

.. _used-by-771:

Used by
-------

-  `Activity <Activity.md>`__

.. _other-properties-594:

Other properties
----------------

============= ==================
**Mappings:**  prov:startedAtTime
============= ==================

.. _slot-stoichiometry-1:

Slot: stoichiometry
===================

URI: `nmdc:stoichiometry <https://microbiomedata/meta/stoichiometry>`__

.. _domain-and-range-772:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-809:

Parents
-------

.. _children-779:

Children
--------

-  `reaction
   participant➞stoichiometry <reaction_participant_stoichiometry.md>`__

.. _used-by-772:

Used by
-------

Slot: store_cond
================

Explain how and for how long the soil sample was stored before DNA
extraction

URI: `nmdc:store_cond <https://microbiomedata/meta/store_cond>`__

.. _domain-and-range-773:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-810:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-780:

Children
--------

.. _used-by-773:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-595:

Other properties
----------------

============= =======================================
**Aliases:**   storage conditions (fresh/frozen/other)
**Mappings:**  mixs:store_cond
============= =======================================

.. _slot-strand-1:

Slot: strand
============

The strand on which a feature is located. Has a value of ‘+’ (sense
strand or forward strand) or ‘-’ (anti-sense strand or reverse strand).

URI: `nmdc:strand <https://microbiomedata/meta/strand>`__

.. _domain-and-range-774:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-811:

Parents
-------

.. _children-781:

Children
--------

-  `genome feature➞strand <genome_feature_strand.md>`__

.. _used-by-774:

Used by
-------

.. _other-properties-596:

Other properties
----------------

=================== ==============
**Exact Mappings:**  biolink:strand
=================== ==============

Slot: study_complt_stat
=======================

“Specification of study completion status, if no the reason should be
specified”

URI:
`nmdc:study_complt_stat <https://microbiomedata/meta/study_complt_stat>`__

.. _domain-and-range-775:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-812:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-782:

Children
--------

.. _used-by-775:

Used by
-------

.. _other-properties-597:

Other properties
----------------

============= =======================
**Aliases:**   study completion status
**Mappings:**  mixs:study_complt_stat
============= =======================

.. _slot-doi-1:

Slot: doi
=========

The dataset citation for this study

URI: `nmdc:study_doi <https://microbiomedata/meta/study_doi>`__

.. _domain-and-range-776:

Domain and Range
----------------

`Study <Study.md>`__ → 0..1 `AttributeValue <AttributeValue.md>`__

.. _parents-813:

Parents
-------

-  is_a: `doi <doi.md>`__

.. _children-783:

Children
--------

.. _used-by-776:

Used by
-------

-  `Study <Study.md>`__

Slot: study identifiers
=======================

URI:
https://microbiomedata/schema/workflow_execution_activity/study_identifiers

.. _domain-and-range-777:

Domain and Range
----------------

None → 0..\* `ExternalIdentifier <types/ExternalIdentifier.md>`__

.. _parents-814:

Parents
-------

-  is_a: `external database
   identifiers <external_database_identifiers.md>`__

.. _children-784:

Children
--------

-  `GOLD study identifiers <GOLD_study_identifiers.md>`__
-  `INSDC SRA ENA study
   identifiers <INSDC_SRA_ENA_study_identifiers.md>`__
-  `INSDC bioproject identifiers <INSDC_bioproject_identifiers.md>`__
-  `MGnify project identifiers <MGnify_project_identifiers.md>`__

.. _used-by-777:

Used by
-------

Slot: study image
=================

Links a study to one or more images.

URI: `nmdc:study_image <https://microbiomedata/meta/study_image>`__

.. _domain-and-range-778:

Domain and Range
----------------

`Study <Study.md>`__ → 0..\* `ImageValue <ImageValue.md>`__

.. _parents-815:

Parents
-------

.. _children-785:

Children
--------

.. _used-by-778:

Used by
-------

-  `Study <Study.md>`__

Slot: study set
===============

This property links a database object to the set of studies within it.

URI: `nmdc:study_set <https://microbiomedata/meta/study_set>`__

.. _domain-and-range-779:

Domain and Range
----------------

`Database <Database.md>`__ → 0..\* `Study <Study.md>`__

.. _parents-816:

Parents
-------

.. _children-786:

Children
--------

.. _used-by-779:

Used by
-------

-  `Database <Database.md>`__

Slot: subject
=============

URI: `nmdc:subject <https://microbiomedata/meta/subject>`__

.. _domain-and-range-780:

Domain and Range
----------------

None → 0..1 `GeneProduct <GeneProduct.md>`__

.. _parents-817:

Parents
-------

.. _children-787:

Children
--------

.. _used-by-780:

Used by
-------

-  `FunctionalAnnotation <FunctionalAnnotation.md>`__

Slot: submitted_to_insdc
========================

“Depending on the study (large-scale e.g. done with next generation
sequencing technology, or small-scale) sequences have to be submitted to
SRA (Sequence Read Archive), DRA (DDBJ Read Archive) or via the
classical Webin/Sequin systems to Genbank, ENA and DDBJ. Although this
field is mandatory, it is meant as a self-test field, therefore it is
not necessary to include this field in contextual data submitted to
databases”

URI:
`nmdc:submitted_to_insdc <https://microbiomedata/meta/submitted_to_insdc>`__

.. _domain-and-range-781:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-818:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-788:

Children
--------

.. _used-by-781:

Used by
-------

.. _other-properties-598:

Other properties
----------------

=============== =======================
**Aliases:**     submitted to insdc
**Mappings:**    mixs:submitted_to_insdc
**In Subsets:**  investigation
=============== =======================

Slot: subspecf_gen_lin
======================

“This should provide further information about the genetic distinctness
of the sequenced organism by recording additional information
e.g. serovar, serotype, biotype, ecotype, or any relevant genetic typing
schemes like Group I plasmid. It can also contain alternative taxonomic
information. It should contain both the lineage name, and the lineage
rank, i.e. biovar:abc123”

URI:
`nmdc:subspecf_gen_lin <https://microbiomedata/meta/subspecf_gen_lin>`__

.. _domain-and-range-782:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-819:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-789:

Children
--------

.. _used-by-782:

Used by
-------

.. _other-properties-599:

Other properties
----------------

=============== ============================
**Aliases:**     subspecific genetic lineage
**Mappings:**    mixs:subspecf_gen_lin
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: substructure_type
=======================

The substructure or under building is that largely hidden section of the
building which is built off the foundations to the ground floor level

URI:
`nmdc:substructure_type <https://microbiomedata/meta/substructure_type>`__

.. _domain-and-range-783:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-820:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-790:

Children
--------

.. _used-by-783:

Used by
-------

.. _other-properties-600:

Other properties
----------------

============= ======================
**Aliases:**   substructure type
**Mappings:**  mixs:substructure_type
============= ======================

Slot: subsurface_depth
======================

URI:
`nmdc:subsurface_depth <https://microbiomedata/meta/subsurface_depth>`__

.. _domain-and-range-784:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-821:

Parents
-------

.. _children-791:

Children
--------

.. _used-by-784:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: subsurface_depth2
=======================

URI:
`nmdc:subsurface_depth2 <https://microbiomedata/meta/subsurface_depth2>`__

.. _domain-and-range-785:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-822:

Parents
-------

.. _children-792:

Children
--------

.. _used-by-785:

Used by
-------

-  `Biosample <Biosample.md>`__

Slot: sulfate
=============

Concentration of sulfate in the sample

URI: `nmdc:sulfate <https://microbiomedata/meta/sulfate>`__

.. _domain-and-range-786:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-823:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-793:

Children
--------

.. _used-by-786:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-601:

Other properties
----------------

============= ============
**Aliases:**   sulfate
**Mappings:**  mixs:sulfate
============= ============

Slot: sulfate_fw
================

Original sulfate concentration in the hydrocarbon resource

URI: `nmdc:sulfate_fw <https://microbiomedata/meta/sulfate_fw>`__

.. _domain-and-range-787:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-824:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-794:

Children
--------

.. _used-by-787:

Used by
-------

.. _other-properties-602:

Other properties
----------------

============= ==========================
**Aliases:**   sulfate in formation water
**Mappings:**  mixs:sulfate_fw
============= ==========================

Slot: sulfide
=============

Concentration of sulfide in the sample

URI: `nmdc:sulfide <https://microbiomedata/meta/sulfide>`__

.. _domain-and-range-788:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-825:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-795:

Children
--------

.. _used-by-788:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-603:

Other properties
----------------

============= ============
**Aliases:**   sulfide
**Mappings:**  mixs:sulfide
============= ============

Slot: surf_air_cont
===================

Contaminant identified on surface

URI: `nmdc:surf_air_cont <https://microbiomedata/meta/surf_air_cont>`__

.. _domain-and-range-789:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-826:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-796:

Children
--------

.. _used-by-789:

Used by
-------

.. _other-properties-604:

Other properties
----------------

============= =======================
**Aliases:**   surface-air contaminant
**Mappings:**  mixs:surf_air_cont
============= =======================

Slot: surf_humidity
===================

Surfaces: water activity as a function of air and material moisture

URI: `nmdc:surf_humidity <https://microbiomedata/meta/surf_humidity>`__

.. _domain-and-range-790:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-827:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-797:

Children
--------

.. _used-by-790:

Used by
-------

.. _other-properties-605:

Other properties
----------------

============= ==================
**Aliases:**   surface humidity
**Mappings:**  mixs:surf_humidity
============= ==================

Slot: surf_material
===================

Surface materials at the point of sampling

URI: `nmdc:surf_material <https://microbiomedata/meta/surf_material>`__

.. _domain-and-range-791:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-828:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-798:

Children
--------

.. _used-by-791:

Used by
-------

.. _other-properties-606:

Other properties
----------------

============= ==================
**Aliases:**   surface material
**Mappings:**  mixs:surf_material
============= ==================

Slot: surf_moisture
===================

Water held on a surface

URI: `nmdc:surf_moisture <https://microbiomedata/meta/surf_moisture>`__

.. _domain-and-range-792:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-829:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-799:

Children
--------

.. _used-by-792:

Used by
-------

.. _other-properties-607:

Other properties
----------------

============= ==================
**Aliases:**   surface moisture
**Mappings:**  mixs:surf_moisture
============= ==================

Slot: surf_moisture_ph
======================

ph measurement of surface

URI:
`nmdc:surf_moisture_ph <https://microbiomedata/meta/surf_moisture_ph>`__

.. _domain-and-range-793:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-830:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-800:

Children
--------

.. _used-by-793:

Used by
-------

.. _other-properties-608:

Other properties
----------------

============= =====================
**Aliases:**   surface moisture pH
**Mappings:**  mixs:surf_moisture_ph
============= =====================

Slot: surf_temp
===============

Temperature of the surface at the time of sampling

URI: `nmdc:surf_temp <https://microbiomedata/meta/surf_temp>`__

.. _domain-and-range-794:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-831:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-801:

Children
--------

.. _used-by-794:

Used by
-------

.. _other-properties-609:

Other properties
----------------

============= ===================
**Aliases:**   surface temperature
**Mappings:**  mixs:surf_temp
============= ===================

Slot: suspend_part_matter
=========================

Concentration of suspended particulate matter

URI:
`nmdc:suspend_part_matter <https://microbiomedata/meta/suspend_part_matter>`__

.. _domain-and-range-795:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-832:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-802:

Children
--------

.. _used-by-795:

Used by
-------

.. _other-properties-610:

Other properties
----------------

============= ============================
**Aliases:**   suspended particulate matter
**Mappings:**  mixs:suspend_part_matter
============= ============================

Slot: suspend_solids
====================

“Concentration of substances including a wide variety of material, such
as silt, decaying plant and animal matter; can include multiple
substances”

URI:
`nmdc:suspend_solids <https://microbiomedata/meta/suspend_solids>`__

.. _domain-and-range-796:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-833:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-803:

Children
--------

.. _used-by-796:

Used by
-------

.. _other-properties-611:

Other properties
----------------

============= ===================
**Aliases:**   suspended solids
**Mappings:**  mixs:suspend_solids
============= ===================

Slot: tan
=========

Total Acid NumberB (TAN) is a measurement of acidity that is determined
by the amount ofB potassium hydroxideB in milligrams that is needed to
neutralize the acids in one gram of oil.B It is an important quality
measurement ofB crude oil. (source:
https://en.wikipedia.org/wiki/Total_acid_number)

URI: `nmdc:tan <https://microbiomedata/meta/tan>`__

.. _domain-and-range-797:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-834:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-804:

Children
--------

.. _used-by-797:

Used by
-------

.. _other-properties-612:

Other properties
----------------

============= =================
**Aliases:**   total acid number
**Mappings:**  mixs:tan
============= =================

Slot: target_gene
=================

Targeted gene or locus name for marker gene studies

URI: `nmdc:target_gene <https://microbiomedata/meta/target_gene>`__

.. _domain-and-range-798:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-835:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-805:

Children
--------

.. _used-by-798:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-613:

Other properties
----------------

=============== ================
**Aliases:**     target gene
**Mappings:**    mixs:target_gene
**In Subsets:**  sequencing
=============== ================

Slot: target_subfragment
========================

Name of subfragment of a gene or locus. Important to e.g. identify
special regions on marker genes like V6 on 16S rRNA

URI:
`nmdc:target_subfragment <https://microbiomedata/meta/target_subfragment>`__

.. _domain-and-range-799:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-836:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-806:

Children
--------

.. _used-by-799:

Used by
-------

-  `OmicsProcessing <OmicsProcessing.md>`__

.. _other-properties-614:

Other properties
----------------

=============== =======================
**Aliases:**     target subfragment
**Mappings:**    mixs:target_subfragment
**In Subsets:**  sequencing
=============== =======================

Slot: tax_class
===============

“Method used for taxonomic classification, along with reference database
used, classification rank, and thresholds used to classify new genomes”

URI: `nmdc:tax_class <https://microbiomedata/meta/tax_class>`__

.. _domain-and-range-800:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-837:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-807:

Children
--------

.. _used-by-800:

Used by
-------

.. _other-properties-615:

Other properties
----------------

=============== ========================
**Aliases:**     taxonomic classification
**Mappings:**    mixs:tax_class
**In Subsets:**  sequencing
=============== ========================

Slot: tax_ident
===============

The phylogenetic marker(s) used to assign an organism name to the SAG or
MAG

URI: `nmdc:tax_ident <https://microbiomedata/meta/tax_ident>`__

.. _domain-and-range-801:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-838:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-808:

Children
--------

.. _used-by-801:

Used by
-------

.. _other-properties-616:

Other properties
----------------

=============== =========================
**Aliases:**     taxonomic identity marker
**Mappings:**    mixs:tax_ident
**In Subsets:**  sequencing
=============== =========================

Slot: temp
==========

Temperature of the sample at the time of sampling

URI: `nmdc:temp <https://microbiomedata/meta/temp>`__

.. _domain-and-range-802:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-839:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-809:

Children
--------

.. _used-by-802:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-617:

Other properties
----------------

============= ===========
**Aliases:**   temperature
**Mappings:**  mixs:temp
============= ===========

Slot: temp_out
==============

The recorded temperature value at sampling time outside

URI: `nmdc:temp_out <https://microbiomedata/meta/temp_out>`__

.. _domain-and-range-803:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-840:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-810:

Children
--------

.. _used-by-803:

Used by
-------

.. _other-properties-618:

Other properties
----------------

============= =========================
**Aliases:**   temperature outside house
**Mappings:**  mixs:temp_out
============= =========================

Slot: term
==========

pointer to an ontology class

URI: https://microbiomedata/schema/workflow_execution_activity/term

.. _domain-and-range-804:

Domain and Range
----------------

`ControlledTermValue <ControlledTermValue.md>`__ → 0..1
`OntologyClass <OntologyClass.md>`__

.. _parents-841:

Parents
-------

.. _children-811:

Children
--------

.. _used-by-804:

Used by
-------

-  `ControlledTermValue <ControlledTermValue.md>`__

.. _other-properties-619:

Other properties
----------------

============= ========
**Mappings:**  rdf:type
============= ========

Slot: tertiary_treatment
========================

The process providing a final treatment stage to raise the effluent
quality before it is discharged to the receiving environment

URI:
`nmdc:tertiary_treatment <https://microbiomedata/meta/tertiary_treatment>`__

.. _domain-and-range-805:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-842:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-812:

Children
--------

.. _used-by-805:

Used by
-------

.. _other-properties-620:

Other properties
----------------

============= =======================
**Aliases:**   tertiary treatment
**Mappings:**  mixs:tertiary_treatment
============= =======================

Slot: texture
=============

“The relative proportion of different grain sizes of mineral particles
in a soil, as described using a standard system; express as % sand (50
um to 2 mm), silt (2 um to 50 um), and clay (<2 um) with textural name
(e.g., silty clay loam) optional.”

URI: `nmdc:texture <https://microbiomedata/meta/texture>`__

.. _domain-and-range-806:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-843:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-813:

Children
--------

.. _used-by-806:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-621:

Other properties
----------------

============= ============
**Aliases:**   texture
**Mappings:**  mixs:texture
============= ============

Slot: texture_meth
==================

Reference or method used in determining soil texture

URI: `nmdc:texture_meth <https://microbiomedata/meta/texture_meth>`__

.. _domain-and-range-807:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-844:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-814:

Children
--------

.. _used-by-807:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-622:

Other properties
----------------

============= =================
**Aliases:**   texture method
**Mappings:**  mixs:texture_meth
============= =================

Slot: tidal_stage
=================

Stage of tide

URI: `nmdc:tidal_stage <https://microbiomedata/meta/tidal_stage>`__

.. _domain-and-range-808:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-845:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-815:

Children
--------

.. _used-by-808:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-623:

Other properties
----------------

============= ================
**Aliases:**   tidal stage
**Mappings:**  mixs:tidal_stage
============= ================

Slot: tillage
=============

Note method(s) used for tilling

URI: `nmdc:tillage <https://microbiomedata/meta/tillage>`__

.. _domain-and-range-809:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-846:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-816:

Children
--------

.. _used-by-809:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-624:

Other properties
----------------

============= ===============
**Aliases:**   history/tillage
**Mappings:**  mixs:tillage
============= ===============

Slot: time_last_toothbrush
==========================

Specification of the time since last toothbrushing

URI:
`nmdc:time_last_toothbrush <https://microbiomedata/meta/time_last_toothbrush>`__

.. _domain-and-range-810:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-847:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-817:

Children
--------

.. _used-by-810:

Used by
-------

.. _other-properties-625:

Other properties
----------------

============= =============================
**Aliases:**   time since last toothbrushing
**Mappings:**  mixs:time_last_toothbrush
============= =============================

Slot: time_since_last_wash
==========================

Specification of the time since last wash

URI:
`nmdc:time_since_last_wash <https://microbiomedata/meta/time_since_last_wash>`__

.. _domain-and-range-811:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-848:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-818:

Children
--------

.. _used-by-811:

Used by
-------

.. _other-properties-626:

Other properties
----------------

============= =========================
**Aliases:**   time since last wash
**Mappings:**  mixs:time_since_last_wash
============= =========================

Slot: tiss_cult_growth_med
==========================

Description of plant tissue culture growth media used

URI:
`nmdc:tiss_cult_growth_med <https://microbiomedata/meta/tiss_cult_growth_med>`__

.. _domain-and-range-812:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-849:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-819:

Children
--------

.. _used-by-812:

Used by
-------

.. _other-properties-627:

Other properties
----------------

============= ===========================
**Aliases:**   tissue culture growth media
**Mappings:**  mixs:tiss_cult_growth_med
============= ===========================

Slot: title
===========

A name given to the entity that differs from the name/label
programatically assigned to it. For example, when extracting study
information for GOLD, the GOLD system has assigned a name/label.
However, for display purposes, we may also wish the capture the title of
the proposal that was used to fund the study.

URI: https://microbiomedata/schema/workflow_execution_activity/title

.. _domain-and-range-813:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-850:

Parents
-------

.. _children-820:

Children
--------

.. _used-by-813:

Used by
-------

.. _other-properties-628:

Other properties
----------------

=================== =============
**Exact Mappings:**  dcterms:title
=================== =============

Slot: toluene
=============

Concentration of toluene in the sample

URI: `nmdc:toluene <https://microbiomedata/meta/toluene>`__

.. _domain-and-range-814:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-851:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-821:

Children
--------

.. _used-by-814:

Used by
-------

.. _other-properties-629:

Other properties
----------------

============= ============
**Aliases:**   toluene
**Mappings:**  mixs:toluene
============= ============

Slot: too short contig num
==========================

URI:
https://microbiomedata/schema/workflow_execution_activity/too_short_contig_num

.. _domain-and-range-815:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-852:

Parents
-------

.. _children-822:

Children
--------

.. _used-by-815:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

Slot: tot_carb
==============

Total carbon content

URI: `nmdc:tot_carb <https://microbiomedata/meta/tot_carb>`__

.. _domain-and-range-816:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-853:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-823:

Children
--------

.. _used-by-816:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-630:

Other properties
----------------

============= =============
**Aliases:**   total carbon
**Mappings:**  mixs:tot_carb
============= =============

Slot: tot_depth_water_col
=========================

Measurement of total depth of water column

URI:
`nmdc:tot_depth_water_col <https://microbiomedata/meta/tot_depth_water_col>`__

.. _domain-and-range-817:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-854:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-824:

Children
--------

.. _used-by-817:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-631:

Other properties
----------------

============= ===========================
**Aliases:**   total depth of water column
**Mappings:**  mixs:tot_depth_water_col
============= ===========================

Slot: tot_diss_nitro
====================

“Total dissolved nitrogen concentration, reported as nitrogen, measured
by: total dissolved nitrogen = NH4 + NO3NO2 + dissolved organic
nitrogen”

URI:
`nmdc:tot_diss_nitro <https://microbiomedata/meta/tot_diss_nitro>`__

.. _domain-and-range-818:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-855:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-825:

Children
--------

.. _used-by-818:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-632:

Other properties
----------------

============= ========================
**Aliases:**   total dissolved nitrogen
**Mappings:**  mixs:tot_diss_nitro
============= ========================

Slot: tot_inorg_nitro
=====================

Total inorganic nitrogen content

URI:
`nmdc:tot_inorg_nitro <https://microbiomedata/meta/tot_inorg_nitro>`__

.. _domain-and-range-819:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-856:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-826:

Children
--------

.. _used-by-819:

Used by
-------

.. _other-properties-633:

Other properties
----------------

============= ========================
**Aliases:**   total inorganic nitrogen
**Mappings:**  mixs:tot_inorg_nitro
============= ========================

Slot: tot_iron
==============

Concentration of total iron in the sample

URI: `nmdc:tot_iron <https://microbiomedata/meta/tot_iron>`__

.. _domain-and-range-820:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-857:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-827:

Children
--------

.. _used-by-820:

Used by
-------

.. _other-properties-634:

Other properties
----------------

============= =============
**Aliases:**   total iron
**Mappings:**  mixs:tot_iron
============= =============

Slot: tot_nitro
===============

“Total nitrogen concentration of water samples, calculated by: total
nitrogen = total dissolved nitrogen + particulate nitrogen. Can also be
measured without filtering, reported as nitrogen”

URI: `nmdc:tot_nitro <https://microbiomedata/meta/tot_nitro>`__

.. _domain-and-range-821:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-858:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-828:

Children
--------

.. _used-by-821:

Used by
-------

.. _other-properties-635:

Other properties
----------------

============= ============================
**Aliases:**   total nitrogen concentration
**Mappings:**  mixs:tot_nitro
============= ============================

Slot: tot_nitro_content
=======================

Total nitrogen content of the sample

URI:
`nmdc:tot_nitro_content <https://microbiomedata/meta/tot_nitro_content>`__

.. _domain-and-range-822:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-859:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-829:

Children
--------

.. _used-by-822:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-636:

Other properties
----------------

============= ======================
**Aliases:**   total nitrogen content
**Mappings:**  mixs:tot_nitro_content
============= ======================

Slot: tot_nitro_content_meth
============================

Reference or method used in determining the total nitrogen

URI:
`nmdc:tot_nitro_content_meth <https://microbiomedata/meta/tot_nitro_content_meth>`__

.. _domain-and-range-823:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-860:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-830:

Children
--------

.. _used-by-823:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-637:

Other properties
----------------

============= =============================
**Aliases:**   total nitrogen content method
**Mappings:**  mixs:tot_nitro_content_meth
============= =============================

Slot: tot_org_c_meth
====================

Reference or method used in determining total organic carbon

URI:
`nmdc:tot_org_c_meth <https://microbiomedata/meta/tot_org_c_meth>`__

.. _domain-and-range-824:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-861:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-831:

Children
--------

.. _used-by-824:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-638:

Other properties
----------------

============= ===========================
**Aliases:**   total organic carbon method
**Mappings:**  mixs:tot_org_c_meth
============= ===========================

Slot: tot_org_carb
==================

“Definition for soil: total organic carbon content of the soil,
definition otherwise: total organic carbon content”

URI: `nmdc:tot_org_carb <https://microbiomedata/meta/tot_org_carb>`__

.. _domain-and-range-825:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-862:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-832:

Children
--------

.. _used-by-825:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-639:

Other properties
----------------

============= ====================
**Aliases:**   total organic carbon
**Mappings:**  mixs:tot_org_carb
============= ====================

Slot: tot_part_carb
===================

Total particulate carbon content

URI: `nmdc:tot_part_carb <https://microbiomedata/meta/tot_part_carb>`__

.. _domain-and-range-826:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-863:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-833:

Children
--------

.. _used-by-826:

Used by
-------

.. _other-properties-640:

Other properties
----------------

============= ========================
**Aliases:**   total particulate carbon
**Mappings:**  mixs:tot_part_carb
============= ========================

Slot: tot_phosp
===============

“Total phosphorus concentration in the sample, calculated by: total
phosphorus = total dissolved phosphorus + particulate phosphorus”

URI: `nmdc:tot_phosp <https://microbiomedata/meta/tot_phosp>`__

.. _domain-and-range-827:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-864:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-834:

Children
--------

.. _used-by-827:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-641:

Other properties
----------------

============= ================
**Aliases:**   total phosphorus
**Mappings:**  mixs:tot_phosp
============= ================

Slot: tot_phosphate
===================

Total amount or concentration of phosphate

URI: `nmdc:tot_phosphate <https://microbiomedata/meta/tot_phosphate>`__

.. _domain-and-range-828:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-865:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-835:

Children
--------

.. _used-by-828:

Used by
-------

.. _other-properties-642:

Other properties
----------------

============= ==================
**Aliases:**   total phosphate
**Mappings:**  mixs:tot_phosphate
============= ==================

Slot: tot_sulfur
================

Concentration of total sulfur in the sample

URI: `nmdc:tot_sulfur <https://microbiomedata/meta/tot_sulfur>`__

.. _domain-and-range-829:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-866:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-836:

Children
--------

.. _used-by-829:

Used by
-------

.. _other-properties-643:

Other properties
----------------

============= ===============
**Aliases:**   total sulfur
**Mappings:**  mixs:tot_sulfur
============= ===============

Slot: train_line
================

The subway line name

URI: `nmdc:train_line <https://microbiomedata/meta/train_line>`__

.. _domain-and-range-830:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-867:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-837:

Children
--------

.. _used-by-830:

Used by
-------

.. _other-properties-644:

Other properties
----------------

============= ===============
**Aliases:**   train line
**Mappings:**  mixs:train_line
============= ===============

Slot: train_stat_loc
====================

The train station collection location

URI:
`nmdc:train_stat_loc <https://microbiomedata/meta/train_stat_loc>`__

.. _domain-and-range-831:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-868:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-838:

Children
--------

.. _used-by-831:

Used by
-------

.. _other-properties-645:

Other properties
----------------

============= =================================
**Aliases:**   train station collection location
**Mappings:**  mixs:train_stat_loc
============= =================================

Slot: train_stop_loc
====================

The train stop collection location

URI:
`nmdc:train_stop_loc <https://microbiomedata/meta/train_stop_loc>`__

.. _domain-and-range-832:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-869:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-839:

Children
--------

.. _used-by-832:

Used by
-------

.. _other-properties-646:

Other properties
----------------

============= ==============================
**Aliases:**   train stop collection location
**Mappings:**  mixs:train_stop_loc
============= ==============================

Slot: travel_out_six_month
==========================

Specification of the countries travelled in the last six months; can
include multiple travels

URI:
`nmdc:travel_out_six_month <https://microbiomedata/meta/travel_out_six_month>`__

.. _domain-and-range-833:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-870:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-840:

Children
--------

.. _used-by-833:

Used by
-------

.. _other-properties-647:

Other properties
----------------

============= =============================================
**Aliases:**   travel outside the country in last six months
**Mappings:**  mixs:travel_out_six_month
============= =============================================

Slot: trna_ext_software
=======================

Tools used for tRNA identification

URI:
`nmdc:trna_ext_software <https://microbiomedata/meta/trna_ext_software>`__

.. _domain-and-range-834:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-871:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-841:

Children
--------

.. _used-by-834:

Used by
-------

.. _other-properties-648:

Other properties
----------------

=============== ========================
**Aliases:**     tRNA extraction software
**Mappings:**    mixs:trna_ext_software
**In Subsets:**  sequencing
=============== ========================

Slot: trnas
===========

The total number of tRNAs identified from the SAG or MAG

URI: `nmdc:trnas <https://microbiomedata/meta/trnas>`__

.. _domain-and-range-835:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-872:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-842:

Children
--------

.. _used-by-835:

Used by
-------

.. _other-properties-649:

Other properties
----------------

=============== ==================================
**Aliases:**     number of standard tRNAs extracted
**Mappings:**    mixs:trnas
**In Subsets:**  sequencing
=============== ==================================

Slot: trophic_level
===================

Trophic levels are the feeding position in a food chain. Microbes can be
a range of producers (e.g. chemolithotroph)

URI: `nmdc:trophic_level <https://microbiomedata/meta/trophic_level>`__

.. _domain-and-range-836:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-873:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-843:

Children
--------

.. _used-by-836:

Used by
-------

.. _other-properties-650:

Other properties
----------------

=============== ============================
**Aliases:**     trophic level
**Mappings:**    mixs:trophic_level
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: turbidity
===============

Measure of the amount of cloudiness or haziness in water caused by
individual particles

URI: `nmdc:turbidity <https://microbiomedata/meta/turbidity>`__

.. _domain-and-range-837:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-874:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-844:

Children
--------

.. _used-by-837:

Used by
-------

.. _other-properties-651:

Other properties
----------------

============= ==============
**Aliases:**   turbidity
**Mappings:**  mixs:turbidity
============= ==============

Slot: tvdss_of_hcr_pressure
===========================

True vertical depth subsea (TVDSS) of the hydrocarbon resource where the
original pressure was measured (e.g. 1578 m )

URI:
`nmdc:tvdss_of_hcr_pressure <https://microbiomedata/meta/tvdss_of_hcr_pressure>`__

.. _domain-and-range-838:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-875:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-845:

Children
--------

.. _used-by-838:

Used by
-------

.. _other-properties-652:

Other properties
----------------

============= ==============================================
**Aliases:**   depth (TVDSS) of hydrocarbon resource pressure
**Mappings:**  mixs:tvdss_of_hcr_pressure
============= ==============================================

Slot: tvdss_of_hcr_temp
=======================

True vertical depth subsea (TVDSS) of the hydrocarbon resource where the
original temperature was measured (e.g. 1345 m)

URI:
`nmdc:tvdss_of_hcr_temp <https://microbiomedata/meta/tvdss_of_hcr_temp>`__

.. _domain-and-range-839:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-876:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-846:

Children
--------

.. _used-by-839:

Used by
-------

.. _other-properties-653:

Other properties
----------------

============= =================================================
**Aliases:**   depth (TVDSS) of hydrocarbon resource temperature
**Mappings:**  mixs:tvdss_of_hcr_temp
============= =================================================

Slot: twin_sibling
==================

Specification of twin sibling presence

URI: `nmdc:twin_sibling <https://microbiomedata/meta/twin_sibling>`__

.. _domain-and-range-840:

Domain and Range
----------------

None → 0..1 `BooleanValue <BooleanValue.md>`__

.. _parents-877:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-847:

Children
--------

.. _used-by-840:

Used by
-------

.. _other-properties-654:

Other properties
----------------

============= =====================
**Aliases:**   twin sibling presence
**Mappings:**  mixs:twin_sibling
============= =====================

Slot: typ_occup_density
=======================

Customary or normal density of occupants

URI:
`nmdc:typ_occup_density <https://microbiomedata/meta/typ_occup_density>`__

.. _domain-and-range-841:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-878:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-848:

Children
--------

.. _used-by-841:

Used by
-------

.. _other-properties-655:

Other properties
----------------

============= ========================
**Aliases:**   typical occupant density
**Mappings:**  mixs:typ_occup_density
============= ========================

.. _slot-type-3:

Slot: type
==========

An optional string that specifies the type object. This is used to allow
for searches for different kinds of objects.

URI: https://microbiomedata/schema/workflow_execution_activity/type

.. _domain-and-range-842:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-879:

Parents
-------

.. _children-849:

Children
--------

-  `attribute value➞type <attribute_value_type.md>`__
-  `workflow execution
   activity➞type <workflow_execution_activity_type.md>`__

.. _used-by-842:

Used by
-------

-  `MAGBin <MAGBin.md>`__

.. _other-properties-656:

Other properties
----------------

============= =================================================
**Examples:**  Example(value=‘nmdc:Biosample’, description=None)
\              Example(value=‘nmdc:Study’, description=None)
============= =================================================

Slot: unbinned contig num
=========================

URI:
https://microbiomedata/schema/workflow_execution_activity/unbinned_contig_num

.. _domain-and-range-843:

Domain and Range
----------------

None → 0..1 `Integer <types/Integer.md>`__

.. _parents-880:

Parents
-------

.. _children-850:

Children
--------

.. _used-by-843:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__

Slot: urine_collect_meth
========================

Specification of urine collection method

URI:
`nmdc:urine_collect_meth <https://microbiomedata/meta/urine_collect_meth>`__

.. _domain-and-range-844:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-881:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-851:

Children
--------

.. _used-by-844:

Used by
-------

.. _other-properties-657:

Other properties
----------------

============= =======================
**Aliases:**   urine/collection method
**Mappings:**  mixs:urine_collect_meth
============= =======================

Slot: url
=========

URI: https://microbiomedata/schema/workflow_execution_activity/url

.. _domain-and-range-845:

Domain and Range
----------------

None → 0..1 `String <types/String.md>`__

.. _parents-882:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-852:

Children
--------

.. _used-by-845:

Used by
-------

-  `ImageValue <ImageValue.md>`__

Slot: urogenit_disord
=====================

“History of urogenital disorders, can include multiple disorders”

URI:
`nmdc:urogenit_disord <https://microbiomedata/meta/urogenit_disord>`__

.. _domain-and-range-846:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-883:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-853:

Children
--------

.. _used-by-846:

Used by
-------

.. _other-properties-658:

Other properties
----------------

============= ====================
**Aliases:**   urogenital disorder
**Mappings:**  mixs:urogenit_disord
============= ====================

Slot: urogenit_tract_disor
==========================

History of urogenitaltract disorders; can include multiple disorders

URI:
`nmdc:urogenit_tract_disor <https://microbiomedata/meta/urogenit_tract_disor>`__

.. _domain-and-range-847:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-884:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-854:

Children
--------

.. _used-by-847:

Used by
-------

.. _other-properties-659:

Other properties
----------------

============= ===============================
**Aliases:**   urine/urogenital tract disorder
**Mappings:**  mixs:urogenit_tract_disor
============= ===============================

.. _slot-used-3:

Slot: used
==========

URI: https://microbiomedata/schema/workflow_execution_activity/used

.. _domain-and-range-848:

Domain and Range
----------------

`Activity <Activity.md>`__ → 0..1 `String <types/String.md>`__

.. _parents-885:

Parents
-------

.. _children-855:

Children
--------

-  `metabolomics analysis
   activity➞used <metabolomics_analysis_activity_used.md>`__
-  `metaproteomics analysis
   activity➞used <metaproteomics_analysis_activity_used.md>`__
-  `nom analysis activity➞used <nom_analysis_activity_used.md>`__

.. _used-by-848:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `Activity <Activity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-660:

Other properties
----------------

============= =========
**Mappings:**  prov:used
============= =========

Slot: ventilation_rate
======================

Ventilation rate of the system in the sampled premises

URI:
`nmdc:ventilation_rate <https://microbiomedata/meta/ventilation_rate>`__

.. _domain-and-range-849:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-886:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-856:

Children
--------

.. _used-by-849:

Used by
-------

.. _other-properties-661:

Other properties
----------------

============= =====================
**Aliases:**   ventilation rate
**Mappings:**  mixs:ventilation_rate
============= =====================

Slot: ventilation_type
======================

Ventilation system used in the sampled premises

URI:
`nmdc:ventilation_type <https://microbiomedata/meta/ventilation_type>`__

.. _domain-and-range-850:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-887:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-857:

Children
--------

.. _used-by-850:

Used by
-------

.. _other-properties-662:

Other properties
----------------

============= =====================
**Aliases:**   ventilation type
**Mappings:**  mixs:ventilation_type
============= =====================

Slot: vfa
=========

Concentration of Volatile Fatty Acids in the sample

URI: `nmdc:vfa <https://microbiomedata/meta/vfa>`__

.. _domain-and-range-851:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-888:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-858:

Children
--------

.. _used-by-851:

Used by
-------

.. _other-properties-663:

Other properties
----------------

============= ====================
**Aliases:**   volatile fatty acids
**Mappings:**  mixs:vfa
============= ====================

Slot: vfa_fw
============

Original volatile fatty acid concentration in the hydrocarbon resource

URI: `nmdc:vfa_fw <https://microbiomedata/meta/vfa_fw>`__

.. _domain-and-range-852:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-889:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-859:

Children
--------

.. _used-by-852:

Used by
-------

.. _other-properties-664:

Other properties
----------------

============= ======================
**Aliases:**   vfa in formation water
**Mappings:**  mixs:vfa_fw
============= ======================

Slot: vir_ident_software
========================

“Tool(s) used for the identification of UViG as a viral genome, software
or protocol name including version number, parameters, and cutoffs used”

URI:
`nmdc:vir_ident_software <https://microbiomedata/meta/vir_ident_software>`__

.. _domain-and-range-853:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-890:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-860:

Children
--------

.. _used-by-853:

Used by
-------

.. _other-properties-665:

Other properties
----------------

=============== =============================
**Aliases:**     viral identification software
**Mappings:**    mixs:vir_ident_software
**In Subsets:**  sequencing
=============== =============================

Slot: virus_enrich_appr
=======================

“List of approaches used to enrich the sample for viruses, if any”

URI:
`nmdc:virus_enrich_appr <https://microbiomedata/meta/virus_enrich_appr>`__

.. _domain-and-range-854:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-891:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-861:

Children
--------

.. _used-by-854:

Used by
-------

.. _other-properties-666:

Other properties
----------------

=============== ============================
**Aliases:**     virus enrichment approach
**Mappings:**    mixs:virus_enrich_appr
**In Subsets:**  nucleic acid sequence source
=============== ============================

Slot: vis_media
===============

The building visual media

URI: `nmdc:vis_media <https://microbiomedata/meta/vis_media>`__

.. _domain-and-range-855:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-892:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-862:

Children
--------

.. _used-by-855:

Used by
-------

.. _other-properties-667:

Other properties
----------------

============= ==============
**Aliases:**   visual media
**Mappings:**  mixs:vis_media
============= ==============

Slot: viscosity
===============

A measure of oil’s resistanceB to gradual deformation byB shear stressB
orB tensile stress (e.g. 3.5 cp; 100 B0C)

URI: `nmdc:viscosity <https://microbiomedata/meta/viscosity>`__

.. _domain-and-range-856:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-893:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-863:

Children
--------

.. _used-by-856:

Used by
-------

.. _other-properties-668:

Other properties
----------------

============= ==============
**Aliases:**   viscosity
**Mappings:**  mixs:viscosity
============= ==============

Slot: volatile_org_comp
=======================

Concentration of carbon-based chemicals that easily evaporate at room
temperature; can report multiple volatile organic compounds by entering
numeric values preceded by name of compound

URI:
`nmdc:volatile_org_comp <https://microbiomedata/meta/volatile_org_comp>`__

.. _domain-and-range-857:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-894:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-864:

Children
--------

.. _used-by-857:

Used by
-------

.. _other-properties-669:

Other properties
----------------

============= ==========================
**Aliases:**   volatile organic compounds
**Mappings:**  mixs:volatile_org_comp
============= ==========================

Slot: votu_class_appr
=====================

“Cutoffs and approach used when clustering new UViGs in Rspecies-levelS
vOTUs. Note that results from standard 95% ANI / 85% AF clustering
should be provided alongside vOTUS defined from another set of
thresholds, even if the latter are the ones primarily used during the
analysis”

URI:
`nmdc:votu_class_appr <https://microbiomedata/meta/votu_class_appr>`__

.. _domain-and-range-858:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-895:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-865:

Children
--------

.. _used-by-858:

Used by
-------

.. _other-properties-670:

Other properties
----------------

=============== ============================
**Aliases:**     vOTU classification approach
**Mappings:**    mixs:votu_class_appr
**In Subsets:**  sequencing
=============== ============================

Slot: votu_db
=============

“Reference database (i.e. sequences not generated as part of the current
study) used to cluster new genomes in”“species-level”” vOTUs, if any”

URI: `nmdc:votu_db <https://microbiomedata/meta/votu_db>`__

.. _domain-and-range-859:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-896:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-866:

Children
--------

.. _used-by-859:

Used by
-------

.. _other-properties-671:

Other properties
----------------

=============== =============
**Aliases:**     vOTU database
**Mappings:**    mixs:votu_db
**In Subsets:**  sequencing
=============== =============

Slot: votu_seq_comp_appr
========================

“Tool and thresholds used to compare sequences when
computing”“species-level”” vOTUs”

URI:
`nmdc:votu_seq_comp_appr <https://microbiomedata/meta/votu_seq_comp_appr>`__

.. _domain-and-range-860:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-897:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-867:

Children
--------

.. _used-by-860:

Used by
-------

.. _other-properties-672:

Other properties
----------------

=============== =================================
**Aliases:**     vOTU sequence comparison approach
**Mappings:**    mixs:votu_seq_comp_appr
**In Subsets:**  sequencing
=============== =================================

Slot: wall_area
===============

The total area of the sampled room’s walls

URI: `nmdc:wall_area <https://microbiomedata/meta/wall_area>`__

.. _domain-and-range-861:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-898:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-868:

Children
--------

.. _used-by-861:

Used by
-------

.. _other-properties-673:

Other properties
----------------

============= ==============
**Aliases:**   wall area
**Mappings:**  mixs:wall_area
============= ==============

Slot: wall_const_type
=====================

The building class of the wall defined by the composition of the
building elements and fire-resistance rating.

URI:
`nmdc:wall_const_type <https://microbiomedata/meta/wall_const_type>`__

.. _domain-and-range-862:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-899:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-869:

Children
--------

.. _used-by-862:

Used by
-------

.. _other-properties-674:

Other properties
----------------

============= ======================
**Aliases:**   wall construction type
**Mappings:**  mixs:wall_const_type
============= ======================

Slot: wall_finish_mat
=====================

The material utilized to finish the outer most layer of the wall

URI:
`nmdc:wall_finish_mat <https://microbiomedata/meta/wall_finish_mat>`__

.. _domain-and-range-863:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-900:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-870:

Children
--------

.. _used-by-863:

Used by
-------

.. _other-properties-675:

Other properties
----------------

============= ====================
**Aliases:**   wall finish material
**Mappings:**  mixs:wall_finish_mat
============= ====================

Slot: wall_height
=================

The average height of the walls in the sampled room

URI: `nmdc:wall_height <https://microbiomedata/meta/wall_height>`__

.. _domain-and-range-864:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-901:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-871:

Children
--------

.. _used-by-864:

Used by
-------

.. _other-properties-676:

Other properties
----------------

============= ================
**Aliases:**   wall height
**Mappings:**  mixs:wall_height
============= ================

Slot: wall_loc
==============

The relative location of the wall within the room

URI: `nmdc:wall_loc <https://microbiomedata/meta/wall_loc>`__

.. _domain-and-range-865:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-902:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-872:

Children
--------

.. _used-by-865:

Used by
-------

.. _other-properties-677:

Other properties
----------------

============= =============
**Aliases:**   wall location
**Mappings:**  mixs:wall_loc
============= =============

Slot: wall_surf_treatment
=========================

The surface treatment of interior wall

URI:
`nmdc:wall_surf_treatment <https://microbiomedata/meta/wall_surf_treatment>`__

.. _domain-and-range-866:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-903:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-873:

Children
--------

.. _used-by-866:

Used by
-------

.. _other-properties-678:

Other properties
----------------

============= ========================
**Aliases:**   wall surface treatment
**Mappings:**  mixs:wall_surf_treatment
============= ========================

Slot: wall_texture
==================

“The feel, appearance, or consistency of a wall surface”

URI: `nmdc:wall_texture <https://microbiomedata/meta/wall_texture>`__

.. _domain-and-range-867:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-904:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-874:

Children
--------

.. _used-by-867:

Used by
-------

.. _other-properties-679:

Other properties
----------------

============= =================
**Aliases:**   wall texture
**Mappings:**  mixs:wall_texture
============= =================

Slot: wall_thermal_mass
=======================

The ability of the wall to provide inertia against temperature
fluctuations. Generally this means concrete or concrete block that is
either exposed or covered only with paint

URI:
`nmdc:wall_thermal_mass <https://microbiomedata/meta/wall_thermal_mass>`__

.. _domain-and-range-868:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-905:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-875:

Children
--------

.. _used-by-868:

Used by
-------

.. _other-properties-680:

Other properties
----------------

============= ======================
**Aliases:**   wall thermal mass
**Mappings:**  mixs:wall_thermal_mass
============= ======================

Slot: wall_water_mold
=====================

Signs of the presence of mold or mildew on a wall

URI:
`nmdc:wall_water_mold <https://microbiomedata/meta/wall_water_mold>`__

.. _domain-and-range-869:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-906:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-876:

Children
--------

.. _used-by-869:

Used by
-------

.. _other-properties-681:

Other properties
----------------

============= ========================
**Aliases:**   wall signs of water/mold
**Mappings:**  mixs:wall_water_mold
============= ========================

Slot: was associated with
=========================

URI:
https://microbiomedata/schema/workflow_execution_activity/was_associated_with

.. _domain-and-range-870:

Domain and Range
----------------

None → 0..1 `Agent <Agent.md>`__

.. _parents-907:

Parents
-------

.. _children-877:

Children
--------

-  `workflow execution activity➞was associated
   with <workflow_execution_activity_was_associated_with.md>`__

.. _used-by-870:

Used by
-------

-  `Activity <Activity.md>`__

.. _other-properties-682:

Other properties
----------------

============= ======================
**Mappings:**  prov:wasAssociatedWith
============= ======================

.. _slot-was-generated-by-1:

Slot: was generated by
======================

URI:
https://microbiomedata/schema/workflow_execution_activity/was_generated_by

.. _domain-and-range-871:

Domain and Range
----------------

None → 0..1 `Activity <Activity.md>`__

.. _parents-908:

Parents
-------

.. _children-878:

Children
--------

.. _used-by-871:

Used by
-------

-  `AttributeValue <AttributeValue.md>`__
-  `BooleanValue <BooleanValue.md>`__
-  `ControlledTermValue <ControlledTermValue.md>`__
-  `GeolocationValue <GeolocationValue.md>`__
-  `ImageValue <ImageValue.md>`__
-  `IntegerValue <IntegerValue.md>`__
-  `PersonValue <PersonValue.md>`__
-  `QuantityValue <QuantityValue.md>`__
-  `TextValue <TextValue.md>`__
-  `TimestampValue <TimestampValue.md>`__
-  `UrlValue <UrlValue.md>`__

.. _other-properties-683:

Other properties
----------------

============= ===================
**Mappings:**  prov:wasGeneratedBy
============= ===================

Slot: was informed by
=====================

URI:
https://microbiomedata/schema/workflow_execution_activity/was_informed_by

.. _domain-and-range-872:

Domain and Range
----------------

None → 0..1 `Activity <Activity.md>`__

.. _parents-909:

Parents
-------

.. _children-879:

Children
--------

-  `workflow execution activity➞was informed
   by <workflow_execution_activity_was_informed_by.md>`__

.. _used-by-872:

Used by
-------

-  `Activity <Activity.md>`__
-  `Agent <Agent.md>`__

.. _other-properties-684:

Other properties
----------------

============= ==================
**Mappings:**  prov:wasInformedBy
============= ==================

Slot: wastewater_type
=====================

“The origin of wastewater such as human waste, rainfall, storm drains,
etc.”

URI:
`nmdc:wastewater_type <https://microbiomedata/meta/wastewater_type>`__

.. _domain-and-range-873:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-910:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-880:

Children
--------

.. _used-by-873:

Used by
-------

.. _other-properties-685:

Other properties
----------------

============= ====================
**Aliases:**   wastewater type
**Mappings:**  mixs:wastewater_type
============= ====================

Slot: water_content
===================

Water content measurement

URI: `nmdc:water_content <https://microbiomedata/meta/water_content>`__

.. _domain-and-range-874:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-911:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-881:

Children
--------

.. _used-by-874:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-686:

Other properties
----------------

============= ==================
**Aliases:**   water content
**Mappings:**  mixs:water_content
============= ==================

Slot: water_content_soil_meth
=============================

Reference or method used in determining the water content of soil

URI:
`nmdc:water_content_soil_meth <https://microbiomedata/meta/water_content_soil_meth>`__

.. _domain-and-range-875:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-912:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-882:

Children
--------

.. _used-by-875:

Used by
-------

-  `Biosample <Biosample.md>`__

.. _other-properties-687:

Other properties
----------------

============= ============================
**Aliases:**   water content method
**Mappings:**  mixs:water_content_soil_meth
============= ============================

Slot: water_current
===================

Measurement of magnitude and direction of flow within a fluid

URI: `nmdc:water_current <https://microbiomedata/meta/water_current>`__

.. _domain-and-range-876:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-913:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-883:

Children
--------

.. _used-by-876:

Used by
-------

.. _other-properties-688:

Other properties
----------------

============= ==================
**Aliases:**   water current
**Mappings:**  mixs:water_current
============= ==================

Slot: water_cut
===============

Current amount of water (%) in a produced fluid stream; or the average
of the combined streams

URI: `nmdc:water_cut <https://microbiomedata/meta/water_cut>`__

.. _domain-and-range-877:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-914:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-884:

Children
--------

.. _used-by-877:

Used by
-------

.. _other-properties-689:

Other properties
----------------

============= ==============
**Aliases:**   water cut
**Mappings:**  mixs:water_cut
============= ==============

Slot: water_feat_size
=====================

The size of the water feature

URI:
`nmdc:water_feat_size <https://microbiomedata/meta/water_feat_size>`__

.. _domain-and-range-878:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-915:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-885:

Children
--------

.. _used-by-878:

Used by
-------

.. _other-properties-690:

Other properties
----------------

============= ====================
**Aliases:**   water feature size
**Mappings:**  mixs:water_feat_size
============= ====================

Slot: water_feat_type
=====================

The type of water feature present within the building being sampled

URI:
`nmdc:water_feat_type <https://microbiomedata/meta/water_feat_type>`__

.. _domain-and-range-879:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-916:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-886:

Children
--------

.. _used-by-879:

Used by
-------

.. _other-properties-691:

Other properties
----------------

============= ====================
**Aliases:**   water feature type
**Mappings:**  mixs:water_feat_type
============= ====================

Slot: water_production_rate
===========================

Water production rates per well (e.g. 987 m3 / day)

URI:
`nmdc:water_production_rate <https://microbiomedata/meta/water_production_rate>`__

.. _domain-and-range-880:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-917:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-887:

Children
--------

.. _used-by-880:

Used by
-------

.. _other-properties-692:

Other properties
----------------

============= ==========================
**Aliases:**   water production rate
**Mappings:**  mixs:water_production_rate
============= ==========================

Slot: water_temp_regm
=====================

“Information about treatment involving an exposure to water with varying
degree of temperature, treatment regimen including how many times the
treatment was repeated, how long each treatment lasted, and the start
and end time of the entire treatment; can include multiple regimens”

URI:
`nmdc:water_temp_regm <https://microbiomedata/meta/water_temp_regm>`__

.. _domain-and-range-881:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-918:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-888:

Children
--------

.. _used-by-881:

Used by
-------

.. _other-properties-693:

Other properties
----------------

============= =========================
**Aliases:**   water temperature regimen
**Mappings:**  mixs:water_temp_regm
============= =========================

Slot: watering_regm
===================

“Information about treatment involving an exposure to watering
frequencies, treatment regimen including how many times the treatment
was repeated, how long each treatment lasted, and the start and end time
of the entire treatment; can include multiple regimens”

URI: `nmdc:watering_regm <https://microbiomedata/meta/watering_regm>`__

.. _domain-and-range-882:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-919:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-889:

Children
--------

.. _used-by-882:

Used by
-------

.. _other-properties-694:

Other properties
----------------

============= ==================
**Aliases:**   watering regimen
**Mappings:**  mixs:watering_regm
============= ==================

Slot: websites
==============

A list of websites that are assocatiated with the entity.

URI: https://microbiomedata/schema/workflow_execution_activity/websites

.. _domain-and-range-883:

Domain and Range
----------------

None → 0..\* `String <types/String.md>`__

.. _parents-920:

Parents
-------

.. _children-890:

Children
--------

.. _used-by-883:

Used by
-------

-  `PersonValue <PersonValue.md>`__

Slot: weekday
=============

The day of the week when sampling occurred

URI: `nmdc:weekday <https://microbiomedata/meta/weekday>`__

.. _domain-and-range-884:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-921:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-891:

Children
--------

.. _used-by-884:

Used by
-------

.. _other-properties-695:

Other properties
----------------

============= ============
**Aliases:**   weekday
**Mappings:**  mixs:weekday
============= ============

Slot: weight_loss_3_month
=========================

“Specification of weight loss in the last three months, if yes should be
further specified to include amount of weight loss”

URI:
`nmdc:weight_loss_3_month <https://microbiomedata/meta/weight_loss_3_month>`__

.. _domain-and-range-885:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-922:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-892:

Children
--------

.. _used-by-885:

Used by
-------

.. _other-properties-696:

Other properties
----------------

============= ================================
**Aliases:**   weight loss in last three months
**Mappings:**  mixs:weight_loss_3_month
============= ================================

Slot: wga_amp_appr
==================

Method used to amplify genomic DNA in preparation for sequencing

URI: `nmdc:wga_amp_appr <https://microbiomedata/meta/wga_amp_appr>`__

.. _domain-and-range-886:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-923:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-893:

Children
--------

.. _used-by-886:

Used by
-------

.. _other-properties-697:

Other properties
----------------

=============== ==========================
**Aliases:**     WGA amplification approach
**Mappings:**    mixs:wga_amp_appr
**In Subsets:**  sequencing
=============== ==========================

Slot: wga_amp_kit
=================

Kit used to amplify genomic DNA in preparation for sequencing

URI: `nmdc:wga_amp_kit <https://microbiomedata/meta/wga_amp_kit>`__

.. _domain-and-range-887:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-924:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-894:

Children
--------

.. _used-by-887:

Used by
-------

.. _other-properties-698:

Other properties
----------------

=============== =====================
**Aliases:**     WGA amplification kit
**Mappings:**    mixs:wga_amp_kit
**In Subsets:**  sequencing
=============== =====================

Slot: win
=========

A unique identifier of a well or wellbore. This is part of the Global
Framework for Well Identification initiative which is compiled by the
Professional Petroleum Data Management Association (PPDM) in an effort
to improve well identification systems. (Supporting information:
https://ppdm.org/ and http://dl.ppdm.org/dl/690)

URI: `nmdc:win <https://microbiomedata/meta/win>`__

.. _domain-and-range-888:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-925:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-895:

Children
--------

.. _used-by-888:

Used by
-------

.. _other-properties-699:

Other properties
----------------

============= ==========================
**Aliases:**   well identification number
**Mappings:**  mixs:win
============= ==========================

Slot: wind_direction
====================

Wind direction is the direction from which a wind originates

URI:
`nmdc:wind_direction <https://microbiomedata/meta/wind_direction>`__

.. _domain-and-range-889:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-926:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-896:

Children
--------

.. _used-by-889:

Used by
-------

.. _other-properties-700:

Other properties
----------------

============= ===================
**Aliases:**   wind direction
**Mappings:**  mixs:wind_direction
============= ===================

Slot: wind_speed
================

Speed of wind measured at the time of sampling

URI: `nmdc:wind_speed <https://microbiomedata/meta/wind_speed>`__

.. _domain-and-range-890:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-927:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-897:

Children
--------

.. _used-by-890:

Used by
-------

.. _other-properties-701:

Other properties
----------------

============= ===============
**Aliases:**   wind speed
**Mappings:**  mixs:wind_speed
============= ===============

Slot: window_cond
=================

The physical condition of the window at the time of sampling

URI: `nmdc:window_cond <https://microbiomedata/meta/window_cond>`__

.. _domain-and-range-891:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-928:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-898:

Children
--------

.. _used-by-891:

Used by
-------

.. _other-properties-702:

Other properties
----------------

============= ================
**Aliases:**   window condition
**Mappings:**  mixs:window_cond
============= ================

Slot: window_cover
==================

The type of window covering

URI: `nmdc:window_cover <https://microbiomedata/meta/window_cover>`__

.. _domain-and-range-892:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-929:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-899:

Children
--------

.. _used-by-892:

Used by
-------

.. _other-properties-703:

Other properties
----------------

============= =================
**Aliases:**   window covering
**Mappings:**  mixs:window_cover
============= =================

Slot: window_horiz_pos
======================

The horizontal position of the window on the wall

URI:
`nmdc:window_horiz_pos <https://microbiomedata/meta/window_horiz_pos>`__

.. _domain-and-range-893:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-930:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-900:

Children
--------

.. _used-by-893:

Used by
-------

.. _other-properties-704:

Other properties
----------------

============= ==========================
**Aliases:**   window horizontal position
**Mappings:**  mixs:window_horiz_pos
============= ==========================

Slot: window_loc
================

The relative location of the window within the room

URI: `nmdc:window_loc <https://microbiomedata/meta/window_loc>`__

.. _domain-and-range-894:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-931:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-901:

Children
--------

.. _used-by-894:

Used by
-------

.. _other-properties-705:

Other properties
----------------

============= ===============
**Aliases:**   window location
**Mappings:**  mixs:window_loc
============= ===============

Slot: window_mat
================

The type of material used to finish a window

URI: `nmdc:window_mat <https://microbiomedata/meta/window_mat>`__

.. _domain-and-range-895:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-932:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-902:

Children
--------

.. _used-by-895:

Used by
-------

.. _other-properties-706:

Other properties
----------------

============= ===============
**Aliases:**   window material
**Mappings:**  mixs:window_mat
============= ===============

Slot: window_open_freq
======================

The number of times windows are opened per week

URI:
`nmdc:window_open_freq <https://microbiomedata/meta/window_open_freq>`__

.. _domain-and-range-896:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-933:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-903:

Children
--------

.. _used-by-896:

Used by
-------

.. _other-properties-707:

Other properties
----------------

============= =====================
**Aliases:**   window open frequency
**Mappings:**  mixs:window_open_freq
============= =====================

Slot: window_size
=================

The window’s length and width

URI: `nmdc:window_size <https://microbiomedata/meta/window_size>`__

.. _domain-and-range-897:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-934:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-904:

Children
--------

.. _used-by-897:

Used by
-------

.. _other-properties-708:

Other properties
----------------

============= ================
**Aliases:**   window area/size
**Mappings:**  mixs:window_size
============= ================

Slot: window_status
===================

Defines whether the windows were open or closed during environmental
testing

URI: `nmdc:window_status <https://microbiomedata/meta/window_status>`__

.. _domain-and-range-898:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-935:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-905:

Children
--------

.. _used-by-898:

Used by
-------

.. _other-properties-709:

Other properties
----------------

============= ==================
**Aliases:**   window status
**Mappings:**  mixs:window_status
============= ==================

Slot: window_type
=================

The type of windows

URI: `nmdc:window_type <https://microbiomedata/meta/window_type>`__

.. _domain-and-range-899:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-936:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-906:

Children
--------

.. _used-by-899:

Used by
-------

.. _other-properties-710:

Other properties
----------------

============= ================
**Aliases:**   window type
**Mappings:**  mixs:window_type
============= ================

Slot: window_vert_pos
=====================

The vertical position of the window on the wall

URI:
`nmdc:window_vert_pos <https://microbiomedata/meta/window_vert_pos>`__

.. _domain-and-range-900:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-937:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-907:

Children
--------

.. _used-by-900:

Used by
-------

.. _other-properties-711:

Other properties
----------------

============= ========================
**Aliases:**   window vertical position
**Mappings:**  mixs:window_vert_pos
============= ========================

Slot: window_water_mold
=======================

Signs of the presence of mold or mildew on the window.

URI:
`nmdc:window_water_mold <https://microbiomedata/meta/window_water_mold>`__

.. _domain-and-range-901:

Domain and Range
----------------

None → 0..1 `TextValue <TextValue.md>`__

.. _parents-938:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-908:

Children
--------

.. _used-by-901:

Used by
-------

.. _other-properties-712:

Other properties
----------------

============= ==========================
**Aliases:**   window signs of water/mold
**Mappings:**  mixs:window_water_mold
============= ==========================

.. _slot-ended-at-time-1:

Slot: ended at time
===================

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_ended_at_time

.. _domain-and-range-902:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..1
`String <types/String.md>`__

.. _parents-939:

Parents
-------

-  is_a: `ended at time <ended_at_time.md>`__

.. _children-909:

Children
--------

.. _used-by-902:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-713:

Other properties
----------------

============= ================
**Mappings:**  prov:endedAtTime
============= ================

.. _slot-execution-resource-1:

Slot: execution resource
========================

Example: NERSC-Cori

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_execution_resource

.. _domain-and-range-903:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..1
`String <types/String.md>`__

.. _parents-940:

Parents
-------

-  is_a: `execution resource <execution_resource.md>`__

.. _children-910:

Children
--------

.. _used-by-903:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _slot-git-url-1:

Slot: git url
=============

Example:
https://github.com/microbiomedata/mg_annotation/releases/tag/0.1

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_git_url

.. _domain-and-range-904:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..1
`String <types/String.md>`__

.. _parents-941:

Parents
-------

-  is_a: `git url <git_url.md>`__

.. _children-911:

Children
--------

.. _used-by-904:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _slot-has-input-4:

Slot: has input
===============

An input to a process.

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_has_input

.. _domain-and-range-905:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..\*
`NamedThing <NamedThing.md>`__

.. _parents-942:

Parents
-------

-  is_a: `has input <has_input.md>`__

.. _children-912:

Children
--------

-  `read QC analysis activity➞has
   input <read_QC_analysis_activity_has_input.md>`__

.. _used-by-905:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-714:

Other properties
----------------

============ =====
**Aliases:**  input
============ =====

.. _slot-has-output-2:

Slot: has output
================

An output biosample to a processing step

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_has_output

.. _domain-and-range-906:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..\*
`NamedThing <NamedThing.md>`__

.. _parents-943:

Parents
-------

-  is_a: `has output <has_output.md>`__

.. _children-913:

Children
--------

-  `read QC analysis activity➞has
   output <read_QC_analysis_activity_has_output.md>`__

.. _used-by-906:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-715:

Other properties
----------------

============ ======
**Aliases:**  output
============ ======

.. _slot-started-at-time-1:

Slot: started at time
=====================

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_started_at_time

.. _domain-and-range-907:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..1
`String <types/String.md>`__

.. _parents-944:

Parents
-------

-  is_a: `started at time <started_at_time.md>`__

.. _children-914:

Children
--------

.. _used-by-907:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-716:

Other properties
----------------

============= ==================
**Mappings:**  prov:startedAtTime
============= ==================

.. _slot-type-4:

Slot: type
==========

An optional string that specifies the type object. This is used to allow
for searches for different kinds of objects.

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_type

.. _domain-and-range-908:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..1
`String <types/String.md>`__

.. _parents-945:

Parents
-------

-  is_a: `type <type.md>`__

.. _children-915:

Children
--------

.. _used-by-908:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-717:

Other properties
----------------

============= =================================================
**Examples:**  Example(value=‘nmdc:Biosample’, description=None)
\              Example(value=‘nmdc:Study’, description=None)
============= =================================================

.. _slot-was-associated-with-1:

Slot: was associated with
=========================

the agent/entity associated with the generation of the file

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_was_associated_with

.. _domain-and-range-909:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 0..1
`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _parents-946:

Parents
-------

-  is_a: `was associated with <was_associated_with.md>`__

.. _children-916:

Children
--------

.. _used-by-909:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-718:

Other properties
----------------

============= ======================
**Mappings:**  prov:wasAssociatedWith
============= ======================

.. _slot-was-informed-by-1:

Slot: was informed by
=====================

URI:
https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_was_informed_by

.. _domain-and-range-910:

Domain and Range
----------------

`WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__ → 1..1
`Activity <Activity.md>`__

.. _parents-947:

Parents
-------

-  is_a: `was informed by <was_informed_by.md>`__

.. _children-917:

Children
--------

.. _used-by-910:

Used by
-------

-  `MAGsAnalysisActivity <MAGsAnalysisActivity.md>`__
-  `MetabolomicsAnalysisActivity <MetabolomicsAnalysisActivity.md>`__
-  `MetagenomeAnnotationActivity <MetagenomeAnnotationActivity.md>`__
-  `MetagenomeAssembly <MetagenomeAssembly.md>`__
-  `MetaproteomicsAnalysisActivity <MetaproteomicsAnalysisActivity.md>`__
-  `MetatranscriptomeActivity <MetatranscriptomeActivity.md>`__
-  `MetatranscriptomeAnnotationActivity <MetatranscriptomeAnnotationActivity.md>`__
-  `MetatranscriptomeAssembly <MetatranscriptomeAssembly.md>`__
-  `NomAnalysisActivity <NomAnalysisActivity.md>`__
-  `ReadQCAnalysisActivity <ReadQCAnalysisActivity.md>`__
-  `ReadBasedAnalysisActivity <ReadBasedAnalysisActivity.md>`__
-  `WorkflowExecutionActivity <WorkflowExecutionActivity.md>`__

.. _other-properties-719:

Other properties
----------------

============= ==================
**Mappings:**  prov:wasInformedBy
============= ==================

Slot: xylene
============

Concentration of xylene in the sample

URI: `nmdc:xylene <https://microbiomedata/meta/xylene>`__

.. _domain-and-range-911:

Domain and Range
----------------

None → 0..1 `QuantityValue <QuantityValue.md>`__

.. _parents-948:

Parents
-------

-  is_a: `attribute <attribute.md>`__

.. _children-918:

Children
--------

.. _used-by-911:

Used by
-------

.. _other-properties-720:

Other properties
----------------

============= ===========
**Aliases:**   xylene
**Mappings:**  mixs:xylene
============= ===========

.. |img| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BAgent%5D,%5BAgent%5D%3Cwas%20associated%20with%200..1-++%5BActivity%7Cid:string;name:string%20%3F;started_at_time:string%20%3F;ended_at_time:string%20%3F;used:string%20%3F%5D,%5BActivity%5D%3Cwas%20informed%20by%200..1-%20%5BActivity%5D,%5BAttributeValue%5D-%20was%20generated%20by%200..1%3E%5BActivity%5D,%5BWorkflowExecutionActivity%5D-%20was%20informed%20by%201..1%3E%5BActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BAttributeValue%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BAgent%5D,%5BAgent%5D%3Cwas%20associated%20with%200..1-++%5BActivity%7Cid:string;name:string%20%3F;started_at_time:string%20%3F;ended_at_time:string%20%3F;used:string%20%3F%5D,%5BActivity%5D%3Cwas%20informed%20by%200..1-%20%5BActivity%5D,%5BAttributeValue%5D-%20was%20generated%20by%200..1%3E%5BActivity%5D,%5BWorkflowExecutionActivity%5D-%20was%20informed%20by%201..1%3E%5BActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BAttributeValue%5D
.. |image1| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BActivity%5D%3Cwas%20informed%20by%200..1-%20%5BAgent%5D,%5BAgent%5D%3Cacted%20on%20behalf%20of%200..1-++%5BAgent%5D,%5BActivity%5D++-%20was%20associated%20with%200..1%3E%5BAgent%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BActivity%5D%3Cwas%20informed%20by%200..1-%20%5BAgent%5D,%5BAgent%5D%3Cacted%20on%20behalf%20of%200..1-++%5BAgent%5D,%5BActivity%5D++-%20was%20associated%20with%200..1%3E%5BAgent%5D,%5BActivity%5D
.. |image2| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BUrlValue%5D,%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BIntegerValue%5D,%5BImageValue%5D,%5BGeolocationValue%5D,%5BControlledTermValue%5D,%5BBooleanValue%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BAttributeValue%7Chas_raw_value:string%20%3F;type:string%20%3F%5D,%5BAttributeValue%5D%5E-%5BUrlValue%5D,%5BAttributeValue%5D%5E-%5BTimestampValue%5D,%5BAttributeValue%5D%5E-%5BTextValue%5D,%5BAttributeValue%5D%5E-%5BQuantityValue%5D,%5BAttributeValue%5D%5E-%5BPersonValue%5D,%5BAttributeValue%5D%5E-%5BIntegerValue%5D,%5BAttributeValue%5D%5E-%5BImageValue%5D,%5BAttributeValue%5D%5E-%5BGeolocationValue%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D%5E-%5BBooleanValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BUrlValue%5D,%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BIntegerValue%5D,%5BImageValue%5D,%5BGeolocationValue%5D,%5BControlledTermValue%5D,%5BBooleanValue%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BAttributeValue%7Chas_raw_value:string%20%3F;type:string%20%3F%5D,%5BAttributeValue%5D%5E-%5BUrlValue%5D,%5BAttributeValue%5D%5E-%5BTimestampValue%5D,%5BAttributeValue%5D%5E-%5BTextValue%5D,%5BAttributeValue%5D%5E-%5BQuantityValue%5D,%5BAttributeValue%5D%5E-%5BPersonValue%5D,%5BAttributeValue%5D%5E-%5BIntegerValue%5D,%5BAttributeValue%5D%5E-%5BImageValue%5D,%5BAttributeValue%5D%5E-%5BGeolocationValue%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D%5E-%5BBooleanValue%5D,%5BActivity%5D
.. |image3| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BGeolocationValue%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BQuantityValue%5D%3Csubsurface_depth2%200..1-++%5BBiosample%7Ctype:string%20%3F;ecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;add_date:string%20%3F;community:string%20%3F;habitat:string%20%3F;host_name:string%20%3F;identifier:string%20%3F;location:string%20%3F;mod_date:string%20%3F;ncbi_taxonomy_name:string%20%3F;proport_woa_temperature:string%20%3F;salinity_category:string%20%3F;sample_collection_site:string%20%3F;soluble_iron_micromol:string%20%3F;GOLD_sample_identifiers:external_identifier%20*;INSDC_biosample_identifiers:external_identifier%20*;INSDC_secondary_sample_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BQuantityValue%5D%3Csubsurface_depth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth2%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cwater_content_soil_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cwater_content%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_phosp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_nitro_content_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_nitro_content%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_org_c_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_diss_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_depth_water_col%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctidal_stage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctillage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctexture_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctexture%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctemp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfate%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cstore_cond%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csodium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_aspect%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_gradient%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_up%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_low%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csieving%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_store_temp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_loc%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_dur%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Csamp_mat_process%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_collect_device%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csalinity_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Credox_potential%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprofile_position%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpressure%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpotassium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cpool_dna_extracts%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphosplipid_fatt_acid%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphaeopigments%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cph_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cph%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpetroleum_hydrocarb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cperturbation%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpart_org_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Coxy_stat_samp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corganism_count%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_matter%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrite%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrate%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cn_alkanes%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmisc_param%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cmicrobial_biomass_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmicrobial_biomass%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_peak_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmagnesium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_climate_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_class_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_addit_analys%200..1-++%5BBiosample%5D,%5BGeolocationValue%5D%3Clat_lon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cheavy_metals_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cheavy_metals%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cglucosidase_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cgeo_loc_name%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cflooding%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cfire%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cfao_class%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cextreme_event%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_medium%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_local_scale%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_broad_scale%201..1-++%5BBiosample%5D,%5BTextValue%5D%3Cenv_package%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Celev%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cdrainage_class%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_oxygen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_phosp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_hydrogen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_carb_dioxide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdensity%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccrop_rotation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_land_use%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Ccollection_date%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchlorophyll%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchloride%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cchem_administration%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccarb_nitro_ratio%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccalcium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbromide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbishomohopanol%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbacteria_carb_prod%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cammonium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Caminopept_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cal_sat_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cal_sat%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkyl_diethers%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Calkalinity_method%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cagrochem_addition%200..1-++%5BBiosample%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BBiosample%5D,%5BBiosampleProcessing%5D-%20has%20input%200..*%3E%5BBiosample%5D,%5BDatabase%5D++-%20biosample%20set%200..*%3E%5BBiosample%5D,%5BOmicsProcessing%5D-%20has%20input(i)%201..*%3E%5BBiosample%5D,%5BNamedThing%5D%5E-%5BBiosample%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BGeolocationValue%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BQuantityValue%5D%3Csubsurface_depth2%200..1-++%5BBiosample%7Ctype:string%20%3F;ecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;add_date:string%20%3F;community:string%20%3F;habitat:string%20%3F;host_name:string%20%3F;identifier:string%20%3F;location:string%20%3F;mod_date:string%20%3F;ncbi_taxonomy_name:string%20%3F;proport_woa_temperature:string%20%3F;salinity_category:string%20%3F;sample_collection_site:string%20%3F;soluble_iron_micromol:string%20%3F;GOLD_sample_identifiers:external_identifier%20*;INSDC_biosample_identifiers:external_identifier%20*;INSDC_secondary_sample_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BQuantityValue%5D%3Csubsurface_depth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth2%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cwater_content_soil_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cwater_content%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_phosp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_nitro_content_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_nitro_content%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_org_c_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_diss_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_depth_water_col%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctidal_stage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctillage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctexture_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctexture%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctemp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfate%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cstore_cond%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csodium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_aspect%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_gradient%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_up%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_low%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csieving%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_store_temp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_loc%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_dur%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Csamp_mat_process%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_collect_device%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csalinity_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Credox_potential%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprofile_position%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpressure%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpotassium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cpool_dna_extracts%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphosplipid_fatt_acid%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphaeopigments%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cph_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cph%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpetroleum_hydrocarb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cperturbation%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpart_org_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Coxy_stat_samp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corganism_count%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_matter%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrite%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrate%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cn_alkanes%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmisc_param%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cmicrobial_biomass_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmicrobial_biomass%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_peak_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmagnesium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_climate_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_class_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_addit_analys%200..1-++%5BBiosample%5D,%5BGeolocationValue%5D%3Clat_lon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cheavy_metals_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cheavy_metals%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cglucosidase_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cgeo_loc_name%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cflooding%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cfire%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cfao_class%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cextreme_event%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_medium%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_local_scale%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_broad_scale%201..1-++%5BBiosample%5D,%5BTextValue%5D%3Cenv_package%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Celev%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cdrainage_class%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_oxygen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_phosp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_hydrogen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_carb_dioxide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdensity%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccrop_rotation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_land_use%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Ccollection_date%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchlorophyll%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchloride%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cchem_administration%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccarb_nitro_ratio%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccalcium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbromide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbishomohopanol%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbacteria_carb_prod%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cammonium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Caminopept_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cal_sat_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cal_sat%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkyl_diethers%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Calkalinity_method%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cagrochem_addition%200..1-++%5BBiosample%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BBiosample%5D,%5BBiosampleProcessing%5D-%20has%20input%200..*%3E%5BBiosample%5D,%5BDatabase%5D++-%20biosample%20set%200..*%3E%5BBiosample%5D,%5BOmicsProcessing%5D-%20has%20input(i)%201..*%3E%5BBiosample%5D,%5BNamedThing%5D%5E-%5BBiosample%5D
.. |image4| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BBiosample%5D%3Chas%20input%200..*-%20%5BBiosampleProcessing%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D%5E-%5BBiosampleProcessing%5D,%5BBiosample%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BBiosample%5D%3Chas%20input%200..*-%20%5BBiosampleProcessing%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D%5E-%5BBiosampleProcessing%5D,%5BBiosample%5D
.. |image5| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BBooleanValue%7Chas_boolean_value:boolean%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BBooleanValue%7Chas_boolean_value:boolean%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image6| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetaboliteQuantification%5D,%5BMetaboliteQuantification%5D-%20metabolite%20quantified%200..1%3E%5BChemicalEntity%7Cinchi:string%20%3F;inchi_key:string%20%3F;smiles:string%20*;chemical_formula:string%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetaboliteQuantification%5D,%5BMetaboliteQuantification%5D-%20metabolite%20quantified%200..1%3E%5BChemicalEntity%7Cinchi:string%20%3F;inchi_key:string%20%3F;smiles:string%20*;chemical_formula:string%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D
.. |image7| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%3Cterm%200..1-++%5BControlledTermValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%3Cterm%200..1-++%5BControlledTermValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image8| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BStudy%5D,%5BPersonValue%5D,%5BPersonValue%5D%3Capplies%20to%20person%201..1-++%5BCreditAssociation%7Capplied_role:credit_enum;type:string%20%3F%5D,%5BStudy%5D++-%20has%20credit%20associations%200..*%3E%5BCreditAssociation%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BStudy%5D,%5BPersonValue%5D,%5BPersonValue%5D%3Capplies%20to%20person%201..1-++%5BCreditAssociation%7Capplied_role:credit_enum;type:string%20%3F%5D,%5BStudy%5D++-%20has%20credit%20associations%200..*%3E%5BCreditAssociation%5D
.. |image9| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D,%5BDatabase%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BDataObject%7Cfile_size_bytes:bytes%20%3F;md5_checksum:string%20%3F;data_object_type:file_type_enum%20%3F;compression_type:string%20%3F;url:string%20%3F;type:string%20%3F;name:string;description:string;id(i):string;alternative_identifiers(i):string%20*%5D,%5BDatabase%5D++-%20data%20object%20set%200..*%3E%5BDataObject%5D,%5BNamedThing%5D%5E-%5BDataObject%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D,%5BDatabase%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BDataObject%7Cfile_size_bytes:bytes%20%3F;md5_checksum:string%20%3F;data_object_type:file_type_enum%20%3F;compression_type:string%20%3F;url:string%20%3F;type:string%20%3F;name:string;description:string;id(i):string;alternative_identifiers(i):string%20*%5D,%5BDatabase%5D++-%20data%20object%20set%200..*%3E%5BDataObject%5D,%5BNamedThing%5D%5E-%5BDataObject%5D,%5BActivity%5D
.. |image10| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BStudy%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BOmicsProcessing%5D,%5BNomAnalysisActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BGenomeFeature%5D,%5BFunctionalAnnotation%5D,%5BGenomeFeature%5D%3Cgenome%20feature%20set%200..*-++%5BDatabase%7Cnmdc_schema_version:string%20%3F;date_created:string%20%3F;etl_software_version:string%20%3F%5D,%5BFunctionalAnnotation%5D%3Cfunctional%20annotation%20set%200..*-++%5BDatabase%5D,%5BOmicsProcessing%5D%3Comics%20processing%20set%200..*-++%5BDatabase%5D,%5BNomAnalysisActivity%5D%3Cnom%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadBasedAnalysisActivity%5D%3Cread%20based%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadQCAnalysisActivity%5D%3Cread%20QC%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetatranscriptomeActivity%5D%3Cmetatranscriptome%20activity%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAssembly%5D%3Cmetagenome%20assembly%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAnnotationActivity%5D%3Cmetagenome%20annotation%20activity%20set%200..*-++%5BDatabase%5D,%5BMetaproteomicsAnalysisActivity%5D%3Cmetaproteomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetabolomicsAnalysisActivity%5D%3Cmetabolomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMAGsAnalysisActivity%5D%3Cmags%20activity%20set%200..*-++%5BDatabase%5D,%5BWorkflowExecutionActivity%5D%3Cactivity%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D%3Cdata%20object%20set%200..*-++%5BDatabase%5D,%5BStudy%5D%3Cstudy%20set%200..*-++%5BDatabase%5D,%5BBiosample%5D%3Cbiosample%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D,%5BBiosample%5D,%5BMAGsAnalysisActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BStudy%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BOmicsProcessing%5D,%5BNomAnalysisActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BGenomeFeature%5D,%5BFunctionalAnnotation%5D,%5BGenomeFeature%5D%3Cgenome%20feature%20set%200..*-++%5BDatabase%7Cnmdc_schema_version:string%20%3F;date_created:string%20%3F;etl_software_version:string%20%3F%5D,%5BFunctionalAnnotation%5D%3Cfunctional%20annotation%20set%200..*-++%5BDatabase%5D,%5BOmicsProcessing%5D%3Comics%20processing%20set%200..*-++%5BDatabase%5D,%5BNomAnalysisActivity%5D%3Cnom%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadBasedAnalysisActivity%5D%3Cread%20based%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadQCAnalysisActivity%5D%3Cread%20QC%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetatranscriptomeActivity%5D%3Cmetatranscriptome%20activity%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAssembly%5D%3Cmetagenome%20assembly%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAnnotationActivity%5D%3Cmetagenome%20annotation%20activity%20set%200..*-++%5BDatabase%5D,%5BMetaproteomicsAnalysisActivity%5D%3Cmetaproteomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetabolomicsAnalysisActivity%5D%3Cmetabolomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMAGsAnalysisActivity%5D%3Cmags%20activity%20set%200..*-++%5BDatabase%5D,%5BWorkflowExecutionActivity%5D%3Cactivity%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D%3Cdata%20object%20set%200..*-++%5BDatabase%5D,%5BStudy%5D%3Cstudy%20set%200..*-++%5BDatabase%5D,%5BBiosample%5D%3Cbiosample%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D,%5BBiosample%5D,%5BMAGsAnalysisActivity%5D
.. |image11| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D
.. |image12| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetagenomeAnnotationActivity%5D,%5BGeneProduct%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BFunctionalAnnotation%7Chas_function:string%20%3F%5D,%5BGeneProduct%5D%3Csubject%200..1-%20%5BFunctionalAnnotation%5D,%5BMetagenomeAnnotationActivity%5D%3Cwas%20generated%20by%200..1-%20%5BFunctionalAnnotation%5D,%5BDatabase%5D++-%20functional%20annotation%20set%200..*%3E%5BFunctionalAnnotation%5D,%5BDatabase%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetagenomeAnnotationActivity%5D,%5BGeneProduct%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BFunctionalAnnotation%7Chas_function:string%20%3F%5D,%5BGeneProduct%5D%3Csubject%200..1-%20%5BFunctionalAnnotation%5D,%5BMetagenomeAnnotationActivity%5D%3Cwas%20generated%20by%200..1-%20%5BFunctionalAnnotation%5D,%5BDatabase%5D++-%20functional%20annotation%20set%200..*%3E%5BFunctionalAnnotation%5D,%5BDatabase%5D
.. |image13| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BReaction%5D,%5BPathway%5D,%5BOrthologyGroup%5D,%5BOntologyClass%5D,%5BFunctionalAnnotationTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D%5E-%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BPathway%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%5D,%5BOntologyClass%5D%5E-%5BFunctionalAnnotationTerm%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BReaction%5D,%5BPathway%5D,%5BOrthologyGroup%5D,%5BOntologyClass%5D,%5BFunctionalAnnotationTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D%5E-%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BPathway%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%5D,%5BOntologyClass%5D%5E-%5BFunctionalAnnotationTerm%5D
.. |image14| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BProteinQuantification%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BPeptideQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BProteinQuantification%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BPeptideQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D
.. |image15| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BGeneProduct%5D%3Cencodes%200..1-%20%5BGenomeFeature%7Cseqid:string;start:integer;end:integer;strand:string%20%3F;phase:integer%20%3F;feature_type:string%20%3F%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BGenomeFeature%5D,%5BDatabase%5D++-%20genome%20feature%20set%200..*%3E%5BGenomeFeature%5D,%5BGeneProduct%5D,%5BDatabase%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BGeneProduct%5D%3Cencodes%200..1-%20%5BGenomeFeature%7Cseqid:string;start:integer;end:integer;strand:string%20%3F;phase:integer%20%3F;feature_type:string%20%3F%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BGenomeFeature%5D,%5BDatabase%5D++-%20genome%20feature%20set%200..*%3E%5BGenomeFeature%5D,%5BGeneProduct%5D,%5BDatabase%5D
.. |image16| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BGeolocationValue%7Clatitude:decimal_degree%20%3F;longitude:decimal_degree%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BGeolocationValue%7Clatitude:decimal_degree%20%3F;longitude:decimal_degree%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image17| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BImageValue%7Curl:string%20%3F;description:string%20%3F;display_order:string%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BImageValue%7Curl:string%20%3F;description:string%20%3F;display_order:string%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image18| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BMetaproteomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNomAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNamedThing%5D%5E-%5BInstrument%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BMetaproteomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNomAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNamedThing%5D%5E-%5BInstrument%5D
.. |image19| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BIntegerValue%7Chas_numeric_value:float%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BIntegerValue%7Chas_numeric_value:float%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image20| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BMAGsAnalysisActivity%5D++-%20mags%20list%200..*%3E%5BMAGBin%7Ctype:string%20%3F;bin_name:string%20%3F;number_of_contig:integer%20%3F;completeness:float%20%3F;contamination:float%20%3F;gene_count:integer%20%3F;bin_quality:string%20%3F;num_16s:integer%20%3F;num_5s:integer%20%3F;num_23s:integer%20%3F;num_tRNA:integer%20%3F;gtdbtk_domain:string%20%3F;gtdbtk_phylum:string%20%3F;gtdbtk_class:string%20%3F;gtdbtk_order:string%20%3F;gtdbtk_family:string%20%3F;gtdbtk_genus:string%20%3F;gtdbtk_species:string%20%3F%5D,%5BMAGsAnalysisActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BMAGsAnalysisActivity%5D++-%20mags%20list%200..*%3E%5BMAGBin%7Ctype:string%20%3F;bin_name:string%20%3F;number_of_contig:integer%20%3F;completeness:float%20%3F;contamination:float%20%3F;gene_count:integer%20%3F;bin_quality:string%20%3F;num_16s:integer%20%3F;num_5s:integer%20%3F;num_23s:integer%20%3F;num_tRNA:integer%20%3F;gtdbtk_domain:string%20%3F;gtdbtk_phylum:string%20%3F;gtdbtk_class:string%20%3F;gtdbtk_order:string%20%3F;gtdbtk_family:string%20%3F;gtdbtk_genus:string%20%3F;gtdbtk_species:string%20%3F%5D,%5BMAGsAnalysisActivity%5D
.. |image21| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BActivity%5D,%5BMAGBin%5D%3Cmags%20list%200..*-++%5BMAGsAnalysisActivity%7Cinput_contig_num:integer%20%3F;binned_contig_num:integer%20%3F;too_short_contig_num:integer%20%3F;lowDepth_contig_num:integer%20%3F;unbinned_contig_num:integer%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BMAGBin%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BActivity%5D,%5BMAGBin%5D%3Cmags%20list%200..*-++%5BMAGsAnalysisActivity%7Cinput_contig_num:integer%20%3F;binned_contig_num:integer%20%3F;too_short_contig_num:integer%20%3F;lowDepth_contig_num:integer%20%3F;unbinned_contig_num:integer%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BMAGBin%5D
.. |image22| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BMetabolomicsAnalysisActivity%5D,%5BChemicalEntity%5D%3Cmetabolite%20quantified%200..1-%20%5BMetaboliteQuantification%7Calternative_identifiers:string%20*;highest_similarity_score:float%20%3F%5D,%5BMetabolomicsAnalysisActivity%5D++-%20has%20metabolite%20quantifications%200..*%3E%5BMetaboliteQuantification%5D,%5BChemicalEntity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BMetabolomicsAnalysisActivity%5D,%5BChemicalEntity%5D%3Cmetabolite%20quantified%200..1-%20%5BMetaboliteQuantification%7Calternative_identifiers:string%20*;highest_similarity_score:float%20%3F%5D,%5BMetabolomicsAnalysisActivity%5D++-%20has%20metabolite%20quantifications%200..*%3E%5BMetaboliteQuantification%5D,%5BChemicalEntity%5D
.. |image23| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BMetaboliteQuantification%5D%3Chas%20metabolite%20quantifications%200..*-++%5BMetabolomicsAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BMetaboliteQuantification%5D,%5BInstrument%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BMetaboliteQuantification%5D%3Chas%20metabolite%20quantifications%200..*-++%5BMetabolomicsAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BMetaboliteQuantification%5D,%5BInstrument%5D,%5BActivity%5D
.. |image24| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
.. |image25| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
.. |image26| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D%3Chas%20peptide%20quantifications%200..*-++%5BMetaproteomicsAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BInstrument%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D%3Chas%20peptide%20quantifications%200..*-++%5BMetaproteomicsAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BInstrument%5D,%5BActivity%5D
.. |image27| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
.. |image28| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
.. |image29| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D
.. |image30| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BReadQCAnalysisActivity%5D,%5BPerson%5D,%5BOntologyClass%5D,%5BWorkflowExecutionActivity%5D-%20has%20input(i)%200..*%3E%5BNamedThing%7Cid:string;name:string%20%3F;description:string%20%3F;alternative_identifiers:string%20*%5D,%5BWorkflowExecutionActivity%5D-%20has%20output(i)%200..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20part%20of%200..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BNamedThing%5D%5E-%5BPerson%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D%5E-%5BInstrument%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D,%5BInstrument%5D,%5BGeneProduct%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BReadQCAnalysisActivity%5D,%5BPerson%5D,%5BOntologyClass%5D,%5BWorkflowExecutionActivity%5D-%20has%20input(i)%200..*%3E%5BNamedThing%7Cid:string;name:string%20%3F;description:string%20%3F;alternative_identifiers:string%20*%5D,%5BWorkflowExecutionActivity%5D-%20has%20output(i)%200..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20part%20of%200..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BNamedThing%5D%5E-%5BPerson%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D%5E-%5BInstrument%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D,%5BInstrument%5D,%5BGeneProduct%5D
.. |image31| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BInstrument%5D%3Cused%200..1-%20%5BNomAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BInstrument%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BInstrument%5D%3Cused%200..1-%20%5BNomAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BInstrument%5D,%5BActivity%5D
.. |image32| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BTextValue%5D%3Cchimera_check%200..1-++%5BOmicsProcessing%7Cadd_date:string%20%3F;mod_date:string%20%3F;instrument_name:string%20%3F;ncbi_project_name:string%20%3F;processing_institution:string%20%3F;type:string%20%3F;GOLD_sequencing_project_identifiers:external_identifier%20*;INSDC_experiment_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BTextValue%5D%3Cseq_quality_check%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cseq_meth%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_cond%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_primers%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_subfragment%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_gene%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_amp%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_ext%200..1-++%5BOmicsProcessing%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BOmicsProcessing%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BOmicsProcessing%5D,%5BControlledTermValue%5D%3Comics%20type%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20output%200..*-%20%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20input%200..*-%20%5BOmicsProcessing%5D,%5BDatabase%5D++-%20omics%20processing%20set%200..*%3E%5BOmicsProcessing%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BBiosample%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BTextValue%5D%3Cchimera_check%200..1-++%5BOmicsProcessing%7Cadd_date:string%20%3F;mod_date:string%20%3F;instrument_name:string%20%3F;ncbi_project_name:string%20%3F;processing_institution:string%20%3F;type:string%20%3F;GOLD_sequencing_project_identifiers:external_identifier%20*;INSDC_experiment_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BTextValue%5D%3Cseq_quality_check%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cseq_meth%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_cond%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_primers%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_subfragment%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_gene%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_amp%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_ext%200..1-++%5BOmicsProcessing%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BOmicsProcessing%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BOmicsProcessing%5D,%5BControlledTermValue%5D%3Comics%20type%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20output%200..*-%20%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20input%200..*-%20%5BOmicsProcessing%5D,%5BDatabase%5D++-%20omics%20processing%20set%200..*%3E%5BOmicsProcessing%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BBiosample%5D
.. |image33| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BControlledTermValue%5D++-%20term%200..1%3E%5BOntologyClass%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D,%5BEnvironmentalMaterialTerm%5D,%5BControlledTermValue%5D,%5BChemicalEntity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BControlledTermValue%5D++-%20term%200..1%3E%5BOntologyClass%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D,%5BEnvironmentalMaterialTerm%5D,%5BControlledTermValue%5D,%5BChemicalEntity%5D
.. |image34| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BFunctionalAnnotationTerm%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BFunctionalAnnotationTerm%5D
.. |image35| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BReaction%5D,%5BReaction%5D%3Chas_part%200..*-%20%5BPathway%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BPathway%5D,%5BFunctionalAnnotationTerm%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BReaction%5D,%5BReaction%5D%3Chas_part%200..*-%20%5BPathway%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BPathway%5D,%5BFunctionalAnnotationTerm%5D
.. |image36| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BPeptideQuantification%7Cpeptide_sequence:string%20%3F;min_q_value:float%20%3F;peptide_spectral_count:integer%20%3F;peptide_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D++-%20has%20peptide%20quantifications%200..*%3E%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BGeneProduct%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BPeptideQuantification%7Cpeptide_sequence:string%20%3F;min_q_value:float%20%3F;peptide_spectral_count:integer%20%3F;peptide_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D++-%20has%20peptide%20quantifications%200..*%3E%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BGeneProduct%5D
.. |image37| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D%5E-%5BPerson%7Cid:string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BNamedThing%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D%5E-%5BPerson%7Cid:string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BNamedThing%5D
.. |image38| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BPersonValue%7Corcid:string%20%3F;profile_image_url:string%20%3F;email:string%20%3F;name:string%20%3F;websites:string%20*;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BPersonValue%7Corcid:string%20%3F;profile_image_url:string%20%3F;email:string%20%3F;name:string%20%3F;websites:string%20*;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image39| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BProteinQuantification%7Cpeptide_sequence_count:integer%20%3F;protein_spectral_count:integer%20%3F;protein_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BProteinQuantification%5D,%5BGeneProduct%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BProteinQuantification%7Cpeptide_sequence_count:integer%20%3F;protein_spectral_count:integer%20%3F;protein_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BProteinQuantification%5D,%5BGeneProduct%5D
.. |image40| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BQuantityValue%7Chas_unit:unit%20%3F;has_numeric_value:double%20%3F;has_minimum_numeric_value:float%20%3F;has_maximum_numeric_value:float%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BQuantityValue%7Chas_unit:unit%20%3F;has_numeric_value:double%20%3F;has_minimum_numeric_value:float%20%3F;has_maximum_numeric_value:float%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image41| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BReactionParticipant%5D,%5BReactionParticipant%5D%3Cright%20participants%200..*-++%5BReaction%7Cdirection:string%20%3F;smarts_string:string%20%3F;is_diastereoselective:boolean%20%3F;is_stereo:boolean%20%3F;is_balanced:boolean%20%3F;is_transport:boolean%20%3F;is_fully_characterized:boolean%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BReactionParticipant%5D%3Cleft%20participants%200..*-++%5BReaction%5D,%5BPathway%5D-%20has_part%200..*%3E%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BReaction%5D,%5BPathway%5D,%5BFunctionalAnnotationTerm%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BReactionParticipant%5D,%5BReactionParticipant%5D%3Cright%20participants%200..*-++%5BReaction%7Cdirection:string%20%3F;smarts_string:string%20%3F;is_diastereoselective:boolean%20%3F;is_stereo:boolean%20%3F;is_balanced:boolean%20%3F;is_transport:boolean%20%3F;is_fully_characterized:boolean%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BReactionParticipant%5D%3Cleft%20participants%200..*-++%5BReaction%5D,%5BPathway%5D-%20has_part%200..*%3E%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BReaction%5D,%5BPathway%5D,%5BFunctionalAnnotationTerm%5D
.. |image42| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BChemicalEntity%5D%3Cchemical%200..1-%20%5BReactionParticipant%7Cstoichiometry:integer%20%3F%5D,%5BReaction%5D++-%20left%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D++-%20right%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D,%5BChemicalEntity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BChemicalEntity%5D%3Cchemical%200..1-%20%5BReactionParticipant%7Cstoichiometry:integer%20%3F%5D,%5BReaction%5D++-%20left%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D++-%20right%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D,%5BChemicalEntity%5D
.. |image43| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D,%5BActivity%5D
.. |image44| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BReadQCAnalysisActivity%7Cinput_read_count:float%20%3F;input_base_count:float%20%3F;output_read_count:float%20%3F;output_base_count:float%20%3F;input_read_bases:float%20%3F;output_read_bases:float%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BNamedThing%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BReadQCAnalysisActivity%7Cinput_read_count:float%20%3F;input_base_count:float%20%3F;output_read_count:float%20%3F;output_base_count:float%20%3F;input_read_bases:float%20%3F;output_read_bases:float%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BNamedThing%5D,%5BActivity%5D
.. |image45| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BImageValue%5D%3Cstudy%20image%200..*-++%5BStudy%7Cecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;title:string%20%3F;alternative_titles:string%20*;alternative_descriptions:string%20*;alternative_names:string%20*;abstract:string%20%3F;objective:string%20%3F;websites:string%20*;publications:string%20*;ess_dive_datasets:string%20*;type:string%20%3F;relevant_protocols:string%20*;funding_sources:string%20*;INSDC_bioproject_identifiers:external_identifier%20*;INSDC_SRA_ENA_study_identifiers:external_identifier%20*;GOLD_study_identifiers:external_identifier%20*;MGnify_project_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BCreditAssociation%5D%3Chas%20credit%20associations%200..*-++%5BStudy%5D,%5BAttributeValue%5D%3Cdoi%200..1-++%5BStudy%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BStudy%5D,%5BDatabase%5D++-%20study%20set%200..*%3E%5BStudy%5D,%5BNamedThing%5D%5E-%5BStudy%5D,%5BPersonValue%5D,%5BNamedThing%5D,%5BImageValue%5D,%5BDatabase%5D,%5BCreditAssociation%5D,%5BAttributeValue%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BImageValue%5D%3Cstudy%20image%200..*-++%5BStudy%7Cecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;title:string%20%3F;alternative_titles:string%20*;alternative_descriptions:string%20*;alternative_names:string%20*;abstract:string%20%3F;objective:string%20%3F;websites:string%20*;publications:string%20*;ess_dive_datasets:string%20*;type:string%20%3F;relevant_protocols:string%20*;funding_sources:string%20*;INSDC_bioproject_identifiers:external_identifier%20*;INSDC_SRA_ENA_study_identifiers:external_identifier%20*;GOLD_study_identifiers:external_identifier%20*;MGnify_project_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BCreditAssociation%5D%3Chas%20credit%20associations%200..*-++%5BStudy%5D,%5BAttributeValue%5D%3Cdoi%200..1-++%5BStudy%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BStudy%5D,%5BDatabase%5D++-%20study%20set%200..*%3E%5BStudy%5D,%5BNamedThing%5D%5E-%5BStudy%5D,%5BPersonValue%5D,%5BNamedThing%5D,%5BImageValue%5D,%5BDatabase%5D,%5BCreditAssociation%5D,%5BAttributeValue%5D
.. |image46| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTextValue%7Clanguage:language_code%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTextValue%7Clanguage:language_code%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image47| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTimestampValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTimestampValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image48| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BUrlValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BUrlValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D
.. |image49| image:: https://yuml.me/diagram/nofunky;dir:TB/class/%5BActivity%5D%3Cwas%20informed%20by%201..1-%20%5BWorkflowExecutionActivity%7Cexecution_resource:string;git_url:string;type:string;started_at_time:string;ended_at_time:string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%3Cwas%20associated%20with%200..1-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetatranscriptomeAssembly%5D,%5BMetatranscriptomeAnnotationActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BActivity%5D,%5BMAGsAnalysisActivity%5D
   :target: https://yuml.me/diagram/nofunky;dir:TB/class/%5BActivity%5D%3Cwas%20informed%20by%201..1-%20%5BWorkflowExecutionActivity%7Cexecution_resource:string;git_url:string;type:string;started_at_time:string;ended_at_time:string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%3Cwas%20associated%20with%200..1-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetatranscriptomeAssembly%5D,%5BMetatranscriptomeAnnotationActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BActivity%5D,%5BMAGsAnalysisActivity%5D
